{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "\n",
    "# Parse"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import sys\n",
    "sys.path.append('..')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[nltk_data] Downloading package stopwords to /root/nltk_data...\n",
      "[nltk_data]   Unzipping corpora/stopwords.zip.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "True"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import nltk\n",
    "nltk.download('stopwords')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "from clib import data, tokenize, context_words, snippets"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "from pathlib import Path\n",
    "\n",
    "DATA_PATH = Path('/kaggle/input/CORD-19-research-challenge/2020-03-13/')\n",
    "DATA_PATH = Path('2020-03-13/').expanduser()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "20c75a596b3047f0bdde8bc0a8a2f715",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "HBox(children=(FloatProgress(value=1.0, bar_style='info', max=1.0), HTML(value='')))"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n"
     ]
    }
   ],
   "source": [
    "1/0 # do not overwrite\n",
    "\n",
    "tokenize.cache_tokenized_papers(DATA_PATH, 'papers_v2.lj', only_some=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Collocations"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "e78d39778f1d4b28b48a94ff906079d0",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "HBox(children=(FloatProgress(value=1.0, bar_style='info', max=1.0), HTML(value='')))"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n"
     ]
    }
   ],
   "source": [
    "papers = tokenize.load_tokenized_papers('papers.jl')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "e98e752098f74d33987b4ec999dd79f2",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "HBox(children=(FloatProgress(value=0.0, max=13202.0), HTML(value='')))"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n"
     ]
    }
   ],
   "source": [
    "full_stats = context_words.get_context_stats(papers, 10)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "saving unigrams...\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "a9479d7061c64277b9d3564742c69d38",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "HBox(children=(FloatProgress(value=0.0, max=10858218.0), HTML(value='')))"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "saving bigrams...\n"
     ]
    },
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "5c7b9d4b324e498f99c07e3f2e1ccd64",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "HBox(children=(FloatProgress(value=0.0, max=10858218.0), HTML(value='')))"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n"
     ]
    }
   ],
   "source": [
    "context_words.save_stats(full_stats, 'full_stats')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "0f1ebaefa9b64eaf964bb4b58e2011fe",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "HBox(children=(FloatProgress(value=0.0, max=13202.0), HTML(value='')))"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "(1278, 11924)"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "from tqdm.auto import tqdm\n",
    "\n",
    "covid_papers = []\n",
    "non_covid_papers = []\n",
    "\n",
    "keywords = [\n",
    "    ['covid'], ['coronavirus'], ['sars'], ['corona', 'virus'], \n",
    "    'severe acute respiratory syndrome'.split(), 'mers'.split(), \n",
    "    'middle east respiratory syndrome'.split()\n",
    "]\n",
    "\n",
    "for p in tqdm(papers):\n",
    "    title = p['metadata.title'][0]['tokens']\n",
    "    \n",
    "    for kwds in keywords:\n",
    "        if all([any([k in tok for tok in title]) for k in kwds]):\n",
    "            covid_papers.append(p)\n",
    "            break\n",
    "    else:\n",
    "        non_covid_papers.append(p)\n",
    "\n",
    "len(covid_papers), len(non_covid_papers)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "000e0343594a4367ac003ee109eb410b",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "HBox(children=(FloatProgress(value=0.0, max=11924.0), HTML(value='')))"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "ename": "KeyboardInterrupt",
     "evalue": "",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mKeyboardInterrupt\u001b[0m                         Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-11-fc13ba108a4c>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[0;32m----> 1\u001b[0;31m \u001b[0mnon_covid_stats\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mcontext_words\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mget_context_stats\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mnon_covid_papers\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;36m10\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m",
      "\u001b[0;32m/alloc/data/fury_fda-deals-ciencia/CORD-19-research-challenge/clib/context_words.py\u001b[0m in \u001b[0;36mget_context_stats\u001b[0;34m(papers, window_size)\u001b[0m\n\u001b[1;32m     30\u001b[0m                 \u001b[0;32mif\u001b[0m \u001b[0mt\u001b[0m \u001b[0;32mnot\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mbigrams\u001b[0m\u001b[0;34m:\u001b[0m \u001b[0mbigrams\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mt\u001b[0m\u001b[0;34m]\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m{\u001b[0m\u001b[0;34m}\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     31\u001b[0m                 \u001b[0mt_bigrams\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mbigrams\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mt\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 32\u001b[0;31m                 \u001b[0;32mfor\u001b[0m \u001b[0mj\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mrange\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mlb\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mub\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     33\u001b[0m                     \u001b[0mt2\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mtokens\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mj\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     34\u001b[0m                     \u001b[0;32mif\u001b[0m \u001b[0mt2\u001b[0m \u001b[0;32mnot\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mt_bigrams\u001b[0m\u001b[0;34m:\u001b[0m \u001b[0mt_bigrams\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0mt2\u001b[0m\u001b[0;34m]\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;36m0\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mKeyboardInterrupt\u001b[0m: "
     ]
    }
   ],
   "source": [
    "non_covid_stats = context_words.get_context_stats(non_covid_papers, 10)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "covid_stats = context_words.get_context_stats(covid_papers, 10)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "context_words.save_stats(non_covid_stats, 'non_covid_stats')\n",
    "context_words.save_stats(covid_stats, 'covid_stats')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Compute scores"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "ename": "FileNotFoundError",
     "evalue": "[Errno 2] No such file or directory: 'covid_stats/unigrams.jl'",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mFileNotFoundError\u001b[0m                         Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-32-d4cf38d931a3>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[1;32m      1\u001b[0m \u001b[0;31m# full_stats = context_words.load_stats('full_stats')\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 2\u001b[0;31m \u001b[0mcovid_stats\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mcontext_words\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mload_stats\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m'covid_stats'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      3\u001b[0m \u001b[0mnon_covid_stats\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mcontext_words\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mload_stats\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m'non_covid_stats'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/alloc/data/fury_fda-deals-ciencia/CORD-19-research-challenge/clib/context_words.py\u001b[0m in \u001b[0;36mload_stats\u001b[0;34m(path)\u001b[0m\n\u001b[1;32m     64\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     65\u001b[0m     \u001b[0munigrams\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m{\u001b[0m\u001b[0;34m}\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 66\u001b[0;31m     \u001b[0;32mwith\u001b[0m \u001b[0;34m(\u001b[0m\u001b[0mpath\u001b[0m \u001b[0;34m/\u001b[0m \u001b[0;34m'unigrams.jl'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mopen\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m \u001b[0;32mas\u001b[0m \u001b[0mf\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     67\u001b[0m         \u001b[0;32mfor\u001b[0m \u001b[0mline\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mtqdm\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mf\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mdesc\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;34m'loading unigrams'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     68\u001b[0m             \u001b[0md\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mjson\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mloads\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mline\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/usr/local/lib/python3.7/pathlib.py\u001b[0m in \u001b[0;36mopen\u001b[0;34m(self, mode, buffering, encoding, errors, newline)\u001b[0m\n\u001b[1;32m   1163\u001b[0m             \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_raise_closed\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1164\u001b[0m         return io.open(self, mode, buffering, encoding, errors, newline,\n\u001b[0;32m-> 1165\u001b[0;31m                        opener=self._opener)\n\u001b[0m\u001b[1;32m   1166\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1167\u001b[0m     \u001b[0;32mdef\u001b[0m \u001b[0mread_bytes\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/usr/local/lib/python3.7/pathlib.py\u001b[0m in \u001b[0;36m_opener\u001b[0;34m(self, name, flags, mode)\u001b[0m\n\u001b[1;32m   1017\u001b[0m     \u001b[0;32mdef\u001b[0m \u001b[0m_opener\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mname\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mflags\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mmode\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;36m0o666\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1018\u001b[0m         \u001b[0;31m# A stub for the opener argument to built-in open()\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m-> 1019\u001b[0;31m         \u001b[0;32mreturn\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_accessor\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mopen\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mflags\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mmode\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m   1020\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m   1021\u001b[0m     \u001b[0;32mdef\u001b[0m \u001b[0m_raw_open\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mflags\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mmode\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0;36m0o777\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mFileNotFoundError\u001b[0m: [Errno 2] No such file or directory: 'covid_stats/unigrams.jl'"
     ]
    }
   ],
   "source": [
    "full_stats = context_words.load_stats('full_stats')\n",
    "covid_stats = context_words.load_stats('covid_stats')\n",
    "non_covid_stats = context_words.load_stats('non_covid_stats')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "10858218"
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(full_stats.bigrams)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [],
   "source": [
    "vocab = dict(sorted(full_stats.unigrams.items(), key=lambda x: -x[1])[:60000])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "71ebf1a189e742d9973be5a9851bba02",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "HBox(children=(FloatProgress(value=0.0, max=10858218.0), HTML(value='')))"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n"
     ]
    }
   ],
   "source": [
    "small_full_stats = context_words.ContextStats(\n",
    "    unigrams=vocab, \n",
    "    bigrams={\n",
    "        center: {\n",
    "            word: score \n",
    "            for word, score in related.items()\n",
    "            if word in vocab\n",
    "        }\n",
    "        for center, related in tqdm(full_stats.bigrams.items())\n",
    "        if center in vocab\n",
    "    }\n",
    ")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "b9d1231750a04a3c88e6a62e487eb768",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "HBox(children=(FloatProgress(value=0.0, max=60000.0), HTML(value='')))"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n"
     ]
    }
   ],
   "source": [
    "scores = context_words.get_scores(small_full_stats)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "6c921cdf66764cbf9c429ca595846ce4",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "HBox(children=(FloatProgress(value=0.0, max=9416.0), HTML(value='')))"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n"
     ]
    }
   ],
   "source": [
    "comparative_scores, out_ref = context_words.get_comparative_scores(covid_stats, non_covid_stats)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "metadata": {},
   "outputs": [],
   "source": [
    "def top(d, t, k=10):\n",
    "    return sorted(d[t].items(), key=lambda x: -x[1])[:k]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 54,
   "metadata": {},
   "outputs": [],
   "source": [
    "keywords = [\n",
    "    'isolation', 'quarantine', 'npi', 'funding', 'growth_rate', 'r0', \n",
    "    'health_care', 'capacity', 'mitigation', 'social_distancing', #'migration'\n",
    "]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 55,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "**mitigation**\n",
       "\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- mitigation"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- containment"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- emissions"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- strategies"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- deployment"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- stockpiles"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- ingredients"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- economic"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- closures"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- conceptual"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "**social_distancing**\n",
       "\n"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- distancing"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- social"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- closures"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- png"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- lockdown"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- rigorous"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- epicenter"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- albany"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- behaviours"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- waning"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "for k in keywords[-2:]:\n",
    "    snippets.dm(f'**{k}**\\n\\n')\n",
    "    for r, s in top(scores, k, k=10):\n",
    "        snippets.dm(f'- {r}')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [],
   "source": [
    "from collections import defaultdict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[('npi', 601.3635569022784),\n",
       " ('npis', 61.30913025184829),\n",
       " ('quadratic', 8.93688326196239),\n",
       " ('facemasks', 6.637902422834385),\n",
       " ('implementation', 5.360520795212643),\n",
       " ('cost', 3.744428171377962),\n",
       " ('strength', 2.637784261220575),\n",
       " ('lost', 1.9641980268738002),\n",
       " ('policy', 1.4548827228130157),\n",
       " ('exponential', 1.2610552510340653),\n",
       " ('cumulative', 1.0389760314001646),\n",
       " ('optimal', 1.0329229855491204),\n",
       " ('triggers', 0.8992446345070141),\n",
       " ('minimize', 0.782040865979919),\n",
       " ('death', 0.7567089124258483),\n",
       " ('terminal', 0.6792959097895032),\n",
       " ('effectiveness', 0.5965034953447873),\n",
       " ('linear', 0.545441794867133),\n",
       " ('recovery', 0.5145082758656173),\n",
       " ('state', 0.46520479583853164),\n",
       " ('hygiene', 0.45619050515495274),\n",
       " ('pandemic', 0.45059546190290467),\n",
       " ('coughing', 0.4438132433871827),\n",
       " ('decision', 0.31884111881348587),\n",
       " ('hand', 0.3101333487251366),\n",
       " ('interventions', 0.3075476026023653),\n",
       " ('applying', 0.2560174774597946),\n",
       " ('intensity', 0.24501529984427917),\n",
       " ('deaths', 0.22985041845545628),\n",
       " ('rate', 0.22788246392217054)]"
      ]
     },
     "execution_count": 33,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "l1 = top(scores, 'npi', 30)\n",
    "l1"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [],
   "source": [
    "l2 = top(comparative_scores, 'environmental', 30)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[('environmental', 'adjust'),\n",
       " ('cues', 'greater'),\n",
       " ('fcv', 'changes'),\n",
       " ('survival', 'viruses'),\n",
       " ('disasters', 'transmission'),\n",
       " ('factors', 'environmental')]"
      ]
     },
     "execution_count": 22,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "list(zip(list(zip(*l1))[0], list(zip(*l2))[0]))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[]"
      ]
     },
     "execution_count": 23,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "top(comparative_scores, 'neurological', 30)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[('individuals', 0.17212272793241304),\n",
       " ('p', 0.1614529646770799),\n",
       " ('fatality', 0.1334063585626627),\n",
       " ('ed', 0.13167464833949932),\n",
       " ('mers', 0.1104246551227619),\n",
       " ('identify', 0.09265019832072449),\n",
       " ('perceived', 0.09099253819678416),\n",
       " ('search', 0.08933853575913712),\n",
       " ('participants', 0.08933853575913712),\n",
       " ('risk', 0.08905988349646546)]"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "top(comparative_scores, 'mers')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[('mers', 0.4980965888121998),\n",
       " ('air', 0.25359821376488123),\n",
       " ('incubation', 0.23734529708392899),\n",
       " ('comorbidity', 0.22122346614920416),\n",
       " ('cases', 0.21046864604492416),\n",
       " ('comorbidities', 0.205240924198479),\n",
       " ('pollution', 0.205240924198479),\n",
       " ('sars', 0.19830044005021039),\n",
       " ('coronaviruses', 0.1894069705270051),\n",
       " ('difference', 0.17373223608561106)]"
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "top(comparative_scores, 'fatality')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 99,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[('economic', 12.649368678901082),\n",
       " ('impact', 4.433679419253568),\n",
       " ('gdp', 3.3477161136735605),\n",
       " ('billion', 3.0224013274860733),\n",
       " ('societal', 3.004155748785821),\n",
       " ('industry', 1.857920677108869),\n",
       " ('tremendous', 1.5350503155381203),\n",
       " ('morbidity', 1.0383317469931643),\n",
       " ('industries', 0.8442261963388724),\n",
       " ('worldwide', 0.6436443902605244),\n",
       " ('substantial', 0.6435282508902141),\n",
       " ('costs', 0.609372143568077),\n",
       " ('livestock', 0.5307305632567649),\n",
       " ('productivity', 0.5173617997292473),\n",
       " ('poultry', 0.4984051036682616),\n",
       " ('bank', 0.3651700779638117),\n",
       " ('burden', 0.350624306056061),\n",
       " ('swine', 0.33930164934531754),\n",
       " ('cattle', 0.30522338960748274),\n",
       " ('mortality', 0.2863867262934438),\n",
       " ('estimated', 0.2690911781820873),\n",
       " ('million', 0.24125701799432703),\n",
       " ('significant', 0.23632951105267866),\n",
       " ('countries', 0.22315846447140136),\n",
       " ('losses', 0.22306242401936183),\n",
       " ('cost', 0.21894960145090603),\n",
       " ('huge', 0.21785068514040476),\n",
       " ('estimates', 0.21636221197907418),\n",
       " ('cause', 0.21276897544646023),\n",
       " ('deaths', 0.2105622714522012)]"
      ]
     },
     "execution_count": 99,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "seeds = ('transmission', 'spillover',\n",
    " 'transmissions',\n",
    " 'spread',\n",
    " 'spreading',\n",
    " 'dispersal',\n",
    " 'transmissibility',\n",
    " 'transmitting',\n",
    " 'transmitted',\n",
    " 'importation',\n",
    " 'contact',\n",
    " 'contagion',\n",
    " 'dissemination',\n",
    " 'fomite',\n",
    " 'reintroduction',\n",
    " 'outbreaks',\n",
    " 'epidemic',\n",
    " 'emergence',\n",
    " 'movement',\n",
    " 'epidemics',\n",
    " 'introductions')\n",
    "\n",
    "\n",
    "seeds = ('immunities', 'immunity',\n",
    " 'responses',\n",
    " 'immune',\n",
    " 'humoral',\n",
    " 'defenses',\n",
    " 'angioplasticity',\n",
    " 'response',\n",
    " 'protection',\n",
    " 'defense',\n",
    " 'immunopathology',\n",
    " 'autoimmunity')\n",
    "# seeds = ('eef1a', )\n",
    "\n",
    "seeds = ('comorbidities',)\n",
    "\n",
    "from nltk.corpus import stopwords\n",
    "\n",
    "stopwords = set(stopwords.words('english'))\n",
    "\n",
    "\n",
    "word_related = defaultdict(int)\n",
    "for seed in ['economic_impact']:\n",
    "    for word, score in sorted(scores.get(seed, {}).items(), key=lambda x: -x[1])[:30]:\n",
    "        if word in stopwords: continue\n",
    "        word_related[word] += score\n",
    "    \n",
    "    \n",
    "sorted(word_related.items(), key=lambda x: -x[1])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 106,
   "metadata": {},
   "outputs": [],
   "source": [
    "impact_papers = []\n",
    "\n",
    "for paper in covid_papers:\n",
    "    for field, value in data.iter_fields(paper, flat=True):\n",
    "        for i, t in enumerate(value['tokens'][:-1]):\n",
    "            if t == 'economic' and value['tokens'][i + 1] == 'impact':\n",
    "                impact_papers.append(paper)\n",
    "                break"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 107,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "17"
      ]
     },
     "execution_count": 107,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(impact_papers)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 108,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "1c21a1d834674faaa96e4a255b494f6e",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "HBox(children=(FloatProgress(value=0.0, max=17.0), HTML(value='')))"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n"
     ]
    }
   ],
   "source": [
    "from importlib import reload\n",
    "\n",
    "reload(snippets)\n",
    "snps = snippets.get_snippets(impact_papers, word_related)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 109,
   "metadata": {},
   "outputs": [],
   "source": [
    "snps.sort(key=lambda x: -x['score'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 110,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "The global macroeconomic **impact** of SARS was **estimated** at USD 30-100 **billion** or around USD 3-10 **million** per cases [14] . The 2003 SARS outbreak caused **losses** of USD 12.3-28.4 **billion** and an **estimated** decrease of 1% in **GDP** in China and 0.5% in Southeast Asia [15] . The social **burden** of SARS in Guangzhou meant less income and spending, with a rough estimate of the total **economic** burden of RMB 11 **billion** [16] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "**keywords**: estimated, impact, losses, economic, million, burden, billion, gdp"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "Influenza virus is a negative sense, segmented RNA virus that presents a **substantial** burden to human health. Despite the availability of successful vaccines and antivirals, infection with seasonal influenza viruses is still the **cause** of 3-5 **million** cases of severe illnesses and up to 300-650 thousand **deaths** worldwide (World Health Organization, 2017) . In the US alone, it is **estimated** that seasonal influenza affects up to 9 **million** people a year resulting in 12,000-56,000 **deaths** annually (Centers for Disease Control and Prevention, 2018c) . The **economic** burden is **estimated** to be in the tens of billions of dollars (Putri et al., 2018) . In addition, pandemic influenza outbreaks emerge at unpredictable intervals causing increased **morbidity**, **mortality**, and a negative **economic** impact (Erbelding et al., 2018) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "**keywords**: cause, deaths, estimated, morbidity, economic, million, substantial, mortality"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "The **economic** impact of major outbreaks can be **substantial** for the affected country. This was seen clearly in SARS, MERS in RoK and Ebola in West Africa. One analyst **estimates** that the current coronavirus outbreak's likely **impact** will range from a 0.8% cut to real **GDP** if the epidemic is controlled within 3 months, to a 1.9% **cost** to **GDP** if the epidemic lasts 9 months [8] . This may increase substantially in the light of the extended restrictions on movement, and therefore trade and commerce, within China."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "**keywords**: impact, cost, economic, substantial, estimates, gdp"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "Impacts. The **economic** impacts of H7N9 were less serious than SARS but still important to characterize. Studies show that the direct medical **costs** of hospitalization of a patient with H7N9 were **estimated** to be RMB 71 060, which is more than a year's income for a person in a rich province in China [20] . In April 2013, the H7N9 avian influenza epidemic caused the price index of meat and **poultry** and their products to fall to 101.5 on a year-on-year basis. As a result of the outbreak, China's **poultry** industry suffered a loss of more than RMB 40 **billion** [27] . However the **economic** impact was little in the global community."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "**keywords**: estimated, costs, economic, poultry, billion"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "Respiratory viral infections are of global public health concern because they are the most common **cause** of symptomatic disease leading to a heavy **economic** burden due to an increased number of sick days (Borchardt and Rolston, 2012; Kim et al., 2017) . In addition, respiratory diseases are one of the most common causes of **mortality** in developing **countries** (Ferkol and Schraufnagel, 2014) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "**keywords**: cause, economic, countries, mortality"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "The high **mortality** of H7N9 changed the attitude of the public towards chickens and it became apparent that few "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "**keywords**: mortality"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "However, the imposition of these dramatic measures does also raise a wider question: if there is an **impact** from these measures, what other **countries** would (or could) implement such measures? Would other **countries** accept the self-imposed **economic** damage that China has accepted to try and contain this outbreak? Is it reasonable to consider that national governments would close down public transport into and out of London, New York or Paris in the week before Christmas even if it were shown to be an effective control measure?"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "**keywords**: economic, impact, countries"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "There were many obvious immediate consequences of the epidemic, such as **substantial** morbidity and **mortality**, fear for becoming infected, panic in the public domain, stringent quarantine measures, travel restrictions, etc. Also two important middle and long-term consequences of the epidemic were identified. First, there was the **economic** impact. This was studied for Beijing by Beutels et al. 21 through associating time series of daily and monthly SARS cases and **deaths** and volume of public train, airplane and cargo transport, tourism, household consumption patterns and gross domestic product growth in Beijing. It was concluded that especially leisure activities, local and international transport and tourism were affected by SARS, particularly in May 2003. Much of this consumption was merely postponed; but irrecoverable **losses** to the tourist sector alone were **estimated** at about US$ 1.4 **billion**, or 300 times the **cost** of treatment for SARS cases in Beijing."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "**keywords**: deaths, estimated, cost, losses, economic, substantial, billion, mortality"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "The 2-3 % prevalence of active MERS-CoV infections is not dissimilar to the hospital-based prevalence of other human CoVs. [112] However, the proportion of **deaths** among those infected with MERS-CoV is much higher than that known for the HCoVs NL63, HKU1, 229E or OC43 in other **countries**, and even above that for SARS-CoV; it is not a virus that could reasonably be described as a \"storm in a teacup\". It is the low transmission rate that has prevented **worldwide** spread, despite many \"opportunities\"."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "**keywords**: deaths, worldwide, countries"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "The effects of MERS mentioned above depend on the social structure [6, 7] . The vulnerability approach states that the effect not only from the physical aspect, but also the **economic** class (status) should not be ignored. The approach asserts that disasters like epidemics combine physical exposure to the epidemic with vulnerability due to structural problems in the **economic** system, resulting in a larger **impact** on certain individuals."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "**keywords**: economic, impact"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "The **impact** of epidemics on labor market: identifying victims of the Middle East Respiratory Syndrome in the Korean labor market"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "**keywords**: impact"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "The vulnerability concept for disasters and natural hazards was introduced by O'Keefe, Westgate and Wisner in the 1970s [6] . In \"Taking the naturalness out of natural disasters\", the authors claimed that the socio-**economic** condition was the **cause** of natural disasters. Natural disasters do not equally affect all people. Instead, the **impact** of the disaster is contingent on the vulnerability of the affected people, which often differs systematically across **economic** class, ethnicity, gender, and other factors [6] . In fact, a vulnerability approach to disasters would suggest that inequalities in exposure and sensitivity to risk as well as inequalities in access to opportunities systematically disadvantage certain groups of people, rendering them more vulnerable to the **impact** of natural disasters [7] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "**keywords**: cause, economic, impact"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "The World **Bank** [5] reported that epidemics affect the economy through two channels. First is the temporary or permanent decrease in the labor supply due to the direct and indirect effects of sickness and **mortality**. Second is the behavioral effect due to people's fear of contagion. Such behavioral effects **cause** a decrease in labor force participation due to the fear of contact with other people, shutting down places of employment. The Severe Acute Respiratory Syndrome (SARS) epidemic between 2002 and 2004 and the H1N1 flu epidemic in 2009 showed that the behavioral effects accounted for 80 or 90% of the **impact** on the economy [4, 5] . In addition, during the Ebola outbreak in 2014, the largest **economic** effects of the crisis was not the direct effect (**mortality**, **morbidity**, caregiving and the associated **losses** to working days), but the decrease in the goods and services demand by behavioral effects (changes in behavior) due to fear, reducing domestic income and employment. In particular, workers in service sectors like hotels and restaurants in Liberia were laid-off and the number of jobs was slashed by half [5] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "**keywords**: cause, impact, morbidity, losses, economic, bank, mortality"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "The risk-hazard model is a type of social vulnerability model. The Risk-Hazard (RH) model seeks to understand the **impact** of a hazard as a function of exposure to the hazardous event and the sensitivity of the entity exposed [9] . The Pressure and Release (PAR) model understands a disaster as the intersection between socio-**economic** pressure and physical exposure [8] . The vulnerability difference is caused by socio-**economic** conditions rather than the physical condition itself and such social vulnerability exacerbates the inequality."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "**keywords**: economic, impact"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "We used chi-square tests to determine the between-site effects and the effect of bird group on the proportion of positive individuals. A value of P,0.001 was considered statistically **significant**."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "**keywords**: significant"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "Infections (acute and chronic conditions) are often associated to comorbidity that increases the risk of medical conditions which can lead to further **morbidity** and **mortality**. Comorbidities related to flu have been recently investigated [51] . Comorbidities for tuberculosis have also been studied recently [52, 53] . To understand the overall mechanism we have studied the comorbidity associations of SARS and HIV infections. Both HIV and SARS are emerging infectious diseases in the modern world; each of these diseases has caused global **societal** and **economic** impact related to unexpected illnesses and **deaths** [54] . SRAS is a **significant** public health threat and HIV is a long term chronic infection. Since these two infections are associated with high **mortality** rates and there are no clinically approved antiviral treatments or vaccines available for either of these infections, we have selected these two infections for our study. Centred on the SARS and HIV-1 infections we have investigated highly heterogeneous disease comorbidity networks using the disease-gene associations, PPI subnetwork, molecular pathways and clinical information."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "**keywords**: deaths, significant, morbidity, societal, economic, mortality"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "Neumayer and Plmper [7] investigated whether socially vulnerable groups were affected more strongly by disasters than other groups. The analysis on 141 **countries** between 1981 and 2002 indicated that women faced a higher probability of death by natural disasters than men in **countries** where the socio-**economic** status (rights) of women was lower than that of men in the society. The authors reported that natural disasters had more adverse impacts on the vulnerable groups in the society and their result was consistent with the vulnerability approach."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "**keywords**: economic, countries"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "After an outbreak that lasted 172 days, the Middle East Respiratory Syndrome (MERS) epidemic was officially declared over on November 25, 2015 when the last MERS patient in Korea died. The Korean economy became sluggish for 6 months from the first MERS outbreak on May 20, 2015 and the **Bank** of Korea reduced the key rates from 1.75% down to 1.5% on June 11, 2015 [1] . This figure marked the lowest interest rates in Korean history [1] . The **bank** announced that the main reason for the interest rate cut was not only sluggish **economic** recovery but also a larger than expect **impact** from MERS on the economy, raising concerns about the worsened **economic** situation. Such events showed that MERS strongly affected the Korean economy [1] ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "**keywords**: economic, impact, bank"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "The strength of this study is that investigated the effect that disasters, such as epidemics, can have on employment status, which directly influences peoples' livelihoods. Although many studies have examined the **economic** impact of epidemics, few studies have focused on the effect of an infectious disease on the labor market. In particular, this study empirically identified the different effects that disasters, such as epidemics, produce not only due to physical vulnerability but also to **economic** class (the channel of the labor market). While we recognize there are limitations to this study, this study cannot narrow the sample by selecting the **industry** and regions that suffered severe **losses** from the contagious disease due to limitations of the available data. If the analysis of specific dimensions of **industry** and regions is further carries out using data, the results are expected to appear in a more remarkable way and to provide more useful information for the policy."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "**keywords**: industry, economic, losses"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "By 26 January also, almost 50 **million** people in Wuhan and neighbouring cities had effectively been placed in quarantine while the WHO had determined that the event should not yet be declared as a Public Health Emergency of International Concern (PHEIC) [2] and had recommended no specific travel restrictions. The WHO have emphasised the importance of exit screening at ports in **countries** showing transmission of the novel coronavirus and have provided guidance for **countries** implementing entry screening at airports while acknowledging that evidence for the effectiveness of entry screening is equivocal."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "**keywords**: million, countries"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "for snippet in snps[:20]:\n",
    "    snippets.dm(snippet['snippet'])\n",
    "    snippets.dm('**keywords**: {}'.format(', '.join(snippet['queries'])))\n",
    "    snippets.dm('---')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 64,
   "metadata": {},
   "outputs": [],
   "source": [
    "sent = 'exponential growth rate (estimated at 0.14 [11] )'"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 93,
   "metadata": {},
   "outputs": [],
   "source": [
    "from collections import defaultdict\n",
    "import re\n",
    "\n",
    "er_pattern = re.compile('growth.{,20}?rate.{,30}(?P<rate>\\d\\.\\d+)', re.I)\n",
    "\n",
    "matching_papers = defaultdict(set)\n",
    "matching_snippets = []\n",
    "\n",
    "for pid, paper in enumerate(covid_papers):\n",
    "    for field, value in data.iter_fields(paper, flat=True):\n",
    "        for match in er_pattern.finditer(value['text']):\n",
    "            matching_papers[float(match.groupdict()['rate'])].add(pid)\n",
    "            matching_snippets.append((paper['metadata.title'][0]['text'], value))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 98,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "**Clinical Medicine Assessing the Impact of Reduced Travel on Exportation Dynamics of Novel Coronavirus Infection (COVID-19)**"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "Here, we consider the median time to a major epidemic in (8) and (9). Since an exponential growth of cases has been observed, we let the hazard be an exponential function. Then, the integral of the hazard function holds the form: C  (exp(rt)  1), where C is a constant (assumed to be one for the following calculation), and r is the exponential growth **rate** estimated at 0.14 per day [16] . The doubling time is then calculated as t d = ln(2)/r = 4.95 days. The difference in the median date between (8) and (9) is thus described as: (2) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "**Clinical Medicine Incubation Period and Other Epidemiological Characteristics of 2019 Novel Coronavirus Infections with Right Truncation: A Statistical Analysis of Publicly Available Case Data**"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "Here, r is the exponential growth **rate** (estimated at 0.14 [11] ), T is the latest time of observation (31 January 2020), and F(.) is the cumulative density function of f (.)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "**MRPrimerV: a database of PCR primers for RNA virus detection**"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "The recent MERS outbreak in Korea, which spread rapidly due to slow and unreliable diagnosis, resulted in a 41% reduction in foreign tourism and decreased the gross domestic product growth **rate** in 2015 by 0.1% (18) . The MRPrimerV database contains 152 380 247 high-quality PCR primer pairs for detection of 1818 viruses, covering 100% of the RNA viruses in the most up-to-date NCBI RefSeq database (Release 76). Because all primers in MRPrimerV were subjected to the same stringent filtering constraints and rigorous similarity testing against all 101 684 human gene sequences, 26 720 camel sequences and all RNA virus sequences, they are all highly target-specific for RNA viruses. The current MRPrimerV database mainly provides primers for detection of RNA viruses that infect human hosts. However, since many human pathogens, such as Zika, JEV, WNV, DENV, MERS-CoV and influenza virus, can also infect other animal species, future work will be directed toward updating the database to include primers against other animal species. MRPrimerV is freely accessible and provides a user-friendly interface. Because the database ranks the CDSs by the penalty scores of their best primer pairs, users can easily select primers for multi-phase PCR experiments to achieve more accu**rate** detection of viruses. MR-PrimerV also supports multi-virus detection and TaqMan probes. MRPrimerV contains not only primers, but also validation results for some viruses, and users are invited to send their own experimental validation data for MRPrimerV primers, so that they can be shared by researchers and international health communities. In addition, MRPrimerV provides the list of genome neighbors that can be detected using each primer pair, covering a total of 22 192 variants of 532 RefSeq RNA viruses. Therefore, this database could be used in viral metagenomics studies aimed at evaluating viral variability, as well as other scientific tasks. In the future, we will add a feature that allows users to design primers using virus sequences they provide. We believe that MRPrimerV, as a public database of high-quality primers for RNA virus detection, will aid future efforts to design primers for reliable diagnoses, facilitating effective responses to potential epidemics."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "**Journal Pre-proof Serial interval of novel coronavirus (COVID-19) infections Title: Serial interval of novel coronavirus (COVID-19) infections Authors**"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "where r is the exponential growth **rate** estimated at 0.14 [7] and T is the latest time of observation (12 February 2020). We account for the exponential growth **rate** of cases, because recently infected individuals are more likely to be sampled during the exponential growth phase of an epidemic. The widely applicable information criterion (WAIC) was used to compare between distributions and the model with the minimal WAIC value was selected as the best-fit model for each set of estimates with and without right truncation."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "**Title: Risk for Transportation of 2019 Novel Coronavirus (COVID-19) from Wuhan to Cities in China**"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       " Epidemic growth **rate**, : 0.095 (0.072 -0.111), corresponding to an epidemic doubling time of incident cases of 7.31 (95% CrI 6.26 -9.66) days; cumulative infections occurred by January 12, 2020 (i.e., 10 days before the quarantine to account for the typical lag between infection and case detection)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "**Title: The Novel Coronavirus, 2019-nCoV, is Highly Contagious and More Infectious Than Initially Estimated**"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "The copyright holder for this preprint (which was not peer-reviewed) is the . https://doi.org/10.1101/2020.02.07.20021154 doi: medRxiv preprint represented the trajectory of infected individuals who travelled from Wuhan using a stochastic agent-based model. The transitions of the infected individuals from symptom onset to hospitalization and then to case confirmation were assumed to follow the distributions inferred from the case report data (see Supplementary Materials for detail). Simulation of the model using best fit parameters showed that the model described the observed case counts over time well (Fig. 2E) . The estimated date of exponential growth initiation is December 16, 2019 (CI: December 12 to Dec 21) and the exponential growth **rate** is 0.30 per day (CI: 0.26 to 0.34 per day). These estimates are consistent with estimates in the 'first arrival' approach ( Fig. 2F and G, and Fig. S4 )."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "**Title: The Novel Coronavirus, 2019-nCoV, is Highly Contagious and More Infectious Than Initially Estimated**"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "Our estimated outbreak growth **rate** is significantly higher than two recent reports where the growth **rate** was estimated to be 0.1 per day (1, 5). This estimate were based on early case counts from Wuhan (1) or international air travel data (5) . However, these data suffer from important limitations. The reported case counts in Wuhan during early outbreak are likely to be underreported because of many factors, and because of the low numbers of individuals traveling abroad compared to the total population size in Wuhan, inference of the infected population size and outbreak growth **rate** from infected cases outside of China suffers from large uncertainty (7, 8) . Our estimated exponential growth **rate**, 0.29/day (a doubling time of 2.4 days) is consistent the rapidly growing outbreak during late January (Fig. 1A) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "**Title: The Novel Coronavirus, 2019-nCoV, is Highly Contagious and More Infectious Than Initially Estimated**"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "Our estimated outbreak growth **rate** is significantly higher than two recent reports where the growth **rate** was estimated to be 0.1 per day (1, 5). This estimate were based on early case counts from Wuhan (1) or international air travel data (5) . However, these data suffer from important limitations. The reported case counts in Wuhan during early outbreak are likely to be underreported because of many factors, and because of the low numbers of individuals traveling abroad compared to the total population size in Wuhan, inference of the infected population size and outbreak growth **rate** from infected cases outside of China suffers from large uncertainty (7, 8) . Our estimated exponential growth **rate**, 0.29/day (a doubling time of 2.4 days) is consistent the rapidly growing outbreak during late January (Fig. 1A) ."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "**Title: The Novel Coronavirus, 2019-nCoV, is Highly Contagious and More Infectious Than Initially Estimated**"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "Fortunately, we see evidence that control efforts have a measurable effect on the **rate** of spread. Since January 23, Wuhan and other cities in Hubei province implemented vigorous control measures, such as closing down transportation and mass gatherings in the city; whereas, other provinces also escalated the public health alert level and implemented strong control measures. We noted that the growth **rate** of the daily number of new cases in provinces outside of Hubei slowed down gradually since late January (Fig. 3B) . Due to the closure of Wuhan (and other cities in Hubei), the number of cases reported in other provinces during this period shall start to track local infection dynamics rather than imports from Wuhan. We estimated that the exponential growth **rate** is decreased to 0.14 per day (CI: 0.12 to 0.15 per day) since January 30. Based on this growth **rate** and an R0 between 4.7 to 6.6 before the control measures, a calculation following the formula in Ref. (14) suggested that a growth **rate** decreasing from 0.29 per day to 0.14 per day translates to a 50%-59% decrease in R0 to between 2.3 to 3.0. This is in agreement with previous estimates of the impact of effective social distancing during 1918 influenza pandemic (18) . Thus, the reduction in growth **rate** may reflect the impact of vigorous control measures implemented and individual behavior changes in China during the course of the outbreak."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "**Title: The Novel Coronavirus, 2019-nCoV, is Highly Contagious and More Infectious Than Initially Estimated**"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "Fortunately, we see evidence that control efforts have a measurable effect on the **rate** of spread. Since January 23, Wuhan and other cities in Hubei province implemented vigorous control measures, such as closing down transportation and mass gatherings in the city; whereas, other provinces also escalated the public health alert level and implemented strong control measures. We noted that the growth **rate** of the daily number of new cases in provinces outside of Hubei slowed down gradually since late January (Fig. 3B) . Due to the closure of Wuhan (and other cities in Hubei), the number of cases reported in other provinces during this period shall start to track local infection dynamics rather than imports from Wuhan. We estimated that the exponential growth **rate** is decreased to 0.14 per day (CI: 0.12 to 0.15 per day) since January 30. Based on this growth **rate** and an R0 between 4.7 to 6.6 before the control measures, a calculation following the formula in Ref. (14) suggested that a growth **rate** decreasing from 0.29 per day to 0.14 per day translates to a 50%-59% decrease in R0 to between 2.3 to 3.0. This is in agreement with previous estimates of the impact of effective social distancing during 1918 influenza pandemic (18) . Thus, the reduction in growth **rate** may reflect the impact of vigorous control measures implemented and individual behavior changes in China during the course of the outbreak."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "**Title: The Novel Coronavirus, 2019-nCoV, is Highly Contagious and More Infectious Than Initially Estimated**"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "Fortunately, we see evidence that control efforts have a measurable effect on the **rate** of spread. Since January 23, Wuhan and other cities in Hubei province implemented vigorous control measures, such as closing down transportation and mass gatherings in the city; whereas, other provinces also escalated the public health alert level and implemented strong control measures. We noted that the growth **rate** of the daily number of new cases in provinces outside of Hubei slowed down gradually since late January (Fig. 3B) . Due to the closure of Wuhan (and other cities in Hubei), the number of cases reported in other provinces during this period shall start to track local infection dynamics rather than imports from Wuhan. We estimated that the exponential growth **rate** is decreased to 0.14 per day (CI: 0.12 to 0.15 per day) since January 30. Based on this growth **rate** and an R0 between 4.7 to 6.6 before the control measures, a calculation following the formula in Ref. (14) suggested that a growth **rate** decreasing from 0.29 per day to 0.14 per day translates to a 50%-59% decrease in R0 to between 2.3 to 3.0. This is in agreement with previous estimates of the impact of effective social distancing during 1918 influenza pandemic (18) . Thus, the reduction in growth **rate** may reflect the impact of vigorous control measures implemented and individual behavior changes in China during the course of the outbreak."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "**Title: The Novel Coronavirus, 2019-nCoV, is Highly Contagious and More Infectious Than Initially Estimated**"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "The 2019-nCoV epidemic is still rapidly growing and spread to more than 20 countries as of February 5, 2020. Here, we estimated the growth **rate** of the early outbreak in Wuhan to be 0.29 per day (a doubling time of 2.4 days), and the reproductive number, R0, to be between 4.7 to 6.6 (CI: 2.8 to 11.3). Among many factors, the Lunar New Year Travel rush in early and mid-January 2020 may or may not play a role in the high outbreak growth **rate**, although SARS epidemic also overlapped with the Lunar New Year Travel rush. How contiguous the 2019-nCoV is in other countries remains to be seen. If the value of R0 is as high in other countries, our results suggest that active and strong population-wide social distancing efforts, such as closing down transportation system, schools, discouraging travel, etc., might be needed to reduce the overall contacts to contain the spread of the virus. , between hospitalization to discharge (E) and between hospitalization to death (F). Filled circles and bars on x-axes denote the estimated mean and 95% confidence intervals. Histograms and the means (stars) of estimated R0 assuming individuals become infectious at symptom onset (blue) or 2 days before symptom onset (orange). The dotted line denotes R0=1. (B) The levels of minimum efforts (lines) of intervention st**rate**gies needed to control the virus spread. St**rate**gies considered are quarantine of symptomatic individuals and individuals who had contacts with them (x-axis) and population-level efforts to reduce overall contact **rate**s (y-axis). Different colored lines denote different assumptions of the fraction of asymptomatic individuals in the infected population. Solid and dashed lines correspond to R 0 =4.7 and 6.3 (i.e. the estimated means of R0), respectively. (C) The cumulative number of cases outside of Hubei province in late January 2020. The growth **rate** decreased to 0.14 per day since January 30. The dashed black line shows January 23 when Wuhan is locked down."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "**Incubation Period and Other Epidemiological Characteristics of 2019 Novel Coronavirus Infections with Right Truncation: A Statistical Analysis of Publicly Available Case Data**"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "Here, r is the exponential growth **rate** (estimated at 0.14 [11] ), and T is the latest time of observation (31 January 2020), and F (.) is the cumulative density function of f (.)."
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "**Title: Serial interval of novel coronavirus (COVID-19) infections Correspondence to**"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "where r is the exponential growth **rate** estimated at 0.14 [5] and T is the latest 10 time of observation (12 February 2020). The widely applicable information 11 criterion (WAIC) was used to compare between distributions and the model with 12 the minimal WAIC value was selected as the best-fit model for each set of 13 estimates with and without right truncation. 14"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "**Assessing the impact of reduced travel on exportation dynamics of novel coronavirus infection (COVID-19)**"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "where C is a constant (assumed to be one for the following calculation), and r is the exponential growth **rate** estimated at 0.14 per day [15] . The doubling time is then calculated as t d =ln(2)/r=4.95 days. The difference in the median date between (8) and (9) is thus described as:"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "for title, s in matching_snippets:\n",
    "    snippets.dm(f'**{title}**')\n",
    "    snippets.dm(re.sub('(rate)', '**\\g<0>**', s['text']))\n",
    "    snippets.dm('---')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 81,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Clinical Medicine Incubation Period and Other Epidemiological Characteristics of 2019 Novel Coronavirus Infections with Right Truncation: A Statistical Analysis of Publicly Available Case Data\n",
      "Journal Pre-proof Serial interval of novel coronavirus (COVID-19) infections Title: Serial interval of novel coronavirus (COVID-19) infections Authors\n",
      "Title: Serial interval of novel coronavirus (COVID-19) infections Correspondence to\n",
      "Assessing the impact of reduced travel on exportation dynamics of novel coronavirus infection (COVID-19)\n",
      "Title: The Novel Coronavirus, 2019-nCoV, is Highly Contagious and More Infectious Than Initially Estimated\n",
      "Incubation Period and Other Epidemiological Characteristics of 2019 Novel Coronavirus Infections with Right Truncation: A Statistical Analysis of Publicly Available Case Data\n",
      "Clinical Medicine Assessing the Impact of Reduced Travel on Exportation Dynamics of Novel Coronavirus Infection (COVID-19)\n"
     ]
    }
   ],
   "source": [
    "for pid in d[0.14]:\n",
    "    print(covid_papers[pid]['metadata.title'][0]['text'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "metadata": {},
   "outputs": [
    {
     "ename": "IndentationError",
     "evalue": "unexpected indent (<ipython-input-68-3c9e630ae444>, line 7)",
     "output_type": "error",
     "traceback": [
      "\u001b[0;36m  File \u001b[0;32m\"<ipython-input-68-3c9e630ae444>\"\u001b[0;36m, line \u001b[0;32m7\u001b[0m\n\u001b[0;31m    if snippet['rgd']:# and not stop_snippet(snippet['rgd']):\u001b[0m\n\u001b[0m    ^\u001b[0m\n\u001b[0;31mIndentationError\u001b[0m\u001b[0;31m:\u001b[0m unexpected indent\n"
     ]
    }
   ],
   "source": [
    "by_seed = defaultdict(list)\n",
    "\n",
    "for snippet in snps:\n",
    "    key = f\"{snippet['lgd']['word']}...{snippet['lgd']['seed']}\"\n",
    "    by_gd[key].append(snippet['sent'])\n",
    "            \n",
    "        if snippet['rgd']:# and not stop_snippet(snippet['rgd']): \n",
    "            key = f\"{snippet['rgd']['seed']}...{snippet['rgd']['word']}\"\n",
    "            by_gd[key].append(snippet['sent'])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 70,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      " 35%|      | 444/1278 [10:14<19:13,  1.38s/it]\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "[None]"
      ]
     },
     "execution_count": 70,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "[e.close() for e in tqdm._instances.copy()]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 73,
   "metadata": {
    "scrolled": false
   },
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> hypertension...diabetes </strong> </summary>\n",
       "  \n",
       "  - Third, although aging impacts immunity to viral infection [30] , patients older than 60 years of age usually have some **comorbidities, such as hypertension, diabetes,** or chronic bronchitis, and seek medical attention only when these primary diseases worsen when combined with ARI.\n",
       "\n",
       "\n",
       "  - **hypertension and diabetes)** or risk of infectious diseases among pregnant women.\n",
       "\n",
       "\n",
       "  - The enrolled patients had various **comorbidities; three patients were complicated with anxiety and depression requiring treatment, sixteen had hypertension, five had diabetes,** seven had peptic ulcers, and one had Sjogren's syndrome.\n",
       "\n",
       "\n",
       "  - Underlying medical conditions in these patients included, in descending order of frequency, **hypertension, diabetes** mellitus, chronic lung diseases, cerebrovascular disorders, cardiovascular disorders, malignancy, pregnancy, chronic hepatic diseases, chronic renal diseases, and neuromuscular diseases.\n",
       "\n",
       "\n",
       "  - In the multivariate analysis, **cardiovascular disorders (OR 4.05, 95% CI 1.72-9.50; P = 0.001), chronic lung diseases (OR 3.38, 95% CI 1.51-7.61; P = 0.003), hypertension (OR 2.37, 95% CI 1.26-4.47; P = 0.008), diabetes** mellitus (OR 3.09, 95% CI 1.57-6.06; P = 0.001), and neuromuscular diseases (OR 10.18, 95% CI 1.30-79.40; P = 0.027) were independently associated with hospitalization of patients with laboratory-confirmed influenza (Table 5 ).\n",
       "\n",
       "\n",
       "  - We were also able to confirm that underlying chronic diseases such as **cardiovascular disorders, chronic lung diseases, hypertension, diabetes** mellitus, and neuromuscular diseases were important factors in determining hospitalization of patients with laboratory-confirmed influenza.\n",
       "\n",
       "\n",
       "  - Patients with **comorbiditie**s (hypertension,** diabe**tes, chronic lung diseases, cardiovascular disorders or neuromuscular diseases) were at greater risk of hospitalization after influenza viral infection.\n",
       "\n",
       "\n",
       "  - A SI measurement 1 was associated with lower positive likelihood ratios in patients 65 years, with **hypertension, diabetes** or using -/Ca 2+ channel-blockers (range 4.9 to 6.5) compared with patients not exposed to these factors (range 7.6 to 11.6).\n",
       "\n",
       "\n",
       "  - Co-morbidities consisted of **hypertension (29.5%), renal insufficiency (13.6%), diabetes** (9.1%), dyslipidemia (5.7%), cardiac diseases (4.5%) and liver diseases (4.5%).\n",
       "\n",
       "\n",
       "  - Except asthmatic subgroup, **comorbidities such as hypertension, diabetes** mellitus, hyperlipidemia and coronary artery disease were frequently observed.\n",
       "\n",
       "\n",
       "  - Our first case is a 69-year-old Caucasian man with a past medical history of **coronary artery disease, chronic renal insufficiency, hypertension and type 1 diabetes.**\n",
       "\n",
       "\n",
       "  - Our second case is a 71-year-old Caucasian woman with a past medical history significant for **hypertension, type 1 diabetes** and chronic hepatitis C.\n",
       "\n",
       "\n",
       "  - The risk of developing type-2 **diabetes (T2D) increases with age, obesity, cardiovascular diseas**e (hypertension,** dyslipidaemia), lack of physical activity, and family history of diabe**tes.\n",
       "\n",
       "\n",
       "  - The signaling complex K/Nox is activated at different pathophysiological states: neurodegenerative diseases [120] , **atherosclerosis [121] , hypertension [122] , diabetes** [123] , and cancer [124] .\n",
       "\n",
       "\n",
       "  - Concomitant medical complications included: **hypertension (n = 5), diabetes** mellitus (DM)(n = 1) and asthma (n = 1).\n",
       "\n",
       "\n",
       "  - Several studies have verified that congenital heart disease has several influence factors, such as increased maternal age, prenatal infections, gestational **hypertension, gestational diabetes,** etc.\n",
       "\n",
       "\n",
       "  - Therefore, we should provide pre-pregnancy education to avoid gestational **hypertension and diabetes** and improve prenatal diagnosis rate to adopt measures in advance.\n",
       "\n",
       "\n",
       "  - It is explained by the high rate of comorbidity among our patients (more than half with arterial **hypertension and diabetes** mellitus), the deep immunosuppression required and the numerous complications of lung grafting.\n",
       "\n",
       "\n",
       "  - A total of 26 cases with fatalities were used to disclose the dangerous **comorbidities, showing that the major comorbidities are hypertension (53.8%), diabetes** (42.3%), coronary heart disease (19.2%), cerebral infarction (15.4%), chronic bronchitis (19.2%) and Parkinson's disease (7.7%) (Figure 2 ).\n",
       "\n",
       "\n",
       "  - Though the symptom compositions reported for the confirmed cases are similar to that of the fatality cases we collected, the percentages of **hypertension, diabetes,** and coronary heart diseases are much higher among the fatality cases than among the\n",
       "\n",
       "\n",
       "  - From the analysis of the fatal cases of this novel coronavirus pneumonia, the **comorbidities of hypertension, diabetes,** coronary heart disease, cerebral infarction, and chronic bronchitis were found to be dangerous factors that resulted in death.\n",
       "\n",
       "\n",
       "  - Patient medical records were reviewed and clinical data were extracted, including demographic characteristics, body mass index (BMI), smoking history, diagnostic method, Sequential Organ Failure Assessment (SOFA) score, nosocomial infection, and **comorbiditie**s (hypertension,** diabe**tes mellitus, chronic obstructive pulmonary disease [COPD], chronic kidney disease, chronic liver disease, current or previous solid organ malignancy).\n",
       "\n",
       "\n",
       "  - With respect to chronic diseases in poor patients, e.g., **hypertension, diabetes,** tuberculosis and severe mental disorders, family doctors were paid to offer systematic health management.\n",
       "\n",
       "\n",
       "  - Based on the involvement of PRCP in the conversion of these peptide hormones, the enzyme has also been associated with diseases, such as **hypertension, diabetes** mellitus, obesity, inflammation, and cardiovascular dysfunction (90, 91).\n",
       "\n",
       "\n",
       "  - Antiretroviral therapy for HIV may increase the risk of developing metabolic syndrome (abdominal **obesity, hyperglycaemia, dyslipidaemia and hypertension) and thus predispose to type 2 diabetes** and cardiovascular disease.\n",
       "\n",
       "\n",
       "  - Results from this study showed that the most common medical condition among the pilgrims was **hypertension (17.9%) followed by diabetes** (7.7%).\n",
       "\n",
       "\n",
       "  - Patients were at significant CV risk with 69% having a smoking history, and a history of CAD in 66%, dyslipidaemia in 60%, **hypertension in 82%, diabetes** in 24% and AF in 11%.\n",
       "\n",
       "\n",
       "  - The prevalence of CAD and cardiac risk factors including dyslipidaemia, **hypertension, diabetes** and cigarette smoking were similar in the two groups, as was LVEF <50%, the use of pharmacological agents, low density lipoprotein cholesterol, triglycerides levels, kidney function and troponin level (all p > 0.05).\n",
       "\n",
       "\n",
       "  - Health practices may differ significantly by group, particularly in health-promotion strategies and approaches to disease screening between source and host nations (for example, **hypertension, diabetes** in pregnancy, colon, uterine or breast cancer).\n",
       "\n",
       "\n",
       "  - In humans, CRF is mainly caused by **hypertension, glomerulonephritis, and diabetes** mellitus 3, 4 .\n",
       "\n",
       "\n",
       "  - However, in two of the papers studied the indices had been developed in populations with one condition only, either **hypertension 23 or type 2 diabetes.**\n",
       "\n",
       "\n",
       "  - In 2009, China launched the NEPHSP with nine items, including health records establishment, health education, immunization, child health, maternal health, geriatric health, **hypertension and type 2 diabetes** management, severe mental illness management, and the surveillance and control of infectious diseases and public health emergencies.\n",
       "\n",
       "\n",
       "  - Standard management of **hypertension and diabetes** has reached 86.27 million and 24.19 million patients respectively, in an equivalent to management rates of 35% and 30%.\n",
       "\n",
       "\n",
       "  - As to outcome indicators for systematic care of patients with **hypertension and diabetes,** the mortality of ischemic stroke and ischemic heart diseases increased in 2000-2013, except the mortality of haemorrhagic stroke has decreased since 2005, and mortality of diabetes increased slightly since 2005 (Fig.\n",
       "\n",
       "\n",
       "  - Cardiovascular risk assessment comprised age, sex, **hypertension, hypercholesterinaemia, diabetes** mellitus or ischaemic cardiomyopathy ( Table 1) .\n",
       "\n",
       "\n",
       "  - Currently, commercially available synthetic protease inhibitors targeting host proteases include (1) thrombin and Factor Xa inhibitors for coagulation (such as dabigatran and rivaroxaban); (2) angiotensin converting enzyme (ACE) inhibitors for **hypertension (such as lisinopril); (3) DPP-4 inhibitors for diabetes** (such as sitagliptin); and (4) proteasome inhibitors for multiple myeloma (such as bortezomib) ( Figure 2 ).\n",
       "\n",
       "\n",
       "  - Dabigatran and Rivaroxaban are anti-coagulants; Lisinopril is for **hypertension; Sitagliptin is for diabetes;** and Bortezomib is for multiple myeloma.\n",
       "\n",
       "\n",
       "  - Coexisting conditions including **coronary heart disease, chronic obstructive pulmonary disease, hypertension, diabetes,** cerebrovascular disease and cancer.\n",
       "\n",
       "\n",
       "  - According to the data from the third and fourth China National Health Services Survey, Chinese adults had worse health status in terms of prevalence of chronic diseases in 2008 than that in 2003, particularly **hypertension, diabetes,** heart disease and stroke.\n",
       "\n",
       "\n",
       "  - To improve the public health care, the Ministry of Health instituted eleven basic items of public health services in 2009, including the establishment of residents' health assessments, health education, health care of children, maternity patients, elderly and patients suffering from **hypertension, type II diabetes** and severe psychosis, vaccine inoculation and immunization, response to public health emergencies and assistance in sanitary control [16] .\n",
       "\n",
       "\n",
       "  - Although some village clinics had the authentic information about patients who suffered from **hypertension and diabetes,** they reported that a large proportion of patients were reluctant to see a doctor in the clinics because they were dissatisfied with the medical conditions there.\n",
       "\n",
       "\n",
       "  - The contract service there is a series of pre-paid package services divided into different kinds of chronic diseases including **hypertension, diabetes,** severe psychosis, chronic obstructive pulmonary disease, malignant tumor and disability.\n",
       "\n",
       "\n",
       "  - According to the statistics collected in March 2014, 76.50% of the contracted families bought **hypertension and/or diabetes** packages charging in middle class.\n",
       "\n",
       "\n",
       "  - However, more than 90% patients suffering from **hypertension and diabetes** in the pilot villages we visited, had been controlled by the village clinics, according to the county health bureau's report.\n",
       "\n",
       "\n",
       "  - County B also applied an audacious way to supplement the funding for public health, which became the most crucial incentive mechanism and underpinned the effectiveness in **hypertension and diabetes** control.\n",
       "\n",
       "\n",
       "  - Distinctively, county B applied an innovative fund raising way and partially solved the problem, also made an initial step toward **hypertension and diabetes** control.\n",
       "\n",
       "\n",
       "  - Inflammation in response to comorbid conditions, such as **hypertension and diabetes,** may play a proportionally larger role in HFPEF as compared to HF with reduced ejection fraction (HFREF).\n",
       "\n",
       "\n",
       "  - TNFa, TNFR1 and TNFR2 were each significantly associated with at least two of the following: age, estimated glomerular filtration rate, **hypertension, diabetes,** smoking, peripheral vascular disease or history of atrial fibrillation.\n",
       "\n",
       "\n",
       "  - Inclusion criteria for healthy controls were as follows: no history of **cardiovascular or renal disease, hypertension, diabetes,** or atrial fibrillation; and no prescriptions for antiarrhythmics; ACEi; ARB; beta-blockers; digoxin; loop or thiazide diuretics; or MRA.\n",
       "\n",
       "\n",
       "  - We explored the relationship between age, eGFR, sex, **obesity, LVH, hypertension, diabetes,** smoking status, peripheral vascular disease or history of atrial fibrillation with TNFa, TNFR1, TNFR2 and IL-6 in the combined HF group (Table 3) .\n",
       "\n",
       "\n",
       "  - TNFa, TNFR1 and TNFR2 increased differentially in response to smoking status or comorbid conditions, such as **hypertension and diabetes,** but were always associated with at least two covariates, while IL-6 was not associated with any covariate (Table 3) .\n",
       "\n",
       "\n",
       "  - The subjects in this study were 50 healthy controls (age .18 yrs), recruited from the hospital or research centre staff, with no history of kidney disease, **hypertension, or diabetes,** and 100 renal transplant recipients from The Ottawa Hospital Renal Transplant Program, age .18 yrs, and .3 months posttransplant.\n",
       "\n",
       "\n",
       "  - After adjusting for potential confounding factors [age, gender, **diabetes, eGFR, albuminuria, hypertension, and use of calcineurin inhibitors] only diabetes** was significantly associated with urinary ACE2 activity (p = 0.003) and protein levels (p,0.001, Table 2 ) in transplant patients.\n",
       "\n",
       "\n",
       "  - His past medical history was significant for **hypertension, diabetes** mellitus, chronic kidney disease, and non-Hodgkin's lymphoma with receipt of an allogeneic stem cell transplant 13 years prior.\n",
       "\n",
       "\n",
       "  - Failure to translate effective therapeutic strategies from the 'bench to bedside' may partly be attributed to the use of animal models that do not incorporate non-modifiable risk factors such as gender and many of the **stroke comorbidities observed in the clinical stroke population, such as hypertension, diabetes,** obesity, etc.\n",
       "\n",
       "\n",
       "  - There is growing evidence that the function of several types of vascular K + channels is altered during major **cardiovascular diseases, such as chronic hypertension, diabetes,** and atherosclerosis [40] .\n",
       "\n",
       "\n",
       "  - Furthermore, one of the pivotal hallmarks of **cardiovascular and metabolic diseases, including hypertension, diabetes,** atherosclerosis, stroke, and heart failure, is reduction-oxidation (redox) imbalance, a condition, known as oxidative stress [17] .\n",
       "\n",
       "\n",
       "  - Among the ESRD affected participants, **hypertension (61.7%) was the most common comorbidity, followed by type 2 diabetes** mellitus (41.2%).\n",
       "\n",
       "\n",
       "  - For the 2019-nCoV, limited comorbidity data is available; however, the twenty-six patients that have succumbed to the novel CoV had significant health conditions including **hypertension, diabetes,** heart and/or kidney function issues that may have made them more susceptible.\n",
       "\n",
       "\n",
       "  - For the MERS-CoV outbreak, smoking, **hypertension, diabetes,** cardiovascular disease, and/or other chronic illnesses have been present in the majority of deaths and correspond to findings in animal models [25] .\n",
       "\n",
       "\n",
       "  - The most common **comorbidities were hypertension in 54(32%) patients, cardiac failure in 36(21.3%) situations and diabetes** in 32(18.9%) cases.\n",
       "\n",
       "\n",
       "  - His past medical history was significant for **hypertension and diabetes** type 2.\n",
       "\n",
       "\n",
       "  - After this, a 61-year-old Saudi male with **hypertension and diabetes** with no history of smoking, reported for surgery.\n",
       "\n",
       "\n",
       "  - Physical inactivity has been linked to **obesity, coronary hearth disease, hypertension, strokes, diabetes,** colon cancer, breast cancer and osteoporotic fractures [49] .\n",
       "\n",
       "\n",
       "  - The fourth adult patient (S1), while not obviously immunosuppressed, also suffered from liver cirrhosis, **hypertension, type 2 diabetes,** and co-infection with herpes simplex virus, and required mechanical ventilation.\n",
       "\n",
       "\n",
       "  - In addition, most of those who have died from MERS-CoV, SARS-CoV and 2019-nCoV were advance in age and had underlying health conditions such as **hypertension, diabetes** or cardiovascular disease that compromised their immune systems [114] .\n",
       "\n",
       "\n",
       "  - In addition, to assess the physical and mental status of the participants, self-rated health status, chronic disease status **(hypertension, diabetes)** and daily life stress level were surveyed (S2 and S3 Files).\n",
       "\n",
       "\n",
       "  - In this survey, if respondents have **hypertension or diabetes,** they categorized in 'Chronic disease present'.\n",
       "\n",
       "\n",
       "  - Totally, 17 participants claimed to be healthy and not suffering from chronic conditions, whereas the remaining 23 participants suffered from various chronic conditions, including **hypertension, diabetes** mellitus, gout, liver disease, and heart disease.\n",
       "\n",
       "\n",
       "  - Consequently, it has a beneficial role in many diseases such as **hypertension, diabetes,** and cardiovascular disease where its expression is decreased.\n",
       "\n",
       "\n",
       "  - Its actions appear critical in a variety of disease states, including **hypertension, diabetes,** ageing, renal impairment, and cardiovascular disease.\n",
       "\n",
       "\n",
       "  - Second, chronic diseases in 2008 were more highly prevalent than in 2003, particularly **hypertension, diabetes,** heart disease, and stroke.\n",
       "\n",
       "\n",
       "  - For example, the prevalence of **hypertension and diabetes** in 2008 approximately doubled since 2003, and that of stroke increased by 1.4 times.\n",
       "\n",
       "\n",
       "  - Therefore a national education program that can eliminate the huge gaps among presence, awareness, treatment, and ability to control of **hypertension and diabetes** should be given to the public, clinicians, and healthcare decision makers.\n",
       "\n",
       "\n",
       "  - Especially, prevalence of **hypertension and diabetes** in 2008 was twice that in 2003.\n",
       "\n",
       "\n",
       "  - s=Serum leptin concentrations and leptin resistance, are independent markers of **obesity, weight gain, systemic hypertension [101] , diabetes** mellitus [102] , obstructive sleep apnoea [103] and myocardial infarction [104] , while polymorphisms of the leptin gene are associated with insulin resistance [105] and long-term risk of developing diabetes mellitus [102] .\n",
       "\n",
       "\n",
       "  - Cardiovascular disease, the leading cause of premature death and disability in most western countries, has a wellestablished multi-factorial basis involving a complex interplay between genetic predisposition, environmental and personal risk factors -including systemic **hypertension, diabetes** mellitus, hyperlipidaemia, obesity and cigarette smoking -and more recently recognized mechanisms, including endothelial dysfunction [118] , vascular inflammation [119] and leptin levels [104] .\n",
       "\n",
       "\n",
       "  - Systemic **hypertension, diabetes** mellitus and hyperlipidaemia have long-established, but complex, patterns of inheritance, a situation further compounded by evidence receptor polymorphisms -including those of angiotensin II type 1 receptor [120] , IRS-1 gene [121] and low density lipoprotein receptor (LDLR) [122] -both confer disease susceptibility and have regionally variable prevalences [123, 124] .\n",
       "\n",
       "\n",
       "  - This hypothesis exists at the confluence of established risk factors for **coronary artery disease, including genetic susceptibility, polymorphisms predisposing to hypertension [126] [127] [128] , diabetes** [126] and hypercholesterolaemia and substantial new data implicating vascular inflammation [119, 129] , endothelial dysfunction [119, 130] , leptin dysregulation [104] and viral infection [131, 132] in the pathogenesis of vascular disease.\n",
       "\n",
       "\n",
       "  - Buraczynska found that an association between renalase gene polymorphism and **hypertension in type 2 diabetes** [32] .\n",
       "\n",
       "\n",
       "  - Our results suggested that history of smoking and/or underlying clinical conditions may increase risk for infection, especially for patients with **hypertension, diabetes,** and heart-pulmonary insufficiency.\n",
       "\n",
       "\n",
       "  - Nearly half of the patients (8/18) had **hypertension, two had diabetes,** three had coronary heart diseases, and three had COPD, indicating that chronic heart and lung diseases would increase risk for H7N9 infection.\n",
       "\n",
       "\n",
       "  - [5, 6] In addition, an elevation of SUA level has been shown to be associated with **hypertension, [7] diabetes** mellitus (DM), metabolic syndrome, coronary disease, [8] and even cardiovascular mortality.\n",
       "\n",
       "\n",
       "  - ICD-9-CM codes were also used to identify patients treated with IABP who underwent PCI or CABG, and to identify patients treated with IABP who had baseline **comorbidities of hypertension (HTN) (ICD-9-CM: 401-405), diabetes** mellitus (DM) (ICD-9-CM: 250), hyperlipidemia (ICD-9-CM: 272), myocardial infarction (MI) (ICD-9-CM: 410), and cerebrovascular attack (CVA) (ICD-9-CM: 430-438).\n",
       "\n",
       "\n",
       "  - A 61-year-old male, who was originally from KSA and had a history of **hypertension and diabetes** without smoking, is reported here.\n",
       "\n",
       "\n",
       "  - 10 Furthermore, the increased maternal age is closely related to the occurrence of gestational **hypertension and gestational diabetes,** all of which are risk factors for congenital heart disease.\n",
       "\n",
       "\n",
       "  - Obesity and overweight (i.e., BMI> 25 kg/m 2 , prevalence: 24.5%, 95% CI: 18.6-31.6%), **hypertension (17.1%, 13.3-21.8%) and diabetes** (13.3%, 9.3-18.8%) were the most prevalent comorbidities in DENV.\n",
       "\n",
       "\n",
       "  - However, **hypertension (45.0%, 39.1-51.0%), diabetes** (24.7%, 20.2-29.8%) and heart diseases (25.6%, 19.5-32.7%) were the most prevalent in WNV.\n",
       "\n",
       "\n",
       "  - Obesity/overweight was the most prevalent comorbidity in DENV patients (24.5%, 95% CI: 18.6-31.6%), followed by **hypertension (17.1%, 95% CI: 13.3-21.8%) and diabetes** (13.3%, 95% CI: 9.3-18.8%) (Fig 2) .\n",
       "\n",
       "\n",
       "  - On the other hand, in WNV cases, **hypertension was the most frequent comorbidity (45.0%, 95% CI: 39.1-51.0%), followed by heart diseases (~25%), diabetes** (~25%) and stroke (10.1%, 95% CI: 7.1-14.3%) (Fig 3) .\n",
       "\n",
       "\n",
       "  - The present study demonstrates that **hypertension and diabetes** are the most prevalent comorbidity in both DENV and WNV with obesity/overweight and heart diseases being present, respectively, in about 20% of the DENV and WNV cases.\n",
       "\n",
       "\n",
       "  - Seven (26%) patients had at least one medical comorbidity **(hypertension [n = 6], diabetes** mellitus [n = 2] and asthma [n = 2]); and nine (33%) were obese (body mass index  27 kg/m 2 [35] ).\n",
       "\n",
       "\n",
       "  - Similarly, most of the pilgrims are older people (>50 years) with **comorbidities such as hypertension, diabetes,** and liver diseases [5, 6] .\n",
       "\n",
       "\n",
       "  - The patient had been receiving routine-biannual health check-ups, and did not have any major disease, except **hypertension and diabetes** mellitus.\n",
       "\n",
       "\n",
       "  - He had been under clinidipin and sitagliptin treatments for 5 years for **hypertension and diabetes,** respectively.\n",
       "\n",
       "\n",
       "  - Co-morbidities included **hypertension, diabetes** mellitus, cerebral infarction, coronary artery disease, and these co-morbidities were in stable state which was confirmed by physicians from related departments.\n",
       "\n",
       "\n",
       "  - Of the 122 cases (47%) with co-morbid conditions, the most common were **hypertension (76 cases, 29%), diabetes** (72 cases, 28%), heart disease (47 cases, 18%), obesity (41 cases, 16%), smoking (41 cases, 16%), chronic renal disease (39 cases, 15%), immunecompromised status (24 cases, 9%), chronic lung disease (19 cases, 7%), steroid use (19 cases, 7%), asthma (12 cases, 5%), chronic liver disease (8 cases, 3%), cancer (5 cases, 2%), pregnancy (5 cases, 2%), chronic hematologic disease (2 cases, 0.7%), systemic lupus erythematosus (2 cases, 0.7%), eclampsia (1 case, 0.4%), and hypothyroid (1 case, 0.4%).\n",
       "\n",
       "\n",
       "  - The most frequent **comorbidities recorded were hypertension (43.6%), psychiatric disorders including anxiety and/or depression (42.0%), heart failure (32.8%), ischemic heart disease (28.3%), and diabetes** mellitus (28.0%).\n",
       "\n",
       "\n",
       "  - Potentially confounding factors were assessed as well, including age, education, smoking status (never or ever smoker), alcohol drinking status (non-drinker or drinker), depression score (Center for Epidemiologic Studies-Depression Scale, CES-D), and selfreported comorbidity **(hypertension, diabetes,** stroke, and arthritis).\n",
       "\n",
       "\n",
       "  - After adjusting for age, education, smoking, alcohol drinking, depression score, comorbidity **(hypertension, diabetes,** stroke, and arthritis), and communication frequency, the men with lower density (<0.71) and higher size (4-6) had a higher BMI (=1.089, p=0.037) than the men with higher density (>0.83) and lower size (1-2), but this was not the case in the women (=0.391, p=0.393).\n",
       "\n",
       "\n",
       "  - We adjusted for **comorbidities, such as hypertension, diabetes,** stroke, and arthritis, in order to overcome this unintended effect.\n",
       "\n",
       "\n",
       "  - He was taking medications for **hypertension and diabetes** mellitus.\n",
       "\n",
       "\n",
       "  - His past medical history included **hypertension, type 2 diabetes** mellitus, hypercholesterolemia, severe coronary triple vessel disease, ischemic cardiomyopathy, and left ventricular dysfunction.\n",
       "\n",
       "\n",
       "  - The fifth topic of \"lifestyle disease\" contained the following words: \"test\", \"study\", \"patient\", \"risk\", **\"hypertension\", \"result\", \"diabetes\"**, \"cholesterol\", \"research team\", \"hepatitis\", \"male\", \"brother\", \"heart disease\", \"death\", \"swine flu\", \"USA\", \"stroke\", \"abnormal\", \"professor\", \"colon cancer\".\n",
       "\n",
       "\n",
       "  - Several studies 9, 11, [16] [17] [18] [19] including the current study (Table 3 , suggest that the male gender; health care worker; MERS patients above the age of 60 with an underlying medical conditions, such as **hypertension, diabetes** and renal failure; those who had exposure to camels and camel milk consumption; and close contact with MERS-CoV cases had a significantly higher risk of severe disease, including death.\n",
       "\n",
       "\n",
       "  - A national public health policy which is being implemented aims to improve the competence of primary health workers in the care of people with chronic diseases such as **hypertension, diabetes** and psychoses (Ministry of Health, 2009) .\n",
       "\n",
       "\n",
       "  - Personal data included sex, age, epidemiological history, history of smoking, and **comorbidities [e.g., chronic obstructive pulmonary disease (COPD), cancer, hypertension and/or diabetes]**.\n",
       "\n",
       "\n",
       "  - There was no significant difference in any comorbidity including **hypertension, diabetes,** chronic obstructive pulmonary disease (COPD), cancer, and others between the two groups (all P > 0.05).\n",
       "\n",
       "\n",
       "  - [14] Probably because of the small sample size, there was no significant difference in any comorbidity including **hypertension, diabetes,** COPD, cancer, and others between the two groups.\n",
       "\n",
       "\n",
       "  - 27 Patients with chronic diseases such as **hypertension, heart disease, diabetes,** and obesity are more vulnerable to excessive temperatures and at risk of complications.\n",
       "\n",
       "\n",
       "  - The reduction in NO due to the competition between Arg and iNOS is also important in the pathogenesis of a variety of diseases, including **atherosclerosis and hypertension, myocardial ischemia/reperfusion injury, diabetes,** allergic asthma and inflammatory diseases (47) (48) (49) (50) .\n",
       "\n",
       "\n",
       "  - 16 This was based on inadequately concentrated urine and failure to identify other metabolic causes of **hypertension such as diabetes** mellitus or hyperthyroidism.\n",
       "\n",
       "\n",
       "  - 13, 14) We estimated potential-outcome means (POMs) and average treatment effect (ATE) in the general population, patients with **hypertension, and those with diabetes** mellitus (DM) using an augmented inverse-probability-weighted (AIPW) estimator that combined elements of regression adjustment on PS and PS weighting estimation among several PS analysis methods.\n",
       "\n",
       "\n",
       "  - The types of medical institutions were determined by assessing which institution the participants used from among clinics, hospitals, and tertiary hospitals if they had a chronic disease such as **hypertension or diabetes** mellitus.\n",
       "\n",
       "\n",
       "  - Continuity of care, accessibility, and patient-centeredness, which are vital in the treatment of chronic diseases such as **hypertension and diabetes** mellitus, are primary health care characteristics that positively influence health outcomes and health care costs.\n",
       "\n",
       "\n",
       "  - **hypertension, diabetes,** epilepsy, hypotiroidism, etc.)\n",
       "\n",
       "\n",
       "  - In addition, most of those who have died had underlying health conditions such as **hypertension, diabetes** or cardiovascular disease that compromised their immune systems.\n",
       "\n",
       "\n",
       "  - The aims of this study were to determine prevalence of **hypertension in feline diabetes** mellitus (DM) and its potential relation with different parameters (age, body condition score (BCS), BCS previous to diagnosis, sex, glycemic control, time since diagnosis, concurrent diseases, chronic corticoid treatments, diabetic neuropathy and remission).\n",
       "\n",
       "\n",
       "  - Of the 28 patients, 10 (35.7%) had at least 1 underlying disease including **hypertension, diabetes,** asthma, chronic rhinitis, dyslipidemia, and hypothyroidism, and 1 patient had undergone surgery for lung cancer (Table 1) .\n",
       "\n",
       "\n",
       "  - Chen et al [2] and Huang et al [4] also reported that 32-51% of the hospitalized patients with confirmed COVID-19 had **hypertension, diabetes, and cardiovascular diseases, while 10 of the 28 Korean patients (35.7%) had at least 1 underlying disease, including hypertension** and diabetes.\n",
       "\n",
       "\n",
       "  - *Any underlying medical condition including chronic lung disease (asthma, chronic obstructive **pulmonary disease, interstitial lung disease, and bronchiectasis), chronic heart disease (coronary artery disease or congestive heart failure, hypertension, and valvulopathy), diabetes** mellitus, chronic kidney disease (with or without dialysis), neurologic disorders (epilepsy, cerebral palsy, dementia, or history of stroke), and chronic liver disease (hepatitis, cirrhosis, or hepatic failure).\n",
       "\n",
       "\n",
       "  - These patients often had underlying diseases such as **hypertension, diabetes,** and coronary heart diseases, with some having multiple underlying diseases.\n",
       "\n",
       "\n",
       "  - We collected well-known risk factors for hematoma expansion from patients' data by using electric medical records and the PACS system: patient's age and sex, location of hematoma, level of consciousness, systolic BP at admission, medical history (e.g., **hypertension, diabetes,** dyslipidemia, cardiovascular disease and chronic kidney disease [CKD]), alcohol consumption, smoking status, use of an antiplatelet agent, use of anticoagulants, and hematoma volumes.\n",
       "\n",
       "\n",
       "  - There was no statistically significant difference between the 2 groups in mean age, sex, size and volume of hematoma, spot signs on hematomas, BP, medical history **(hypertension, diabetes** mellitus, dyslipidemia, cardiovascular disease, stroke, alcohol), and drug use (antiplatelet or anticoagulant), except smoking status and CKD.\n",
       "\n",
       "\n",
       "  - [10] Concomitant diseases included arterial **hypertension (n = 5), diabetes** mellitus (n = 3), hypercholesterolemia (n = 2), coronary heart disease (n = 2), sleep apnea (n = 2), and celiac disease (n = 2).\n",
       "\n",
       "\n",
       "  - The most common coexisting medical conditions were **hypertension (31.7%), diabetes** (18.8%), solid organ malignancy (13.4%), and chronic lung disease (10.2%).\n",
       "\n",
       "\n",
       "  - 18 In the Fukuoka ischemic **stroke registry, mean age was 70 years, 76% had hypertension, 27% diabetes** mellitus, 46% hyperlipidemia, and 26% atrial fibrillation.\n",
       "\n",
       "\n",
       "  - Experience with HIV/AIDS treatment scale-up (40) could provide useful lessons for a standardized approach to management of **hypertension and diabetes,** thereby enhancing cost-effectiveness; facilitating supervision, monitoring, and evaluation; and ensuring accountability.\n",
       "\n",
       "\n",
       "  - Fifteen patients had **hypertension, 5 had diabetes** mellitus, 1 patient was HIV infected and 1 had congenital kidney disease.\n",
       "\n",
       "\n",
       "  - Three had asthma, five had **hypertension, and one had diabetes** mellitus type 2.\n",
       "\n",
       "\n",
       "  - The ratio of **hypertension, diabetes,** earlier myocardial infarction and ACBG were 58%, 27%, 32% and 15%, respectively, in the monitored population (13 females, 28 males; mean age: 68 years).\n",
       "\n",
       "\n",
       "  - The presence of microalbuminuria (MA) is a known risk factor in patients with **hypertension and diabetes.**\n",
       "\n",
       "\n",
       "  - Smoking, high cholesterol, **hypertension, and diabetes** did not significantly differ between the groups.\n",
       "\n",
       "\n",
       "  - Risk factors including **hypertension and diabetes** were documented.\n",
       "\n",
       "\n",
       "  -  all persons aged 50 years, because in this group greatly increases the number of people belonging to high risk groups  adults and children who have chronic **pulmonary(including asthma), or cardiovascular (except isolated hypertension), renal, hepatic, neurological, hematologic or metabolic disorders (including diabetes** mellitus);\n",
       "\n",
       "\n",
       "  - All case-patients had concurrent medical conditions; the most common were heart disease (70%, 14/20), dementia (65%, 13/20), **hypertension (40%, 8/20), diabetes** (35%, 7/20), and lung disease (35%, 7/20).\n",
       "\n",
       "\n",
       "  - 37.2% of patients had at least one underlying disorder (i.e., **hypertension, diabetes,** cardiovascular diseases and chronic lung diseases).\n",
       "\n",
       "\n",
       "  - Of these deceased patients, 64.9% had at least one underlying disorder (i.e., **hypertension, diabetes,** cardiovascular disease, or chronic obstructive pulmonary disease) ( Table 2 ).\n",
       "\n",
       "\n",
       "  - 57.0% of the patients had at least one of the concomitant diseases including **hypertension, diabetes,** cardiovascular diseases and chronic lung diseases.\n",
       "\n",
       "\n",
       "  - Meanwhile, numbers of patients with COVID-19 infection had chronic **comorbidities, mainly hypertension, diabetes** and cardiovascular disease, which is similar to MERS-COV population.\n",
       "\n",
       "\n",
       "  - Seventy-one patients had underlying diseases, including 28 patients with **hypertension, 12 patients with diabetes,** and 31 patients with other conditions.\n",
       "\n",
       "\n",
       "  - The most common chronic medical diseases were **hypertension (n =12, 34.3%) and diabetes** (n =7, 20%).\n",
       "\n",
       "\n",
       "  - All the six AMI patients were aged >60 years; five of them had two or more underlying **comorbiditie**s (hypertension,** diabe**tes, cardiovascular diseases, and chronic obstructive pulmonary disease).\n",
       "\n",
       "\n",
       "  - Patients who progressed to severe NCP were more likely to have underlying **hypertension (P = 0.004), but did not otherwise have significant differences in other co-morbidities including diabetes,** chronic obstructive pulmonary disease (COPD), cardiovascular disease, cerebrovascular disease and chronic hepatitis B infection.\n",
       "\n",
       "\n",
       "  - All of those who died had underlying diseases, the most common of which was **hypertension (16/25, 64%), followed by diabetes** (10/25, 40%), heart diseases (8/25, 32%), kidney diseases (5/25, 20%), cerebral infarction (4/25, 16%), chronic obstructive pulmonary disease (COPD, 2/25, 8%), malignant tumors (2/25, 8%) and acute pancreatitis (1/25, 4%).\n",
       "\n",
       "\n",
       "  - Interpretation The age and underlying diseases **(hypertension, diabetes,** etc.)\n",
       "\n",
       "\n",
       "  - As concerning the underlying diseases, the most common one was **hypertension, followed by diabetes,** Heart disease, kidney disease, cerebral infarction, COPD, malignant tumors and acute pancreatitis.\n",
       "\n",
       "\n",
       "  - In conclusion, the age and underlying diseases **(hypertension, diabetes,** etc.)\n",
       "\n",
       "\n",
       "  - The analysis of COVID-19 patients with cardiac-related chronic diseases **(coronary artery diseas**e, hypertension** and type 2 diabe**tes mellitus) showed that they were more prone to be in the severe or critical conditions.\n",
       "\n",
       "\n",
       "  - Percentage of patients with **hypertension was higher in severe group (22 [52.4%] vs 4 [12.9%], P < 0.001), while other coexisting disorders, including diabetes,** cardiovascular disease and chronic obstructive pulmonary disease showed no significant differences.\n",
       "\n",
       "\n",
       "  - Thirty-three severe patients with a median age of 66 years(range,33-82 years,IQR, 57-76) were treated in intensive care units; out of these patients, 66.7 %(22) were men and 19 (57.5%) had chronic diseases, including **hypertension, diabetes,** heart failure, stroke, and renal insufficiency.\n",
       "\n",
       "\n",
       "  - Including 33 patients in the intensive care unit (ICU), the median age of the patients was 66 years(IQR, 57-76), ranging from 33-82 years old, with 22 (66.7%) being men and 19 (57.5%) patients having chronic diseases, including **hypertension, diabetes,** heart failure, stroke, and renal insufficiency.\n",
       "\n",
       "\n",
       "  - The bulk of death cases had comorbidity (76.8%), including **hypertension (56.1%), heart disease (20.7%), diabetes** (18.3%), cerebrovascular disease (12.2%), and cancer (7.3%).\n",
       "\n",
       "\n",
       "  - https://doi.org/10.1101/2020.02.26.20028191 doi: medRxiv preprint of death cases had comorbidity (75.6%), including **hypertension (56.1%), heart disease (20.7%), diabetes** (18.3%), cerebrovascular disease (12.2%), and cancer (7.3%).\n",
       "\n",
       "\n",
       "  - 16 Moreover, we found underlying diseases such as **hypertension, heart disease and diabetes** were very common in our death cases, and 30.5% of patients had 2 or more comorbidities.\n",
       "\n",
       "\n",
       "  - Consistent with recent reports, our results suggest that 2019-nCoV is more likely to infect elderly people with chronic **comorbidities as a result of the weaker immune functions of these patients [5, 6] , and 2019-nCoV-associated death is also related to elder age and underlying illnesses, especially hypertension, diabetes,** and cardiac disease.\n",
       "\n",
       "\n",
       "  - There were 10 patients with basic diseases (27.78%), including 7 patients with **hypertension (19.44%), 4 patients with diabetes** (11.11%), 1 patient with hypothyroidism (2.78%), and 1 patient with hyperlipidemia (2.78%).\n",
       "\n",
       "\n",
       "  - Thirty-nine cases (38.2%) had **comorbidities, including hypertension, diabetes,** coronary artery disease, tuberculosis, hepatitis B and chronic bronchitis.\n",
       "\n",
       "\n",
       "  - The majority of our patients were 50 to 70 years old with common chronic diseases, such as **hypertension, diabetes,** tuberculosis, hepatitis B, chronic bronchitis, etc.\n",
       "\n",
       "\n",
       "  - Participants were assigned a '1' if they endorsed a history of each of the following medical conditions: **cerebrovascular risk factors including hypertension, heart failure, COPD, anemia, diabetes;** endocrine dysfunction including thyroid disease, testosterone therapy, estrogen therapy; kidney disease.\n",
       "\n",
       "\n",
       "  - Thus, our main goal is to establish the odds ratio for COVID-19 mortality, adjusted to age and gender-specific prevalence of the morbidities reported in the CDC paper 11 , namely **hypertension, diabetes,** cardiovascular disease, chronic respiratory disease and cancer.\n",
       "\n",
       "\n",
       "  - Age is described by intervals of decades (0-9; 10-19; up to  80 years old) and the **comorbidities are: hypertension, diabetes,** cardiovascular disease, chronic respiratory disease and cancer.\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> diabetes...hypertension </strong> </summary>\n",
       "  \n",
       "  - Exclusion criteria for the patients and the control subjects were history of a chronic disease (tuberculosis, **diabetes, hypertension,** heart disease, hepatic diseases, etc.\n",
       "\n",
       "\n",
       "  - We focused on the most common chronic diseases that were set as nation health priorities for treatment and prevention, such as **diabetes, heart disease, asthma, hypertension,** and cancer.\n",
       "\n",
       "\n",
       "  - A 57-year-old patient presenting with type 2 **diabetes mellitus and hypertension** was admitted to Saint-Pierre Hospital with fever (39  C), an elevated C-reactive protein (248 mg/L), and acute urinary retention requiring the placement of a urinary catheter.\n",
       "\n",
       "\n",
       "  - Of the 79 hospitalized patients, the frequency of **diabetes, hypertension,** cases having at least one of the comorbid conditions, and the proportion of elderly were significantly higher in patients with influenza A (H3N2) (P,0.05).\n",
       "\n",
       "\n",
       "  - Because old age, **diabetes, essential hypertension,** and -/Ca 2+ channel-blockers might reduce the compensatory increase in heart rate and mask blood pressure reductions in shock or pre-shock states, we hypothesized that these factors weaken the association between SI and mortality, reducing the utility of SI to identify patients at risk.\n",
       "\n",
       "\n",
       "  - We examined whether age 65 years, **diabetes, essential hypertension,** and use of -/Ca 2+ channel-blockers modifi ed the association between SI and mortality.\n",
       "\n",
       "\n",
       "  - The risk of developing type-2 **diabetes (T2D) increases with age, obesity, cardiovascular diseas**e (hypertension,** dyslipidaemia), lack of physical activity, and family history of diabe**tes.\n",
       "\n",
       "\n",
       "  - Allergy was not evaluated because the symptoms are very broad to specifically define as an allergy, and there would be significant potential information bias as it is less likely to clinically diagnosed allergy as compared with asthma, **diabetes, hypertension** and heart disease.\n",
       "\n",
       "\n",
       "  - We identified older age, male sex and underlying medical conditions, including **diabetes mellitus, renal disease, respiratory disease, heart disease and hypertension,** as clinical predictors of death associated with MERS.\n",
       "\n",
       "\n",
       "  - Several clinical predictors of death were identified including older age, male sex and underlying medical conditions, including **diabetes mellitus, renal disease, respiratory disease, heart disease and hypertension.**\n",
       "\n",
       "\n",
       "  - Exclusion criteria were ischemic heart disease, respiratory, renal or neurological disease, **diabetes mellitus, hypertension,** BMI > 30, BMI < 18; smoker; pregnant; use of vasoactive medication or NSAIDS/aspirin within 48 h of study; current involvement in other research studies.\n",
       "\n",
       "\n",
       "  - The chemical chaperone TUDCA is a classic ER stress inhibitor that improves ER folding capacity, which has been protective in various diseases, including **diabetes mellitus, hypertension,** calcification, and even cardiac dysfunction by preventing ER stress (135) (136) (137) .\n",
       "\n",
       "\n",
       "  - The comparison of probability of malaria infection was made between patients with **diabetes (n=675) and a control group (n=791) comprising patients attending hypertension** clinics, patients attending other hospital clinics, and hospital staff.\n",
       "\n",
       "\n",
       "  - The one person in the general population group who was seropositive for MERS-CoV was female in her mid-50s with **diabetes and hypertension.**\n",
       "\n",
       "\n",
       "  - Other studies have shown no change in blood pressure in ACE2 knock-out (KO) mice compared to control C57BL/6 (Tikellis et al., 2008; Shiota et al., 2010) , but once **diabetes was induced in ACE2 KO mice, hypertension** did develop (Tikellis et al., 2008) .\n",
       "\n",
       "\n",
       "  - However, in our own study of 503 Australian Caucasians with type 2 **diabetes , the prevalence of hypertension** was significantly higher in both men and women and was associated with the G allele of the ACE2 SNP rs4240157 .\n",
       "\n",
       "\n",
       "  - reported that, among 41 confirmed cases, the symptom composition is as follows: pneumonia (100%), fever (98%), cough (76%), lymphopenia (63%), Dyspnea (55%), fatigue (44%), sputum production (28%), headache (8%), hemoptysis (5%), and diarrhea (3%); and the underlying diseases included **diabetes (20%), hypertension (15%), and cardiovascular** disease (15%) [21] .\n",
       "\n",
       "\n",
       "  - Hypoalbuminaemia is also an important marker of severe infection and/or byproduct of chronic disease such as **diabetes, hypertension** and CKD.\n",
       "\n",
       "\n",
       "  - Seventeen patients had **diabetes (33.3 %), 25 had hypertension (49 %), 14 (27.5 %) had end stage renal disease, eight (15.7 %) had coronary** artery disease and six (11.8 %) patients were immunosuppressed, two of whom were pregnant.\n",
       "\n",
       "\n",
       "  - Analysis of the clinical features of 41 hospitalized patients with laboratory-confirmed 2019-nCoV infection revealed that 30 were men (73%); less than one-half had underlying co-morbid conditions (13; 32%) which included **diabetes (8, 20%) , hypertension (6, 15%), and cardiovascular** disease (6; 15%); and the average age was 49.0 years old.\n",
       "\n",
       "\n",
       "  - All of the normal control mothers, which had no pregnancy-related problems, such as **diabetes or hypertension,** had or were carrying fetuses lacking any congenital defects.\n",
       "\n",
       "\n",
       "  - Until more is understood about MERS-CoV infection, we believe that patients with **diabetes mellitus, hypertension,** ischemic heart disease, and end-stage renal diseases should be considered at a higher risk for MERS-CoV infection and in the triage, these patients seem to be prioritized for treatment.\n",
       "\n",
       "\n",
       "  - First, long-term treatment with GCs is often accompanied by severe side effects, such as **diabetes, increased risk of infection, osteoporosis, hypertension,** and so forth [25, 26] .\n",
       "\n",
       "\n",
       "  - Linagliptin has also demonstrated efficacy and tolerability in type 2 **diabetes patients with hypertension** and microalbuminuria [149] .\n",
       "\n",
       "\n",
       "  - The study authors collected various clinical data: fever (temperature  38C); presence of any of the following respiratory symptoms: cough, bloody cough, shortness of breath, or chest pain; presence any of the following gastrointestinal symptoms: diarrhea, vomiting, or nausea; MERS-CoV infection; intensive care unit (ICU) admission; hospital: KAMC-Riyadh or KFGH-Jeddah; abnormal radiology findings; **diabetes; renal disease; and hypertension.**\n",
       "\n",
       "\n",
       "  - Alterations in the composition of the gut microbiota and reductions in microbial diversity lead to different disorders such as inflammatory conditions of the intestine (inflammatory bowel disease, irritable bowel syndrome, colorectal cancer) (Gagnire et al., 2016; Ghoshal et al., 2017; Rapozo et al., 2017) , type 2 **diabetes, obesity, anorexia nervosa, forms of severe acute malnutrition, cardiovascular diseases (atherosclerosis, hypertension,** heart failure) (Tang et al., 2017) , neurobehavioral diseases (autism spectrum disorder, major depression) (Clemente et al., 2012) .\n",
       "\n",
       "\n",
       "  - The newest study including the first 41 confirmed cases admitted to Wuhan between 16 December 2019 and 2 January 2020 showed the median age of patients was 49 years; and the main underlying diseases were **diabetes, hypertension, and cardiovascular** diseases.\n",
       "\n",
       "\n",
       "  - Age above 65 years, history of dengue infection, **diabetes mellitus, hypertension** and renal insufficiency have been identified as being significantly associated with DHF/DSS (Lee et al., 2006) .\n",
       "\n",
       "\n",
       "  - The one person in the general population group who was seropositive for MERS-CoV was female in her mid-50s with **diabetes and hypertension.**\n",
       "\n",
       "\n",
       "  - In fact, the contract service policy aims to construct a closer relationship between rural residents and village doctors, which can urge village doctors to improve the efficacy of public health services, for example, the early detection of **diabetes and hypertension,** and other chronic diseases control [23] .\n",
       "\n",
       "\n",
       "  - Twenty-eight (24.3%) of 115 patients of had at least one underling illness, including **diabetes, ischemic heart disease, hypertension, chronic obstructive pulmonary** disease, asthma, cerebrovascular disease, and chronic hepatitis.\n",
       "\n",
       "\n",
       "  - The tree shows that age is the most discriminatory attribute, followed by BMI, family history of **diabetes, work-related stress, physical activity, salty food preference, hypertension, gender, and history of cardiovascular** disease or stroke.\n",
       "\n",
       "\n",
       "  - 1 Including: **diabetes, hypertension, cardiovascular,** neurological, and immunodeficiency diseases; 2 This was calculated as collection date minus admission date; 3 Included: cephalosporin, piperacillin/tazobactam, and meropenem; 4 Included: fluconazole, voriconazole, and liposomal amphotericin B.\n",
       "\n",
       "\n",
       "  - Overactivation of the RAS in **diabetes, especially Ang II and ACE [3] , leads to hypertension,** fluid retention, and inflammation, causing renal and vascular end-stage disease in the long term [35] .\n",
       "\n",
       "\n",
       "  - ACE2 activity) while the following explanatory variables were entered into the model: age, gender, **diabetes, eGFR, albuminuria, hypertension,** and use of calcineurin inhibitors.\n",
       "\n",
       "\n",
       "  - The median age for transplant subjects was 54 (IQR, 42-62) yrs, 65% were male, 38% had **diabetes, and 88% had hypertension.**\n",
       "\n",
       "\n",
       "  - After adjusting for potential confounding factors [age, gender, **diabetes, eGFR, albuminuria, hypertension, and use of calcineurin inhibitors] only diabetes** was significantly associated with urinary ACE2 activity (p = 0.003) and protein levels (p,0.001, Table 2 ) in transplant patients.\n",
       "\n",
       "\n",
       "  - Some studies suggest that **diabetes, hypertension,** chronic heart disease, and chronic kidney disease are risk factors for MERC-CoV infection and a severe clinical presentation [19] ; however, this theory needs to be further investigated.\n",
       "\n",
       "\n",
       "  - Both groups were matched regarding the age, gender, body mass index, the association of **diabetes or hypertension,** and Euro score.\n",
       "\n",
       "\n",
       "  - The AKI group match the other group with respect to age, gender, body mass index, the association of **diabetes or hypertension.**\n",
       "\n",
       "\n",
       "  - **diabetes, hypertension,** chronic respiratory disease, etc.)\n",
       "\n",
       "\n",
       "  - Recognized risk factors for IVIG-induced thrombosis Frontiers in Immunology | Primary Immunodeficiencies include male gender; age >60; **diabetes; renal insufficiency, dyslipidemia; hypertension; immobility; coronary** disease; pre-existing vascular disease, family history of early thromboembolic disease; atrial fibrillation, high-dose and high-speed IVIG infusions.\n",
       "\n",
       "\n",
       "  - The comorbidity conditions that might have effect on disease susceptibility, such as **diabetes mellitus, heart disease, hypertension,** asthma, tuberculosis, and malignancy, were traced by reviewing medical records and self reporting.\n",
       "\n",
       "\n",
       "  - They had no history of cardiorespiratory illness, influenza-like illness in the week prior to the trial, or any pregnancyrelated complications such as gestational **diabetes, hypertension,** intrauterine growth restriction, placenta praevia, ruptured membranes, or threatened preterm labor.\n",
       "\n",
       "\n",
       "  - Case report A 60 year old man with a history of **diabetes and hypertension** was referred to our clinic with the suspicion of infective encephalitis.\n",
       "\n",
       "\n",
       "  - According to the pre-embarkation medical assessment, 28% of pilgrims were classified as high risk due to underlying health conditions such as **diabetes, hypertension,** other chronic diseases or if they were 60 years or older.\n",
       "\n",
       "\n",
       "  - The covariate list included the following: gestational age at first antenatal visit, maternal age less than 19 y, maternal age more than 35 y, multiple births, maternal medical risk factors, labor/delivery complications, birth defects, maternal **diabetes (gestational and/or non-gestational), hypertension** (treated or untreated), mother insured, multivitamin use in pregnancy, history of smoking during pregnancy, history of alcohol use during pregnancy, black race, education less than 12 th grade, mother's marital status, and maternal weight before pregnancy.\n",
       "\n",
       "\n",
       "  - b In the secondary adjusted models, the covariates included gestational age at first antenatal visit, maternal age less than 19 y, maternal age more than 35 y, multiple births, maternal medical risk factors, labor/delivery complications, birth defects, maternal **diabetes (gestational and/or non-gestational), hypertension** (treated or untreated), mother insured, multivitamin use in pregnancy, history of smoking during pregnancy, history of alcohol use during pregnancy, black race, education less than 12th grade, mother's marital status, and maternal weight before pregnancy.\n",
       "\n",
       "\n",
       "  - b In the secondary adjusted models, the covariates included gestational age at first antenatal visit, maternal age less than 19 y, maternal age more than 35 y, multiple births, maternal medical risk factors, labor/delivery complications, birth defects, maternal **diabetes (gestational and/or non-gestational), hypertension** (treated or untreated), mother insured, multivitamin use in pregnancy, history of smoking during pregnancy, history of alcohol use during pregnancy, black race, education less than 12th grade, mother's marital status, and maternal weight before pregnancy.\n",
       "\n",
       "\n",
       "  - Altered G6PD activity is associated with pathophysiology, such as autophagy, insulin resistance, infection, inflammation, as well as **diabetes and hypertension.**\n",
       "\n",
       "\n",
       "  - ), pre-existing diseases such **diabetes or hypertension,** or choice of treatment [26, 30] .\n",
       "\n",
       "\n",
       "  - The covariates were age (in years), sex (male, female), nationality (Saudi, Non-Saudi), admission year (2014-2015, 2016-2017), vital signs on admission (body temperature, heart rate, respiratory rate, systolic and diastolic blood pressure), MERS-CoV associated conditions on admission (pneumonia/acute respiratory distress syndrome, septic shock/multiple organ failure, abdominal pain/diarrhoea), pre-existing comorbid conditions on admission **(diabetes, hypertension,** hepatitis C, chronic renal failure, and chronic heart disease), receipt of the treatment regimen (yes, no), duration of hospital stay (in days), status at discharge (alive, dead).\n",
       "\n",
       "\n",
       "  - Variables that showed a significant association (P <0.10) in univariate analysis (age, sex, the dialysis type, BMI, SGA, albumin level, TA-SUA, and **diabetes) or were of considerable theoretical relevance (total Kt/V, calcium level, phosphorus level, hypertension,** gout, uricosuric drugs, and antiuricosuric drugs) were entered into the multivariate stepwise linear regression models.\n",
       "\n",
       "\n",
       "  - 1. any known cardiorespiratory conditions (including but not limited to the following: asthma, bronchitis, cystic fibrosis, congenital heart disease, emphysema) 2. any known medical conditions that may be exacerbated by strenuous physical activity, including but not limited to the following: exercise-induced asthma, lower respiratory infections (including pneumonia, bronchitis) in the past 2 weeks, anxiety disorders, **diabetes, hypertension,** or epilepsy/seizure disorders 3. any physical disability from medical, orthopaedic or neuromuscular disorders 4. have an acute upper respiratory tract infection or symptomatic rhinitis (i.e.\n",
       "\n",
       "\n",
       "  - MERS-CoV may infect people of any age, but older age, underlying comorbidity (such as **diabetes mellitus, renal disease, respiratory disease, heart disease, and hypertension)**, and delayed confirmation or late diagnosis are all factors that affect MERS disease outcomes and mortality [2] [3] [4] [5] [6] [7] .\n",
       "\n",
       "\n",
       "  - The OR of severe WNV was 4.21 in patients with **diabetes (95% CI: 2.22-7.92) and 2.72 in those with hypertension** (95% CI: 1.78-4.14) (Table 5 ).\n",
       "\n",
       "\n",
       "  - Although the prevalence of **diabetes, hypertension** and heart diseases varied widely among the DENV selected studies (ranging from 60-to 100-fold), the vast majority of the reports showed values clustering around the pooled estimated averages for each comorbidity, as evidenced by the low I 2 index values.\n",
       "\n",
       "\n",
       "  - In line with these findings, the results of the present study strongly suggest that **diabetes, heart diseases and hypertension** are more prevalent in severe DENV and WNV cases than in the non-severe disease and may present significant risk factors-together with advanced age-in complication of flavivirus infection.\n",
       "\n",
       "\n",
       "  - We provided evidence for a higher prevalence of **diabetes, hypertension,** and heart diseases in severe cases of flavivirus infections such as DENV and WNV than in nonsevere cases.\n",
       "\n",
       "\n",
       "  - Most of the patients had no major comorbidity, except one had **diabetes mellitus and the other two had hypertension.**\n",
       "\n",
       "\n",
       "  - CHD and **diabetes were the commonest medical risk factors reported among H7N9 patients, and the prevalence of smoking and hypertension** was higher in subjects with H7N9 compared with the other patient groups.\n",
       "\n",
       "\n",
       "  - Moreover, red ginseng is believed to improve immune function, physical performance and sexual function, and manage cancer, **diabetes (DM) and hypertension.**\n",
       "\n",
       "\n",
       "  - The second topic of \"lifestyle disease\" contained the following words: **\"diabetes\", \"osteoporosis\", \"hypertension\", \"exercise\", \"risk\", \"cholesterol\", \"obesity\"**, \"blood vessels\", \"blood pressure\", \"metabolic syndrome\", \"abnormal\", \"stroke\", \"gout\", \"bone\", \"study\", \"fat\", \"intake\", \"smoking\", \"weight\", and \"fracture\".\n",
       "\n",
       "\n",
       "  - Healthy staff members, excluding those having underlying high-risk illnesses such as **diabetes, chronic lung, heart (except for hypertension)**, renal, and liver diseases, as well as those taking immunosuppressive drugs and pregnant women are recommended to be assigned to related task areas (BII) [39, 40] .\n",
       "\n",
       "\n",
       "  - The prevalence of **diabetes, hypertension,** and chronic kidney disease (CKD) was 13.3%, 13.3%, and 10%, respectively.\n",
       "\n",
       "\n",
       "  - The prevalence of **diabetes, hypertension,** and CKD was not different between the two groups.\n",
       "\n",
       "\n",
       "  - Baseline characteristics of the 3 patients who underwent CRRT were as follows: #1, 71-yr old male who had **diabetes, hypertension, CKD, and heart disease; #2, 76-yr old male who had CKD, chronic pulmonary** disease, and chronic liver disease; #3, 86-yr old male who had diabetes, hypertension, and heart disease.\n",
       "\n",
       "\n",
       "  - The prevalence of **diabetes, hypertension,** and renal disease were 67%, 50%, and 42%, respectively.\n",
       "\n",
       "\n",
       "  - She had underlying **diabetes mellitus and hypertension,** and was diagnosed with atri-37 al fibrillation 7 days prior to admission.\n",
       "\n",
       "\n",
       "  - (a) General treatment principles: treatment according to the symptoms and the underlying disease **(diabetes, hypertension,** etc.\n",
       "\n",
       "\n",
       "  - Other chronic conditions have been reported in 96% of clinical cases including **diabetes, hypertension,** heart disease and kidney disease.\n",
       "\n",
       "\n",
       "  - The relationship between **diabetes mellitus (DM) and hypertension** is complex.\n",
       "\n",
       "\n",
       "  - Hypertension was not significantly correlated with the control of DM; however, none of the dogs considered as in good control of the **diabetes had severe hypertension** (P = 0.067).\n",
       "\n",
       "\n",
       "  - As with most chronic diseases, such as **diabetes and hypertension,** constant maintenance and management are required.\n",
       "\n",
       "\n",
       "  - Moreover, the percentage of patients with **diabetes (87.5% vs 43.75%), hypertension** (75% vs 18.75%) as well as those on mechanical ventilation (25% vs 0%), interferon and ribavirin (25% vs 0%), oseltamivir (50% vs.6.25%), and those with chronic kidney disease (37.5% vs 6.25%), and cancer (25% vs 6.25%) were higher among those cases compared to their counterparts with medical costs lower than the mean cost.\n",
       "\n",
       "\n",
       "  - 31 Most of the patients in the study sample had chronic health conditions, such as **diabetes (58%), hypertension** (37%), chronic kidney disease (17%), and cancer (12%).\n",
       "\n",
       "\n",
       "  - Uncontrolled **diabetes mellitus and hypertension were the most common comorbidities** reported among three of the cases.\n",
       "\n",
       "\n",
       "  - Uncontrolled **diabetes mellitus, hypertension, and ischemic heart diseases were the most common comorbidities** present and were reported among three cases including the deceased.\n",
       "\n",
       "\n",
       "  - Finally, Saudis live longer now, which explains the rise in cancer cases and chronic illnesses like **diabetes, hypertension and hyperlipidemia.**\n",
       "\n",
       "\n",
       "  - Comorbid medical conditions were documented for 88 (62%) cases, with **diabetes (n = 63; 44%) and hypertension** (n = 57; 40%) most common.\n",
       "\n",
       "\n",
       "  - 39 Results emanating from different studies have shown that influenza outbreaks are characterized by high severity of symptoms with increased mortality, 6, 8, 13, 25, 32 Several factors have been associated with H1N1 disease severity, such as factors due to the virus (ie, viral pathogenic mutations, resistance to antivirals), factors inherent to host susceptibility (eg, age, sex, race), including physiological immunosuppression or acquired diseases (ie, **diabetes, hypertension, obesity,** asthma), factors associated with available medical services and public health facilities, and factors arising from the presence of bacterial coinfections.\n",
       "\n",
       "\n",
       "  - Patient B had no underlying disease such as **diabetes, hypertension** or chronic renal disease.\n",
       "\n",
       "\n",
       "  - This striking difference in the incidence of renal impairment is likely due to the low prevalence of underlying conditions such as chronic renal diseases, **diabetes and hypertension** in our population.\n",
       "\n",
       "\n",
       "  - He had a history of atrial fibrillation, **diabetes mellitus, hypertension,** chronic kidney disease, and hypothyroidism.\n",
       "\n",
       "\n",
       "  - AD genes are also expressed in the blood-brain barrier (BBB), which is disrupted by AD risk factors (age, alcohol, aluminum, concussion, cerebral hypoperfusion, **diabetes, homocysteine, hypercholesterolemia, hypertension, obesity,** pesticides, pollution, physical inactivity, sleep disruption, smoking) and by pathogens, directly or via olfactory routes to basal-forebrain BBB control centers.\n",
       "\n",
       "\n",
       "  - These include **diabetes, midlife hypertension or obesity,** smoking, and physical inactivity [206] .\n",
       "\n",
       "\n",
       "  - HAC dogs were at increased risk for concurrent **diabetes mellitus, urinary tract infection, urolithiasis, hypertension,** gall bladder mucocele, and thromboembolic disease compared with dogs without HAC.\n",
       "\n",
       "\n",
       "  - In traditional Chinese medicine, it has been used in treatments for hyperlipemia, **diabetes mellitus, Alzheimer's disease, acute liver injury, inflammation, photophobia, headache, dizziness and hypertension** (4) (5) (6) .\n",
       "\n",
       "\n",
       "  - A 74-year-old Saudi male patient with **diabetes, hypertension, and dyslipidemia** presented with a 3 days of ataxia, vomiting, confusion, and fever of 39.5 C (Fig.\n",
       "\n",
       "\n",
       "  - Ten participants had coexisting conditions **(diabetes mellitus, hypertension,** asthma).\n",
       "\n",
       "\n",
       "  - One additional patient had **diabetes mellitus type 2 and hypertension.**\n",
       "\n",
       "\n",
       "  - Among the 42 sporadic case-patients, median age was 57 (range 25-89) years; 35 (83%) were male, and 33 (79%) reported underlying conditions, most commonly **diabetes (n = 24) and hypertension** (n = 23).\n",
       "\n",
       "\n",
       "  - Median patient age was 56 (range 29-84) years; 28 (88%) were male, and 23 (72%) reported underlying conditions, most commonly **diabetes (n = 19) and hypertension** (n = 15).\n",
       "\n",
       "\n",
       "  - Compared with NICUGP, patients in ICUGP were more likely to have developed postoperative arrhythmia (57.9% vs 12.9%, P < 0.001), were older, of higher ASA status, and more likely to have **diabetes, coronary artery disease, hypertension,** a higher cancer stage, and to have received more intraoperative blood products.\n",
       "\n",
       "\n",
       "  - Sixteen percent (16/88) of the infected patients reported underlying conditions such as human immunodeficiency virus (HIV) and human T-cell leukemia virus (HTLV)-type infection, **diabetes mellitus, liver transplant, megacolon syndrome, hypertension,** Wiskott-Aldrich syndrome, and congenital kidney and lung diseases.\n",
       "\n",
       "\n",
       "  - It is governmentally funded offering free screening for breast cancer, **diabetes, hypertension and obesity** for all women above 45 years.\n",
       "\n",
       "\n",
       "  - Six (13%) HCWs reported having a chronic medical condition (e.g., asthma, **diabetes, hypertension)**, and 6 (13%) reported smoking tobacco.\n",
       "\n",
       "\n",
       "  - A total of 49 participants (1 case-patient and 48 non-case-patients) reported having >1 chronic condition (e.g., asthma, **diabetes, heart disease, hypertension,** breast cancer) ( Table 1 ).\n",
       "\n",
       "\n",
       "  - The MERS case-patient reported having asthma; among non-case-patients, the most common chronic conditions reported were asthma (31%), **diabetes (25%), and hypertension** (18%).\n",
       "\n",
       "\n",
       "  - Certain underlying conditions, including **diabetes mellitus (DM), hypertension,** chronic cardiac disease, and chronic renal disease, are recognized risk factors for illness and death caused by infection with MERS-CoV (2,3).\n",
       "\n",
       "\n",
       "  - The first case-patient was a 65-year-old male retiree with a history of **diabetes, heart disease, smoking, and hypertension.**\n",
       "\n",
       "\n",
       "  - The third case-patient was a 76-year-old male farmer with a history of **diabetes, heart disease, and hypertension.**\n",
       "\n",
       "\n",
       "  - The fifth case was a 73-year-old male retiree with **diabetes and hypertension.**\n",
       "\n",
       "\n",
       "  - In addition, she also suffered from type 2 **diabetes mellitus, hypertension,** osteoporosis, bilateral osteoarthritis of the knees, and an old traumatic fracture of the left humeral neck.\n",
       "\n",
       "\n",
       "  - All but two patients (96%) had one or more chronic medical conditions, including **diabetes (68%), hypertension** (34%), heart disease (28%), and kidney disease (49%).\n",
       "\n",
       "\n",
       "  - 2 patients had **diabetes and hypertension** respectively.\n",
       "\n",
       "\n",
       "  - 24 Similarly, in our study, all the six AMI patients were aged >60 years and had one or more underlying conditions, including **diabetes, hypertension, COPD, and cardiovascular** diseases.\n",
       "\n",
       "\n",
       "  - Epidemiological survey shows that COVID-19 patients with underlined conditions such as **diabetes, hypertension, cardiovascular** disease or the elderly are not only susceptible, but also often more serious.\n",
       "\n",
       "\n",
       "  - There was no significant difference of **diabetes, cardiovascular disease and chronic obstructive pulmonary disease between the two groups, but the patients with hypertension** in the severe group were more than those in the mild group (p = 0.056) ( Table 1) .\n",
       "\n",
       "\n",
       "  - After adjusting for age and smoking status, patients with COPD (HR 2.681, 95%CI 1.424-5.048), **diabetes (HR 1.59, 95%CI 1.03-2.45), hypertension** (HR 1.58, 95%CI 1.07-2.32) and malignancy (HR 3.50, 95%CI 1.60-7.64) were more likely to reach to the composite endpoints than those without.\n",
       "\n",
       "\n",
       "  - On admission, 20-51% of patients reported as having at least one comorbidity, with **diabetes (10-20%), hypertension (10-15%) and cardiovascular** and cerebrovascular diseases (7-40%) being most common [3, 4, 6] .\n",
       "\n",
       "\n",
       "  - For instance, **diabetes [29] and COPD [30] frequently co-exist with hypertension or coronary** heart diseases.\n",
       "\n",
       "\n",
       "  - Among three deaths, one was infected with SARS-CoV-2 after recent lung cancer surgery, one had **diabetes, ischemic cardiomyopathy, and hyperthyroidism, and one had hypertension and coronary** heart disease [16, 17] .\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> pulmonary...hypertension </strong> </summary>\n",
       "  \n",
       "  - In addition to their role in vascular diseases, chymases were also found to be associated with lung diseases such as **pulmonary fibrosis, pulmonary arterial hypertension** (PAH), asthma, and COPD [116, [120] [121] [122] .\n",
       "\n",
       "\n",
       "  - For example, aldosterone has been implicated in the pathogenesis of **pulmonary hypertension [62] , and spironolactone has been shown to attenuate experimental pulmonary hypertension** via MR inhibition in pulmonary artery smooth muscle cells [63] .Admittedly, bleomycin is also a frequently used tool drug to induce pulmonary hypertension [64, 65] .\n",
       "\n",
       "\n",
       "  - Attenuation or lack of CD39 activity is associated with vascular dysfunction and remodeling in **pulmonary arterial hypertension** [32] , decreased hepatic regeneration, and failure of vascular reconstitution [33] .\n",
       "\n",
       "\n",
       "  - Expression was undetectable in lung from patients with **pulmonary arterial hypertension.**\n",
       "\n",
       "\n",
       "  - Apelin is down-regulated in **pulmonary arterial hypertension (PAH), a devastating disease characterized by vascular remodeling resulting in progressive obliteration of the pulmonary** circulation, leading to right ventricle (RV) hypertrophy and right heart failure (Alastalo et al., 2011; Chandra et al., 2011) .\n",
       "\n",
       "\n",
       "  - The importance of BMP signaling for the vascular system is demonstrated by the association of mutations in genes encoding for BMPRII and ALK1 with the development of two genetic vascular diseases, namely, **pulmonary arterial hypertension** and hereditary hemorrhagic telangiectasia.\n",
       "\n",
       "\n",
       "  - Also, in **pulmonary arterial hypertension virus infection is considered to be a possible risk factor (203) , and pulmonary** hypertension has been reported to be a side effect of prolonged treatment with type I IFN (204, 205) .\n",
       "\n",
       "\n",
       "  - TRAIL has been found to be increased within **pulmonary vascular lesions of patients with pulmonary hypertension (206) and also in a mouse model of hypoxia-induced pulmonary** hypertension, levels of soluble TRAIL correlated with right ventricular systolic pressure, right ventricular hypertrophy, and pathologic alterations (33, 34) .\n",
       "\n",
       "\n",
       "  - MiRNAs have shown to have an effect also upon **pulmonary arterial hypertension** (PAH).\n",
       "\n",
       "\n",
       "  - [78] , where AntagomiR-223 was efficient in reducing levels of miR-223 in **pulmonary artery and lungs of rats as compared to controls, but failed to attenuate pulmonary hypertension.**\n",
       "\n",
       "\n",
       "  - shown that hypoxia results in reduced nitric oxide levels, a principle cause of **pulmonary vasoconstriction and that inhalation of nitric oxide can reduce hypoxic pulmonary hypertension,** improve the patient's ventilation to perfusion ratio, improve hypoxemia, and improve HAPE in patients with signs and symptoms, as well as decrease hospitalization time with few adverse effects.\n",
       "\n",
       "\n",
       "  - Anisodamine can improve **pulmonary vascular spasms, decrease pulmonary vascular resistance and hypoxic pulmonary hypertension, improve the pulmonary** microcirculation, and maintain smooth pulmonary blood flow.\n",
       "\n",
       "\n",
       "  - cystic fibrosis, **pulmonary sarcoidosis, pulmonary hypertension f All solid tumors including bronchial carcinoma g Patients with one of the following conditions: primary or secondary antibody deficiency, congenital immunodeficiency, immunosuppressive therapy other than steroids, severe malnutrition with cachexia h Highest/lowest value within a time period of 3 days before and 3 days after date of rtPCR i Other infections (n = 10); neoplastic diseases (n =6); collagen vascular; other rheumatologic or autoimmune (n = 14); sarcoidosis (n = 3); non-infectious nonneoplastic pulmonary** diseases (n = 23); cardiovascular diseases (n = 3) Other chronic lung diseases, e.g.\n",
       "\n",
       "\n",
       "  - cystic fibrosis, **pulmonary sarcoidosis, pulmonary hypertension** f All solid tumors including bronchial carcinoma g Patients with one of the following conditions: primary or secondary antibody deficiency, congenital immunodeficiency, immunosuppressive therapy other than steroids, severe malnutrition with cachexia h Highest/lowest value within a time period of 3 days before and 3 days after date of rtPCR i Other infections (n = 14); neoplastic diseases (n = 1); collagen vascular; other rheumatologic or autoimmune (n = 1); gastroesophageal diseases (n = 1) Other chronic lung diseases, e.g.\n",
       "\n",
       "\n",
       "  - cystic fibrosis, **pulmonary sarcoidosis, pulmonary hypertension f All solid tumors including bronchial carcinoma g Patients with one of the following conditions: primary or secondary antibody deficiency, congenital immunodeficiency, immunosuppressive therapy other than steroids, severe malnutrition with cachexia h Highest/lowest value within a time period of 3 days before and 3 days after date of rtPCR i Other infections (n = 24); neoplastic diseases (n = 7); collagen vascular; other rheumatologic or autoimmune (n = 15); sarcoidosis (n = 3); non-infectious nonneoplastic pulmonary** diseases (n = 23); cardiovascular diseases (n = 3); gastroesophageal diseases (n = 1) Pathogens associated with mixed infections were counted individually 31 of 40 adults (78%).\n",
       "\n",
       "\n",
       "  - (2) neurological diseases (Alzheimer's disease, Parkinson's disease, ischemia, central nerve system injury): a disintegrin and metalloproteinase domain-containing protein 10 (ADAM10), -secretase, -secretase 1, and MMP-24; (3) inflammation (arthritis, chronic obstructive **pulmonary disease (COPD), asthma, Crohn's disease, inflammatory bowel disease, colitis, diverticulitis, chronic liver disease): MMPs, cathepsins, neutrophil elastase, and ADAM17; (4) blood coagulation: thrombin, factor Xa; (5) diabetes: dipeptidyl peptidase 4 (DPP4); (6) cardiovascular disease (atherosclerosis, hypertension, cardiomyopathy, congestive** heart failure, myocardial infarction, neovascularization and cardiac remodeling, cardiac fibrosis, aneurism, ischemia-reperfusion injury): angiotensin converting enzyme, renin, MMPs, chymase, neprilysin, calpain; (7) osteoporosis: cathepsin K; and (8) autoimmune diseases (multiple sclerosis, lupus, Guillain-Barre syndrome, psoriasis): activated protein C, kallikreins; ageing (skin), MMPs, neutrophil elastase [1, 2] .\n",
       "\n",
       "\n",
       "  - Chronic lung tissue changes and **pulmonary alveolar proteinosis (PAP), as well as pulmonary arterial hypertension,** have been described in adult patients [1, [7] [8] [9] .\n",
       "\n",
       "\n",
       "  - PAP was found in 18% and **pulmonary arterial hypertension** in 9% of patients.\n",
       "\n",
       "\n",
       "  - In dogs, heart disease has been related to serious conditions, including progressive heart failure, **pulmonary edema and/or congestion, hypertension, left arterial** rupture, and pericardial tamponade (Borgarelli et al., 2008; Guglielmini, 2003; Pouchelon et al., 2015) .\n",
       "\n",
       "\n",
       "  - Acute lung injury (ALI) is a syndrome characterized by diffuse **pulmonary edema and severe hypoxemia in the absence of clinical evidence of left atrial hypertension** [1] .\n",
       "\n",
       "\n",
       "  - [14] found an incidence of **pulmonary hypertension of 92.2% but did not find any association between pulmonary hypertension** and death.\n",
       "\n",
       "\n",
       "  - Clinically, right ventricular failure has no agreed definition, but criteria (using **pulmonary artery catheter data) include pulmonary hypertension** associated with an RV cardiac index <2.5 L min 1 m 2 and a right atrial pressure >8 mmHg [9] .\n",
       "\n",
       "\n",
       "  - Keywords: **pulmonary fibrosis, canine infectious respiratory disease complex (CIRDC), pulmonary hypertension,** pneumonia, interstitial lung disease, PIBO BACKGROUND Post-infectious bronchiolitis obliterans (PIBO), a syndrome in children most commonly caused by Mycoplasma pneumonia (1, 2) and adenovirus (1) and occasionally Bordetella pertussis (3, 4) is associated with chronic inflammatory and fibrotic lesions of small airways leading to chronic airflow obstruction (5) .\n",
       "\n",
       "\n",
       "  - Another important cause of severe complications is **pulmonary edema, which can be consequence of left ventricular systolic or diastolic dysfunction, or due to the use of tocolytic agent, fluid overload, severe hypertension** or severe renal disease.\n",
       "\n",
       "\n",
       "  - Vascular injury and remodeling may lead to **pulmonary arterial hypertension** which may compromise right ventricular function associated with a poor clinical outcome [13] .\n",
       "\n",
       "\n",
       "  - Most hospitals and intensive care units worldwide use the standard criteria for the diagnosis of acute lung injury (ALI)/ARDS: presence of acute hypoxemia (PaO 2 /FIO 2 less than 300 mmHg or 39.99 Kpa for ALI or less than 200 mmHg or 26.66 Kpa for ARDS), bilateral infiltrates seen on a frontal chest radiograph that are consistent with **pulmonary edema, and no clinical evidence of left atrial hypertension, or (if it is measured) a pulmonary** artery wedge pressure (PAWP) of less than 18 mmHg according to the 1994-1998 American-European Consensus Conference on ARDS (AECC) [25, 26] .\n",
       "\n",
       "\n",
       "  - (b) In cases of severe ARDS with right ventricular dysfunction and **pulmonary artery hypertension** consider prone position and inhaled nitric oxide.\n",
       "\n",
       "\n",
       "  - IL-17 is also important for iBALT formation in patients with **pulmonary arterial hypertension** (92, 93) .\n",
       "\n",
       "\n",
       "  - Chronic thromboembolic **pulmonary hypertension (CTEPH) is one of the primary causes of severe pulmonary hypertension.**\n",
       "\n",
       "\n",
       "  - Pulmonary endarterectomy (PEA) can prevent mortality due to right ventricle failure (1) , which occurs in chronic thromboembolic **pulmonary hypertension (CTEPH), a type-4 pulmonary hypertension** (2) .\n",
       "\n",
       "\n",
       "  - Mean **pulmonary artery pressure >25 mmHg at rest or >30 mmHg during exercise was considered to indicate the presence of pulmonary hypertension.**\n",
       "\n",
       "\n",
       "  - The natural killer cell has been found to be involved in vascular remodeling, which may lead to **pulmonary arterial hypertension** (52, 53) .\n",
       "\n",
       "\n",
       "  - Numerous studies have demonstrated that the polyamine regulatory pathway is a pharmacological target in **pulmonary arterial hypertension** (54, 55) .\n",
       "\n",
       "\n",
       "  - Altered G6PD status is implicated in many cellular pathophysiological processes and diseases, including hypoxia, inflammation, microbial infection, sepsis, **pulmonary vessel dilation, diabetes, hypertension,** kidney disease, and brain injuries [24,30,.\n",
       "\n",
       "\n",
       "  - Consequently, G6PD overactivation contributes to remodeling of **pulmonary arterial and development of pulmonary hypertension** [138] .\n",
       "\n",
       "\n",
       "  - The severity of SNV-associated HCPS may have made it more apparent that the onset of **pulmonary edema, tachycardia and hypertension** seemed to be all but universally temporally associated with the appearance of a spectrum of highly-activated cells of the lymphoid lineage in the peripheral blood.\n",
       "\n",
       "\n",
       "  - The indication for thrombolytic therapy was: (1) patients with haemodynamic instability or cardiogenic shock; (2) patients with massive **pulmonary artery obstruction (obstructions in more than 2 lobes on CTPA, or perfusion defects in more than 7 segments on V/Q scan) and combined right ventricular dysfunction (RVD) and pulmonary hypertension** on echocardiographic examination.\n",
       "\n",
       "\n",
       "  - Using the current subset of data, five major classes of diseases were observed in the disease Ontology-based enrichment analysis as follows: cancer (prostate cancer, thyroid cancer, breast cancer and neoplasm metastasis), diseases of the respiratory or **pulmonary system (asthma, bronchial hyperreactivity, pulmonary alveolar proteinosis), degenerative diseases of the central nervous system (Amyotrophic lateral sclerosis, Parkinson's Disease), vascular disease (atherosclerosis, hypertension)** and encephalitis (rabies).\n",
       "\n",
       "\n",
       "  - This process reduces **pulmonary ventilatory efficiency and leads progressively to general hypoxia and cardio-pulmonary hypertension** and failure.\n",
       "\n",
       "\n",
       "  - The histological features of **pulmonary arteriopathy observed in 3 adult marmosets (0.47 %) were indicative of pulmonary hypertension,** and concurrent hypertrophic cardiomyopathy was present in at least one of the affected animals.\n",
       "\n",
       "\n",
       "  -  Chronic lung disease included asthma, reactive airway disease, bronchopulmonary dysplasia, chronic obstructive **pulmonary disease, or emphysema, or the requirement for home oxygen combined with other lung conditions such as chronic respiratory failure or pulmonary hypertension.**\n",
       "\n",
       "\n",
       "  - For example, apelin administration showed cardioprotective effects in heart failure 12 , and ameliorated the development of **pulmonary arterial hypertension** in rats 13,14 and humans 15 .\n",
       "\n",
       "\n",
       "  - A previous study had used a concentration of up to 100 nmol/min for systemic infusion, where they obtained a therapeutic response in patients with **pulmonary arterial hypertension** and the highest dose was well tolerated 15 .\n",
       "\n",
       "\n",
       "  - Furthermore, ER stress markers correlated with disease severity (the presence of **pulmonary arterial hypertension)** and IL-6 levels (260) .\n",
       "\n",
       "\n",
       "  - ET A R/ET B R and selective ET A R antagonists, such as bosentan, macitentan, and ambrisentan, are used clinically for the treatment of **pulmonary arterial hypertension,** but their use is associated with side effects, including edema, anemia, increased risk of heart failure, and hepatic transaminitis (Wei et al., 2016; Packer et al., 2017) .\n",
       "\n",
       "\n",
       "  - Based on the dual-feedback system linking the endothelin system and the RAS, dual AT 1 R/ET A R antagonists have been developed and tested in the clinic for **pulmonary arterial hypertension, essential hypertension,** and chronic kidney disease (Murugesan et al., 2002 (Murugesan et al., , 2005 Neutel et al., 2008; Komers and Plotkin, 2016; Komers et al., 2017) ; however, it is still to be established whether the protective benefits outweigh the risk of adverse reactions.\n",
       "\n",
       "\n",
       "  - Therefore, recombinant human angiotensin-converting enzyme 2 (rhACE2) is currently considered for treating acute respiratory distress syndrome and **pulmonary arterial hypertension.**\n",
       "\n",
       "\n",
       "  - The second study is an openlabel dose escalation study for rhACE2 (NCT03177603) in patients with **pulmonary arterial hypertension, in which changes in pulmonary** vascular resistance, cardiac output, and mean pulmonary artery pressure in response to rhACE2 serve as primary outcome measures.\n",
       "\n",
       "\n",
       "  - Recombinant ACE2 and brain-selective APA inhibition are currently being tested clinically in **pulmonary arterial hypertension and overweight patients with hypertension,** respectively, based on the concept that ACE2 degrades Ang II and upregulates the protective Ang 1-7, whereas APA blockade prevents brain AT 1 R stimulation by Ang III.\n",
       "\n",
       "\n",
       "  - In model 2, BNP was included as an additional variable in the regression model, since **pulmonary hypertension is associated with poor prognosis in chronic pulmonary tuberculosis 16, 17 and BNP is a non-invasive marker of pulmonary hypertension** 18, 19 (Table 3 ).\n",
       "\n",
       "\n",
       "  - These findings are consistent with the results of previous studies 19, 25, 26 , which demonstrated that BNP levels were a prognostic factor in patients with idiopathic **pulmonary arterial hypertension, idiopathic pulmonary** fibrosis, and chronic cor pulmonale.\n",
       "\n",
       "\n",
       "  - Even so, depending on severity, development of pre-capillary **pulmonary arterial hypertension** may also occur.\n",
       "\n",
       "\n",
       "  - Unfortunately, right ventricular function in dogs with **pulmonary arterial hypertension** has been poorly investigated.\n",
       "\n",
       "\n",
       "  - Venous thromboembolism (VTE) including acute **pulmonary embolism (APE), chronic thromboembolic pulmonary hypertension** (CTEPH) and deep venous thrombosis (DVT) is a global disease.\n",
       "\n",
       "\n",
       "  - iNO therapy is currently FDA approved for the treatment of term and near-term neonates with hypoxemic respiratory failure associated with clinical or echocardiographic evidence of **pulmonary arterial hypertension** (26, 27) .\n",
       "\n",
       "\n",
       "  - respiratory failure, **pulmonary heart disease cachexia) or COPD-non related (eg obesity, diabetes mellitus, arterial hypertension)** [23] [24] [25] .\n",
       "\n",
       "\n",
       "  - Therefore, we evaluated the ability of transverse RV strain and strain rate, obtained by 2-dimensional speckle tracking echocardiography in healthy dogs and dogs with **pulmonary hypertension, to identify dogs with pulmonary hypertension.**\n",
       "\n",
       "\n",
       "  - We classified dogs as having moderate **pulmonary hypertension if TR velocity > 3.5 m/sec, and severe pulmonary hypertension** if TR velocity > 4.5 m/sec.\n",
       "\n",
       "\n",
       "  - Although transverse RV strain (but not strain rate) showed a negative relationship with class of **pulmonary hypertension, it was not useful in identifying dogs with pulmonary hypertension.**\n",
       "\n",
       "\n",
       "  - In **pulmonary hypertension (PH) patients, this led to the reconceptualization of the RV as part of right ventricular pulmonic circulation unit and the acknowledgement that RV function is a major determinant of prognosis in pulmonary arterial hypertension.**\n",
       "\n",
       "\n",
       "  - Over-secretion of ANP is correlated with hypoxia, which leads to **pulmonary vasoconstriction, pulmonary hypertension,** and right-heart overload [22, 23] .\n",
       "\n",
       "\n",
       "  - It is also described in previous studies that approximately 14% of affected dogs also have evidence of associated **pulmonary arterial hypertension.**\n",
       "\n",
       "\n",
       "  - Echocardiographic evaluation of the right **pulmonary artery distensibility index (RPAD index) was recently described as a valuable method for early detection and severity evaluation of pulmonary arterial hypertension** in dogs.\n",
       "\n",
       "\n",
       "  - It is also described in previous studies that approximately 14% of affected dogs also have evidence of associated **pulmonary arterial hypertension.**\n",
       "\n",
       "\n",
       "  - Echocardiographic evaluation of the right **pulmonary artery distensibility index (RPAD index) was recently described as a valuable method for early detection and severity evaluation of pulmonary arterial hypertension** in dogs.\n",
       "\n",
       "\n",
       "  - Although endothelium-derived TXA 2 causes excessive **pulmonary vascular resistance by acting as a potent vasoconstrictor, mitogen of vascular smooth muscle cells and promoter of platelet aggregation, there is no current therapeutic strategy for pulmonary hypertension,** targeting TXA 2 formation.\n",
       "\n",
       "\n",
       "  - These results suggest that TXA 2 synthase inhibitor may contribute to the improvement of **pulmonary hemodynamics in pulmonary hypertension by alleviating pulmonary** vasoconstriction and preventing thrombosis.\n",
       "\n",
       "\n",
       "  - A new echocardiographic index, the right **pulmonary vein to pulmonary artery ratio (PV/PA) has been described for the detection of pulmonary venous hypertension.**\n",
       "\n",
       "\n",
       "  - ARDS is defined as rapid onset of hypoxia with PaO 2 /fraction inspired oxygen (FiO 2 ) ratio <300 and bilateral **pulmonary infiltrates in the absence of left atrial hypertension.**\n",
       "\n",
       "\n",
       "  - *Any underlying medical condition including chronic lung disease (asthma, chronic obstructive **pulmonary disease, interstitial lung disease, and bronchiectasis), chronic heart disease (coronary artery disease or congestive heart failure, hypertension, and valvulopathy), diabetes** mellitus, chronic kidney disease (with or without dialysis), neurologic disorders (epilepsy, cerebral palsy, dementia, or history of stroke), and chronic liver disease (hepatitis, cirrhosis, or hepatic failure).\n",
       "\n",
       "\n",
       "  - The **pulmonary vein-to-pulmonary artery ratio (PV/PA) has been shown to be decreased in West Highland white terriers affected with pulmonary fibrosis in comparison with age-and breedmatched controls, suggesting its potential use in the non-invasive diagnosis of pulmonary hypertension** (PH).\n",
       "\n",
       "\n",
       "  - Echocardiographic evaluations were performed on 127 healthy dogs (50 dogs had been included in a previous study of TAPSE), 55 dogs with heart disease but no **pulmonary hypertension (tricuspid regurgitant velocity <3 m/s) and 39 dogs with pulmonary hypertension** of various etiologies and severities (tricuspid regurgitant velocity >2.9 m/s).\n",
       "\n",
       "\n",
       "  - Tricuspid annular plane excursion, fractional area change, tissue Doppler imaging and speckle-tracking echocardiography have been used to assess RV systolic function in healthy dogs and dogs with **pulmonary artery hypertension** (PAH).\n",
       "\n",
       "\n",
       "  - Pulmonary-vein-to**-pulmonary-artery ratio (PV:PA) has been proposed as a method of identifying various stages of cardiovascular disease, including congestive heart failure (CHF) and pulmonary hypertension** (PH).\n",
       "\n",
       "\n",
       "  - have pointed out that DPP-4 inhibitors that can alleviate **pulmonary artery remodeling and, finally, delay the development of pulmonary hypertension.**\n",
       "\n",
       "\n",
       "  - neg., negative; pos., positive; COPD, chronic obstructive **pulmonary disease; AT, antitrypsin; PAH, pulmonary artery hypertension.**\n",
       "\n",
       "\n",
       "  - ALI or **pulmonary embolism (PE) has been reported in humans and animals with intracranial disorders such as head trauma, brain tumor, intracranial hypertension** or cerebral compression.\n",
       "\n",
       "\n",
       "  - The purpose of this study was to determine whether oral supplementation with L-arginine could reach plasma concentrations of L-citrulline, as a surrogate of nitric oxide generation, that are associated with **pulmonary vascular hemodynamic improvement in humans with pulmonary hypertension.**\n",
       "\n",
       "\n",
       "  - Studies are underway to dose L-arginine in client-owned dogs with **pulmonary hypertension to see whether peak L-citrulline concentrations, or a specific % change in L-citrulline concentrations, are associated with clinical and hemodynamic improvement in pulmonary hypertension.**\n",
       "\n",
       "\n",
       "  - One female child had end-stage **pulmonary hypertension, and it was unclear whether her respiratory distress was due to worsening pulmonary hypertension** or an infectious process.\n",
       "\n",
       "\n",
       "  - ALI = acute lung injury; ARDS = acute respiratory distress syndrome; CMV = conventional mechanical ventilation; COPD = chronic obstructive **pulmonary disease; ECMO = extracorporeal membrane oxygenation; ET = endothelin; HFOV = high-frequency oscillatory ventilation; IAP = intraabdominal pressure; ICU = intensive care unit; IPAH = idiopathic pulmonary arterial hypertension; IPV = intrapulmonary percussive ventilation; PCP = pulmonary** capillary pressure; PEEP = positive end-expiratory pressure; SARS = severe acute respiratory syndrome; VAT = ventilator-associated tracheobronchitis.\n",
       "\n",
       "\n",
       "  - Souza and coworkers analyzed **pulmonary artery pressure decay curves in patients with pulmonary hypertension due to either idiopathic pulmonary** arterial hypertension (IPAH; n = 12) or ARDS (n = 11) [24] .\n",
       "\n",
       "\n",
       "  - So far, the mechanism underlying sildenafi l's eff ects on **pulmonary vascular remodeling and potassium channel activity in pulmonary artery smooth muscle cells (PASMCs) has not been clearly addressed in pulmonary hypertension secondary to increased pulmonary** blood fl ow.\n",
       "\n",
       "\n",
       "  - Conclusions Oral sildenafi l attenuated **pulmonary vascular remodeling and upregulated Kv1.5 mRNA expression in the rats with pulmonary hypertension** secondary to left-to-right shunt.\n",
       "\n",
       "\n",
       "  -  all persons aged 50 years, because in this group greatly increases the number of people belonging to high risk groups  adults and children who have chronic **pulmonary(including asthma), or cardiovascular (except isolated hypertension), renal, hepatic, neurological, hematologic or metabolic disorders (including diabetes** mellitus);\n",
       "\n",
       "\n",
       "  - A compartment-specific transcriptional profiling approach revealed that lncRNA PAXIP1-AS1 regulates **pulmonary arterial hypertension** by modulating smooth muscle cell function (Jandl et al., 2019) .\n",
       "\n",
       "\n",
       "  - Long non-coding RNAs influence the transcriptome in **pulmonary arterial hypertension:** the role of PAXIP1-AS1.\n",
       "\n",
       "\n",
       "  - History of chronic obstructive **pulmonary disease (COPD) was non-significant higher but hypertension and cardiovascular** disease (CVD) was lower in men than in women.\n",
       "\n",
       "\n",
       "  - The AT1R blocker valsartan has recently been reported to improve clinical outcomes for patients with chronic obstructive **pulmonary disease (COPD) complicated with pulmonary hypertension** (DOI:10.1183/13993003.congress-2019.PA2469).\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> hypertension...hypertension </strong> </summary>\n",
       "  \n",
       "  - Specifically, NaCl consumption varied from roughly 5 g/d in Alaskan Eskimos where **hypertension was extremely rare, to 25 g/d in Northern Japanese, where nearly one third of the population exhibited hypertension.**\n",
       "\n",
       "\n",
       "  - Specifically, NaCl consumption varied from roughly 5 g/d in Alaskan Eskimos where **hypertension was extremely rare, to 25 g/d in Northern Japanese, where nearly one third of the population exhibited hypertension.**\n",
       "\n",
       "\n",
       "  - It is well accepted that excess salt exerts **hypertension and promotes renal damage in spontaneously hypertensive rats (SHR), which is used worldwide as a characterized experimental model of naturally occurring hypertension** [27] .\n",
       "\n",
       "\n",
       "  - Another recent study indicated that prevalence of **hypertension among Arab American is 36.5% which is higher than the overall worldwide estimates of hypertension** prevalence [16] .\n",
       "\n",
       "\n",
       "  - Another recent study indicated that prevalence of **hypertension among Arab American is 36.5% which is higher than the overall worldwide estimates of hypertension** prevalence [16] .\n",
       "\n",
       "\n",
       "  - The effect of short rACE2 on acute **hypertension was examined in a model of angiotensin II-induced hypertension** as previously described by us [6, 7, 15] .\n",
       "\n",
       "\n",
       "  - The effect of short rACE2 on acute **hypertension was examined in a model of angiotensin II-induced hypertension** as previously described by us [6, 7, 15] .\n",
       "\n",
       "\n",
       "  - Experimental **hypertension studies have measured ACE2 in either the heart or kidney and/or plasma, and have reported that deletion or inhibition of ACE2 leads to hypertension,** whilst enhancing ACE2 protects against the development of hypertension, hence increasing ACE2 may be a therapeutic option for the management of high blood pressure in man.\n",
       "\n",
       "\n",
       "  - Experimental **hypertension studies have measured ACE2 in either the heart or kidney and/or plasma, and have reported that deletion or inhibition of ACE2 leads to hypertension,** whilst enhancing ACE2 protects against the development of hypertension, hence increasing ACE2 may be a therapeutic option for the management of high blood pressure in man.\n",
       "\n",
       "\n",
       "  - This review discusses the evidence regarding ACE2 in experimental **hypertension models and the association between circulating ACE2 activity and ACE2 polymorphisms with blood pressure and arterial hypertension** in man.\n",
       "\n",
       "\n",
       "  - This review will discuss the evidence regarding ACE2 in experimental models of primary and secondary **arterial hypertension, the association between circulating ACE2 activity and blood pressure and hypertension** in man, as well as genetic association studies that have investigated ACE2 polymorphisms with blood pressure and/or arterial hypertension.\n",
       "\n",
       "\n",
       "  - Although a potentially reversible cause of **hypertension can be identified in less than 10% of patients with hypertension,** the overall high prevalence of hypertension means that secondary forms can affect millions of patients worldwide.\n",
       "\n",
       "\n",
       "  - Although a potentially reversible cause of **hypertension can be identified in less than 10% of patients with hypertension,** the overall high prevalence of hypertension means that secondary forms can affect millions of patients worldwide.\n",
       "\n",
       "\n",
       "  - Dysregulation of ACE2 has been reported in experimental models of secondary **hypertension, such as endocrine hypertension** [deoxycorticosterone (DOCA)-salt, Ang II infusion], renal failure (subtotal nephrectomy), and renal artery stenosis (Goldblatt hypertension).\n",
       "\n",
       "\n",
       "  - Dysregulation of ACE2 has been reported in experimental models of secondary **hypertension, such as endocrine hypertension** [deoxycorticosterone (DOCA)-salt, Ang II infusion], renal failure (subtotal nephrectomy), and renal artery stenosis (Goldblatt hypertension).\n",
       "\n",
       "\n",
       "  - One study had a combined total of 2528 Chinese Han subjects with **hypertension and 2024 ethnicity matched normotensive controls and reported no association with hypertension** (Zhou and Yang, 2009 ).\n",
       "\n",
       "\n",
       "  - In a small case-control study conducted in Anglo-Celtic Australian subjects with **hypertension (n = 152) or normotension (n = 193), four ACE2 SNPs were not associated with hypertension** (Benjafield et al., 2004) .\n",
       "\n",
       "\n",
       "  - In a small case-control study conducted in Anglo-Celtic Australian subjects with **hypertension (n = 152) or normotension (n = 193), four ACE2 SNPs were not associated with hypertension** (Benjafield et al., 2004) .\n",
       "\n",
       "\n",
       "  - Large carefully conducted clinical studies, designed with sufficient numbers and power to allow separate analyses of males and females are urgently needed in patients with **hypertension to more precisely clarify the potential role of ACE2 in blood pressure variation and hypertension,** and to uncover mechanisms that might offer novel treatment targets.\n",
       "\n",
       "\n",
       "  - Large carefully conducted clinical studies, designed with sufficient numbers and power to allow separate analyses of males and females are urgently needed in patients with **hypertension to more precisely clarify the potential role of ACE2 in blood pressure variation and hypertension,** and to uncover mechanisms that might offer novel treatment targets.\n",
       "\n",
       "\n",
       "  - Ang-(1-7) had no effect on the salt-induced hypertrophy in Ang II-independent **hypertension but it significantly reduced myocyte hypertrophy in Ang II-induced hypertension.**\n",
       "\n",
       "\n",
       "  - Ang-(1-7) had no effect on the salt-induced hypertrophy in Ang II-independent **hypertension but it significantly reduced myocyte hypertrophy in Ang II-induced hypertension.**\n",
       "\n",
       "\n",
       "  - Modulation of sACE2 activity in the nervous system (overexpression) also attenuated **hypertension [12] indicating a possibly protective role of sACE2 in hypertension.**\n",
       "\n",
       "\n",
       "  - Modulation of sACE2 activity in the nervous system (overexpression) also attenuated **hypertension [12] indicating a possibly protective role of sACE2 in hypertension.**\n",
       "\n",
       "\n",
       "  - We found that low eGFR, **hypertension, smoking status and history of atrial fibrillation were significantly associated with elevated TNFa levels, while elevated TNFR1 levels were associated with aging, low eGFR, hypertension,** diabetes and PVD, and elevated TNFR2 levels were associated with aging, low eGFR, diabetes and atrial fibrillation.\n",
       "\n",
       "\n",
       "  - We found that low eGFR, **hypertension, smoking status and history of atrial fibrillation were significantly associated with elevated TNFa levels, while elevated TNFR1 levels were associated with aging, low eGFR, hypertension,** diabetes and PVD, and elevated TNFR2 levels were associated with aging, low eGFR, diabetes and atrial fibrillation.\n",
       "\n",
       "\n",
       "  - For instance, **hypertension is the single most important modifiable risk factor for stroke, acting as a contributing factor in over 75 % of first time stroke patients [6] with hypertension** during acute stroke is associated with poorer clinical outcome [7] .\n",
       "\n",
       "\n",
       "  - Our results indicate that the prevalence of **hypertension in the last 6 months before the survey has doubled from 3.6% in 2003 to 7.1% in 2008; and 17.5% of Chinese people had been diagnosed with hypertension** by medical doctors.\n",
       "\n",
       "\n",
       "  - Our results indicate that the prevalence of **hypertension in the last 6 months before the survey has doubled from 3.6% in 2003 to 7.1% in 2008; and 17.5% of Chinese people had been diagnosed with hypertension** by medical doctors.\n",
       "\n",
       "\n",
       "  - Although this was attributed to a possible \"white-coat **hypertension\" phenomenon, she was referred to the hypertension** clinic for further investigation and management of hypertension.\n",
       "\n",
       "\n",
       "  - Although this was attributed to a possible \"white-coat **hypertension\" phenomenon, she was referred to the hypertension** clinic for further investigation and management of hypertension.\n",
       "\n",
       "\n",
       "  - Liqourice-induced **hypertension with hypokalemia must be differentiated from other genetic deficiencies which may present with similar findings: three monogenetic types of mineralo-corticoid hypertension** have been identified, Liddle's syndrome, glucocorticoid-remediable hypertension, and apparent mineralo-corticoid excess, an autosomal recessive disorder with mutations in the 11-HSD2 gene [36] [37] [38] [39] .\n",
       "\n",
       "\n",
       "  - Liqourice-induced **hypertension with hypokalemia must be differentiated from other genetic deficiencies which may present with similar findings: three monogenetic types of mineralo-corticoid hypertension** have been identified, Liddle's syndrome, glucocorticoid-remediable hypertension, and apparent mineralo-corticoid excess, an autosomal recessive disorder with mutations in the 11-HSD2 gene [36] [37] [38] [39] .\n",
       "\n",
       "\n",
       "  - Zhao et al have found that renalase gene polymorphysim is associated with primary **hypertension, indicat-ing this gene may become the novel marker of genetic susceptibility in essential hypertension** [30] .\n",
       "\n",
       "\n",
       "  - Zhao et al have found that renalase gene polymorphysim is associated with primary **hypertension, indicat-ing this gene may become the novel marker of genetic susceptibility in essential hypertension** [30] .\n",
       "\n",
       "\n",
       "  - In turn, deficiencies in Ang-(1-7) that contribute to autonomic dysfunction were apparent in **hypertension (15) and aging (16) ; Ang-(1-7) deficiency in hypertension** was restored by ACE inhibitor treatment (17) or chronic Ang-(1-7) replacement (18) .\n",
       "\n",
       "\n",
       "  - In turn, deficiencies in Ang-(1-7) that contribute to autonomic dysfunction were apparent in **hypertension (15) and aging (16) ; Ang-(1-7) deficiency in hypertension** was restored by ACE inhibitor treatment (17) or chronic Ang-(1-7) replacement (18) .\n",
       "\n",
       "\n",
       "  - The endothelin system has been a target for therapeutic intervention due to its pathophysiological role in **hypertension, pulmonary arterial hypertension,** heart failure, renal disease, and diabetes.\n",
       "\n",
       "\n",
       "  - Based on the dual-feedback system linking the endothelin system and the RAS, dual AT 1 R/ET A R antagonists have been developed and tested in the clinic for **pulmonary arterial hypertension, essential hypertension,** and chronic kidney disease (Murugesan et al., 2002 (Murugesan et al., , 2005 Neutel et al., 2008; Komers and Plotkin, 2016; Komers et al., 2017) ; however, it is still to be established whether the protective benefits outweigh the risk of adverse reactions.\n",
       "\n",
       "\n",
       "  - Sacubitril/ valsartan has been evaluated in the management of **hypertension, heart failure with reduced ejection fraction (HFrEF), and heart failure with preserved ejection fraction (HFpEF) and has demonstrated clinical efficacy in the reduction of blood pressure in patients with essential hypertension** and without HFpEF and a reduction in hospitalizations and mortality for patients with HFrEF.\n",
       "\n",
       "\n",
       "  - Sacubitril/ valsartan has been evaluated in the management of **hypertension, heart failure with reduced ejection fraction (HFrEF), and heart failure with preserved ejection fraction (HFpEF) and has demonstrated clinical efficacy in the reduction of blood pressure in patients with essential hypertension** and without HFpEF and a reduction in hospitalizations and mortality for patients with HFrEF.\n",
       "\n",
       "\n",
       "  - Brain RAS hyperactivity has been implicated in the development and maintenance of **hypertension in several experimental and genetic animal models of hypertension,** such as SHRs, deoxycorticosterone acetate (DOCA)-salt rats, and transgenic mice overexpressing both angiotensinogen and renin human genes (Basso et al., 1981; Ganten et al., 1983; Davisson et al., 1998) .\n",
       "\n",
       "\n",
       "  - Brain RAS hyperactivity has been implicated in the development and maintenance of **hypertension in several experimental and genetic animal models of hypertension,** such as SHRs, deoxycorticosterone acetate (DOCA)-salt rats, and transgenic mice overexpressing both angiotensinogen and renin human genes (Basso et al., 1981; Ganten et al., 1983; Davisson et al., 1998) .\n",
       "\n",
       "\n",
       "  - Recombinant ACE2 and brain-selective APA inhibition are currently being tested clinically in **pulmonary arterial hypertension and overweight patients with hypertension,** respectively, based on the concept that ACE2 degrades Ang II and upregulates the protective Ang 1-7, whereas APA blockade prevents brain AT 1 R stimulation by Ang III.\n",
       "\n",
       "\n",
       "  - 27 The most common cause of retinal detachment in a geriatric cat is **hypertension, with renal disease a common cause of secondary hypertension.**\n",
       "\n",
       "\n",
       "  - 27 The most common cause of retinal detachment in a geriatric cat is **hypertension, with renal disease a common cause of secondary hypertension.**\n",
       "\n",
       "\n",
       "  - Angiotensin-converting enzyme inhibitors (ACEi) are recommended to treat **hypertension associated with CKD to limit target-organ damage and especially glomerular hypertension.**\n",
       "\n",
       "\n",
       "  - Angiotensin-converting enzyme inhibitors (ACEi) are recommended to treat **hypertension associated with CKD to limit target-organ damage and especially glomerular hypertension.**\n",
       "\n",
       "\n",
       "  - Angiotensin-converting enzyme inhibitors (ACEi) are recommended to treat **hypertension associated with CKD to limit target-organ damage and especially glomerular hypertension.**\n",
       "\n",
       "\n",
       "  - Angiotensin-converting enzyme inhibitors (ACEi) are recommended to treat **hypertension associated with CKD to limit target-organ damage and especially glomerular hypertension.**\n",
       "\n",
       "\n",
       "  - Prevalence of **hypertension was 30% (9/30) and 4/30 had severe hypertension** (180 mm Hg).\n",
       "\n",
       "\n",
       "  - Prevalence of **hypertension was 30% (9/30) and 4/30 had severe hypertension** (180 mm Hg).\n",
       "\n",
       "\n",
       "  - Prevalence of **hypertension was different in diabetic cats without concurrent diseases (0%), diabetic cats with a disease not associated with hypertension** (29%) and those with a disease known to increase SBP (56%)( P = 0.023).\n",
       "\n",
       "\n",
       "  - Prevalence of **hypertension was different in diabetic cats without concurrent diseases (0%), diabetic cats with a disease not associated with hypertension** (29%) and those with a disease known to increase SBP (56%)( P = 0.023).\n",
       "\n",
       "\n",
       "  - Aims of this study were to identify the frequency of **hypertension in dogs with DM, and characterize the relationship between hypertension** and age, sex, reproductive status, duration of DM, body condition score, concurrent diseases and treatment with corticoids.\n",
       "\n",
       "\n",
       "  - Aims of this study were to identify the frequency of **hypertension in dogs with DM, and characterize the relationship between hypertension** and age, sex, reproductive status, duration of DM, body condition score, concurrent diseases and treatment with corticoids.\n",
       "\n",
       "\n",
       "  - Systemic **arterial hypertension is not frequently recognized in dogs with mitral valve degeneration (MVD), although borderline hypertension** is difficult to assess, mainly because of different measurement techniques, inter-operator variability and, most importantly, examination-related stress.\n",
       "\n",
       "\n",
       "  - Chen et al [2] and Huang et al [4] also reported that 32-51% of the hospitalized patients with confirmed COVID-19 had **hypertension, diabetes, and cardiovascular diseases, while 10 of the 28 Korean patients (35.7%) had at least 1 underlying disease, including hypertension** and diabetes.\n",
       "\n",
       "\n",
       "  - Since the most common cause of cardiomyocyte hypertrophy is **hypertension, and since an increased Ang II level in hypertension** is a critical pro-hypertrophy trigger, we next explored whether there might be an association between HGF and Ang II in cardiomyocytes.\n",
       "\n",
       "\n",
       "  - Since the most common cause of cardiomyocyte hypertrophy is **hypertension, and since an increased Ang II level in hypertension** is a critical pro-hypertrophy trigger, we next explored whether there might be an association between HGF and Ang II in cardiomyocytes.\n",
       "\n",
       "\n",
       "  - Early studies in our laboratory demonstrated that acute PE of hemorrhagic and fulminant type occurred accompanying severe **hypertension and bradycardia (Cushing responses) in rats following cerebral compression (CC) or intracranial hypertension** (ICH).\n",
       "\n",
       "\n",
       "  - Early studies in our laboratory demonstrated that acute PE of hemorrhagic and fulminant type occurred accompanying severe **hypertension and bradycardia (Cushing responses) in rats following cerebral compression (CC) or intracranial hypertension** (ICH).\n",
       "\n",
       "\n",
       "  - Seven of our 28 patients had associated comorbid diseases; these include two cases of malignancy including one associated with **hypertension, and one case each of hypertension,** chronic lung disease, valvular heart disease, thalassemia A close-up view of a chest radiograph in the posteroanterior projection obtained 4 days after fever onset shows a subtle ground glass density in the lateral aspect of the right upper lung (arrows) (radiographic score = 1).\n",
       "\n",
       "\n",
       "  - Seven of our 28 patients had associated comorbid diseases; these include two cases of malignancy including one associated with **hypertension, and one case each of hypertension,** chronic lung disease, valvular heart disease, thalassemia A close-up view of a chest radiograph in the posteroanterior projection obtained 4 days after fever onset shows a subtle ground glass density in the lateral aspect of the right upper lung (arrows) (radiographic score = 1).\n",
       "\n",
       "\n",
       "  - More than half of patients had history of **hypertension, which was from 59% to 63% and the frequency of hypertension** in ischemic stroke was not much different in three time period.\n",
       "\n",
       "\n",
       "  - More than half of patients had history of **hypertension, which was from 59% to 63% and the frequency of hypertension** in ischemic stroke was not much different in three time period.\n",
       "\n",
       "\n",
       "  - Underlying disease status among the 32 patients were no DM, **hypertension, or COD (n = 11); DM without hypertension** or COD (n = 5); DM and hypertension without COD (n = 5); and DM or hypertension with COD (n = 11).\n",
       "\n",
       "\n",
       "  - Underlying disease status among the 32 patients were no DM, **hypertension, or COD (n = 11); DM without hypertension** or COD (n = 5); DM and hypertension without COD (n = 5); and DM or hypertension with COD (n = 11).\n",
       "\n",
       "\n",
       "  - Of the 11 case-patients with no DM or **hypertension, 100% survived, as did all patients with DM but without hypertension** or COD.\n",
       "\n",
       "\n",
       "  - Of the 11 case-patients with no DM or **hypertension, 100% survived, as did all patients with DM but without hypertension** or COD.\n",
       "\n",
       "\n",
       "  - In comparison, 2 (40%) of 5 with DM and **hypertension but without COD died, and 9 (82%) of 11 with DM or hypertension** and with COD died (Appendix Table) .\n",
       "\n",
       "\n",
       "  - In comparison, 2 (40%) of 5 with DM and **hypertension but without COD died, and 9 (82%) of 11 with DM or hypertension** and with COD died (Appendix Table) .\n",
       "\n",
       "\n",
       "  - High occurrence of **hypertension was detected in the AMI patients (total patients with hypertension:** n=8, 15.09%; AMI group: n=4, 66.67%; cardiac marker abnormalities group: n=2, 8.33%; without cardiac marker abnormalities group: n=2, 8.69%; p=0.001).\n",
       "\n",
       "\n",
       "  - High occurrence of **hypertension was detected in the AMI patients (total patients with hypertension:** n=8, 15.09%; AMI group: n=4, 66.67%; cardiac marker abnormalities group: n=2, 8.33%; without cardiac marker abnormalities group: n=2, 8.69%; p=0.001).\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> comorbidities...hypertension </strong> </summary>\n",
       "  \n",
       "  - Third, although aging impacts immunity to viral infection [30] , patients older than 60 years of age usually have some **comorbidities, such as hypertension, diabetes,** or chronic bronchitis, and seek medical attention only when these primary diseases worsen when combined with ARI.\n",
       "\n",
       "\n",
       "  - The overseas KT recipients also suffered from more **comorbidities: they had a higher CCI score (1.01 vs 0.73, p<0.001), as well as higher percentages of diabetes (14.2% vs 8.0%, p<0.001) and hypertension** (38.6% vs 26.5%, p<0.001).\n",
       "\n",
       "\n",
       "  - The enrolled patients had various **comorbidities; three patients were complicated with anxiety and depression requiring treatment, sixteen had hypertension, five had diabetes,** seven had peptic ulcers, and one had Sjogren's syndrome.\n",
       "\n",
       "\n",
       "  - Patients with **comorbiditie**s (hypertension,** diabe**tes, chronic lung diseases, cardiovascular disorders or neuromuscular diseases) were at greater risk of hospitalization after influenza viral infection.\n",
       "\n",
       "\n",
       "  - Except asthmatic subgroup, **comorbidities such as hypertension, diabetes** mellitus, hyperlipidemia and coronary artery disease were frequently observed.\n",
       "\n",
       "\n",
       "  - Of the 281 confirmed cases, 160 (56.9%) involved **comorbidities, wherein diabetes mellitus, hypertension, ischemic heart disease, congestive** heart failure, end-stage renal disease and chronic kidney disease were significantly associated with mortality from MERS-CoV and two or three comorbidities significantly affected the fatality rates from MERS-CoV.\n",
       "\n",
       "\n",
       "  - Pooled estimates of the OR were greater than the value of 1 for all five **comorbidities, including diabetes mellitus (n = 8 studies), renal disease (6 studies), respiratory disease (5 studies), heart disease (5 studies) and hypertension** (5 studies).\n",
       "\n",
       "\n",
       "  - A total of 26 cases with fatalities were used to disclose the dangerous **comorbidities, showing that the major comorbidities are hypertension (53.8%), diabetes** (42.3%), coronary heart disease (19.2%), cerebral infarction (15.4%), chronic bronchitis (19.2%) and Parkinson's disease (7.7%) (Figure 2 ).\n",
       "\n",
       "\n",
       "  - From the analysis of the fatal cases of this novel coronavirus pneumonia, the **comorbidities of hypertension, diabetes,** coronary heart disease, cerebral infarction, and chronic bronchitis were found to be dangerous factors that resulted in death.\n",
       "\n",
       "\n",
       "  - About the **comorbidities, patients with AMI and sepsis have significant higher rates of hypertension, dyslipidemia,** diabetes, peripheral vascular disease, congestive heart failure, end-stage renal disease, cerebral vascular accident and chronic obstructive pulmonary disease (all P < 0.05 Conclusions In patients with ACS, clinical manifestations of AHF at admission constitute a strong predictor of adverse outcome in the follow up that may be significantly modified by the coronary revascularization strategy\n",
       "\n",
       "\n",
       "  - Patient medical records were reviewed and clinical data were extracted, including demographic characteristics, body mass index (BMI), smoking history, diagnostic method, Sequential Organ Failure Assessment (SOFA) score, nosocomial infection, and **comorbiditie**s (hypertension,** diabe**tes mellitus, chronic obstructive pulmonary disease [COPD], chronic kidney disease, chronic liver disease, current or previous solid organ malignancy).\n",
       "\n",
       "\n",
       "  - 26 We have encountered poor prognosis and long hospital stay in patients with scrub typhus if AKI was accompanied, especially in patients with **comorbidities such as diabetes mellitus (DM), hypertension** and CKD.\n",
       "\n",
       "\n",
       "  - The most common **comorbidities reported were hypertension** (41%), metabolic disorders (33%), and chronic lung disease (30%).\n",
       "\n",
       "\n",
       "  - As has been shown by others, **comorbidities such as chronic lung disease, diabetes, and hypertension** likely contributed to the poor survival of patients with influenza in our population.\n",
       "\n",
       "\n",
       "  - The second patient, a 39-year-old at 24 weeks gestation, had several **comorbidities, including end stage renal disease, hypertension,** and hemodialysis.\n",
       "\n",
       "\n",
       "  - While most of the patients in this study had some kind of underlying **comorbidities (52.7% have at least one comorbidities), around 38% of all patients had more than one comorbidities with the most common being obesity, diabetes and hypertension** (which occurred in more than 50% of those with any underlying comorbidity) ( Table 1) .\n",
       "\n",
       "\n",
       "  - Severe and fatal disease is strongly linked to underlying **comorbidities including diabetes mellitus, hypertension, obesity,** and chronic cardiac, pulmonary, or renal disease [23] .\n",
       "\n",
       "\n",
       "  - The CFR was also increased in those with **comorbidities including cardiovascular, diabetes, chronic respiratory disease, hypertension,** and cancer.\n",
       "\n",
       "\n",
       "  - A total of 75% (n = 24) of affected patients had **comorbidities that included diabetes mellitus, hypertension, and cardiovascular,** neurological and immunodeficiency diseases.\n",
       "\n",
       "\n",
       "  - The most common **comorbidities were hypertension in 54(32%) patients, cardiac failure in 36(21.3%) situations and diabetes** in 32(18.9%) cases.\n",
       "\n",
       "\n",
       "  - Pneumonia is a common illness that is life-threatening to the elderly, especially those with **comorbidities such as diabetes or hypertension** [4] .\n",
       "\n",
       "\n",
       "  - Potential predictors examined in bivariate analysis included age, sex, race/ethnicity, length of hospital stay, time from symptom onset to hospital admission, ICU admission, presence of **comorbidities ( 1,  2 and  3 comorbidities), individual comorbiditie**s (hypertension,** cardiac disease, lung disease, obes**ity, immunodeficiency and neurologic disorder), and the presence of individual viral pathogens (influenza viruses, parainfluenza viruses, coronaviruses, HMPV, RSV and rhinoviruses).\n",
       "\n",
       "\n",
       "  - ICD-9-CM codes were also used to identify patients treated with IABP who underwent PCI or CABG, and to identify patients treated with IABP who had baseline **comorbidities of hypertension (HTN) (ICD-9-CM: 401-405), diabetes** mellitus (DM) (ICD-9-CM: 250), hyperlipidemia (ICD-9-CM: 272), myocardial infarction (MI) (ICD-9-CM: 410), and cerebrovascular attack (CVA) (ICD-9-CM: 430-438).\n",
       "\n",
       "\n",
       "  - The objective of this study is to systematically review the existing literature on the prevalence of the most common non-communicable **comorbidities related to the cluster of metabolic syndromes-associated diseases, such as diabetes mellitus, heart diseases, hypertension, asthma, stroke** and obesity in flavivirus infections and to evaluate the difference of their prevalence in severe vs. non-severe clinical outcomes to infection.\n",
       "\n",
       "\n",
       "  - Data extracted from the selected studies in duplicate by two reviewers and included the first author's name, publication date, country, dates of recruitment, total sample size (divided to males and females), age estimates (from reported mean, median or the mid-point for age range of the highest subject frequency), procedures for case identification, type of flavivirus infection, severity of infection, prevalence of clinical manifestations (mild symptoms such as fever, headache, muscle pain, rash, and malaise together with severe symptoms as described below) and percentage of **comorbidities including diabetes (both type I and type II, if mentioned), hypertension,** heart diseases (due to the small sample size of individual conditions, we (Table 1) .\n",
       "\n",
       "\n",
       "  - Results of Egger's test, however, for most of the associations between DENV or WNV and the **comorbidities showed p>0.1 (except for the prevalence of hypertension** in DENV and heart diseases in WNV, p = 0.03), given the assumption for evidence of smallstudy effects is based on p<0.1 as previously reported [109] .\n",
       "\n",
       "\n",
       "  - However, in WNV studies, the among-studies prevalence of **comorbidities varied by 3-to 12-fold for diabetes, hypertension, heart diseases and stroke** with low to moderate I 2 indexes.\n",
       "\n",
       "\n",
       "  - Similarly, most of the pilgrims are older people (>50 years) with **comorbidities such as hypertension, diabetes,** and liver diseases [5, 6] .\n",
       "\n",
       "\n",
       "  - The concomitant **comorbidities at the start of continuous ambulatory peritoneal dialysis (CAPD) encompassed diabetes mellitus (n = 179, 34.6%), hypertension (n = 419, 85.2%), and cardiovascular** disease (n = 186, 37.9%).\n",
       "\n",
       "\n",
       "  - The **comorbidities included diabetes mellitus (DM), hypertension,** CVD and Charlson Comorbidity Index, which is one of the most commonly used scoring systems to predict the risk of mortality.\n",
       "\n",
       "\n",
       "  - The concomitant **comorbidities at the start of CAPD encompassed diabetes mellitus (n = 179, 34.6%), hypertension (n = 419, 85.2%), and cardiovascular** disease (n = 186, 37.9%).\n",
       "\n",
       "\n",
       "  - The most frequent **comorbidities recorded were hypertension (43.6%), psychiatric disorders including anxiety and/or depression (42.0%), heart failure (32.8%), ischemic heart disease (28.3%), and diabetes** mellitus (28.0%).\n",
       "\n",
       "\n",
       "  - We adjusted for **comorbidities, such as hypertension, diabetes,** stroke, and arthritis, in order to overcome this unintended effect.\n",
       "\n",
       "\n",
       "  - 10 Many infected individuals exhibit **comorbidities (e.g., diabetes, hypertension, asthma, and obesity)**, rendering them more susceptible to complications.\n",
       "\n",
       "\n",
       "  - Data extracted from the selected studies included the first author's name, publication date, country, dates of recruitment, ethnicity, total sample size (divided to males and females), age, prevalence of symptoms (including fever, shortness of breath, sore throat, cough and Acute Physiology and Chronic Health Evaluation II, APACHEII) and percentage of **comorbidities at the time of presentation (including obesity, immunosuppression or HIV and heart conditions such as hypertension, cardiovascular** diseases and/or coronary arterial diseases) (Tables 1 and 2).\n",
       "\n",
       "\n",
       "  - A recent study noted that, compared to subjects with no **comorbidities, severe cases of influenza are more likely to manifest with obesity (OR for mortality 2.74, 95%CI: 1.56-4.80), cardiovascular diseases (2.92, 1.76-4.86), hypertension** (1.49, 1.10-2.01) and neuromuscular disease (2.68, 1.91-3.75).\n",
       "\n",
       "\n",
       "  - Personal data included sex, age, epidemiological history, history of smoking, and **comorbidities [e.g., chronic obstructive pulmonary disease (COPD), cancer, hypertension and/or diabetes]**.\n",
       "\n",
       "\n",
       "  - Twenty patients (25.6%) had **comorbidities, of which hypertension** was the most common.\n",
       "\n",
       "\n",
       "  - The independent variables included the following major diseases and/or **comorbidities: BBP, cancer, liver cirrhosis, HIV/AIDS, acute myocardial infarction, systemic arterial hypertension, dilated cardiomyopathy, pulmonary** embolism, chronic obstructive pulmonary disease, diabetes mellitus, sepsis and septic shock, chronic kidney failure, and tuberculosis.\n",
       "\n",
       "\n",
       "  - The most frequent **comorbidities among the patients were diabetes (58.33%), hypertension** (37.50%), and chronic kidney disease (16.66%) ( Table 2 ).\n",
       "\n",
       "\n",
       "  - -Particular attention should also be paid to certify that staff deployed to provide psychological-psychiatric care in institutions are informed and/or sensitized of known potential risk factors for a more severe course of a Covid-19 infection of their own (advanced age, somatic **comorbidities, chronic respiratory diseases, hypertension,** cancer, known immune deficiencies etc.).\n",
       "\n",
       "\n",
       "  - They showed no infi ltrations on chest X-ray, had moderately to very severely impaired lung function according to GOLD stages II-IV and reported various **comorbidities (arterial hypertension, cor pulmonale, obstructive sleep apnea, cardiomyopathy and ischemic stroke)**.\n",
       "\n",
       "\n",
       "  - Interestingly, other statistically signifi cant associated **comorbidities were cancer, CHF, arrhythmias, and hypertension,** which were more commonly seen in the HDLVEF group.\n",
       "\n",
       "\n",
       "  - Meanwhile, numbers of patients with COVID-19 infection had chronic **comorbidities, mainly hypertension, diabetes** and cardiovascular disease, which is similar to MERS-COV population.\n",
       "\n",
       "\n",
       "  - All the six AMI patients were aged >60 years; five of them had two or more underlying **comorbiditie**s (hypertension,** diabe**tes, cardiovascular diseases, and chronic obstructive pulmonary disease).\n",
       "\n",
       "\n",
       "  - The **comorbidities with high incidence in the elderly like diabetes mellitus (DM), hypertension, coronary** artery disease (CAD), chronic obstructive pulmonary disease (COPD) and malignant tumor were all lower than 5.0%.\n",
       "\n",
       "\n",
       "  - Patients who are older (60 years) or carry **comorbidities of disorders (69.57% vs 11.29%, p<0.001), especially hypertension** (39.13% vs 12.90%, p=0.0007) or Coronary heart disease (21.74% vs 4.84%, p=0.018) are more easily developed ARF.\n",
       "\n",
       "\n",
       "  - Severe infiltration of lymphocytes in tubulointerstitium was only observed in one patient who carries **comorbidities of hypertension (>10 years) and coronary** disease (>10 year).\n",
       "\n",
       "\n",
       "  - While minimal infiltration was seen in three cases who are **comorbidities of disorders like hypertension** and gout, and the other two cases are absence tubulointerstitial lymphocyte infiltration, moreover, no obvious lymphocytes were seen in glomeruli, suggesting lymphocyte infiltration is uncommon after viral infection.\n",
       "\n",
       "\n",
       "  - Consistent with recent reports, our results suggest that 2019-nCoV is more likely to infect elderly people with chronic **comorbidities as a result of the weaker immune functions of these patients [5, 6] , and 2019-nCoV-associated death is also related to elder age and underlying illnesses, especially hypertension, diabetes,** and cardiac disease.\n",
       "\n",
       "\n",
       "  - Thirty-nine cases (38.2%) had **comorbidities, including hypertension, diabetes,** coronary artery disease, tuberculosis, hepatitis B and chronic bronchitis.\n",
       "\n",
       "\n",
       "  - Age is described by intervals of decades (0-9; 10-19; up to  80 years old) and the **comorbidities are: hypertension, diabetes,** cardiovascular disease, chronic respiratory disease and cancer.\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> hypertension...pulmonary </strong> </summary>\n",
       "  \n",
       "  - For example, aldosterone has been implicated in the pathogenesis of **pulmonary hypertension [62] , and spironolactone has been shown to attenuate experimental pulmonary hypertension** via MR inhibition in pulmonary artery smooth muscle cells [63] .Admittedly, bleomycin is also a frequently used tool drug to induce pulmonary hypertension [64, 65] .\n",
       "\n",
       "\n",
       "  - Apelin is down-regulated in **pulmonary arterial hypertension (PAH), a devastating disease characterized by vascular remodeling resulting in progressive obliteration of the pulmonary** circulation, leading to right ventricle (RV) hypertrophy and right heart failure (Alastalo et al., 2011; Chandra et al., 2011) .\n",
       "\n",
       "\n",
       "  - Also, in **pulmonary arterial hypertension virus infection is considered to be a possible risk factor (203) , and pulmonary** hypertension has been reported to be a side effect of prolonged treatment with type I IFN (204, 205) .\n",
       "\n",
       "\n",
       "  - TRAIL has been found to be increased within **pulmonary vascular lesions of patients with pulmonary hypertension (206) and also in a mouse model of hypoxia-induced pulmonary** hypertension, levels of soluble TRAIL correlated with right ventricular systolic pressure, right ventricular hypertrophy, and pathologic alterations (33, 34) .\n",
       "\n",
       "\n",
       "  - Among all patients, 68% had one or more coexisting medical conditions such as **hypertension, cardiac disease, and chronic obstructive pulmonary** disease.\n",
       "\n",
       "\n",
       "  - (6) electrocardiographic findings including sinus **tachycardia, clockwise rotation and sharp P waves; (7) persistent pulmonary hypertension on echocardiography; (8) symptoms resolve rapidly following rest, oxygen treatment, pulmonary** artery pressure lowering treatment, and diuresis.\n",
       "\n",
       "\n",
       "  - Aminophylline can dilate the bronchus, decrease hypoxic pulmonary **hypertension, enhance diaphragm function, suppress lipid peroxidation and hypoxia-induced pulmonary** vascular inflammation, enhance cardiac function and diuresis, clear bronchial mucus, and subsequently resolve moist rales.\n",
       "\n",
       "\n",
       "  - Anisodamine can improve **pulmonary vascular spasms, decrease pulmonary vascular resistance and hypoxic pulmonary hypertension, improve the pulmonary** microcirculation, and maintain smooth pulmonary blood flow.\n",
       "\n",
       "\n",
       "  - cystic fibrosis, **pulmonary sarcoidosis, pulmonary hypertension f All solid tumors including bronchial carcinoma g Patients with one of the following conditions: primary or secondary antibody deficiency, congenital immunodeficiency, immunosuppressive therapy other than steroids, severe malnutrition with cachexia h Highest/lowest value within a time period of 3 days before and 3 days after date of rtPCR i Other infections (n = 10); neoplastic diseases (n =6); collagen vascular; other rheumatologic or autoimmune (n = 14); sarcoidosis (n = 3); non-infectious nonneoplastic pulmonary** diseases (n = 23); cardiovascular diseases (n = 3) Other chronic lung diseases, e.g.\n",
       "\n",
       "\n",
       "  - cystic fibrosis, **pulmonary sarcoidosis, pulmonary hypertension f All solid tumors including bronchial carcinoma g Patients with one of the following conditions: primary or secondary antibody deficiency, congenital immunodeficiency, immunosuppressive therapy other than steroids, severe malnutrition with cachexia h Highest/lowest value within a time period of 3 days before and 3 days after date of rtPCR i Other infections (n = 24); neoplastic diseases (n = 7); collagen vascular; other rheumatologic or autoimmune (n = 15); sarcoidosis (n = 3); non-infectious nonneoplastic pulmonary** diseases (n = 23); cardiovascular diseases (n = 3); gastroesophageal diseases (n = 1) Pathogens associated with mixed infections were counted individually 31 of 40 adults (78%).\n",
       "\n",
       "\n",
       "  - Nearly half (48%) of enrolled case-patients had at least one underlying medical condition, the most common being **hypertension (18%), chronic bronchitis (11%), and chronic obstructive pulmonary** disease (8%).\n",
       "\n",
       "\n",
       "  - Twenty-eight (24.3%) of 115 patients of had at least one underling illness, including **diabetes, ischemic heart disease, hypertension, chronic obstructive pulmonary** disease, asthma, cerebrovascular disease, and chronic hepatitis.\n",
       "\n",
       "\n",
       "  - [14] found an incidence of **pulmonary hypertension of 92.2% but did not find any association between pulmonary hypertension** and death.\n",
       "\n",
       "\n",
       "  - Hemilla et al., in a review of patients with severe ARDS who subsequently received ECMO, found evidence of moderate pulmonary **hypertension using pulmonary** artery catheter data acquired prior to the institution of extracorporeal support [12] .\n",
       "\n",
       "\n",
       "  - Most hospitals and intensive care units worldwide use the standard criteria for the diagnosis of acute lung injury (ALI)/ARDS: presence of acute hypoxemia (PaO 2 /FIO 2 less than 300 mmHg or 39.99 Kpa for ALI or less than 200 mmHg or 26.66 Kpa for ARDS), bilateral infiltrates seen on a frontal chest radiograph that are consistent with **pulmonary edema, and no clinical evidence of left atrial hypertension, or (if it is measured) a pulmonary** artery wedge pressure (PAWP) of less than 18 mmHg according to the 1994-1998 American-European Consensus Conference on ARDS (AECC) [25, 26] .\n",
       "\n",
       "\n",
       "  - Chronic thromboembolic **pulmonary hypertension (CTEPH) is one of the primary causes of severe pulmonary hypertension.**\n",
       "\n",
       "\n",
       "  - Pulmonary endarterectomy (PEA) can prevent mortality due to right ventricle failure (1) , which occurs in chronic thromboembolic **pulmonary hypertension (CTEPH), a type-4 pulmonary hypertension** (2) .\n",
       "\n",
       "\n",
       "  - After enrollment, subjects were followed daily for the development of ARDS, as defined by the American-European Consensus Committee (AECC) as follows [16] : a) evidence of hypoxemia with Pao 2 /Fio 2 #200 mm Hg; b) evidence of bilateral infiltrates on chest radiographs; and c) absence of left atrial **hypertension with pulmonary** arterial occlusion pressure #18 mm Hg or no congestive heart failure.\n",
       "\n",
       "\n",
       "  - Total costs associated with RV infection in the USA were estimated at approximately US$40 billion per year, which is greater than many other conditions, such as **hypertension, asthma, or chronic obstructive pulmonary** disease [2] .\n",
       "\n",
       "\n",
       "  - The majority of participants had underlying medical conditions, with **hypertension and chronic pulmonary** disease being the most common (medical conditions present in  10% of participants are presented in Table 1 ).\n",
       "\n",
       "\n",
       "  - The endothelin system has been a target for therapeutic intervention due to its pathophysiological role in **hypertension, pulmonary arterial hypertension,** heart failure, renal disease, and diabetes.\n",
       "\n",
       "\n",
       "  - The second study is an openlabel dose escalation study for rhACE2 (NCT03177603) in patients with **pulmonary arterial hypertension, in which changes in pulmonary** vascular resistance, cardiac output, and mean pulmonary artery pressure in response to rhACE2 serve as primary outcome measures.\n",
       "\n",
       "\n",
       "  - In model 2, BNP was included as an additional variable in the regression model, since **pulmonary hypertension is associated with poor prognosis in chronic pulmonary tuberculosis 16, 17 and BNP is a non-invasive marker of pulmonary hypertension** 18, 19 (Table 3 ).\n",
       "\n",
       "\n",
       "  - These findings are consistent with the results of previous studies 19, 25, 26 , which demonstrated that BNP levels were a prognostic factor in patients with idiopathic **pulmonary arterial hypertension, idiopathic pulmonary** fibrosis, and chronic cor pulmonale.\n",
       "\n",
       "\n",
       "  - Baseline characteristics of the 3 patients who underwent CRRT were as follows: #1, 71-yr old male who had **diabetes, hypertension, CKD, and heart disease; #2, 76-yr old male who had CKD, chronic pulmonary** disease, and chronic liver disease; #3, 86-yr old male who had diabetes, hypertension, and heart disease.\n",
       "\n",
       "\n",
       "  - Pulmonary **hypertension (PH) is a hemodynamic and pathophysiologic condition defined an increase in systolic pulmonary** artery pressure greater than 30 mmHg.\n",
       "\n",
       "\n",
       "  - The independent variables included the following major diseases and/or **comorbidities: BBP, cancer, liver cirrhosis, HIV/AIDS, acute myocardial infarction, systemic arterial hypertension, dilated cardiomyopathy, pulmonary** embolism, chronic obstructive pulmonary disease, diabetes mellitus, sepsis and septic shock, chronic kidney failure, and tuberculosis.\n",
       "\n",
       "\n",
       "  - Therefore, we evaluated the ability of transverse RV strain and strain rate, obtained by 2-dimensional speckle tracking echocardiography in healthy dogs and dogs with **pulmonary hypertension, to identify dogs with pulmonary hypertension.**\n",
       "\n",
       "\n",
       "  - We classified dogs as having moderate **pulmonary hypertension if TR velocity > 3.5 m/sec, and severe pulmonary hypertension** if TR velocity > 4.5 m/sec.\n",
       "\n",
       "\n",
       "  - Although transverse RV strain (but not strain rate) showed a negative relationship with class of **pulmonary hypertension, it was not useful in identifying dogs with pulmonary hypertension.**\n",
       "\n",
       "\n",
       "  - In **pulmonary hypertension (PH) patients, this led to the reconceptualization of the RV as part of right ventricular pulmonic circulation unit and the acknowledgement that RV function is a major determinant of prognosis in pulmonary arterial hypertension.**\n",
       "\n",
       "\n",
       "  - The aim of this study was to evaluate for echocardiographic evidence of pulmonary **hypertension in cats in L-CHF with pleural effusion in comparison to cats with pulmonary** edema, other pulmonary disease, and healthy cats.\n",
       "\n",
       "\n",
       "  - Twenty of the 62 patients (32%) had underlying diseases-seven (11%) had liver disease, five (8%) Of the 33 patients with symptoms for more than 10 days after illness onset, 13 (39%) had underlying diseases: four (12%) patients had liver disease, four (12%) had **hypertension, and one each had chronic obstructive pulmonary** disease (3%), diabetes (3%), and cerebrovascular disease (3%).\n",
       "\n",
       "\n",
       "  - These results suggest that TXA 2 synthase inhibitor may contribute to the improvement of **pulmonary hemodynamics in pulmonary hypertension by alleviating pulmonary** vasoconstriction and preventing thrombosis.\n",
       "\n",
       "\n",
       "  - Pulmonary **hypertension (PH) may develop secondary to hypoxic vasoconstriction and/or pulmonary** parenchymal infiltration.\n",
       "\n",
       "\n",
       "  - Of those, 3 patients had right heart failure due to pulmonary **hypertension, and 1 had a severe pulmonary** thromboembolism ( Table 2) .\n",
       "\n",
       "\n",
       "  - PVOD should be suspected in those who are progressively dyspneic, have evidence of pulmonary **hypertension in the absence of left heart failure, and imaging suggestive of pulmonary** edema [64, 83, 85] .\n",
       "\n",
       "\n",
       "  - Despite their use in primary pul monary **hypertension, pulmonary** vasodilators may be detrimental in PVOD and should be avoided.\n",
       "\n",
       "\n",
       "  - This system controls blood pressure and is involved in the development of pulmonary **hypertension and pulmonary** fibrosis.\n",
       "\n",
       "\n",
       "  - The sub-categories in the respiratory disorders category included severe acute respiratory syndrome, infectious disease of lung, lower respiratory tract infection, pneumonia, interstitial lung disease, injury of lung, asthma, pulmonary **hypertension, chronic obstructive pulmonary** disease and fibrosis of lung.\n",
       "\n",
       "\n",
       "  - A 46-year-old man with **hypertension and a history of pulmonary** tuberculosis was admitted due to infection with laboratory-confirmed MERS-CoV.\n",
       "\n",
       "\n",
       "  - Patients were regarded as having ARDS when they satisfied the American-European Consensus Conference definition for the condition: acute onset of respiratory failure, bilateral infiltrates on chest radiograph, hypoxemia as defined by a ratio of **arterial oxygen partial pressure to fractional inspired oxygen (PaO2/ FiO2 ratio)  200 mm Hg, and no evidence of left atrial hypertension or pulmonary** capillary pressure < 18 mm Hg (if measured) to rule out cardiogenic edema (11) .\n",
       "\n",
       "\n",
       "  - Studies are underway to dose L-arginine in client-owned dogs with **pulmonary hypertension to see whether peak L-citrulline concentrations, or a specific % change in L-citrulline concentrations, are associated with clinical and hemodynamic improvement in pulmonary hypertension.**\n",
       "\n",
       "\n",
       "  - The diagnosis of pre-capillary **hypertension (PH) can be challenging in dogs due to the non-specific nature of associated clinical signs, and the cost and technical requirements for direct and indirect evaluations of pulmonary** pressure.\n",
       "\n",
       "\n",
       "  - Tufts University, North Grafton, MA, USA Pulmonary **hypertension (PHTN), defined as an increased in pulmonary** arterial pressure >25 mmHg at rest may develop due to a variety of causes in dogs and is often classified into one of five mechanisms using the WHO classification scheme.\n",
       "\n",
       "\n",
       "  - One female child had end-stage **pulmonary hypertension, and it was unclear whether her respiratory distress was due to worsening pulmonary hypertension** or an infectious process.\n",
       "\n",
       "\n",
       "  - CT can also participate in the noninvasive approach of pulmonary **hypertension by means of the analysis of the degree of distensibility of central pulmonary** arteries.\n",
       "\n",
       "\n",
       "  - ALI = acute lung injury; ARDS = acute respiratory distress syndrome; CMV = conventional mechanical ventilation; COPD = chronic obstructive **pulmonary disease; ECMO = extracorporeal membrane oxygenation; ET = endothelin; HFOV = high-frequency oscillatory ventilation; IAP = intraabdominal pressure; ICU = intensive care unit; IPAH = idiopathic pulmonary arterial hypertension; IPV = intrapulmonary percussive ventilation; PCP = pulmonary** capillary pressure; PEEP = positive end-expiratory pressure; SARS = severe acute respiratory syndrome; VAT = ventilator-associated tracheobronchitis.\n",
       "\n",
       "\n",
       "  - Souza and coworkers analyzed **pulmonary artery pressure decay curves in patients with pulmonary hypertension due to either idiopathic pulmonary** arterial hypertension (IPAH; n = 12) or ARDS (n = 11) [24] .\n",
       "\n",
       "\n",
       "  - So far, the mechanism underlying sildenafi l's eff ects on **pulmonary vascular remodeling and potassium channel activity in pulmonary artery smooth muscle cells (PASMCs) has not been clearly addressed in pulmonary hypertension secondary to increased pulmonary** blood fl ow.\n",
       "\n",
       "\n",
       "  - Results Eleven weeks later, the rats in the shunt group developed pulmonary **hypertension as evidenced by signifi cantly increased mPAP, RV/(LV+S), as well as higher RMT of middle and small pulmonary** muscularized arteries (all P = 0.01).\n",
       "\n",
       "\n",
       "  - 25.2% of patients had at least one underlying disorder (i.e., **hypertension, chronic obstructive pulmonary** disease).\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> hypertension...cardiovascular </strong> </summary>\n",
       "  \n",
       "  - Despite clinical consequences of uncontrolled **hypertension as expressed in morbidity and mortality from cardiovascular** disease and stroke being well recognized [2, 9, 10] , very small proportions of patients achieve control at less than 140/90 mmHg [11] .\n",
       "\n",
       "\n",
       "  - Moreover, ADAM17 can control the transcription of matrix metalloproteinase-2 (MMP-2), which in turn mediated angiotensin-II (Ang-II)-induced **hypertension in mice independently of cardiac hypertrophy or fibrosis, showing that the effects of ADAM17 in the cardiovascular** system may be connected to other metalloproteinases (Odenbach et al., 2011) .\n",
       "\n",
       "\n",
       "  - The importance of salt restriction is well recognized, and the World Health Organization (WHO) recommended that salt intake should be kept below 5 g/day in 2016, which is based on the WHO Guideline (WHO Guideline: Sodium intake for adults and children, 2012) in order to avoid health problems, especially those related to CKD as well as **hypertension and the cardiovascular** system.\n",
       "\n",
       "\n",
       "  - Elevated levels of soluble ICAM-1 are linked to oxidative stress, **hypertension, cardiovascular** disease, type 2 diabetes, organ transplant dysfunction, increased abdominal fat mass, liver disease, certain malignancies, and sepsis [38] .\n",
       "\n",
       "\n",
       "  - Diabetes is interrelated with **obesity, hypertension and cardiovascular** diseases that share similar risk factors and are each linked to adverse disease outcomes, rendering elucidation of the precise contribution of diabetes to excess morbidity and mortality difficult.\n",
       "\n",
       "\n",
       "  - reported that, among 41 confirmed cases, the symptom composition is as follows: pneumonia (100%), fever (98%), cough (76%), lymphopenia (63%), Dyspnea (55%), fatigue (44%), sputum production (28%), headache (8%), hemoptysis (5%), and diarrhea (3%); and the underlying diseases included **diabetes (20%), hypertension (15%), and cardiovascular** disease (15%) [21] .\n",
       "\n",
       "\n",
       "  - Within a few years, data from Framingham identified cigarette smoking (Dawber 1960) and elevated cholesterol and **hypertension as preventable causes of cardiovascular** disease (CVD) (Kannel et al.\n",
       "\n",
       "\n",
       "  - On the other hand, many ACE inhibitors are currently used to treat **hypertension and other cardiovascular** diseases [25] .\n",
       "\n",
       "\n",
       "  - Analysis of the clinical features of 41 hospitalized patients with laboratory-confirmed 2019-nCoV infection revealed that 30 were men (73%); less than one-half had underlying co-morbid conditions (13; 32%) which included **diabetes (8, 20%) , hypertension (6, 15%), and cardiovascular** disease (6; 15%); and the average age was 49.0 years old.\n",
       "\n",
       "\n",
       "  - This method weighted complete cases (those not lost to follow-up) by the inverse of their probability of being a complete case and included the sociodemographic variables in Additional file 2: Table S1 and additional predictors that may influence the likelihood of loss to follow-up: province, marital status, ethnicity, co-residence with children, current smoking and alcohol use, self-reported **hypertension, self-reported cardiovascular** disease, and primary occupation (agricultural vs. non-agricultural) [52] .\n",
       "\n",
       "\n",
       "  - The newest study including the first 41 confirmed cases admitted to Wuhan between 16 December 2019 and 2 January 2020 showed the median age of patients was 49 years; and the main underlying diseases were **diabetes, hypertension, and cardiovascular** diseases.\n",
       "\n",
       "\n",
       "  - Experimental and clinical studies have demonstrated a role for the Ang-(1-7)/ACE2/Mas axis in the evolution of **hypertension, the regulation of cardiovascular** and renal function, and the progression of cardiovascular and renal disease including diabetic nephropathy [77] .\n",
       "\n",
       "\n",
       "  - The tree shows that age is the most discriminatory attribute, followed by BMI, family history of **diabetes, work-related stress, physical activity, salty food preference, hypertension, gender, and history of cardiovascular** disease or stroke.\n",
       "\n",
       "\n",
       "  - Although all nine variables such as age, gender, BMI, **hypertension, history of cardiovascular** disease or stroke, family history of diabetes, physical activity, work-related stress, and salty food preference carried certain critical values in making an early and noninvasive prediction of diabetes before the expensive laboratory tests become necessary, age plays a dominant role on the potential development of diabetes by serving as a single but the most discriminatory attribute to diabetes.\n",
       "\n",
       "\n",
       "  - A total of 75% (n = 24) of affected patients had **comorbidities that included diabetes mellitus, hypertension, and cardiovascular,** neurological and immunodeficiency diseases.\n",
       "\n",
       "\n",
       "  - 1 Including: **diabetes, hypertension, cardiovascular,** neurological, and immunodeficiency diseases; 2 This was calculated as collection date minus admission date; 3 Included: cephalosporin, piperacillin/tazobactam, and meropenem; 4 Included: fluconazole, voriconazole, and liposomal amphotericin B.\n",
       "\n",
       "\n",
       "  - The majority of patients who are hospitalized with MERS-CoV infection had chronic co-morbidities such as **obesity, diabetes, hypertension, cardiovascular** diseases or end-stage renal disease [40, [52] [53] [54] .\n",
       "\n",
       "\n",
       "  - Thus, it is conceivable that in **hypertension and other cardiovascular** diseases an imbalance between the protective ACE2/ Ang-(1-7)/Mas arm, and the deleterious ACE/AngII/AT1 axis of the RAS induces a significant reduction in  NO bioavailability associated with oxidative stress and causes endothelial dysfunction.\n",
       "\n",
       "\n",
       "  - The most common underlying medical condition was **hypertension (48%, n = 125) followed by chronic lung disease (33%, n = 84) and cardiovascular** disease (26%, n = 69).\n",
       "\n",
       "\n",
       "  - In addition, ACE2 deficiency is associated with modest systolic **hypertension [30] , although the mouse genetic background significantly alters the cardiovascular** phenotype [30] [31] [32] [33] .\n",
       "\n",
       "\n",
       "  - Other ailments associated with wealthier Western nations, including lung and breast cancer, **obesity, hypertension, and cardiovascular** disease are rising in incidence (Normile 2010 ).\n",
       "\n",
       "\n",
       "  - Recent studies showed that elevated levels of catecholamine in renal failure patients compared to normal individuals may be associated with **hypertension and cardiovascular** complications of CKD.\n",
       "\n",
       "\n",
       "  - On the other hand, humans with an APOC3 mutation benefit from a favorable lipoprotein profile, increased insulin sensitivity, lower incidence of **hypertension, and protection against cardiovascular** diseases (68) (69) (70) .\n",
       "\n",
       "\n",
       "  - Diabetes, **hypertension, heart disease (including: cardiovascular** disease, coronary artery disease, coronary vascular disease, atrial fibrillation, chronic ischemic heart disease, acute coronary syndrome for duration less than 6 months, cardiac disorder, cardiac heart failure, congestive cardiac failure), stroke, obesity, asthma (does not include chronic obstructive pulmonary disease; COPD).\n",
       "\n",
       "\n",
       "  - The concomitant **comorbidities at the start of continuous ambulatory peritoneal dialysis (CAPD) encompassed diabetes mellitus (n = 179, 34.6%), hypertension (n = 419, 85.2%), and cardiovascular** disease (n = 186, 37.9%).\n",
       "\n",
       "\n",
       "  - The concomitant **comorbidities at the start of CAPD encompassed diabetes mellitus (n = 179, 34.6%), hypertension (n = 419, 85.2%), and cardiovascular** disease (n = 186, 37.9%).\n",
       "\n",
       "\n",
       "  - As the primary **cardiovascular risk factor, the risk level of hypertension is equivalent to three other cardiovascular** risk factors together.\n",
       "\n",
       "\n",
       "  - Ang (1-7) is hypothesized to exhibit antioxidant effects in diabetic nephropathy, **hypertension, cardiovascular** diseases and in the brain (16) (17) (18) (19) .\n",
       "\n",
       "\n",
       "  - Abstract--Despite the success of **renin-angiotensin system (RAS) blockade by angiotensin-converting enzyme (ACE) inhibitors and angiotensin II type 1 receptor (AT 1 R) blockers, current therapies for hypertension and related cardiovascular** diseases are still inadequate.\n",
       "\n",
       "\n",
       "  - Because of the counter-regulatory actions of the NPs on detrimental Ang II/AT 1 R effects, augmentation of the NP system has been explored as an additional therapeutic strategy for the treatment of **hypertension and cardiovascular** disease.\n",
       "\n",
       "\n",
       "  - ACE inhibitors, the first drugs targeting the RAS, have been used effectively for the treatment of a wide range of indications related to **hypertension, cardiovascular** disease, and renal disease for over 30 years.\n",
       "\n",
       "\n",
       "  - Consequently, dual ACE C-domain-selective/NEP inhibitors could offer a promising alternative for the treatment of **hypertension and cardiovascular** disease by potentiating NP levels in addition to blocking Ang II formation.\n",
       "\n",
       "\n",
       "  - Data extracted from the selected studies included the first author's name, publication date, country, dates of recruitment, ethnicity, total sample size (divided to males and females), age, prevalence of symptoms (including fever, shortness of breath, sore throat, cough and Acute Physiology and Chronic Health Evaluation II, APACHEII) and percentage of **comorbidities at the time of presentation (including obesity, immunosuppression or HIV and heart conditions such as hypertension, cardiovascular** diseases and/or coronary arterial diseases) (Tables 1 and 2).\n",
       "\n",
       "\n",
       "  - The aim of the study is to assess the SDMA in dogs with myxomatous mitral valve disease (MMVD) at various disease stages, to evaluate the effect of pulmonary **hypertension (PH) and the possible influence of cardiovascular** therapies.\n",
       "\n",
       "\n",
       "  - 55, 56 These immune-enhancing effects come at the price of increased rates of microalbunemia, **hypertension and cardiovascular** disease in T allele carriers.\n",
       "\n",
       "\n",
       "  - On the other hand, humans with an APOC3 mutation benefit from a favorable lipoprotein profile, increased insulin sensitivity, lower incidence of **hypertension, and protection against cardiovascular** diseases (Atzmon et al., 2006; Jrgensen et al., 2014; Pollin et al., 2008) .\n",
       "\n",
       "\n",
       "  - In our cohort, the median age of non-survivor was 71 years, 81.19% of them combined with one or more underlying medical conditions including **hypertension, cardiovascular** disease, diabetes, chronic pulmonary disease, cerebrovascular disease, chronic kidney disease and malignancy.\n",
       "\n",
       "\n",
       "  - 24 Similarly, in our study, all the six AMI patients were aged >60 years and had one or more underlying conditions, including **diabetes, hypertension, COPD, and cardiovascular** diseases.\n",
       "\n",
       "\n",
       "  - History of chronic obstructive **pulmonary disease (COPD) was non-significant higher but hypertension and cardiovascular** disease (CVD) was lower in men than in women.\n",
       "\n",
       "\n",
       "  - Epidemiological survey shows that COVID-19 patients with underlined conditions such as **diabetes, hypertension, cardiovascular** disease or the elderly are not only susceptible, but also often more serious.\n",
       "\n",
       "\n",
       "  - The most common chronic diseases were **hypertension (21, 58.33%), cardiovascular** (8, 22 .22%) and diabetes (7, 19.44%) ( Table 1 ).\n",
       "\n",
       "\n",
       "  - On admission, 20-51% of patients reported as having at least one comorbidity, with **diabetes (10-20%), hypertension (10-15%) and cardiovascular** and cerebrovascular diseases (7-40%) being most common [3, 4, 6] .\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> cardiovascular...hypertension </strong> </summary>\n",
       "  \n",
       "  - A total of 16,492 subjects with HbA1c 6.5-12% and one of the following **cardiovascular risks: a history of MI, atherosclerosis, hypertension, smoking, or dyslipidemia,** were recruited in SAVOR-TIMI 53 to evaluate the effect of saxagliptin on cardiovascular outcomes.\n",
       "\n",
       "\n",
       "  - The frequency of **cardiovascular disorders, diabetes, and hypertension** were significantly higher in patients with influenza A (H3N2).\n",
       "\n",
       "\n",
       "  - The frequency of **cardiovascular disorders, cerebrovascular disorders, chronic lung diseases, chronic renal diseases, diabetes, and hypertension** were statistically higher in patients who had been tested for routine laboratory findings at initial presentation.\n",
       "\n",
       "\n",
       "  - In the multivariate analysis, **cardiovascular disorders (OR 4.05, 95% CI 1.72-9.50; P = 0.001), chronic lung diseases (OR 3.38, 95% CI 1.51-7.61; P = 0.003), hypertension (OR 2.37, 95% CI 1.26-4.47; P = 0.008), diabetes** mellitus (OR 3.09, 95% CI 1.57-6.06; P = 0.001), and neuromuscular diseases (OR 10.18, 95% CI 1.30-79.40; P = 0.027) were independently associated with hospitalization of patients with laboratory-confirmed influenza (Table 5 ).\n",
       "\n",
       "\n",
       "  - We were also able to confirm that underlying chronic diseases such as **cardiovascular disorders, chronic lung diseases, hypertension, diabetes** mellitus, and neuromuscular diseases were important factors in determining hospitalization of patients with laboratory-confirmed influenza.\n",
       "\n",
       "\n",
       "  - Calcium channel blockers (CCBs) reduce the intracellular Ca 2+ level and are widely used for treating various **cardiovascular diseases, including hypertension,** angina, and supraventricular arrhythmias.\n",
       "\n",
       "\n",
       "  - The focus was on chronic diseases that are most affected by nutrition and physical activity -namely **cardiovascular diseases, some cancers, diabetes, hypertension, stroke** and chronic respiratory disease.\n",
       "\n",
       "\n",
       "  - One of the most frequent **cardiovascular diseases is hypertension.**\n",
       "\n",
       "\n",
       "  - We review here the current experimental evidence suggesting that activation of the ACE2/Ang-(1-9) axis protects the heart and vessels (and possibly the kidney) from adverse **cardiovascular remodeling in hypertension** as well as in heart failure.\n",
       "\n",
       "\n",
       "  - We review here the current experimental evidence suggesting that activation of the ACE2/Ang-(1-9) pathway protects the heart and vessels (and possibly the kidney) from adverse **cardiovascular remodeling in hypertension** as well as in heart failure.\n",
       "\n",
       "\n",
       "  - This effect was associated with a decrease in aortic medial thickness, suggesting that this may be a protective mechanism in the prevention of **cardiovascular events during hypertension** [94] .\n",
       "\n",
       "\n",
       "  - Inclusion criteria for healthy controls were as follows: no history of **cardiovascular or renal disease, hypertension, diabetes,** or atrial fibrillation; and no prescriptions for antiarrhythmics; ACEi; ARB; beta-blockers; digoxin; loop or thiazide diuretics; or MRA.\n",
       "\n",
       "\n",
       "  - An imbalance of these two opposing axes is thought to exist between genders and is thought to contribute to the pathology of **cardiovascular conditions such as hypertension, a stroke** co-morbidity.\n",
       "\n",
       "\n",
       "  - There is growing evidence that the function of several types of vascular K + channels is altered during major **cardiovascular diseases, such as chronic hypertension, diabetes,** and atherosclerosis [40] .\n",
       "\n",
       "\n",
       "  - Furthermore, one of the pivotal hallmarks of **cardiovascular and metabolic diseases, including hypertension, diabetes,** atherosclerosis, stroke, and heart failure, is reduction-oxidation (redox) imbalance, a condition, known as oxidative stress [17] .\n",
       "\n",
       "\n",
       "  - Inhibi-tors of ACE, such as captopril and lisinopril, are front line therapeutics in a range of **cardiovascular disorders, including hypertension, congestive** heart failure, left ventricular hypertrophy and myocardial infarction [2] .\n",
       "\n",
       "\n",
       "  - Under pathophysiological conditions, ROS play important roles in various **cardiovascular complications including atherosclerosis, hypertension, congestive** heart failure, ischemic heart disease, and ischemia-reperfusion injury.\n",
       "\n",
       "\n",
       "  - Many acute **cardiovascular events, such as acute ischemic stroke (AIS) and myocardial infarction (MI), are due to, e.g., unhealthy vascular states, resulting from prolonged atherosclerosis, coronary artery disease, diabetes, uncontrolled hypertension,** and aging [1] .\n",
       "\n",
       "\n",
       "  - In addition to their glucose-lowering action, DPP-4 inhibitors have been demonstrated to play a protective role in **cardiovascular diseases, including hypertension [44] , abdominal aortic aneurysm [45] , cardiomyopathy [46] , atherosclerosis** [47] , and peripheral vascular disease [48] , via both GLP-1-dependent and GLP-1-independent pathways due to their diverse, widely distributed, and pleiotropic actions [49] .\n",
       "\n",
       "\n",
       "  - Other studies have reported that this association is not independent and may be attributed to some concomitant **cardiovascular risk factors, such as hyperglycemia, hypertension,** and insulin resistance [10, 11] .\n",
       "\n",
       "\n",
       "  - First, hyperuricemia is closely related to the established conventional **cardiovascular risk factors, such as DM, hypertension,** older age, and male gender.\n",
       "\n",
       "\n",
       "  - Taking **cardiovascular disease, for example, there are about 30% of the population in the world died from cardiovascular and cerebrovascular events, among which 62% of stroke and 49% of cardiovascular events were directly caused by hypertension** [65] .\n",
       "\n",
       "\n",
       "  - As the primary **cardiovascular risk factor, the risk level of hypertension is equivalent to three other cardiovascular** risk factors together.\n",
       "\n",
       "\n",
       "  - Treatment of **cardiovascular disease and hypertension** is therefore a priority for patients with COPD, especially frequent exacerbators.\n",
       "\n",
       "\n",
       "  - Hypertension is the main risk factor for **cardiovascular disease; despite the availability of more than 100 commercial drugs and drug combinations for treating hypertension, a substantial proportion of the hypertensive** population has uncontrolled or suboptimally controlled hypertension (Oparil and Schmieder, 2015) .\n",
       "\n",
       "\n",
       "  - Peptidases and receptors within these systems are important drug targets for the treatment of various **cardiovascular diseases, including hypertension, heart failure, and coronary** artery syndrome.\n",
       "\n",
       "\n",
       "  - ACE2 also plays a role in the conversion of Ang I to Ang 1-9 (albeit with much lower efficiency than conversion of Ang II to Ang 1-7), an additional counterregulatory peptide that reduces adverse **cardiovascular remodeling, cardiomyocyte hypertrophy, and cardiac fibrosis in various animal models of hypertension** and myocardial infarction after subcutaneous administration (Ocaranza et al., 2010 (Ocaranza et al., , 2014 Flores-Muoz et al., 2011 .\n",
       "\n",
       "\n",
       "  - The 40 subjects having mean ACE2 levels of 33.0 pM showed a stronger exposure to **cardiovascular risk factors including abdominal adiposity, hypertension,** and elevated fasting glucose and lipid levels (Rice et al., 2006) .\n",
       "\n",
       "\n",
       "  - The goal was to create a bispecific drug (i.e., NPA7) that would possess greater systemic and renal vasodilating, natriuretic, diuretic, and cardiac unloading properties compared with either Ang 1-7 or BNP alone and which potentially would have beneficial efficacy for the treatment of **cardiovascular disease such as hypertension** and heart failure.\n",
       "\n",
       "\n",
       "  - NPA7 as a therapeutic has implications for the treatment of **cardiovascular disease, especially for those disease states in which RAS activation plays a pivotal role (i.e., hypertension** and heart failure).\n",
       "\n",
       "\n",
       "  - In particular, ACE inhibitors have had a major impact on **cardiovascular medicine, especially in the treatment of heart failure, hypertension,** and ischemic heart disease (Ferrario and Mullick, 2017; Oparil et al., 2018) .\n",
       "\n",
       "\n",
       "  - Many major evidence-based **cardiovascular guidelines recommend ACE inhibitors as firstline therapy, at least for heart failure and hypertension** (Rosendorff et al., 2015; Yancy et al., 2017; Whelton et al., 2018; Wright et al., 2018) .\n",
       "\n",
       "\n",
       "  - Clinical studies have shown that the local compliance of the basement membrane of VECs in the brain tissue is as soft as 1 kPa, while bigger vessels such as the aorta are as stiff as 20 kPa (Wells, 2008; Wood et al., 2010; Janmey and Miller, 2011; Onken et al., 2014; Kohn et al., 2015) , and their stiffness can increase up to 70 kPa due to aging (Huynh et al., 2011) and in the context of **cardiovascular diseases, such as atherosclerosis or hypertension** (Blacher et al., 1999; Boutouyrie et al., 2002) .\n",
       "\n",
       "\n",
       "  - The combined frequencies of **cardiovascular diseases, hypertension,** and cardio-artillery diseases were ~3-fold higher in MERS-CoV cases than in H1N1 (31.35.9% vs. 7.22.7, P=0.0004).\n",
       "\n",
       "\n",
       "  - Client-owned dogs with MMVD not previously receiving **cardiovascular medications and without moderate-or-severe pulmonary hypertension** were eligible for enrollment.\n",
       "\n",
       "\n",
       "  - In fact, severe clinical forms are associated with the presence of several underlying medical conditions in about 90% of the cases, especially in elderly patients, and preexisting respiratory or **cardiovascular diseases and hypertension** have been recognized as risk factors of developing severe illness [7, 9] .\n",
       "\n",
       "\n",
       "  - There were no significant differences in the underlying diseases except **cardiovascular diseases (13.3% vs. 0%; P = 0.025) and hypertension** (16.7% vs. 0%; P = 0.009).\n",
       "\n",
       "\n",
       "  - The metabolic syndrome refers to the clustering of **cardiovascular risk factors that include diabetes, obesity, dyslipidaemia and hypertension.**\n",
       "\n",
       "\n",
       "  - The rest of the patients could be divided into two groups: patients with no heart condition (group B, n = 62) discharged as musculoskeletal pain (n = 54), cholecystitis (n = 3), gastritis (n = 2), depression (n = 1), sepsis (n = 1) or meningitis (n = 1); and patients with **cardiovascular disease but no actual ischemia (group C, n = 46) discharged as heart failure (n = 18), hypertension (n = 16), tachycardia** (n = 5), bradycardia (n = 3), dissection of aorta (n = 2) and pulmonary embolism (n = 2).\n",
       "\n",
       "\n",
       "  - We considered patients with **cardiovascular disease or CKD to have chronic organ dam-Diabetes mellitus and hypertension** are recognized risk factors for severe clinical outcomes, including death, associated with Middle East respiratory syndrome coronavirus infection.\n",
       "\n",
       "\n",
       "  - COD, chronic organ damage; CKD, chronic kidney disease; CVD, **cardiovascular disease; DM, diabetes mellitus; HTN, hypertension.**\n",
       "\n",
       "\n",
       "  - The concurrent medical conditions of those who died included dementia, **cardiovascular disease, cancer, multiple sclerosis, Parkinson disease, diabetes mellitus, hypertension,** asthma, and chronic kidney disease.\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> obesity...hypertension </strong> </summary>\n",
       "  \n",
       "  - In the diseasome, **obesity (classified as a body mass index greater than 30 [62] ) is connected to a number of diseases with which it has an observed co-morbidity, including type II diabetes and hypertension** [63] .\n",
       "\n",
       "\n",
       "  - Cardiovascular complications of **obesity include hypertension, dyslipidemia,** and endothelial dysfunction.\n",
       "\n",
       "\n",
       "  - A recent study found that **obesity, cardiovascular, hypertension** and neuromuscular disease were strongly associated with severe pandemic influenza [26] .\n",
       "\n",
       "\n",
       "  - PPARs played a vitally important role in treating **obesity, atherogenic dyslipidemia, hypertension,** and insulin resistance as main therapeutic targets [4] .\n",
       "\n",
       "\n",
       "  - In addition, several studies have demonstrated that high salt intake in **obesity also plays a crucial role in the development of hypertension.**\n",
       "\n",
       "\n",
       "  - Hypertension could have been confounded by some other explanatory factor (s), for example, **obesity could have likely led to both hypertension** and MERS death.\n",
       "\n",
       "\n",
       "  - Diabetes is interrelated with **obesity, hypertension and cardiovascular** diseases that share similar risk factors and are each linked to adverse disease outcomes, rendering elucidation of the precise contribution of diabetes to excess morbidity and mortality difficult.\n",
       "\n",
       "\n",
       "  - Metabolic syndrome is characterized by the presence of abdominal **obesity, IR, hyperglycemia, dyslipidemia, and hypertension** [64] .\n",
       "\n",
       "\n",
       "  - Diabetes, **obesity, and hypertension** are also more prevalent among Canary Islanders than in other Spanish populations in all age groups (Marcelino-Rodr guez et al.\n",
       "\n",
       "\n",
       "  - Antiretroviral therapy for HIV may increase the risk of developing metabolic syndrome (abdominal **obesity, hyperglycaemia, dyslipidaemia and hypertension) and thus predispose to type 2 diabetes** and cardiovascular disease.\n",
       "\n",
       "\n",
       "  - We explored the relationship between age, eGFR, sex, **obesity, LVH, hypertension, diabetes,** smoking status, peripheral vascular disease or history of atrial fibrillation with TNFa, TNFR1, TNFR2 and IL-6 in the combined HF group (Table 3) .\n",
       "\n",
       "\n",
       "  - The majority of patients who are hospitalized with MERS-CoV infection had chronic co-morbidities such as **obesity, diabetes, hypertension, cardiovascular** diseases or end-stage renal disease [40, [52] [53] [54] .\n",
       "\n",
       "\n",
       "  - Physical inactivity has been linked to **obesity, coronary hearth disease, hypertension, strokes, diabetes,** colon cancer, breast cancer and osteoporotic fractures [49] .\n",
       "\n",
       "\n",
       "  - However, in recent years, this Na/K-ATPase/ROS amplification loop has demonstrated significance in oxidative stress related disease states, including **obesity, atherosclerosis, heart failure, uremic cardiomyopathy, and hypertension.**\n",
       "\n",
       "\n",
       "  - Other ailments associated with wealthier Western nations, including lung and breast cancer, **obesity, hypertension, and cardiovascular** disease are rising in incidence (Normile 2010 ).\n",
       "\n",
       "\n",
       "  - s=Serum leptin concentrations and leptin resistance, are independent markers of **obesity, weight gain, systemic hypertension [101] , diabetes** mellitus [102] , obstructive sleep apnoea [103] and myocardial infarction [104] , while polymorphisms of the leptin gene are associated with insulin resistance [105] and long-term risk of developing diabetes mellitus [102] .\n",
       "\n",
       "\n",
       "  - Could it also be that ethnically based epidemics of **obesity, diabetes mellitus, hypertension** and reno-vascular disease (the 'metabolic syndrome'), as seen in PIMA Indians, Nauruans and Australian Aborigines [115] have developed not primarily because of exposure to \"Western\" foods and lifestyles -that, after all, are all-pervasive without necessarily having so dramatic an effect on other groups -but because of chronic or recurrent exposure to viruses, or genotypes of viruses to which their particular repertoire of receptor polymorphisms confer no protection?\n",
       "\n",
       "\n",
       "  - Of the 187 patients who have so far died of influenza A(H1N1), 35% of them had a history of metabolic conditions like **obesity and diabetes mellitus, 23% of smoking and 16% of cardiovascular disease, e.g., angina and arterial hypertension,** followed by respiratory tract and infectious conditions.\n",
       "\n",
       "\n",
       "  - There was a high prevalence of overweight (46.3%), **obesity (28.1%), diabetes mellitus (25.6%) and hypertension** (25.6%) and 13.2% participants reported that they suffered from chronic respiratory disease.\n",
       "\n",
       "\n",
       "  - 54-4.59) in DENV patients with **obesity/overweight, 2.76 (95%CI: 2.54-299) in patients with diabetes, 2.4 in those with heart diseases (95% CI: 1.65-1.70) and 1.61 (95% CI: 1.52-1.70) in patients with hypertension.**\n",
       "\n",
       "\n",
       "  - Moreover, administration of D-Ala 7 -Ang-(1-7) increased blood pressure in female mice maintained on the HF diet suggesting that the ACE2-Ang-(1-7)-Mas axis may buffer **obesity-induced hypertension** to a greater extent in females.\n",
       "\n",
       "\n",
       "  - Patients with MERS-CoV had 2-fold higher rates of **obesity (P>0.05) and 3-fold higher combined frequencies of cardiovascular diseases, hypertension,** and cardio-artillery diseases (P<0.05) (Tables 1 and 2 ).\n",
       "\n",
       "\n",
       "  - In previous of clinical trials, anti**-obesity, anti-hypertension,** anti-hypercholesterol effects of Kimchi have been investigated however, their immunomodulatory effects are not reported.\n",
       "\n",
       "\n",
       "  - ( In humans the metabolic syndrome (MS) is a well-recognised and extensively studied entity that comprises **obesity, hypertension,** dyslipidaemia, and glucose intolerance.\n",
       "\n",
       "\n",
       "  - ( In humans the metabolic syndrome (MS) is a well-recognised and extensively studied entity that comprises **obesity, hypertension,** dyslipidaemia, and glucose intolerance.\n",
       "\n",
       "\n",
       "  - An alarming increase was observed in the incidence rate of non-communicable diseases namely, **obesity, diabetes, and hypertension.**\n",
       "\n",
       "\n",
       "  - Data on 5 common non-communicable diseases, namely, **obesity, diabetes, hypertension (HTN), coronary** artery diseases (CAD), and pulmonary disease, were retrieved from studies (PubMed) published over the last decade, reviewed, and expressed as prevalence percentage.\n",
       "\n",
       "\n",
       "  - As a disease with metabolic disturbance, **obesity could lead to insulin resistance, glucose intolerance, dyslipidemia as well as hypertension.**\n",
       "\n",
       "\n",
       "  - A history of lung diseases, **obesity, diabetes, hypertension, neurological diseases, malignancy, and heart diseases were the most common comorbidities** in our study (83.6%).\n",
       "\n",
       "\n",
       "  - Cardiovascular risk reduction in individuals with metabolic syndrome should include 1) control of **obesity, unhealthy diet and lack of physical activity; 2) control of the individual components of metabolic syndrome, especially atherogenic dyslipidaemia, hypertension,** hyperglycaemia and prothrombotic state; and 3) control of insulin resistance, a defect closely linked to metabolic syndrome.\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> hypertension...coronary </strong> </summary>\n",
       "  \n",
       "  - Apart from the patient who died from aplastic anemia, who had coexisting **hypertension and coronary** heart disease, the other patients with severe disease had no evidence of pre-existing comorbidity.\n",
       "\n",
       "\n",
       "  - The **renin angiotensin system (RAS) plays a major role in the pathogenesis of hypertension, which is a risk factor for coronary** events, stroke, kidney failure, and heart failure (Johnston, 1994; Dzau, 2005) .\n",
       "\n",
       "\n",
       "  - Seventeen patients had **diabetes (33.3 %), 25 had hypertension (49 %), 14 (27.5 %) had end stage renal disease, eight (15.7 %) had coronary** artery disease and six (11.8 %) patients were immunosuppressed, two of whom were pregnant.\n",
       "\n",
       "\n",
       "  - After controlling for age, gender, urbanization level, socioeconomic status, liver cirrhosis, **hypertension, coronary** artery disease, hyperlipidemia, malignancy, smoking, chronic obstructive pulmonary disease, obesity, history of alcohol intoxication, chronic renal insufficiency, biliary tract disease, chronic pancreatitis and Charlson Comorbidity Index score, the adjusted hazard ratio of pancreatic cancer was 2.48 (95 % confidence interval, 1.84X3.34) in patients with diabetes.\n",
       "\n",
       "\n",
       "  - Recognized risk factors for IVIG-induced thrombosis Frontiers in Immunology | Primary Immunodeficiencies include male gender; age >60; **diabetes; renal insufficiency, dyslipidemia; hypertension; immobility; coronary** disease; pre-existing vascular disease, family history of early thromboembolic disease; atrial fibrillation, high-dose and high-speed IVIG infusions.\n",
       "\n",
       "\n",
       "  - Patient and control groups were genotyped to establish associations between genetic variations and susceptibility to diseases such as esophageal squamous cell carcinoma [68] , hepatocellular carcinoma in a particularly high-risk region of China [69] , **hypertension [70] , and coronary** heart disease [71] .\n",
       "\n",
       "\n",
       "  -  Diverse reviews on APOE gene effects show that the ApoE4 allele is linked to moderate excess risk for arterial **hypertension and ischemic heart disease (Haan and Mayeda, 2010) , intracerebral and lobar bleeding (Biffi et al., 2010) , coronary** heart disease, stroke, peripheral artery disease, and diabetes mellitus (Eichner et al., 2002) .\n",
       "\n",
       "\n",
       "  - Their underlying medical conditions, described in Table 2 , included **hypertension (n 5 54, 48.2%), coronary** heart disease (n 5 12, 10.7%), chronic obstructive pulmonary disease (n 5 6, 5.4%), cerebrovascular disease (n 5 5, 4.5%), chronic liver disease (n 5 7, 6.3%), chronic renal disease (n 5 4, 3.6%), diabetes mellitus (n 5 17, 15.2%), rheumatoid arthritis (n 5 3, 2.7%) and cancer (n 5 7, 6.3%).\n",
       "\n",
       "\n",
       "  - The patient had a 20-year history of **hypertension and coronary** artery disease status and had twice undergone placement of intracoronary stents at six and two years previously.\n",
       "\n",
       "\n",
       "  - In group A, seven patients had underlying conditions: five had **hypertension, one had coronary** heart disease, and one Average time from onset of symptoms to treatment with antiviral agents was 5 or 6 days in both groups.\n",
       "\n",
       "\n",
       "  - Exclusion criteria: (i) patients with a demonstrable history of metabolic bone diseases; (ii) those who had a chronic disease history of **hypertension, coronary** heart disease, and diabetes; and (iii) patients younger than 20 years of age or older than 50 years of age.\n",
       "\n",
       "\n",
       "  - One patient had arterial **hypertension, the second was a transplanted patient, and the third had coronary** insufficiency.\n",
       "\n",
       "\n",
       "  - The age-and sex-stratified population prevalence of chronic heart disease (CHD; excluding isolated **hypertension) was estimated from a study that recorded a prior history of hospitalization with coronary** artery disease (A history of hospitalization for myocardial infarction or a surgical history of coronary balloon angioplasty, or coronary stent implantation or coronary artery bypass.)\n",
       "\n",
       "\n",
       "  - Peptidases and receptors within these systems are important drug targets for the treatment of various **cardiovascular diseases, including hypertension, heart failure, and coronary** artery syndrome.\n",
       "\n",
       "\n",
       "  - The patient had a history of **hypertension, ischemic heart disease, and coronary** artery bypass graft surgery (CABG), chronic obliterative arteriopathy of the inferior limbs, and bilateral carotid atherosclerosis; in July, he underwent contrast cardiac ventriculography for acute myocardial infarction.\n",
       "\n",
       "\n",
       "  - Data on 5 common non-communicable diseases, namely, **obesity, diabetes, hypertension (HTN), coronary** artery diseases (CAD), and pulmonary disease, were retrieved from studies (PubMed) published over the last decade, reviewed, and expressed as prevalence percentage.\n",
       "\n",
       "\n",
       "  - Methods A 77-year-old male with a history of **hypertension, previous replacement of aortic valve and a right coronary** artery bypass was admitted to our ICU after cardiac arrest.\n",
       "\n",
       "\n",
       "  - For example, in our analysis, we do not find evidence of **hypertension or blood pressure, history of coronary** heart disease and stroke to be casually linked to ACE2 expression, although patients with severe infections have been reported to be enriched for these comorbidities.\n",
       "\n",
       "\n",
       "  - Patients in NT-proBNP higher group were significantly older with more **hypertension (HP) and coronary** heart disease (CHD) histories, higher levels of diastolic blood pressure (DBP), myohemoglobin (MYO), CK-MB, hs-TnI, blood urea, creatinine, white blood cell (WBC), C-Reactive Protein (CRP) and procalcitonin (PCT), lower level of lymphocyte (LYM) and higher rate of in-hospital death.\n",
       "\n",
       "\n",
       "  - Abbreviations: NT-proBNP, N-terminal pro-brain natriuretic peptide; HP, **hypertension; CHD, coronary** heart disease; MYO, myoglobin; CK-MB, creatine kinase-MB; Hs-TnI, high-sensitivity troponin-I; WBC, white blood cell; LYM, lymphocytes; CRP, c-reactive protein; PCT, procalcitonin; HR, hazards ratio; 95%CI, 95% confidence interval.\n",
       "\n",
       "\n",
       "  - Two reports showed that patients with COVID-19 and **hypertension or coronary** artery disease had worse in-hospital outcomes [8, 9] .\n",
       "\n",
       "\n",
       "  - The **comorbidities with high incidence in the elderly like diabetes mellitus (DM), hypertension, coronary** artery disease (CAD), chronic obstructive pulmonary disease (COPD) and malignant tumor were all lower than 5.0%.\n",
       "\n",
       "\n",
       "  - Severe infiltration of lymphocytes in tubulointerstitium was only observed in one patient who carries **comorbidities of hypertension (>10 years) and coronary** disease (>10 year).\n",
       "\n",
       "\n",
       "  - Despite considerable variations in the proportion in individual studies due to the limited sample size and the region where patients were managed, circulatory diseases (including **hypertension and coronary** heart diseases) remained the most .\n",
       "\n",
       "\n",
       "  - For instance, **diabetes [29] and COPD [30] frequently co-exist with hypertension or coronary** heart diseases.\n",
       "\n",
       "\n",
       "  - Among three deaths, one was infected with SARS-CoV-2 after recent lung cancer surgery, one had **diabetes, ischemic cardiomyopathy, and hyperthyroidism, and one had hypertension and coronary** heart disease [16, 17] .\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> hypertension...hypertensive </strong> </summary>\n",
       "  \n",
       "  - It is well accepted that excess salt exerts **hypertension and promotes renal damage in spontaneously hypertensive rats (SHR), which is used worldwide as a characterized experimental model of naturally occurring hypertension** [27] .\n",
       "\n",
       "\n",
       "  - The role of ACE2 in **hypertension has been clarified by its overexpression in vivo, reducing blood pressure in hypertensive** models [22] [23] [24] but not in normotensive animals [25] .\n",
       "\n",
       "\n",
       "  - Comparison of ACE2 expression levels in the kidneys of three rat strains showed that ACE2 expression was International Journal of Hypertension 5 lower in the **hypertensive-prone strains and moreover that ACE2 expression decreased significantly when hypertension was initiated in salt-sensitive hypertensive** rats.\n",
       "\n",
       "\n",
       "  - On the other hand, cardiac overexpression of ACE2 prevented **hypertension-induced cardiac hypertrophy and fibrosis in spontaneously hypertensive** rats (SHR) and in Ang-II-infused rats [59, 60] .\n",
       "\n",
       "\n",
       "  - Take the **hypertension package in middle class as an example, the contracted family pay 100 (US $16.4) for a hypertensive** family member (the original value is 150).\n",
       "\n",
       "\n",
       "  - In a recent study from Xia et al., overexpression of ACE2 in the brain was observed to prevent the development of **hypertension in a triple transgenic mouse model, which was generated by transposing the hACE2 phenotype onto the R + A + chronically hypertensive** mouse background [41] .\n",
       "\n",
       "\n",
       "  - Preclinical models of genetic **hypertension have demonstrated increased AT 1 R expression in the vasculature of spontaneously hypertensive** rats (SHR) compared to age-matched normotensive controls [41] .\n",
       "\n",
       "\n",
       "  - Renal denervation, a potential anti**-hypertension treatment [37] , was reported to increase plasma renalase content and renalase expression in the kidneys in spontaneously hypertensive** rats [38] .\n",
       "\n",
       "\n",
       "  - Although these cases may represent an unusual clinical population, they were included here for our report to be inclusive as the number of subjects was extremely small (n = 7) and they only had 3 cases of **hypertension (representing <0.08% of the entire hypertensive** cases).\n",
       "\n",
       "\n",
       "  - In WNV, patients with **hypertension had higher odds of developing neuroinvasive outcome (OR 1.88, 95% CI 0.63-5.58) [78] and encephalitis (OR = 5.1, 95% CI 2.5-10.4) [87] than non-hypertensive** cases.\n",
       "\n",
       "\n",
       "  - Hypotheses were that EMS causes **hypertension, higher sympathetic tone, resting and exercising arrythmias and echocardiographic changes consistent with hypertensive** cardiomyopathy.\n",
       "\n",
       "\n",
       "  - Hypertension is the main risk factor for **cardiovascular disease; despite the availability of more than 100 commercial drugs and drug combinations for treating hypertension, a substantial proportion of the hypertensive** population has uncontrolled or suboptimally controlled hypertension (Oparil and Schmieder, 2015) .\n",
       "\n",
       "\n",
       "  - The presence of retinal detachment and **hypertension was consistent with target organ damage, 16 and hypertensive** seizures or hypertensive encephalopathy were initially suspected as the trigger for seizure activity.\n",
       "\n",
       "\n",
       "  - Elevations in SAP, DAP, and MAP were noted following drug administration at most doses; while profound **hypertension was observed at 4 mg/kg (SAP/DAP, MAP [mmHg] = 291-300/163-213, 218-279), all mares became hypertensive** in the 30-45 minutes following CoCl 2 administration.\n",
       "\n",
       "\n",
       "  - Arteritis lesions may be due to immune-mediated etiology as seen in (NZBXNZW)F1 hybrid, and MRL/Mp mice or due to **hypertension as seen in spontaneously hypertensive** rat (SHR) strains.\n",
       "\n",
       "\n",
       "  - Cats with **hypertension and TOD have significantly higher NT-proBNP concentrations than non-hypertensive** cats.\n",
       "\n",
       "\n",
       "  - Prevalence of **hypertension was similar throughout the study (73% 1-MAT, 84% 3-MAT, 80% 6-MAT) whilst the percentage of hypertensive** animals at severe RTOD decreased (75% before treatment; 63% 1-MAT, 31% 3-MAT and 42% 6-MAT; but not significantly).\n",
       "\n",
       "\n",
       "  - A variety of underlying reasons for **hypertension have been suggested including hyperthyroidism, where it is estimated around 20% are hypertensive** at diagnosis.\n",
       "\n",
       "\n",
       "  - Further studies are warranted to determine whether differences exist in the pathophysi-ological mechanisms underlying **hypertension in the more severely hypertensive** group.\n",
       "\n",
       "\n",
       "  - For example, haematomas with a lobar distribution are suggestive of cerebral amyloid angiopathy, those located in the basal ganglia in a patient with **hypertension are suggestive of hypertensive** microangiopathy, and blood in the subarachnoid space is suggestive of a ruptured aneurysm.\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> hypertension...stroke </strong> </summary>\n",
       "  \n",
       "  - It has been estimated by the American Heart Association that the yearly direct and indirect costs of **hypertension exceeded 93.5 billion in 2011, and together with stroke** and cardiovascular diseases, accounts for 17% of the total healthcare expenditure per year in the US [14] .\n",
       "\n",
       "\n",
       "  - In adult rats with established **hypertension, there was an association between reduced kidney ACE2 and blood pressure in SHR and in stroke-**prone SHR, compared to levels in normotensive rats (Crackower et al., 2002) .\n",
       "\n",
       "\n",
       "  - As we know, the **hypertension is the leading risk factor of haemorrhagic stroke** (RR = 2.74) [52] .\n",
       "\n",
       "\n",
       "  - The focus was on chronic diseases that are most affected by nutrition and physical activity -namely **cardiovascular diseases, some cancers, diabetes, hypertension, stroke** and chronic respiratory disease.\n",
       "\n",
       "\n",
       "  - Overactivation of brain ACE-Ang II-AT 1 R axis was found to take part in the development and maintenance of **hypertension, an important risk factor for ischemic stroke,** by elevating oxidative stress in brain and increasing the activity of sympathetic nervous system [13] .\n",
       "\n",
       "\n",
       "  - As the most important modifiable risk factor for **stroke [36] , hypertension is involved in the pathogenesis of ischemic stroke** through accelerating the progression of atherosclerosis, which leads to the thrombogenesis in cerebral circulation [37] .\n",
       "\n",
       "\n",
       "  - An imbalance of these two opposing axes is thought to exist between genders and is thought to contribute to the pathology of **cardiovascular conditions such as hypertension, a stroke** co-morbidity.\n",
       "\n",
       "\n",
       "  - For instance, **hypertension is the single most important modifiable risk factor for stroke, acting as a contributing factor in over 75 % of first time stroke patients [6] with hypertension** during acute stroke is associated with poorer clinical outcome [7] .\n",
       "\n",
       "\n",
       "  - Hypertension is the single most important modifiable risk factor for the development of **stroke, clinical outcome is poorer in patients with hypertension during acute stroke** and in experimental animal models of stroke, ischaemic damage is significantly increased in hypertensive animals [39, 40] .\n",
       "\n",
       "\n",
       "  - Raised blood pressure, especially systolic pressure **(hypertension), is one of the striking factors inducing various diseases like heart failure, stroke,** myocardial infarction and arterial aneurysm, and is a leading cause of chronic kidney failure [1] .\n",
       "\n",
       "\n",
       "  - Medical history revealed stage 2 **hypertension, right sylvian hemorrhagic stroke** (2010) with left sequel hemiparesis and bilateral pulmonary thromboembolism (2010).\n",
       "\n",
       "\n",
       "  - The objective of this study is to systematically review the existing literature on the prevalence of the most common non-communicable **comorbidities related to the cluster of metabolic syndromes-associated diseases, such as diabetes mellitus, heart diseases, hypertension, asthma, stroke** and obesity in flavivirus infections and to evaluate the difference of their prevalence in severe vs. non-severe clinical outcomes to infection.\n",
       "\n",
       "\n",
       "  - However, in WNV studies, the among-studies prevalence of **comorbidities varied by 3-to 12-fold for diabetes, hypertension, heart diseases and stroke** with low to moderate I 2 indexes.\n",
       "\n",
       "\n",
       "  - Suboptimal control of **hypertension is associated with target organ damage leading to heart failure, ischemic heart disease, stroke,** kidney dysfunction, retinopathy, and vascular dementia, all of which are major causes of disability and premature death.\n",
       "\n",
       "\n",
       "  - Tanshinones have been investigated for the treatment of cardio**-cerebrovascular diseases, such as myocardial infarction, atherosclerosis, hyperlipidemi**a, hypertension,** and str**oke.\n",
       "\n",
       "\n",
       "  - With respect to general clinical, 58% family history of **hypertension, 41% for DM, 22% to 27.5% for AMI and stroke.**\n",
       "\n",
       "\n",
       "  - Moreover, a comparison of survey data obtained in 2002 vs. 1993-1996 showed a significant improvement in **hypertension awareness, treatment, and control in Taiwan, which may have contributed to its declining incidence of ischemic stroke.**\n",
       "\n",
       "\n",
       "  - They showed no infi ltrations on chest X-ray, had moderately to very severely impaired lung function according to GOLD stages II-IV and reported various **comorbidities (arterial hypertension, cor pulmonale, obstructive sleep apnea, cardiomyopathy and ischemic stroke)**.\n",
       "\n",
       "\n",
       "  - The percentage of **hypertension was 72.5% in stroke** patients and 43.7% in controls.\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> hypertensive...hypertension </strong> </summary>\n",
       "  \n",
       "  - Also, **hypertensive patients in Hong Kong sometimes attend both the private and public sectors for antihypertensive medication refills, although more than 90% of hypertensive patients attended the public sector for management of hypertension.**\n",
       "\n",
       "\n",
       "  - PST2238 is being evaluated as an anti**-hypertensive drug in phase II clinical trials for ouabain-and adducin-induced hypertension** patients [38] .\n",
       "\n",
       "\n",
       "  - The ACE2 gene maps to a quantitative trait locus on the X chromosome in three strains of genetically **hypertensive rats, suggesting that ACE2 may be a candidate gene for hypertension.**\n",
       "\n",
       "\n",
       "  - A recent study in DOCAsalt **hypertensive mice, demonstrated that low-renin hypertension** was associated with reduced ACE2 expression and activity in the hypothalamus and increased ACE2 activity in the cerebrospinal fluid, as a result of increased shedding of ACE2 from the tissue membrane by ADAM17 (a disintegrin and metalloproteinase) (Xia et al., 2013) .\n",
       "\n",
       "\n",
       "  - For example, monocyte-derived macrophage ACE2 activity in untreated **hypertensive patients was not different to levels in normotensive individuals (Keidar et al., 2007) , and in patients with ST-elevation myocardial infarction (n = 95), there was no association between ACE2 activity and hypertension** (Ortiz-Perez et al., 2013) .\n",
       "\n",
       "\n",
       "  - The other study, a meta-analysis of 7251 **hypertensive patients, reported a significant association between the rs2285666 AA genotype with hypertension** in females only .\n",
       "\n",
       "\n",
       "  - A recent study investigated the ACE2 gene in 246 **hypertensive subjects from Odisha, India, and reported that the SNP rs2106809 TT genotype in females and the T allele in males was associated with hypertension compared to normotensive** controls (Patnaik et al., 2014) .\n",
       "\n",
       "\n",
       "  - This reduction in blood pressure may be the result of increased sensitivity of the baroreflex, which has been seen upon ACE2 delivery in **hypertensive models [26] , and a reduction in neuronally induced hypertension** has been observed in transgenic mice [19] .\n",
       "\n",
       "\n",
       "  - Comparison of ACE2 expression levels in the kidneys of three rat strains showed that ACE2 expression was International Journal of Hypertension 5 lower in the **hypertensive-prone strains and moreover that ACE2 expression decreased significantly when hypertension was initiated in salt-sensitive hypertensive** rats.\n",
       "\n",
       "\n",
       "  - The initial study did not see any genetic changes associated with the ACE2 gene in these **hypertensive strains, supporting subsequent data, which have, up until now, failed to show any link between ACE2 polymorphisms and hypertension** [54] .\n",
       "\n",
       "\n",
       "  - Health care professionals should check diets of all **hypertensive patients and besides giving advice about eschewing liquorice products could refer affected people for early diagnosis and treatment for hypertension.**\n",
       "\n",
       "\n",
       "  - Moreover, the blood pressure decrease was greater in **hypertensive DOCA-salt rats than in SHRs, suggesting that RB150/firibastat may be especially effective in saltdependent hypertension.**\n",
       "\n",
       "\n",
       "  - The low number of **hypertensive patients and low mean SBP in our study indicate that hypertension** is uncommon and unlikely to be the cause of renal damage in clinically ill FIV-infected cats.\n",
       "\n",
       "\n",
       "  - Three patients were **hypertensive, 3 had psoriatic arthropathy, 1 had hepatitis C virus (HCV) with anaemia of chronic disorder type and 1 had diabetes mellitus with hypertension.**\n",
       "\n",
       "\n",
       "  - Three patients were **hypertensive, three had psoriatic arthropathy, one had hepatitis C virus (HCV) with anaemia of chronic disorder type and one had diabetes mellitus with hypertension.**\n",
       "\n",
       "\n",
       "  - At the same time, dilated capillary vessels were observed in the glomeruli of 6 cases, benign **hypertensive glomerulosclerosis and autolysis were only noted in three cases who have coexisting hypertension,** whereas, severe glomerular injury was absence in all of these six cases ( Figure 2, Table 2) .\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> hypertension...obesity </strong> </summary>\n",
       "  \n",
       "  - The same analysis found **hypertension, heart disease and obesity** (conditions interrelated with diabetes) to be significantly more prevalent in severe dengue, and also reported a four-fold increased prevalence of diabetes in severe West Nile fever cases by the same methodology, supporting an earlier study 48 .\n",
       "\n",
       "\n",
       "  - Severe and fatal disease is strongly linked to underlying **comorbidities including diabetes mellitus, hypertension, obesity,** and chronic cardiac, pulmonary, or renal disease [23] .\n",
       "\n",
       "\n",
       "  - Potential predictors examined in bivariate analysis included age, sex, race/ethnicity, length of hospital stay, time from symptom onset to hospital admission, ICU admission, presence of **comorbidities ( 1,  2 and  3 comorbidities), individual comorbiditie**s (hypertension,** cardiac disease, lung disease, obes**ity, immunodeficiency and neurologic disorder), and the presence of individual viral pathogens (influenza viruses, parainfluenza viruses, coronaviruses, HMPV, RSV and rhinoviruses).\n",
       "\n",
       "\n",
       "  - Diabetes, **hypertension, cardiac diseases and obesity** are interrelated as they share similar cardiometabolic risk factors that result in the development of metabolic syndrome and the subsequent manifestation of this range of chronic diseases.\n",
       "\n",
       "\n",
       "  - Data on age-and sex-specific population prevalence were available for **coronary heart disease, chronic renal disease, diabetes, hypertension, smoking, and obesity;** agespecific but not sex-specific population prevalence was available for asthma and chronic obstructive pulmonary disease (COPD) [10] [11] [12] [13] .\n",
       "\n",
       "\n",
       "  - A further study, identifying physical inactivity, depression, smoking, mid-life **hypertension or obesity,** low education, and diabetes as key risk factors, has estimated that 50% of Alzheimer's disease cases may be preventable [317] .\n",
       "\n",
       "\n",
       "  - The second topic of \"lifestyle disease\" contained the following words: **\"diabetes\", \"osteoporosis\", \"hypertension\", \"exercise\", \"risk\", \"cholesterol\", \"obesity\"**, \"blood vessels\", \"blood pressure\", \"metabolic syndrome\", \"abnormal\", \"stroke\", \"gout\", \"bone\", \"study\", \"fat\", \"intake\", \"smoking\", \"weight\", and \"fracture\".\n",
       "\n",
       "\n",
       "  - 10 Many infected individuals exhibit **comorbidities (e.g., diabetes, hypertension, asthma, and obesity)**, rendering them more susceptible to complications.\n",
       "\n",
       "\n",
       "  - Rare etiologies identified included primary pulmonary **hypertension, primary ciliary dyskinesia, excitement-induced cough, and obesity.**\n",
       "\n",
       "\n",
       "  - Rare etiologies identified included primary pulmonary **hypertension, primary ciliary dyskinesia, excitement-induced cough, and obesity.**\n",
       "\n",
       "\n",
       "  - The relation between **hypertension, obesity** and diabetes in cats is not similar to observed in humans.\n",
       "\n",
       "\n",
       "  - No significant correlation between **hypertension and sex, reproductive status, concurrent diseases, time since DM diagnosis or obesity** was observed.\n",
       "\n",
       "\n",
       "  - 39 Results emanating from different studies have shown that influenza outbreaks are characterized by high severity of symptoms with increased mortality, 6, 8, 13, 25, 32 Several factors have been associated with H1N1 disease severity, such as factors due to the virus (ie, viral pathogenic mutations, resistance to antivirals), factors inherent to host susceptibility (eg, age, sex, race), including physiological immunosuppression or acquired diseases (ie, **diabetes, hypertension, obesity,** asthma), factors associated with available medical services and public health facilities, and factors arising from the presence of bacterial coinfections.\n",
       "\n",
       "\n",
       "  - AD genes are also expressed in the blood-brain barrier (BBB), which is disrupted by AD risk factors (age, alcohol, aluminum, concussion, cerebral hypoperfusion, **diabetes, homocysteine, hypercholesterolemia, hypertension, obesity,** pesticides, pollution, physical inactivity, sleep disruption, smoking) and by pathogens, directly or via olfactory routes to basal-forebrain BBB control centers.\n",
       "\n",
       "\n",
       "  - These include **diabetes, midlife hypertension or obesity,** smoking, and physical inactivity [206] .\n",
       "\n",
       "\n",
       "  - It is governmentally funded offering free screening for breast cancer, **diabetes, hypertension and obesity** for all women above 45 years.\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> coronary...hypertension </strong> </summary>\n",
       "  \n",
       "  - Our first case is a 69-year-old Caucasian man with a past medical history of **coronary artery disease, chronic renal insufficiency, hypertension and type 1 diabetes.**\n",
       "\n",
       "\n",
       "  - Coexisting conditions including **coronary heart disease, chronic obstructive pulmonary disease, hypertension, diabetes,** cerebrovascular disease and cancer.\n",
       "\n",
       "\n",
       "  - This hypothesis exists at the confluence of established risk factors for **coronary artery disease, including genetic susceptibility, polymorphisms predisposing to hypertension [126] [127] [128] , diabetes** [126] and hypercholesterolaemia and substantial new data implicating vascular inflammation [119, 129] , endothelial dysfunction [119, 130] , leptin dysregulation [104] and viral infection [131, 132] in the pathogenesis of vascular disease.\n",
       "\n",
       "\n",
       "  - Data on age-and sex-specific population prevalence were available for **coronary heart disease, chronic renal disease, diabetes, hypertension, smoking, and obesity;** agespecific but not sex-specific population prevalence was available for asthma and chronic obstructive pulmonary disease (COPD) [10] [11] [12] [13] .\n",
       "\n",
       "\n",
       "  - Diabetic cardiomyopathy has been defined as the existence of left ventricular dysfunction in diabetics without **coronary artery disease, hypertension** or other potential etiological conditions.\n",
       "\n",
       "\n",
       "  - Comorbidities include asthma 2 (6.7%), **coronary artery disease 2 (6.7%), diabetes mellitus 4 (13.3%) and hypertension** 5 (16.6) , and one each of gout, hypothyroidism, eczema, fibromyalgia and Crohn's disease.\n",
       "\n",
       "\n",
       "  - Recent studies have shown that alkaloids in CR can protect against CVDs, such as **coronary heart diseases, myocardial ischemia-reperfusion injury, heart failure, arrhythmia, and hypertension** (Feng 2008; Mei 2011; Yong et al.\n",
       "\n",
       "\n",
       "  - Co-morbidities: asthma 2 (6.7%); **coronary artery disease 2 (6.7%); diabetes mellitus 4 (13.3%); hypertension** 5 (16.6) , and one each of gout, hypothyroidism, eczema, fibromyalgia, and Crohn's disease.\n",
       "\n",
       "\n",
       "  - Co-morbidities: asthma 2 (6.7%); **coronary artery disease 2 (6.7%); diabetes mellitus 4 (13.3%); hypertension** 5 (16.6), and one each of gout, hypothyroidism, eczema, fibromyalgia, and Crohn's disease.\n",
       "\n",
       "\n",
       "  - Co-morbidities: asthma 2 (6.7%); **coronary artery disease 2 (6.7%); diabetes mellitus 4 (13.3%); hypertension** 5 (16.6), and one each of gout, hypothyroidism, eczema, fibromyalgia, and Crohn's disease.\n",
       "\n",
       "\n",
       "  - High cholesterol accumulation in blood serum is strongly correlated with **coronary heart disease, arteriosclerosis, myocardial infarction, brain thrombosis, lipid metabolism dysfunction, hypertension,** etc [111] .\n",
       "\n",
       "\n",
       "  - Comorbidity was extensive, including **coronary artery disease and hypertension** in one-half of the patients.\n",
       "\n",
       "\n",
       "  - The analysis of COVID-19 patients with cardiac-related chronic diseases **(coronary artery diseas**e, hypertension** and type 2 diabe**tes mellitus) showed that they were more prone to be in the severe or critical conditions.\n",
       "\n",
       "\n",
       "  - **coronary heart disease, hypertension)** would be merged into a single category.\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> hypertension...congestive </strong> </summary>\n",
       "  \n",
       "  - Amiloride and its derivatives are used as potassiumsparing diuretics to treat **hypertension and congestive** heart failure.\n",
       "\n",
       "\n",
       "  - Of the 281 confirmed cases, 160 (56.9%) involved **comorbidities, wherein diabetes mellitus, hypertension, ischemic heart disease, congestive** heart failure, end-stage renal disease and chronic kidney disease were significantly associated with mortality from MERS-CoV and two or three comorbidities significantly affected the fatality rates from MERS-CoV.\n",
       "\n",
       "\n",
       "  - Amiloride and its derivatives are potassium-sparing diuretics used to treat **hypertension and to prevent hypokalemia associated with congestive** heart failure.\n",
       "\n",
       "\n",
       "  - (2) neurological diseases (Alzheimer's disease, Parkinson's disease, ischemia, central nerve system injury): a disintegrin and metalloproteinase domain-containing protein 10 (ADAM10), -secretase, -secretase 1, and MMP-24; (3) inflammation (arthritis, chronic obstructive **pulmonary disease (COPD), asthma, Crohn's disease, inflammatory bowel disease, colitis, diverticulitis, chronic liver disease): MMPs, cathepsins, neutrophil elastase, and ADAM17; (4) blood coagulation: thrombin, factor Xa; (5) diabetes: dipeptidyl peptidase 4 (DPP4); (6) cardiovascular disease (atherosclerosis, hypertension, cardiomyopathy, congestive** heart failure, myocardial infarction, neovascularization and cardiac remodeling, cardiac fibrosis, aneurism, ischemia-reperfusion injury): angiotensin converting enzyme, renin, MMPs, chymase, neprilysin, calpain; (7) osteoporosis: cathepsin K; and (8) autoimmune diseases (multiple sclerosis, lupus, Guillain-Barre syndrome, psoriasis): activated protein C, kallikreins; ageing (skin), MMPs, neutrophil elastase [1, 2] .\n",
       "\n",
       "\n",
       "  - Medical history: **hypertension and congestive** heart failure class III NYHA.\n",
       "\n",
       "\n",
       "  - Inhibi-tors of ACE, such as captopril and lisinopril, are front line therapeutics in a range of **cardiovascular disorders, including hypertension, congestive** heart failure, left ventricular hypertrophy and myocardial infarction [2] .\n",
       "\n",
       "\n",
       "  - Under pathophysiological conditions, ROS play important roles in various **cardiovascular complications including atherosclerosis, hypertension, congestive** heart failure, ischemic heart disease, and ischemia-reperfusion injury.\n",
       "\n",
       "\n",
       "  - Third, amiloride, a drug used in the treatment of **hypertension and congestive** heart failure, blocks the ion channel activity and enhances the virus-like particle budding caused by human immunodeficiency virus 1 (HIV-1) Vpu [58] .\n",
       "\n",
       "\n",
       "  - The signs of BS include laryngeal stridor, rales, exercise intolerance, and higher risk of heat **stroke, cyanosis, vomiting, pulmonary hypertension, and right ventricle congestive** heart failure.\n",
       "\n",
       "\n",
       "  - Suppression of angiotensin-converting enzyme (ACE) is widely used to treat **hypertension, congestive** heart failure, myocardial infarction, endothelial dysfunction and renal disease (73) .\n",
       "\n",
       "\n",
       "  - Pulmonary **hypertension (PH) may lead to right ventricular (RV) remodeling, dysfunction and right-sided congestive** heart failure (R-CHF).\n",
       "\n",
       "\n",
       "  - Pulmonary **hypertension (PH) is present in approximately 70-80% of dogs with left-sided congestive** heart failure (L-CHF) and contributes to clinical signs and outcome.\n",
       "\n",
       "\n",
       "  - HS-CHD was defined as uncorrected or palliated cyanotic CHD, or acyanotic CHD associated with documented pulmonary **hypertension, and/or a requirement for medication to manage congestive** heart failure (CHF).\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> arterial...hypertension </strong> </summary>\n",
       "  \n",
       "  - The standard hemodynamic protocol involves maintaining the mean **arterial blood pressure (MAP), but in patients with intracranial hypertension** it may not be enough to maintain adequate cerebral perfusion.\n",
       "\n",
       "\n",
       "  - This review will discuss the evidence regarding ACE2 in experimental models of primary and secondary **arterial hypertension, the association between circulating ACE2 activity and blood pressure and hypertension** in man, as well as genetic association studies that have investigated ACE2 polymorphisms with blood pressure and/or arterial hypertension.\n",
       "\n",
       "\n",
       "  - Nowadays, it has also been evident that a variety of illnesses have shown inflammatory response such as venous and chronic **arterial diseases [20] [21] [22] , myocardial ischemia [23, 24] , Alzheimer's disease, acute cerebral stroke [22] , cancer, and arterial hypertension** [25] .\n",
       "\n",
       "\n",
       "  - As secondary outcome measures, we searched the clinical records for data on the following clinical variables: heart rate, respiratory rate, **arterial oxygen saturation in room air, retractions, oxygen-therapy, fluid therapy, episodes of paroxysmal cough, cyanosis, apnea, post-cough vomit, fever (body temperature > 37.5C), presence of skin petechiae, conjunctival hemorrhage and complications (hypoxia, bradycardia, pneumonia, gastroenteritis, thrombocytopenia, urinary tract infection, anemia, dehydration, feeding difficulties, transient hypertension)**.\n",
       "\n",
       "\n",
       "  - In our experience, the prone position can be an acceptable alternative to improve oxygenation in severe ARDS patients with **arterial pulmonary hypertension** and right ventricular dysfunction, which associated with the use of inhaled nitric oxide, can minimize intrathoracic pressures to facilitate right ventricular performance.\n",
       "\n",
       "\n",
       "  - After confirmation of diagnosis, these 259 patients were divided into two groups: A diagnosis of ALI included the following: acute hypoxia with a ratio of partial pressure of **arterial oxygen to the fraction of inspired oxygen (PaO 2 : F I O 2 )  300 mmHg with or without of elevation of arterial PaCO 2 , the presence of bilateral infiltrates seen by CXR, and no evidence of left atrial hypertension** 36 .\n",
       "\n",
       "\n",
       "  - Assessment of systolic **arterial blood pressure (SAP) is an important tool in small animal internal medicine practice, especially with diseases or clinical conditions that can cause hypertension** or hypotension.\n",
       "\n",
       "\n",
       "  - Assessment of systolic **arterial blood pressure (SAP) is an important tool in small animal internal medicine practice, especially with diseases or clinical conditions that can cause hypertension** or hypotension.\n",
       "\n",
       "\n",
       "  - Systemic **arterial hypertension is not frequently recognized in dogs with mitral valve degeneration (MVD), although borderline hypertension** is difficult to assess, mainly because of different measurement techniques, inter-operator variability and, most importantly, examination-related stress.\n",
       "\n",
       "\n",
       "  - Patients were regarded as having ARDS when they satisfied the American-European Consensus Conference definition for the condition: acute onset of respiratory failure, bilateral infiltrates on chest radiograph, hypoxemia as defined by a ratio of **arterial oxygen partial pressure to fractional inspired oxygen (PaO2/ FiO2 ratio)  200 mm Hg, and no evidence of left atrial hypertension or pulmonary** capillary pressure < 18 mm Hg (if measured) to rule out cardiogenic edema (11) .\n",
       "\n",
       "\n",
       "  - Clevidipine is an ultrashort-acting, vascular and **arterial-selective calcium antagonist currently under development for treating acute hypertension.**\n",
       "\n",
       "\n",
       "  - Conclusion Our simple model explains many features of the **arterial pulse observed in clinical practice such as the development of the dicrotic notch, the change in shape along the arterial tree and the steepening and acceleration with hypertension.**\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> renin...hypertension </strong> </summary>\n",
       "  \n",
       "  - Furthermore, following the administration of GLA, various side effects such as weight gain, alkalosis, **renin-angiotensinaldosterone system suppression, hypertension** and muscular paralysis may occur.\n",
       "\n",
       "\n",
       "  - This may reflect the universal disruption of **renin in genetic knockout mice, versus a modulation of predominantly the circulating RAS with dietary sodium changes, as salt-sensitive models of hypertension** exhibit suppressed circulating RAS activity but elevated brain RAS activity [65] [66] [67] .\n",
       "\n",
       "\n",
       "  - The **renin angiotensin system (RAS) plays a major role in the pathogenesis of hypertension, which is a risk factor for coronary** events, stroke, kidney failure, and heart failure (Johnston, 1994; Dzau, 2005) .\n",
       "\n",
       "\n",
       "  - The **renin-angiotensin system (RAS) is a critical regulator of hypertension,** primarily through the actions of the vasoactive peptide Ang II, which is generated by the action of angiotensin-converting enzyme (ACE) mediating an increase in blood pressure.\n",
       "\n",
       "\n",
       "  - Pharmacological therapies based on the **renin angiotensin system (RAS) blockade are used extensively for the treatment of hypertension,** heart failure, and CV remodeling but in spite of their success the prevalence of end-organ damage and residual risk remain still high.\n",
       "\n",
       "\n",
       "  - demonstrated that intracerebroventricular (I.C.V) infusion of Ang-(1-7) led to a significant decrease in BP in TGRmRen2 (27) rats with **renin dependent transgenic hypertension** [47] .\n",
       "\n",
       "\n",
       "  - In the circulation, a drop in blood pressure (systemic hypotension) and/or blood volume results in juxtaglomerular cells within the kidneys to release **renin (protease) whereas increased blood pressur**e (hypertension)** inhibits re**nin release under normal circumstances.\n",
       "\n",
       "\n",
       "  - Activation of the **renin-angiotensin system is a major mediator of NADPH oxidase activation and ROS production in human hypertension** [72] .\n",
       "\n",
       "\n",
       "  - Abstract--Despite the success of **renin-angiotensin system (RAS) blockade by angiotensin-converting enzyme (ACE) inhibitors and angiotensin II type 1 receptor (AT 1 R) blockers, current therapies for hypertension and related cardiovascular** diseases are still inadequate.\n",
       "\n",
       "\n",
       "  - The aim of the present study was to perform a pilot study on the plasma aldosterone concentration (PAC), plasma **renin activity (PRA), and hypertension** in cats with chronic azotemia in Sweden.\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> hypertension...atherosclerosis </strong> </summary>\n",
       "  \n",
       "  - There is good evidence that DPP4 inhibition mediates protective effect on myocardial infarction, **hypertension, and atherosclerosis.**\n",
       "\n",
       "\n",
       "  - Notably, nifedipine is one of the most widely used drugs for treating **hypertension and atherosclerosis** in China.\n",
       "\n",
       "\n",
       "  - This novel effect of RhoA/Rho-kinase inhibition on ACE2 gene expression, enzymatic activity, and Ang-(1-9) levels might additionally contribute to its benefits in **hypertension, atherosclerosis,** and in cardiovascular and renal pathologic remodeling.\n",
       "\n",
       "\n",
       "  - Overactivation of brain Angiotensin-converting enzyme (ACE) -Angiotensin II (Ang II) -Angiotensin II type 1 receptor (AT 1 R) axis was found to be involved in the progress of **hypertension, atherosclerosis** and thrombogenesis, which increased the susceptibility to ischemic stroke.\n",
       "\n",
       "\n",
       "  - Several physiopathologic conditions, such as **hypertension, atherosclerosis** and prothrombotic state, have been found to increase the risk for ischemic stroke.\n",
       "\n",
       "\n",
       "  - Activation of this axis was reported to attenuate the development of **hypertension [4] and the pathologic progress of atherosclerosis** [5] [6] [7] .\n",
       "\n",
       "\n",
       "  - Emerging evidence suggested that activation of ACE2-Ang-(1-7)-Mas axis could attenuate the development of **hypertension and the pathologic progress of atherosclerosis.**\n",
       "\n",
       "\n",
       "  - In addition to their glucose-lowering action, DPP-4 inhibitors have been demonstrated to play a protective role in **cardiovascular diseases, including hypertension [44] , abdominal aortic aneurysm [45] , cardiomyopathy [46] , atherosclerosis** [47] , and peripheral vascular disease [48] , via both GLP-1-dependent and GLP-1-independent pathways due to their diverse, widely distributed, and pleiotropic actions [49] .\n",
       "\n",
       "\n",
       "  - Involvement of MIF in several human diseases like pulmonary **hypertension, endothelial cell growth, atherosclerosis,** wound healing, viral infection, many cancers including lung, colon, prostate, breast and pancreatic cancer has gained significant interest in this molecule 22 .\n",
       "\n",
       "\n",
       "  - Some investigators reported that some chronic diseases are more common in males, such as **hypertension [10] , atherosclerosis,** and chronic heart diseases [11] .\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> hypertension...dyslipidemia </strong> </summary>\n",
       "  \n",
       "  - A total of 16,492 subjects with HbA1c 6.5-12% and one of the following **cardiovascular risks: a history of MI, atherosclerosis, hypertension, smoking, or dyslipidemia,** were recruited in SAVOR-TIMI 53 to evaluate the effect of saxagliptin on cardiovascular outcomes.\n",
       "\n",
       "\n",
       "  - These conditions include gastroesophageal reflux, a risk factor for aspiration pneumonia and asthma (31) , which may be worsened by **hypertension and dyslipidemia,** two conditions affecting immune responses in the lungs and increasing susceptibility to asthma and aspiration pneumonia (32) .\n",
       "\n",
       "\n",
       "  - Cardiovascular complications of **obesity include hypertension, dyslipidemia,** and endothelial dysfunction.\n",
       "\n",
       "\n",
       "  - About the **comorbidities, patients with AMI and sepsis have significant higher rates of hypertension, dyslipidemia,** diabetes, peripheral vascular disease, congestive heart failure, end-stage renal disease, cerebral vascular accident and chronic obstructive pulmonary disease (all P < 0.05 Conclusions In patients with ACS, clinical manifestations of AHF at admission constitute a strong predictor of adverse outcome in the follow up that may be significantly modified by the coronary revascularization strategy\n",
       "\n",
       "\n",
       "  - The patient had 10-year and 5-year histories of **hypertension and dyslipidemia,** respectively.\n",
       "\n",
       "\n",
       "  - Patient #1 was a 68-year-old man and had underlying diseases including asthma, **hypertension, dyslipidemia,** and benign prostate hypertrophy.\n",
       "\n",
       "\n",
       "  - The main risk factors observed in this review were **hypertension (19 cases, 33% of prevalence), previous vascular disease (18%), and dyslipidemia** (17%).\n",
       "\n",
       "\n",
       "  - The proportion of diabetic patients did not differ significantly among four patient groups (chisquare = 1.69, P = 0.639) as well as the proportion of patients with **hypertension and dyslipidemia** (chi-square = 1.63, P = 0.652).\n",
       "\n",
       "\n",
       "  - A 74-year-old Saudi male patient with **diabetes, hypertension, and dyslipidemia** presented with a 3 days of ataxia, vomiting, confusion, and fever of 39.5 C (Fig.\n",
       "\n",
       "\n",
       "  - Aged subjects with **hypertension and dyslipidemia** were also not different in their induction of IFN-I and IFN-III (p>0.8).\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> stroke...hypertension </strong> </summary>\n",
       "  \n",
       "  - As the most important modifiable risk factor for **stroke [36] , hypertension is involved in the pathogenesis of ischemic stroke** through accelerating the progression of atherosclerosis, which leads to the thrombogenesis in cerebral circulation [37] .\n",
       "\n",
       "\n",
       "  - Failure to translate effective therapeutic strategies from the 'bench to bedside' may partly be attributed to the use of animal models that do not incorporate non-modifiable risk factors such as gender and many of the **stroke comorbidities observed in the clinical stroke population, such as hypertension, diabetes,** obesity, etc.\n",
       "\n",
       "\n",
       "  - This imbalance in the RAS may be exacerbated in the presence of known **stroke risk factors such as gender and hypertension** and therefore provide a potential therapeutic target in a subset of patients.\n",
       "\n",
       "\n",
       "  - Hypertension is the single most important modifiable risk factor for the development of **stroke, clinical outcome is poorer in patients with hypertension during acute stroke** and in experimental animal models of stroke, ischaemic damage is significantly increased in hypertensive animals [39, 40] .\n",
       "\n",
       "\n",
       "  - Interestingly, the pathology of **stroke co-morbidities such as hypertension** and gender are all influenced by RAS dysfunction, where there is an increased/overactive ACE/Ang II/ AT 1 R pathway.\n",
       "\n",
       "\n",
       "  - ARBs have been extensively studied in the context of experimental **stroke due to its current clinical use in hypertension** management.\n",
       "\n",
       "\n",
       "  - Atherosclerosis, myocardial infarction and **stroke can all result from chronic undetected hypertension** [15, [26] [27] [28] [29] .\n",
       "\n",
       "\n",
       "  - The signs of BS include laryngeal stridor, rales, exercise intolerance, and higher risk of heat **stroke, cyanosis, vomiting, pulmonary hypertension, and right ventricle congestive** heart failure.\n",
       "\n",
       "\n",
       "  - [2] [3] [4] [5] The positive change in the occurrence of **stroke in these countries has been attributed primarily to better screening and treatment of established risk factors for stroke, particularly hypertension.**\n",
       "\n",
       "\n",
       "  - 18 In the Fukuoka ischemic **stroke registry, mean age was 70 years, 76% had hypertension, 27% diabetes** mellitus, 46% hyperlipidemia, and 26% atrial fibrillation.\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> tachycardia...hypertension </strong> </summary>\n",
       "  \n",
       "  - The illness showed a very short incubation period and the symptoms includedheadache, dizziness, fever, vomiting, doll's-eye reflex, hypotonia, **tachycardia, lowering of consciousness, areflexia (loss of all spinal reflexes), hypertension** and high mortality (Goh et al.\n",
       "\n",
       "\n",
       "  - (6) electrocardiographic findings including sinus **tachycardia, clockwise rotation and sharp P waves; (7) persistent pulmonary hypertension on echocardiography; (8) symptoms resolve rapidly following rest, oxygen treatment, pulmonary** artery pressure lowering treatment, and diuresis.\n",
       "\n",
       "\n",
       "  - Within the MERS-CoV positive cohort, shortness of breath, **tachycardia and underlying co-morbidities (such as cancer, diabetes, hypertension** and heart disease) were significantly more frequent amongst non-HCWs than in HCWs.\n",
       "\n",
       "\n",
       "  - 13 Findings may include **tachycardia, jugular venous distension, pulsus paradoxus, tachypnea, hypotension, hypertension,** and pericardial rub.\n",
       "\n",
       "\n",
       "  - The main symptoms were mydriasis (79%), agitation (71.4%), **tachycardia (64.3%) with an average of 114 bpm [102, 145], hallucinations (57%) and hypertension** (57%).\n",
       "\n",
       "\n",
       "  - Twentyfour trials reported the twenty-seven specific symptoms including diarrhea, dizziness and headache, somnolence, dry mouth, Bloating, constipation, **tachycardia, blurred vision, insomnia, prolonged QT, naupathia, fatigue, anxiety, tremor, anorexia, palpitation, asthenia, oscitancy, sweat, akathisia, tetter, excitation, hypertension,** bellyache, dysuria, transaminase increased, and sexual dysfunction (Figure 3) .\n",
       "\n",
       "\n",
       "  - Ketamine in a mixture with midazolam led to insignificant changes in hemodynamics and only in group 2 patients were more significant changes noted **(tachycardia and mild hypertension)**.\n",
       "\n",
       "\n",
       "  - These include but are not limited to: high fever, seizures, encephalitis, meningitis, tremor, ptosis, dysarthria, dysphasia, respiratory tract lesions, severe acute respiratory distress syndrome (ARDS), **tachycardia, myocarditis, vomiting, hypertension,** and segmental myoclonus along with or without other clinical features [15-58-98] .\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> hypertension...normotensive </strong> </summary>\n",
       "  \n",
       "  - Table 1 summarizes the experimental studies that have measured ACE2 gene, protein and/or activity, in either the heart, kidney and/or plasma, usually at one time point, and generally in Zhong et al., 2004; Diez-Freire et al., 2006; Ferreira et al., 2011; Tan et al., 2011; Yang et al., 2013  Kidney ACE2 mRNA, protein and activity Crackower et al., 2002; Zhong et al., 2004; Tikellis et al., 2006; Tan et al., 2011  ACE2 protein animals with established **hypertension, and compared results to a \"normotensive\"** control.\n",
       "\n",
       "\n",
       "  - Both rhACE2 (Wysocki et al., 2010) and mouse recombinant ACE2 (Ye et al., 2012) attenuated Ang II induced **hypertension in mice, with no effect on blood pressure in normotensive** mice (Wysocki et al., 2010) .\n",
       "\n",
       "\n",
       "  - One study had a combined total of 2528 Chinese Han subjects with **hypertension and 2024 ethnicity matched normotensive controls and reported no association with hypertension** (Zhou and Yang, 2009 ).\n",
       "\n",
       "\n",
       "  - A recent study investigated the ACE2 gene in 246 **hypertensive subjects from Odisha, India, and reported that the SNP rs2106809 TT genotype in females and the T allele in males was associated with hypertension compared to normotensive** controls (Patnaik et al., 2014) .\n",
       "\n",
       "\n",
       "  - Besides, in experimental **hypertension (DOCA salt model) and in normotensive** sham animals, RhoA/Rho-kinase inhibition (a signaling pathway that participates in pathological cardiovascular and renal remodeling and also in blood pressure regulation) by fasudil reduced BP and increased vascular and plasma ACE2 enzymatic activity.\n",
       "\n",
       "\n",
       "  - In conclusion, CKD cats with **hypertension had higher FGF-23 concentrations than their normotensive** counterparts.\n",
       "\n",
       "\n",
       "  - Cats that required a higher dose to control their SBP below 160 mmHg had a significantly higher SBP at diagnosis of **hypertension and therefore required a greater decrease in SBP in order to achieve normotensive** control.\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> atherosclerosis...hypertension </strong> </summary>\n",
       "  \n",
       "  - On the other hand, due to the structural and functional heterogeneity of ADAM17 substrates, the sheddase is also involved in various pathological processes such as cancer, inflammatory diseases, neurological diseases, cardiac failure, **atherosclerosis, diabetes, cardiac hypertrophy, and hypertension** (Sandgren et al., 1990; Black et al., 1997; Peschon et al., 1998; Li et al., 2006; Ohtsu et al., 2006b; Zhan et al., 2007; Wang et al., 2009; Scheller et al., 2011; Giricz et al., 2013; Menghini et al., 2013; Xia et al., 2013) .\n",
       "\n",
       "\n",
       "  - Next, articles detail the roles of macrophages in protection against (or causation of) various diseases including wounds, cancer, infections, **atherosclerosis, obesity, hypertension,** and other conditions.\n",
       "\n",
       "\n",
       "  - The signaling complex K/Nox is activated at different pathophysiological states: neurodegenerative diseases [120] , **atherosclerosis [121] , hypertension [122] , diabetes** [123] , and cancer [124] .\n",
       "\n",
       "\n",
       "  - These drugs had the promise of being highly effective in the treatment of endothelial dysfunction, **atherosclerosis, hypertension,** and heart failure and were termed by some as the \"super ACE inhibitors.\"\n",
       "\n",
       "\n",
       "  - The reduction in NO due to the competition between Arg and iNOS is also important in the pathogenesis of a variety of diseases, including **atherosclerosis and hypertension, myocardial ischemia/reperfusion injury, diabetes,** allergic asthma and inflammatory diseases (47) (48) (49) (50) .\n",
       "\n",
       "\n",
       "  - An increase in the total numbers of PDMPs is associated with disease states such as **atherosclerosis, diabetes, cancer, sepsis, and pulmonary hypertension** (7) .\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> hypertension...arterial </strong> </summary>\n",
       "  \n",
       "  - This review discusses the evidence regarding ACE2 in experimental **hypertension models and the association between circulating ACE2 activity and ACE2 polymorphisms with blood pressure and arterial hypertension** in man.\n",
       "\n",
       "\n",
       "  - Two of these Post-SARS subjects (ages 63 yrs., BMI = 28, and 49 yrs, BMI = 32) who were being treated for **hypertension had moderate and mild sleep hypopnea/apnea disorder (with a respiratory distress index (RDI) = 18.8 and 8.4 respectively) and arterial** oxygen desaturation (minimum of 81.2% and 83% respectively).\n",
       "\n",
       "\n",
       "  - The only other person (age 57 yrs, BMI 25.5) being treated for **hypertension had very mild elevation of RDI (7.5) but no snoring, no significant sleep-related arterial** oxygen desaturations or daytime sleepiness.\n",
       "\n",
       "\n",
       "  - Therefore, it is an effective vasodilator, reduces hypoxic pulmonary **hypertension, increases arterial** oxygen tension, improves the symptoms and signs of HAPE, lowers right atrial pressure, and increases cardiac output.\n",
       "\n",
       "\n",
       "  - In dogs, heart disease has been related to serious conditions, including progressive heart failure, **pulmonary edema and/or congestion, hypertension, left arterial** rupture, and pericardial tamponade (Borgarelli et al., 2008; Guglielmini, 2003; Pouchelon et al., 2015) .\n",
       "\n",
       "\n",
       "  - Conclusions Despite decreasing MAP, infusion of AM reversed pulmonary **hypertension and improved the oxygen supply-demand relationship in a dose-dependent manner, as indicated by a reduced arterial** lactate concentration.\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> tricuspid...hypertension </strong> </summary>\n",
       "  \n",
       "  - The thoracic radiographs of Dog 13 revealed mild right-sided cardiomegaly and mild generalised bronchointerstitial lung changes; echocardiography showed mild **tricuspid and aortic regurgitation but no signs of pulmonary hypertension** or right ventricular pressure overload.\n",
       "\n",
       "\n",
       "  - Thus, the aim of the present study was to assess the right ventricular systolic function in dogs with **tricuspid regurgitation and pulmonary hypertension** by echocardiographic evaluation.\n",
       "\n",
       "\n",
       "  - Additionally, we examined relationships of transverse strain and strain rate variables with heart rate, age and bodyweight in healthy dogs, and with **tricuspid regurgitation (TR) velocity and left atrial size in dogs with pulmonary hypertension.**\n",
       "\n",
       "\n",
       "  - We also observed that 42.1% of the dogs (n = 278) had concomitant degeneration of the **tricuspid valve and 19.4% (n = 105) pulmonary arterial hypertension** (PH).\n",
       "\n",
       "\n",
       "  - We also observed that 42.1% of the dogs (n = 278) had concomitant degeneration of the **tricuspid valve and 19.4% (n = 105) pulmonary arterial hypertension** (PH).\n",
       "\n",
       "\n",
       "  - Echocardiographic parameters measured/calculated comprised: left ventricular diameter diastole/systole (LVDd/s), left atrial (LA) diameter long-axis view, LA to aorta ratio (LA:Ao) in 2D short-axis, mitral valve (MV) thickening, MV prolapse, MV regurgitation, **tricuspid valve regurgitation (TR), pulmonary hypertension** (TR velocity> 2.8 m/s) and fractional shortening.\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> hypertension...antihypertensive </strong> </summary>\n",
       "  \n",
       "  - Patients with previously untreated essential **hypertension and eutrophic inward remodeling appears to respond to antihypertensive** medication.\n",
       "\n",
       "\n",
       "  - BMI was calculated as weight in kilograms divided by the square of height in meters (kg/m 2 ), and a B MI  25 was defined as overweight; **hypertension was defined as systolic blood pressure  140 mmHg or diastolic blood pr essure  90 mmHg and/or current use of antihypertensive** drugs for blood pressure control.\n",
       "\n",
       "\n",
       "  - Brain APA, the enzyme responsible for generating brain Ang III, therefore constitutes a promising target for **hypertension treatment, justifying the development of potent and selective APA inhibitors as central-acting antihypertensive** agents.\n",
       "\n",
       "\n",
       "  - This is in agreement with the observation that, in experimental models of **hypertension, firibastat acted as an antihypertensive** agent and not as a hypotensive agent.\n",
       "\n",
       "\n",
       "  - Feline **hypertension is commonly associated with chronic kidney disease (CKD) Amlodipine is the first choice antihypertensive** treatment in cats but could have a negative effect on the kidney by increasing glomerular pressure through afferent arteriolar dilatation.\n",
       "\n",
       "\n",
       "  - The aim of this study was to investigate the utility of NT-proBNP as a biomarker of **hypertension, TOD and efficacy of antihypertensive** treatment.\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> amlodipine...hypertension </strong> </summary>\n",
       "  \n",
       "  - The patient was initially treated (day 0) with a combination of prednisolone (2 mg/kg q24h) and melphalan (0.1 mg/kg q48h) by mouth, plus **amlodipine (0.18 mg/ kg) and spironolactone (2 mg/kg) by mouth for management of MM and systemic hypertension,** respectively, but no improvement in clinical and biological condition was observed at days 10, 30 and 45.\n",
       "\n",
       "\n",
       "  - A randomised, double blinded, placebo controlled parallel group study (n = 72) was conducted in 2 phases to determine the efficacy and safety of **amlodipine in cats with naturally occurring hypertension.**\n",
       "\n",
       "\n",
       "  - UVC does not significantly change with **amlodipine treatment suggesting that amlodipine may not cause renal stress when used in cats with hypertension** and concurrent CKD.\n",
       "\n",
       "\n",
       "  - The aim of the study was to assess the efficacy of chewable **amlodipine tablets in reducing systolic blood pressure (SBP) in cats diagnosed with hypertension.**\n",
       "\n",
       "\n",
       "  - The present study is the first large clinical trial to show that **amlodipine is clearly superior to placebo in the treatment of cats with hypertension.**\n",
       "\n",
       "\n",
       "  - The present study related blood pressure reduction to serum **[amlodipine] in feline clinical hypertension.**\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> hypertension...comorbidities </strong> </summary>\n",
       "  \n",
       "  - Diabetes (28.6%) and **hypertension (29.5%) were the most common comorbidities** (Table 1) .\n",
       "\n",
       "\n",
       "  - Uncontrolled **diabetes mellitus and hypertension were the most common comorbidities** reported among three of the cases.\n",
       "\n",
       "\n",
       "  - Uncontrolled **diabetes mellitus, hypertension, and ischemic heart diseases were the most common comorbidities** present and were reported among three cases including the deceased.\n",
       "\n",
       "\n",
       "  - A history of lung diseases, **obesity, diabetes, hypertension, neurological diseases, malignancy, and heart diseases were the most common comorbidities** in our study (83.6%).\n",
       "\n",
       "\n",
       "  - reported that DM, **hypertension, cardiac disease, renal disease, and bronchial asthma were frequent underlying conditions associated with death of MERS-CoV-infected patients and also found that the risk for death increased for patients with multiple comorbidities** (3).\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> hypertension...hyperlipidemia </strong> </summary>\n",
       "  \n",
       "  - The most frequent chronic diseases in our patient population were **hypertension, hyperlipidemia,** lung diseases (e.g., COPD, asthma, etc.\n",
       "\n",
       "\n",
       "  - Patient zero was a 68-year-old man with **hypertension, hyperlipidemia,** and benign prostatic hypertrophy who often used an inhaler for asthma, although asthma did not impede his daily life.\n",
       "\n",
       "\n",
       "  - Side effects were reported for 15 of these cases (8 cases of **hypertension, 4 hepatic dysfunction, 2 hypertrichosis, and 1 hyperlipidemia)**.\n",
       "\n",
       "\n",
       "  - Finally, Saudis live longer now, which explains the rise in cancer cases and chronic illnesses like **diabetes, hypertension and hyperlipidemia.**\n",
       "\n",
       "\n",
       "  - Among Tianjin residents aged 18 years or older, approximately 37% have **hypertension and hyperlipidemia** and 30% are obese (7) .\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> congestive...hypertension </strong> </summary>\n",
       "  \n",
       "  - Patients with **congestive heart failure, coronary artery disease, hypertension,** and aortic valvular disease display increased autophagosomal accumulation in their myocardial biopsies [368] .\n",
       "\n",
       "\n",
       "  - B-type (formerly known as brain-type) natriuretic peptide (BNP or pro-BNP) is a protein that has been commonly used to assess fluid status and cardiac strain in **congestive heart failure and pulmonary hypertension,** among others.\n",
       "\n",
       "\n",
       "  - Based on diagnosis, cats were allocated to 5 groups, namely symptomatic cardiomyopathy with signs of **congestive heart failure (group CM-Sx), asymptomatic cardiomyopathy (group CM-aSx), respiratory diseases (group Resp-Ds), systemic diseases without systemic hypertension** (group Sys-Ds), and healthy controls (group Healthy).\n",
       "\n",
       "\n",
       "  - A previous study of small-breed dogs with mitral valve disease demonstrated that clinically useful information is obtained when classifying murmurs on a 2-level scale, with a 4-level scale providing additional probabilistic information (probability of **congestive heart failure or pulmonary hypertension)**.\n",
       "\n",
       "\n",
       "  - Cardiac disease included **congestive heart failure, coronary artery disease, or a history of myocardial infarction; reports of isolated hypertension** were not included.\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> hypertension...renin </strong> </summary>\n",
       "  \n",
       "  - While markedly increased circulatory levels of ACE2 activity after native rACE2 administration are capable of effectively lowering blood pressure in models of Ang II-induced **hypertension [6, 7, 14] or renin** overexpression in the circulation [15] , their use is not suitable for treatment of other forms of kidney disease.\n",
       "\n",
       "\n",
       "  - Neuronal overexpression of brain ACE-2 is also neuroprotective in a chronic **hypertension mouse model (transgenic for renin** and angiotensinogen that overproduces Ang II) following experimental induction of ischaemic stroke [34, 35, 62] .\n",
       "\n",
       "\n",
       "  - In the circulation, a drop in blood pressure (systemic hypotension) and/or blood volume results in juxtaglomerular cells within the kidneys to release **renin (protease) whereas increased blood pressur**e (hypertension)** inhibits re**nin release under normal circumstances.\n",
       "\n",
       "\n",
       "  - Second, UA can lead to endothelial dysfunction and induce **hypertension through activation of the renin-**angiotensin system [26, 27] .\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> normotensive...hypertension </strong> </summary>\n",
       "  \n",
       "  - In a study that examined ACE2 expression in the kidneys of SHR and **normotensive Wistar-Kyoto (WKY) rats at birth, 6 weeks and adulthood (Tikellis et al., 2006) , kidney ACE2 expression fell with the onset of hypertension** in SHR compared to WKY, and remained reduced in the adult SHR kidney (Tikellis et al., 2006) .\n",
       "\n",
       "\n",
       "  - For example, it has been shown that Ang-(1-7) causes vasodilation in forearm circulation of **normotensive subjects and patients with essential hypertension** [95] while other studies were unable to report any significant effect of Ang-(1-7) in the same vascular territory in ACEi-treated patients [43] .\n",
       "\n",
       "\n",
       "  - The **normotensive group was subsequently used to identify predictors of incident hypertension** using Cox regression analysis (hazard ratio (HR) and 95% confidence intervals (CI) reported).\n",
       "\n",
       "\n",
       "  - Moreover, in azotaemic cats that were **normotensive, FGF-23 concentration appeared associated with the risk of incident hypertension** in a model adjusted for baseline systolic blood pressure, PCV and body weight.\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> antihypertensive...hypertension </strong> </summary>\n",
       "  \n",
       "  - This bears a significant practice implication as thiazide diuretics have been recommended as one of the first-line **antihypertensive therapy for management of arterial hypertension** according to the guidelines issued by the National Institute for Health and Clinical Excellence (2006) [33] and the seventh report of the Joint National Committee 7 th Report (2003) [34] .\n",
       "\n",
       "\n",
       "  - Omapatrilat displayed potent inhibition in the low nanomolar range against both ACE and NEP in vitro, as well as chronic potent **antihypertensive and cardioprotective effects in experimental models of hypertension** and heart failure (Robl et al., 1997 (Robl et al., , 1999 Trippodo et al., 1998; Intengan and Schiffrin, 2000; Pu et al., 2002) .\n",
       "\n",
       "\n",
       "  - In human medicine the recommendation exists to initiate a more aggressive **antihypertensive plan in patients with severe hypertension.**\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> cerebrovascular...hypertension </strong> </summary>\n",
       "  \n",
       "  - Feline ischemic **cerebrovascular disease is commonly associated with an underlying disease, including hypertension,** hyperthyroidism, neoplasia, cardiac disease, and chronic nephropathies (1, 4) .\n",
       "\n",
       "\n",
       "  - Tanshinones have been investigated for the treatment of cardio**-cerebrovascular diseases, such as myocardial infarction, atherosclerosis, hyperlipidemi**a, hypertension,** and str**oke.\n",
       "\n",
       "\n",
       "  - Participants were assigned a '1' if they endorsed a history of each of the following medical conditions: **cerebrovascular risk factors including hypertension, heart failure, COPD, anemia, diabetes;** endocrine dysfunction including thyroid disease, testosterone therapy, estrogen therapy; kidney disease.\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> portal...hypertension </strong> </summary>\n",
       "  \n",
       "  - Ultrasound can also be used to assess hepatomegaly, splenomegaly and dilation of the **portal vein, all of which result from portal hypertension.**\n",
       "\n",
       "\n",
       "  - Portal vein thrombosis (PVT) causes partial or total obstruction of **portal blood flow, leading to pre-hepatic portal hypertension.**\n",
       "\n",
       "\n",
       "  - In fact, CH is a risk factor for **portal vein thrombosis presumably related to decreased production of anticoagulants or to endothelial dysfunction secondary to portal hypertension.**\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> hypertension...tachycardia </strong> </summary>\n",
       "  \n",
       "  - Patients with central nervous system involvement typically present with features of brainstem encephalitis, including myoclonus and autonomic dysregulation **(hypertension, tachycardia)**.\n",
       "\n",
       "\n",
       "  - Reasons for these were: obstetric bleeding (4), **hypertension (2), infection (4), tachycardia** (1), preterm labour (9), exacerbation of pre-existing condition (2) and pre-eclampsia (1).\n",
       "\n",
       "\n",
       "  - The rest of the patients could be divided into two groups: patients with no heart condition (group B, n = 62) discharged as musculoskeletal pain (n = 54), cholecystitis (n = 3), gastritis (n = 2), depression (n = 1), sepsis (n = 1) or meningitis (n = 1); and patients with **cardiovascular disease but no actual ischemia (group C, n = 46) discharged as heart failure (n = 18), hypertension (n = 16), tachycardia** (n = 5), bradycardia (n = 3), dissection of aorta (n = 2) and pulmonary embolism (n = 2).\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> hypertension...tricuspid </strong> </summary>\n",
       "  \n",
       "  - Echocardiographic evaluations were performed on 127 healthy dogs (50 dogs had been included in a previous study of TAPSE), 55 dogs with heart disease but no **pulmonary hypertension (tricuspid regurgitant velocity <3 m/s) and 39 dogs with pulmonary hypertension** of various etiologies and severities (tricuspid regurgitant velocity >2.9 m/s).\n",
       "\n",
       "\n",
       "  - TAPSE:Ao was regressed against bodyweight and, in dogs with pulmonary **hypertension, against tricuspid** velocity.\n",
       "\n",
       "\n",
       "  - In dogs with pulmonary **hypertension TAPSE:Ao had a negative association with tricuspid** velocity.\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> hypertension...amlodipine </strong> </summary>\n",
       "  \n",
       "  - In addition, the patient had >20 years history of **hypertension, up to a maximum of 150/90 mmHg; thus, amlodipine** (5 mg every day) treatment had been used to control the blood pressure, which was determined to be of proper control.\n",
       "\n",
       "\n",
       "  - No further seizures had been observed and asymptomatic **hypertension had recurred but was well controlled after amlodipine** was increased to 1.25 mg q12h.\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> hypertension...cerebrovascular </strong> </summary>\n",
       "  \n",
       "  - The most common preexisting medical conditions were **hypertension (65.7% of subjects), other cardiopulmonary diseases (52.5%), noninflammatory cerebrovascular/** neurological disorders (30.9%), coronary artery disease (28.6%), and autoimmune/inflammatory disease (24.2%).\n",
       "\n",
       "\n",
       "  - The remaining 6 had the following comorbid conditions: **hypertension (3 patients), alcohol dependency and cerebrovascular** disease (1 patient), bipolar disorder (1 patient) and kyphoscoliosis (1 patient).\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> hypertension...portal </strong> </summary>\n",
       "  \n",
       "  - However, symptoms associated with portal **hypertension must be monitored in animals with primary portal** vein hypoplasia.\n",
       "\n",
       "\n",
       "  - However, symptoms associated with portal **hypertension must be monitored in animals with primary portal** vein hypoplasia.\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> artery...hypertension </strong> </summary>\n",
       "  \n",
       "  - He reasoned in his thesis that the severity of the brain pathology in these patients correlates with the diameter of their aneurismal internal carotid **artery, which is similar to the situation of a cirrhotic liver and the dilated portal vein due to portal hypertension.**\n",
       "\n",
       "\n",
       "  - Doppler evaluation of the renal **artery has been performed in humans to identify renal artery stenosis or kinking of an artery as potential causes of renal hypertension.**\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> mellitus...hypertension </strong> </summary>\n",
       "  \n",
       "  - Diabetes **mellitus (DM), hypertension** (HTN), cardiac diseases, renal disease and bronchial asthma were the most frequent comorbid disorders (Table 4) .\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> hypertension...artery </strong> </summary>\n",
       "  \n",
       "  - Paroxysmal **hypertension is a known complication following head and neck radiation, and can be attributed to damage to the carotid artery** baroreceptors [13] .\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "\n",
       "<details>\n",
       "  <summary> <strong> hyperlipidemia...hypertension </strong> </summary>\n",
       "  \n",
       "  - Increased ALP (86.7%) and ALT (40%), **hyperlipidemia (45.6%), low urinary density (57.5%), thrombocytosis (30.2%) and hypertension** (51.4%) were reported.\n",
       "</details>\n",
       "    "
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "for k, sents in sorted(by_gd.items(), key=lambda x: -len(x[1]))[:50]:\n",
    "    sents = '\\n  - ' + ('\\n\\n\\n  - '.join(sents))\n",
    "    dm(f\"\"\"\n",
    "<details>\n",
    "  <summary> <strong> {k} </strong> </summary>\n",
    "  {sents}\n",
    "</details>\n",
    "    \"\"\")\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 152,
   "metadata": {
    "collapsed": true
   },
   "outputs": [
    {
     "data": {
      "text/markdown": [
       "### High-Resolution Analysis of Coronavirus Gene Expression by RNA Sequencing and Ribosome Profiling"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Ribosome profiling is a technique which** exploits the capacity of the translating ribosome to protect around 30 nucleotides of mRNA from ribonuclease digestion.', 'lgd': {'word': 'Ribosome', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'which'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Ribosome-protected mRNA fragments are purified, subjected to deep sequencing and mapped back to the transcriptome to give a global \"snap-**shot\" of translation.', 'lgd': {'word': 'Ribosome', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'snap'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Here we apply ribosome profiling to cells infected with Murine coronavirus, mouse hepatitis virus, strain A59 (MHV-A59), a model coronavirus** in the same genus as SARS-CoV and MERS-CoV.', 'lgd': {'word': 'Here', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'coronavirus'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Ribosome profiling is emerging as a powerful technique** to monitor translation in living cells at sub-codon resolution.', 'lgd': {'word': 'Ribosome', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'technique'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In this work, we carried out the first ribosome profiling analysis of an RNA virus, using as a model system** the murine coronavirus strain MHV-A59, a betacoronavirus in the same genus as the medically important SARS-CoV and MERS-CoV.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'system'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Polyproteins pp1a and pp1b are translated from the genomic RNA, with pp1b being expressed as a transframe fusion** with pp1a (i.e.', 'lgd': {'word': 'Polyproteins', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'fusion'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**pp1ab) vi**a 1** programmed riboso**mal frameshifting (1 PRF).', 'lgd': {'word': 'pp1ab', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'ribosomal'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The 3 0 ORFs are expressed from a series of** subgenomic RNAs produced during infection.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Each subgenomic RNA contains a 5 0** leader sequence that is identical to the 5 0 leader of the genome, appended via polymerase \"jumping\" between body transcription regulatory sequences (TRSs) (green diamonds) and the leader TRS (orange diamond) during negative-strand synthesis.', 'lgd': {'word': 'Each', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': '0'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '(repeat **1) in** reads per million mapped reads (RPM).', 'lgd': None, 'rgd': {'seed': '1', 'word': 'in'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Read densities are plotted on a log(1+**x) scale to cover the wide range in expression across the genome.', 'lgd': {'word': 'Read', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': '1'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Histograms show the positions of the 5 0 ends of reads with a +12** nt offset to map (for RPFs) approximate P-site positions.', 'lgd': {'word': 'Histograms', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': '12'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**(C) The positive-sense RiboSeq/RNASeq ratio, after first applyi**ng a** 15**-nt running mean (RM) filter to each individual distribution.', 'lgd': {'word': 'C', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'nt'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Coronavirus infections are frequent in bats and other mammal**s [1]**** and interactions between humans and non-human animal populations presents a constant risk of new zoonotic outbreaks [2] .', 'lgd': {'word': 'Coronavirus', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'and'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Therefore, a useful model** for these two viruses, especially with regard to their structure and replication, is Murine coronavirus, a betacoronavirus that is commonly referred to as mouse hepatitis virus (MHV).', 'lgd': {'word': 'Therefore', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'model'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Like all coronaviruses, MHV has a monopartite, positive-**sense, single-stranded RNA genome (gRNA) (Fig 1A) .', 'lgd': {'word': 'Like', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'positive'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**These ORFs are expressed as two polyproteins pp1a and pp1ab, where pp1ab is a \"transframe\"** fusion of the ORF1a and ORF1b products, produced via 1 programmed ribosomal frameshifting (1 PRF) [4, 5] .', 'lgd': {'word': 'These', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'transframe'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**These ORFs are translated from a series of** subgenomic mRNAs (mRNAs 2 to 7) produced during virus infection.', 'lgd': {'word': 'These', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Each subgenomic mRNA is identical to a 3 0** -coterminal region of the virus genome with the exception of a 65 nucleotide (nt) leader sequence at the 5 0 end that is identical to the 5 0 end of the gRNA.', 'lgd': {'word': 'Each', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': '0'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**These leader sequences are added (as a reverse Ribosome** profiling is an emerging methodology that facilitates global mapping of the positions of translating ribosomes on the transcriptome, defining at the codon level the extent to which individual mRNAs species are engaged in protein synthesis [7] [8] [9] .', 'lgd': {'word': 'These', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'Ribosome'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The technique exploits the knowledge that translating ribosomes can protect from RNase digestion a defined fragment** of mRNA of around 28-30 nt in length [10] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'fragment'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In ribosome profiling, often referred to as RiboSeq, cells are lysed under conditions optimised to minimise further ribosome movement (addition of translation inhibitors, rapid freezing), the lysate is treated with ribonuclease (often RNase 1) to** degrade regions of mRNAs that are not physically protected, and the ribosomes harvested on sucrose gradients or through a sucrose cushion.', 'lgd': {'word': 'In', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'to'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The ribosome pellet is deproteinized, the ribosome-protected fragments (RPFs) harvested by elution from a polyacrylamide gel,** ligated to adapters, subjected to RT-PCR, deep sequenced and mapped back to the genome.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'gel'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The corresponding transcriptome is also determined from the same lysate: total RNA is harvested, fragmented, cloned and sequenced to generate a parallel RNA** sequencing (RNASeq) library.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'RNA'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Ribosome profiling has been applied to a variety of** cellular organisms to address a range of questions in translational control and global gene expression [9, [11] [12] [13] [14] [15] [16] .', 'lgd': {'word': 'Ribosome', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': \"**Also, it has been employed in the study of the replication of large DNA viruses; namely, human cytomegalovirus [17] [18] , Kaposi's sarcoma-associated herpesvirus [19] , herpes simplex virus 1 [20]** , vaccinia virus [21] , and bacteriophage lambda [22] , providing insights into the temporal regulation of gene expression in these viruses and identifying numerous previously unrecognized translated ORFs, including novel protein-coding ORFs and short regulatory uORFs.\", 'lgd': {'word': 'Also', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '20'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In this paper, we describe the first analysis of RNA virus replication and gene expression by ribosome profiling (and parallel RNASeq), using MHV as a model system.**', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'system'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The data obtained allowed us to determine the time course of virus positive and negative-sense RNA production, as well as the translation of each of the virus genes, the expression of short and/or previously unannotated ORFs, and the efficiency of 1** PRF.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Surprisingly, we found that ribosomes do not pause appreciably durin**g 1** PRF, argu**ing against a requirement for pausing in frameshifting.', 'lgd': {'word': 'Surprisingly', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'arguing'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**To study the kinetics of virus RNA and protein synthesis in a single cycle** of virus replication, we performed two independent biological repeats (repeats 1 and 2) of an MHV infection time course in which murine 17 clone 1 cells (17Cl-1) were infected with recombinant MHV-A59 at a multiplicity of infection (MOI) of 10 and cells harvested at 1, 2.5, 5 and 8 h post-infection (p.i.', 'lgd': {'word': 'To', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'cycle'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '), **with mock-infected cells harvested at 1 and 8** h. For all time points, two dishes were prepared and, immediately prior to harvesting, cells were treated with cycloheximide (CHX) alone, or harringtonine (HAR) then CHX (as detailed in Materials and Methods), for analysis of elongating (CHX) and initiating (HAR) ribosomes, respectively.', 'lgd': {'word': 'with', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '8'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In general, there is a 5 0** to 3 0 increasing gradient in total ribosome density, with the N ORF being expressed at the highest level, the M, 5, S and 2 ORFs at intermediate levels, and ORFs 1a and 1b at the lowest levels.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': '0'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Thus, these are unlikely to derive from ribosomes loading onto negative-sense RNAs (e.**g.', 'lgd': {'word': 'Thus', 'seed': 'e'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This ratio was substantially lower in ORF1a and ORF1b than in the 3 0 coding ORFs (except the defective genes HE and 4), which may indicate that a substantial proportion** of genomic RNA is not being translated (e.g.', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'proportion'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**sequestered in replication-transcription complexes [RTCs] or destined for packaging) or that genomic RNA intrinsically has a relatively low** translation efficiency.', 'lgd': {'word': 'sequestered', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'low'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Note, however, that this simple calculation ignores the fact that RNASeq density is present for all ORFs on a transcript whereas** RiboSeq density is only present for the translatable ORFs (normally the 5 0 proximal ORF).', 'lgd': {'word': 'Note', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'whereas'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This analysis shows that there is significant variability in the RNASeq read depth within a transcript, which** we ascribe to biases such as fragmentation bias, PCR bias and ligation bias.', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'which'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Similarly, variability in the RiboSeq data within a CDS may** be partly due to nuclease bias, PCR bias and ligation bias but also reflects real variations in ribosome progressivity.', 'lgd': {'word': 'Similarly', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'may'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**S2 Fig (repeat 1) and** S3 Fig (repeat 2) show, for each library, histograms of the codon positions to which the 5 0 ends of host mRNA reads map for different read lengths.', 'lgd': {'word': 'S2', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'and'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Conversely, and as expected, the 5 0 ends of RNASeq reads had a nearly uniform** distribution between the three possible codon positions.', 'lgd': {'word': 'Conversely', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'uniform'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The RiboSeq read length distributions were typically sharply peaked at around 29 nt consistent with other analyses [8] , while those of RNASeq were much broader, consistent with a length distribution** set by the size of the gel slice excised during purification of fragmented RNA in the RNASeq protocol (approximately 28-34 nt).', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'distribution'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**For all samples, a discrete peak** in RPF abundance was observed just upstream of the initiation site.', 'lgd': {'word': 'For', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'peak'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**As noted previously, the peak is probably largely a result of** drug treatment-either HAR which specifically arrests initiating ribosomes, or CHX which arrests elongating ribosomes but allows ribosomes to continue to accumulate at initiation sites [8] .', 'lgd': {'word': 'As', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This peak corresponds to the 5 0 ends of RPFs derived from initiating ribosomes with the AUG codon in the ribosomal P-site, and allows calibration of the offset between the RPF 5 0 end and RPF P-site position, which, for these libraries, is normally 12 nt (e.**g.', 'lgd': {'word': 'This', 'seed': 'e'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**For many samples, a discrete peak** was also observed 15 nt upstream of the stop codon, corresponding to ribosomes pausing during termination (with the stop codon in the ribosomal A-site).', 'lgd': {'word': 'For', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'peak'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**repeat 1 and repeat 2, RiboSeq CHX mock 1** h in S4 Fig) .', 'lgd': {'word': 'repeat', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'repeat'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**For 30-nt reads, a trough was** also apparent four codons upstream of the termination peak (S5 Fig); this corresponds to reads that all end in UAG-3 0 , UAA-3 0 or UGA-3 0 , and again is likely to be an artefact of ligation bias.', 'lgd': {'word': 'For', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'was'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A peak 12 nt upstream of the AUG (more noticeable in repeat 1 samples, S6** Fig, left ) together with a very low level of phasing within the CDS (S6 Fig) likely represents a low level of contamination of RNASeq samples by RiboSeq samples, although the latter could potentially also be a result of codon usage bias, e.g.', 'lgd': {'word': 'A', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'S6'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**a preference for** RNY codons [25] , compounded with ligation biases.', 'lgd': None, 'rgd': {'seed': 'a', 'word': 'for'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Averaged over all host mRNAs, very few RPFs were observed in 3 0 UTRs while a larger but** still low level of RPFs were observed in 5 0 UTRs (S4 Fig and S5 Fig) .', 'lgd': {'word': 'Averaged', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'but'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**We also observed a remarkable perturbation** in host cell translation at late time points (S4 Fig, lower panels-RiboSeq CHX, compare 5 and 8 h p.i.', 'lgd': {'word': 'We', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'perturbation'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**with 1 and 8 h mocks) that was not mirrored in RNASeq data (S6 Fig) and could be a** consequence of cell stress [26] [27] [28] .', 'lgd': {'word': 'with', 'seed': 'a'}, 'rgd': {'seed': '1', 'word': '8'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Indeed, a comparison of** read length distributions of host-derived RNA and virusderived RNA revealed contamination of this type in some of our libraries, despite great care in processing experimental samples (S7 Fig and S8 Fig) .', 'lgd': {'word': 'Indeed', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**However, for the 1 and 2.**5 h p.i.', 'lgd': {'word': 'However', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '2'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The absolute amount of contamination was very low and would have little effect on host mRNA analyses; however, relative to the amount of virus RNA at 1 and 2.**5 h p.i., it was significant.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '2'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Contamination was also apparent for the 1 and 2.**5 h p.i.', 'lgd': {'word': 'Contamination', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '2'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**RiboSeq HAR samples and the 1 h p.**i.', 'lgd': {'word': 'RiboSeq', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'p'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Nevertheless, traces of contamination were still apparent in the 1 and 2.**5 h p.i.', 'lgd': {'word': 'Nevertheless', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '2'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**RiboSeq CHX and 1 h p.** i. RiboSeq HAR samples (S8 Fig).', 'lgd': {'word': 'RiboSeq', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'p'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This suggests that this sample is contaminated with RNASeq material from a sample containing** little or no virus RNA, thus affecting the host mRNA length distribution but not the virus RNA length distribution.', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'containing'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Since the analysis is based on mapping to NCBI RefSeq mRNAs, a low level** of 3 0 UTR occupancy derives from genuine RPFs derived from coding exons in one splice form that have alternative mappings to the 3 0 UTR in another splice form.', 'lgd': {'word': 'Since', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'level'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Thus, for mock infections, the 3 0 UTR RiboSeq read length distributions largely matched those of the CDSs (S9 Fig and S10 Fig, 1 and 8** h mock CHX), albeit with some differences (e.g.', 'lgd': {'word': 'Thus', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '8'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**a high-end** tail) arising from unknown sources of contamination potentially including host protein:mRNA RNPs.', 'lgd': None, 'rgd': {'seed': 'a', 'word': 'end'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**For infected samples, the host mRNA 3 0 UTR density for CHX samples was similar in magnitude (0.5-1.2%) to** that of the mocks (0.7-0.9%), except for the 8 h p.i.', 'lgd': {'word': 'For', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'to'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**CHX repeat 1 sample was** not qualitatively different from that of the mocks, suggesting that the increase in 3 0 UTR occupancy might not simply be explained by virus-induced RNPs, but rather, or as well, by an increase in bona fide RPFs in the 3 0 UTRs.', 'lgd': {'word': 'CHX', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'was'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': 'If **a proportion of** late time-point contamination results from virus proteins interacting with mRNA to form RNPs, it may be significantly higher for virus RNA than for host mRNA, as virus proteins are likely to interact selectively with virus RNA; for example, through specific binding signals or via compartmentalization within the cell.', 'lgd': None, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Reassuringly, in all cases, the virus positive-sense RiboSeq reads showed a similar or** even tighter length distribution at late time points than the host RiboSeq reads (S7 Fig and S8 Fig; 5 h.p.i and 8 h p.i., CHX and HAR).', 'lgd': {'word': 'Reassuringly', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'or'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Fig 3A shows a time course** of the total amount of virus RNA expressed as a fraction of total virus RNA plus host mRNA, for both RiboSeq CHX and RNASeq samples.', 'lgd': {'word': 'Fig', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'course'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Total virus translation as a fraction of** total cellular translation increased 700 to 20,000-fold from 1 to 5 h p.i., while virus positive-sense RNA increased 80 to 200-fold over the same time period.', 'lgd': {'word': 'Total', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In repeat 2, virus translation and RNA appeared to have reached a maximum by** 5 h p.i., while infection progressed a little slower in repeat 1.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'by'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': 'From **1 h p.**i.', 'lgd': None, 'rgd': {'seed': '1', 'word': 'p'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**to 2.5 h p.i., the positive-sense RNA fraction remained roughly constant (presumably reflecting the input RNA) while the negative-sense RNA fraction grew from essentially negligible amounts to~0.1% of** total virus RNA and host mRNA (Fig 3A) .', 'lgd': {'word': 'to', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Note, again, that the data only provide upper bounds for the Raw RNASeq densities represent the cumulative sum of densities for all mRNAs that cover a given genome** region.', 'lgd': {'word': 'Note', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'genome'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Subtraction of the density for the immediately upstream inter-TRS region gives an estimate of the RNASeq density for a specific mRNA,** herein referred to as the \"decumulated\" density.', 'lgd': {'word': 'Subtraction', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'mRNA'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**RNASeq densities for mRNA4 are omitted as it is not expressed at a sufficiently high** level relative to mRNA3 to apply the \"decumulation\" procedure.', 'lgd': {'word': 'RNASeq', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'high'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Circles and solid lines correspond to repeat 2; crosses and dotted lines correspond to repeat 1.**', 'lgd': {'word': 'Circles', 'seed': '1'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Due to low levels of reads and contamination (see text), values for 1 h p.**i.', 'lgd': {'word': 'Due', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'p'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The particularly contaminated repeat 1 RiboSeq 1** h p.i.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '1'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**We then followed a similar procedure** for all other mRNAs.', 'lgd': {'word': 'We', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'procedure'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**We also omitted the 1 h p.**i.', 'lgd': {'word': 'We', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'p'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**As mentioned above, this could be due to a proportion of** gRNA being present in an untranslatable pool, perhaps as RTCs or RNPs destined for packaging, but may also indicate a real restraint on ORF1a/1ab translatablity (see below).', 'lgd': {'word': 'As', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**On the assumption that gRNA will not be directed to a packaging pathway** at early time points, this suggests that incoming and early synthesis gRNA is largely involved in RNA synthesis, or is, indeed, inherently poorly translated.', 'lgd': {'word': 'On', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'pathway'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Such \"chimeric\" reads (where the 5 0 part maps to the leader sequence, and the 3 0 part maps just downstream of a body TRS)** were not included in the initial mapping to the virus genome ( Fig 1B) , but were identified subsequently (see Materials and Methods).', 'lgd': {'word': 'Such', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'TRS'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The \"TRS method\" has the advantage that it avoids the potential inaccuracies introduced by decumulation but may be more subject to fragmentation, ligation and PCR biases due to the relatively short window in which to calculate a mean RNASeq** density.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'RNASeq'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Nonetheless there is a good correlation** between the two estimates (R 2 = 0.99).', 'lgd': {'word': 'Nonetheless', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'correlation'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Heterogeneity in the number of copies of the pentanucleotide has previously been observed to occur in mRNA6 for MHV-A59, and both mRNA6 and mRNA7 for MHV-JHM, and this is presumably due to heterogeneity in the site of re-annealing following a polymerase jump** [39] .', 'lgd': {'word': 'Heterogeneity', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'jump'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Consistent with this, we also observed significant usage of a junction site** 5 nt upstream of the canonical site for mRNA6 (13-17% of mRNA6 transcripts) (S3 Table) .', 'lgd': {'word': 'Consistent', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'site'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The greater usage for mRNA6 is likely due to it having an imperfect pentanucleotide UCCAA at the canonical junction site but a perfect pentanucleotide** UCUAA 5 nt upstream; in contrast, other mRNAs have a better pentanucleotide match at the canonical site than at the site 5 nt upstream (S2 Table) .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'pentanucleotide'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**For mRNA7, where the upstream pentanucleotide is CCUAA instead of UCUAA, we observed that the first nucleotide could be templated either by the body sequence (i.e.**', 'lgd': {'word': 'For', 'seed': 'e'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': \"**'C';~40%) or by the leader sequence (i.e.**\", 'lgd': {'word': 'C', 'seed': 'e'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The sequences at non-canonical junctions often partly resembled canonical TRSs (e.**g.', 'lgd': {'word': 'The', 'seed': 'e'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In cases where the nucleotides at +1 to +2** in the genome sequence differed from UC, the RNASeq read generally contained nucleotides templated by the genome sequence rather than the UC in the leader sequence (e.g.', 'lgd': {'word': 'In', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '2'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**CC instead of UC for the nt 22483 junction), although there were exceptions (e.**g.', 'lgd': {'word': 'CC', 'seed': 'e'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This latter site, AAUAAGC, aligns with a TRS previously** identified for an HE mRNA in the JHM strain of MHV [38] .', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'previously'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The sequence in MHV-JHM is AAUAAAC, differing from the MHV-A59 sequence by a G to** A substitution.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'to'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The latter are calculated on a per gene** (rather than per transcript) basis, using RNASeq and RiboSeq reads contained entirely within annotated CDS regions (i.e.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'gene'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Fig 7A shows the distributions of RiboSeq and RNASeq reads on the virus genome at 1 h p.**i.', 'lgd': {'word': 'Fig', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'p'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A 5 0 to 3 0 decreasing gradient in RPF density is visible within ORF1a in the RiboSeq CHX density profile, while very few RPFs were found within ORF1b, indicating that, at 1 h p.**i., ribosomes have only had time to translate part of the 4470-codon ORF1a.', 'lgd': {'word': 'A', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'p'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This does not indicate the translate rate per se, as time is also required for uncoating, recruitment of ribosomes, translation of a uORF on** the gRNA (see below), and potential delays with initiation and reinitiation (see also below).', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'on'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In the RiboSeq HAR data, a clear trough** in RPF density is visible after the ORF1a initiation peak, followed by higher density further downstream in ORF1a.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'trough'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Taking the width of the trough as~750 codons, this gives an elongation rate of 4.2 amino acids **s 1** , simi**lar to that determined previously in mouse embryonic stem cells (5.6 amino acids s 1 ) [8] .', 'lgd': {'word': 'Taking', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'similar'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**While it is impossible to definitively rule out contamination in this way, the analysis suggests that the RNASeq density in the N region at 1 h p.**i.', 'lgd': {'word': 'While', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'p'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**is not a result of** contamination.', 'lgd': {'word': 'is', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Since, for mRNA7, negative-sense RNA is present at >0.1% of** positive-sense RNA at 2.5, 5 and 8 h p.i.', 'lgd': {'word': 'Since', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**(Fig 4) , the absence of appreciable levels of negative-sense reads mapping to the N region in the high M**OI 1**** h p.i.', 'lgd': {'word': 'Fig', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'p'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The near-complete absence of negative-sense reads also argues against this phenomenon being a result of** early synthesis.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Closer inspection revealed a number of** a relatively abundant chimeric reads suggesting a mosaic structure of rearranged N-ORF sequences, reminiscent of defective interfering (DI) RNAs [43, 44] .', 'lgd': {'word': 'Closer', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**However, since coronavirus DI RNAs are expected to include parts of the 5 0 end of the genome and a packaging signal,** and only arise after multiple high-MOI passages, we believe the N ORF transcripts we have identified must represent a novel class of packaged transcripts.', 'lgd': {'word': 'However', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'signal'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**An alternative, albeit unlikely, explanation is that the excess 3 0 density may be a result of** selective degradation (either natural or artifactual) of~96% of the input gRNA.', 'lgd': {'word': 'An', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Using these RPF counts, the N ORF translation efficiency (normalized to total virus RNA and host mRNA) was calculated to be only 0.0005, compared to values in the range 1.1 to** 1.7 at the 2.5, 5 and 8 h p.i.', 'lgd': {'word': 'Using', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'to'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Th**e 1** PRF sig**nal that facilitates expression of MHV pp1ab comprises two elements, a heptanucleotide slippery sequence (U_UUA_AAC), identical in all known coronaviruses, and an RNA pseudoknot structure located a few nucleotides downstream [5, 45, 46] (Fig 8A) .', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'signal'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Frameshifting likely provides a fixed ratio** of translation products for assembly into a macromolecular complex [47, 48] .', 'lgd': {'word': 'Frameshifting', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'ratio'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Such a calculation is, therefore, invalid immediately after infection (as ribosomes begin to translate ORF1a of the input gRNA but have not yet reached ORF1b; Fig 7) and may also be compromised if newly synthesised gRNA entering the translation pool represents a** significant fraction of the gRNA undergoing translation.', 'lgd': {'word': 'Such', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'is'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Thus, we measured the frameshifting efficiency at 5 h p.i., calculating values of 48% for repeat 1, and** 70% for repeat 2 ( Fig 8B) .', 'lgd': {'word': 'Thus', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'and'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**As the infection appeared to be more advanced in repeat 2 (Fig 3) , it is possible that the higher level measured (70%) is a consequence of** depletion of the host cell protein synthesis resources, e.g.', 'lgd': {'word': 'As', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**exhaustion of initiation factors (including free ribosomes) could decrease the density of ribosomes in ORF1a as elongating ribosomes run off, and a partial exhaustion** of elongation factors could slow the establishment of a new steady state.', 'lgd': {'word': 'exhaustion', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'exhaustion'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**We began by cloning a 100-bp** fragment including the MHV frameshift signal (Fig 8A) into a dual luciferase frameshift reporter plasmid (pDluc; [52, 53] ) between the Renilla (Rluc) and firefly (Fluc) luciferase genes.', 'lgd': {'word': 'We', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'bp'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In this plasmid, frameshifting permits expression of Fluc as a fusion with** Rluc (analogous to the expression of MHV pp1ab), while failure to frameshift results in expression of Rluc alone.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'with'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Frameshifting efficiencies were calculated from the ratio of Fluc activity to Rluc activity, normalized by a control construct** in which an extra C residue was inserted immediately downstream of the slippery sequence to place Rluc and Fluc in the same reading frame.', 'lgd': {'word': 'Frameshifting', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'construct'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The well-studied coronavirus frameshifting signal from Avian coronavirus, infectious bronchitis virus (IBV) served as a positive control,** alongside a lower efficiency control (the gag/pol 1 PRF signal of HIV isolate HXB2) [54, 55] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'control'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The MHV frameshifting efficiency was found to be 38% in 17Cl-1 and 45%** in BHK-21 cells, and similar in both instances to that of IBV ( Fig 8C) .', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '45'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The relevance of ribosomal pausing to the mechanism o**f 1** PRF** has long been a subject of debate [56] [57] [58] .', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'has'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Frameshift signal-associated pauses have been documented in a number of** in vitro assays [59] [60] [61] [62] [63] [64] , but there is, as yet, little evidence for a causal relationship, and pausing has not been examined in infected cells.', 'lgd': {'word': 'Frameshift', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**However, reasoning that the frameshift-stimulatory pseudoknot beginning 6 nt 3 0 of the slippery heptanucleotide U_UUA_AAC would be partly inside the mRNA entrance channel at the onset of frameshifting, and might, due to its compact structure be somewhat resistant to RNase 1 digestion, we** considered the possibility that frameshift-associated pauses may generate longer RPFs, which would be excluded from the profiling analysis as a result of gel size selection (28-34 nt).', 'lgd': {'word': 'However', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'we'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**repeat 2 RiboSeq CHX samples (see S1C Table for composition) using a larger gel** slice (nominally 28-80 nt).', 'lgd': {'word': 'repeat', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'gel'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Although we failed to identify significant pausing at the frameshift site, there were other sites at which RPFs accumulated to a much higher** level than at neighbouring sites.', 'lgd': {'word': 'Although', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'higher'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**We frequently observed such accumulations at initiation sites (possibly an artifact of CHX treatment; [8] ) (Fig  1 and Fig** 2) , but also at internal sites within ORFs.', 'lgd': {'word': 'We', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'Fig'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Besides ribosome pausing, fluctuations in RPF density may occur as a result of** nuclease, ligation, and PCR biases.', 'lgd': {'word': 'Besides', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Following [65] , we compared the distributions of variability in RiboSeq and RNASeq densities within ORF1a, which revealed that RiboSeq densities were more variable than RNASeq densities ( Fig  9A) , with the extra variability presumed to be a result of** fluctuations in ribosome progressivity.', 'lgd': {'word': 'Following', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The 3 0 end positions of these RPFs were, as is normal, more variable, with the most abundant 3 0 ends mapping to nt 11393-11394 for repeat 1 and nt** 11394-11395 for repeat 2, giving read lengths of 28-29 and 29-30 nt which are within the typical range for the respective samples ( S2 Fig and S3 Fig) .', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'nt'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**repeats 1 and 2, 64% and 66%, respectively, of host mRNA RPFs have 5 0 ends mapping to codon position 1,** with only 7% and 8% mapping to codon position 2.', 'lgd': {'word': 'repeats', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '2'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The reason for the deviation at the pause site is unknown but may be a result of** \"tension\" within the mRNA or perturbation of the ribosome conformation [66] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Due to the unusual codon position of the 5 0 end, it was not possible to definitively predict the P-site position of ribosomes at this pause site, but it is more likely to be at nt 11377 to 11379 (AAA codon) than at nt 11380 to 11382 (CAG codon) as the lengths of the most abundant reads (28-29 and 29-30 nt in repeats 1 and 2,** respectively) are more consistent with reads being 1 nt shorter than normal at the 5 0 end, rather than 2 nt longer.', 'lgd': {'word': 'Due', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '2'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The nascent peptide sequence (i.e.**', 'lgd': {'word': 'The', 'seed': 'e'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**An additional smaller peak was apparent 30 nt upstream (in repeat 1) and** potentially corresponds to a following ribosome stacking behind a proportion of paused ribosomes.', 'lgd': {'word': 'An', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'and'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**RPFs at the first of the two pause sites, located in the nsp3 region, have 5 0 ends that map to nt 4704 in repeat 1 but nt** 4699 in repeat 2 (Fig 9C, left) .', 'lgd': {'word': 'RPFs', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'nt'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**However, this 5 nt difference is made up in the length of the reads (top three read lengths 28, 27 and 29 nt in repeat 1, but** 33, 34 and 32 nt in repeat 2) so that the 3 0 ends of the RPFs map to similar positions in both repeats.', 'lgd': {'word': 'However', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'but'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In general the nuclease trimming seems to be less stringent in repeat 2 than in repeat 1 (host** mRNA RPF lengths peak at 29-30 nt in repeat 1 but at 30 nt in repeat 2; S2 Fig and S3 Fig).', 'lgd': {'word': 'In', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'host'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The pause site read length in repeat 2 is unusually long, indicating that the extra~5 nt at the 5 0 end are, for some reason, partially protected, resisting trimming in repeat 2 but not in the more stringently trimmed repeat 1.**', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**An alternative explanation is that these pauses are induced by trans-acting factors, e.**g.', 'lgd': {'word': 'An', 'seed': 'e'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Coronaviruses induce substantial membrane rearrangements in the infected cell, including formation of a reticulovesicular network** composed of two types of membrane modifications, double-membrane vesicles (DMVs) and convoluted membranes (CM).', 'lgd': {'word': 'Coronaviruses', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'network'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Nsp3 is the largest protein encoded in MHV ORF1a and contains multiple domains, including two small ubiquitin-like domains (Ubl1 and Ubl2), two papain-like cysteine proteinase domains (PLP1 and PLP2), a poly-ADP-**ribose-binding activity (ADRP domain), the newly determined domain preceding Ubl2 and PLP2 (DPUP; [76] ), the nucleic-acid binding domain (NAB), the betacoronavirus marker (G2M), a transmembrane domain (TM) and domain Y (Fig 10A) .', 'lgd': {'word': 'Nsp3', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'ADP'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Initially, a cDNA fragment** comprising the first 1,125 residues of nsp3 (nsp3  ; excluding the NAB, G2M, TM and Y domains) was cloned into pcDNA3.1 and synthetic mRNAs translated in RRL for 3 min prior to addition of the translation initiation inhibitor edeine.', 'lgd': {'word': 'Initially', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'fragment'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Incubation was continued, samples withdrawn at the indicated times post-edeine addition and translation products separated on a 10% SDS-**PAGE gel (Fig 10B) .', 'lgd': {'word': 'Incubation', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'SDS'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**To accurately mark the position of the predicted pause product, a control mRNA** in which a UAA stop codon had been introduced at the pausing A-site was also translated.', 'lgd': {'word': 'To', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'mRNA'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**As seen in Fig 10B  (marked by a red asterisk)** a distinct translational pause was observed during translation of nsp3  , migrating at the same position as the \"pause control\" and accumulating and then Samples were withdrawn at the indicated times after edeine addition, and translation products separated on a 10% SDS-PAGE gel and detected by autoradiography.', 'lgd': {'word': 'As', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'asterisk'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**MW indicates 14 C-labelled molecular weight standards and H 2 O as a negative control.**', 'lgd': {'word': 'MW', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'control'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**pPS0 contains, under the control of the SP6 promoter, a copy of** the influenza virus PB1 gene into which has been inserted cDNA encoding the nsp3 pause region (red) plus 30 upstream residues.', 'lgd': {'word': 'pPS0', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In mutant 1, Lys-**Phe at the pausing site was changed to Ala-Ala, and subsequent mutants were prepared sequentially from this clone, thus mutant 5 contains six substitutions (see text).', 'lgd': {'word': 'In', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'Lys'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In all panels, the pause product is indicated by a red asterisk.**', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'asterisk'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**To more closely define the stalling sequence, a region encoding** 46 amino acids of nsp3 including the putative pausing peptide was cloned into the influenza PB1 reporter gene in transcription vector pPS0 (Fig 10C) [60] and edeine assays performed as above.', 'lgd': {'word': 'To', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'encoding'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Once again, a clear ribosomal** pause was evident (Fig 10C, red asterisk) .', 'lgd': {'word': 'Once', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'ribosomal'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**These residues were mutated to alanine sequentially and incrementally (pPS0-nsp3 mutants), such that in Mut 1, Lys-**Phe adjacent to the pausing site was changed to Ala-Ala, Mut 2 had these changes plus Arg to Ala, and so on, as shown in Fig 10D.', 'lgd': {'word': 'These', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'Lys'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Edeine assays were performed and a single time** point (20 min) from each mutant analysed by SDS-PAGE.', 'lgd': {'word': 'Edeine', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'time'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**As seen in Fig  10D, pausing was obviated in Mut 3, Mut 4 and Mut 5, indicating that the residue substituted by alanine Mut 3 is likely to be a major contributor** to the ribosomal pause.', 'lgd': {'word': 'As', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'contributor'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The plot excludes \"chimeric\" reads (i.e.**', 'lgd': {'word': 'The', 'seed': 'e'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**reads that span a TRS transcriptional** discontinuity), so the RNASeq density drops close to the TRS site and the same is also expected to happen for RiboSeq.', 'lgd': {'word': 'reads', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'transcriptional'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Probing of initiation sites through harringtonine treatment revealed unexpectedly that a substantial number** of reads accumulate at or near the 5 0 end of the leader, despite an absence of AUG codons.', 'lgd': {'word': 'Probing', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'number'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**These 5 0 -proximal reads have a tight length** distribution characteristic of true RPFs (Fig 11B; left panel) so are likely to be bona fide RPFs rather than some form of contamination.', 'lgd': {'word': 'These', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'length'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The 5 0 portion of the leader contains a number of** potential near-cognate non-AUG initiation codons, but most of the harringtonine reads do not obviously map to these.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**For example, the most abundant RPF position corresponds to a GCG codon** (genome coordinates [16] [17] [18] ; initiation at this point would generate a 12 amino acid peptide, but it should be noted that GCG is not a recognised non-AUG initiation codon.', 'lgd': {'word': 'For', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'codon'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Elongation profiling with cycloheximide revealed a similar pattern** of reads in the 5 0 part of the leader but also a larger peak on a UUG codon close to the 3 0 end of the leader sequence ( Fig  11A) .', 'lgd': {'word': 'Elongation', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'pattern'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**UUG is a known, albeit** quite inefficient non-AUG initiation codon [8, 78] and, in this case, it is also in a poor initiation context (cucUUGuag; in mammals contexts with an A at 3, or a G at 3 and a G at +4, may be regarded as \"strong\"; [79] ), so only a very small proportion of ribosomes would be expected to initiate here.', 'lgd': {'word': 'UUG', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'albeit'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Consistent with this, the HAR peak is very small compared to that seen at the N initiation codon (1.4%; Fig** 11C) (though similar in magnitude to initiation peaks at the uORF and ORF1a on the genomic RNA; Fig 11A) .', 'lgd': {'word': 'Consistent', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'Fig'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The reasons for this are unknown, but may be related to the UUG codon being immediately followed by a termination codon,** with the peak potentially being derived from both initiation and termination pauses (UUG in P-site, UAG in A-site).', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'codon'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Downstream of the leader TRS but upstream of ORF1a, is a single, short** AUG-initiated uORF that is present in many coronaviruses and believed to play a role as a regulator of genomic [80] .', 'lgd': {'word': 'Downstream', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'short'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This effect appeared less pronounced at early time points, suggesting a potential role** for temporal regulation of replication protein synthesis (Fig 12A, bottom panel) .', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'role'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In the laboratory-adapted strain MHV-A59, the HE ORF is disrupted by a premature termination** codon (red diamond, Fig  13A) [85] , and, furthermore, the TRS upstream of HE in MHV-A59 is defective (open green box, Fig 13A) [38] , leading to only very low levels of HE mRNAs (see above).', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'termination'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Translation of the annotated HE ORF (i.e.**', 'lgd': {'word': 'Translation', 'seed': 'e'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The low level of initiation noted at the \"authentic\" HE start codon is likely explained by the very low levels of HE mRNA production inferred from the observation of a few RNASeq** reads crossing the HE leader/body transcriptional discontinuity (see above and S3 Table) , since leaky scanning on mRNA2 is unlikely to allow access to HE due to the large number of intervening AUG codons.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'RNASeq'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Similarly, in MHV-A59 the natural ORF4 coding sequence is split by a frameshift mutation** into a short 5 0 ORF4a (pale yellow, Fig 13B) and a longer 3 0 ORF4b (grey, Fig 1A and Fig 13B) [86] .', 'lgd': {'word': 'Similarly', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'mutation'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Upstream of ORF4a, but downstream of the mRNA TRS junction, a low level** of initiation appeared to occur on an AUU codon (RiboSeq, HAR, orange peak).', 'lgd': {'word': 'Upstream', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'level'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Ribosomes initiating here would translate a 15-codon** ORF resulting in the peptide MYSILIATWPRKRQS (assuming the initiating codon AUU is decoded as Met).', 'lgd': {'word': 'Ribosomes', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'codon'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Upstream of ORF5, we identified an alternative initiation site at a CUG codon** (Fig 13C,  HAR , blue peak) which may have some bearing on the mechanism of expression of the E ORF, which lies downstream of ORF5 on the bicistronic mRNA5 (Fig 13C) .', 'lgd': {'word': 'Upstream', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'codon'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The level of initiation at the CUG was comparable to that at the ORF5 AUG ( Fig 13C) and translation of this short ORF might be utilized to shunt a proportion of** ribosomes past the ORF5 AUG codon.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**After translating a very short** ORF (e.g.', 'lgd': {'word': 'After', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'short'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**<30 codons), the small subunit of the ribosome can remain associated with the message, resume scanning, and reinitiate translation **at a** downstream** AUG codon [87] .', 'lgd': {'word': '30', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'AUG'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**After translation of a short ORF,** the 40S subunit of the ribosome is not immediately competent to reinitiate, but becomes competent after scanning for some distance.', 'lgd': {'word': 'After', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'ORF'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The presence of an upstream CUG-initiated short ORF is preserved in other strains of MHV, though most (other than MHV-A59) also have a separate AUG-**initiated (albeit in a weak initiation context) short ORF that could be used to shunt even more ribosomes past the ORF5 initiation codon.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'AUG'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**These viruses also preserve a conserved absence** of AUG codons (in any reading frame) throughout ORF5 except for the 5 0 -most 44 nt (where there are from one to five AUG codons, depending on species and strain) and the very 3 0 end where the E ORF AUG is situated [88] .', 'lgd': {'word': 'These', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'absence'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In contrast, related viruses such as Betacoronavirus 1 (including** bovine coronavirus and equine coronavirus) have AUG codons spaced throughout ORF5, but produce a separate mRNA for E protein expression so that bicistronic expression from the same mRNA as ORF5 is not required [89, 90] .', 'lgd': {'word': 'In', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'including'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A long internal ORF (I) is present within the N ORF of MHV and many other coronaviruses, encoding a largely hydrophobic** polypeptide that is thought to confer a minor growth advantage to the virus [92, 93] .', 'lgd': {'word': 'A', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'hydrophobic'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**To further confirm expression of the I protein, the N coding sequence was cloned into pcDNA.3 and the mRNA translated in RRL ( Fig 14C) and immunoprecipitated (Fig 14D) with anti-N and anti-I sera, and, as a negative control,** anti-S serum.', 'lgd': {'word': 'To', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'control'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**As shown, both N protein (50 kDa) and I protein (23 kDa) were expressed from the synthetic N mRNA, with I produced at a level of** about 2% of N.', 'lgd': {'word': 'As', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Although we were unable to obtain strong evidence for the expression of I from the profiling data, a comparison of** the phasing of RPFs (a) in the region where the I ORF overlaps the N ORF, and (b) in the region of the N ORF downstream of the I termination codon, revealed in the former a slight excess of RPFs with 5 0 ends mapping to the second position of N-frame codons (blue in Fig 14E; upper panels) .', 'lgd': {'word': 'Although', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The excess is consistent with 6-12% of ribosomes translating the +1 (i.**e.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'i'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**It is possible that 5 0 leader sequence present in mRNA7 (e.**g.', 'lgd': {'word': 'It', 'seed': 'e'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**To ensure that the phasing difference was not due to a single RPF** peak (as individual peaks can sometimes map to a non-standard phase; cf.', 'lgd': {'word': 'To', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'RPF'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Fig 9C) , mean phasing was also determined in a 55-codon** sliding window and, consistent with the previous result, the proportion of RPFs mapping to the second position of N-frame codons (blue in Fig 14E; lower panels) was found to decrease abruptly around the I ORF stop codon.', 'lgd': {'word': 'Fig', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'codon'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A caveat to note is that, in MHV-A59, an upstream AUG (bold) is present in the I frame followed by a stop codon** (asterisk) prior to the \"designated\" I AUG codon (underlined;.', 'lgd': {'word': 'A', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'codon'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**.MPVAEAPL  TALVMESSRRP; both AUGs are **in a** strong cont**ext).', 'lgd': {'word': 'MPVAEAPL', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'context'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In some related virus sequences, the stop codon is replaced with a sense codon** such that I is probably initiated from the upstream AUG.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'codon'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A defective TRS for a very low** abundance HE mRNA is annotated with an open green box.', 'lgd': {'word': 'A', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'low'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In MHV-A59, the HE ORF is disrupted with a premature termination** codon (red diamond).', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'termination'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In MHV-A59, ORF4 is split by a frameshift mutation** into ORF 4b (grey) and a very short ORF4a (pale yellow).', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'mutation'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**An upstream AUU-initiated short ORF and a short out-**of-frame AUG-initiated ORF are shown in orange.', 'lgd': {'word': 'An', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'out'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A CUG codon in the same frame as the upstream ORF4, and a short out-**of-frame AUG-initiated ORF are indicated.', 'lgd': {'word': 'A', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'out'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**We have used ribosome profiling to investigate virus gene expression kinetics, relative translational efficiencies, ribosomal frameshifting, ribosome pausing, and uORF translation in cells infected with MHV, a representative of** the betacoronavirus genus of the coronavirus family of RNA viruses.', 'lgd': {'word': 'We', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**We explored a number of** data quality issues that can arise when applying ribosome profiling to the study of RNA viruses that replicate to high titres in cell culture and describe ways to bioinformatically assess and quantify potential contamination.', 'lgd': {'word': 'We', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The first was lowlevel contamination of one sample by another, a problem that** is compounded by the high levels of virus RNA synthesised in infected cells at late time points.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'that'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Although we were mindful of the possibility of such contamination, we found little evidence for it occurring as a result of** MHV infection.', 'lgd': {'word': 'Although', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The high level of phasing in our RiboSeq data (S4 Fig) allowed us to carefully assess contamination issues, and our observations reinforce the essentiality of basic data quality checks (e.**g.', 'lgd': {'word': 'The', 'seed': 'e'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Despite these challenges, the profiling and RNA-Seq analysis of MHV infection still showed itself to be a powerful tool** to investigate specific aspects of MHV replication at high resolution.', 'lgd': {'word': 'Despite', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'tool'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Up to 2.5 h p.i., there was little amplification of positive-sense RNA, whilst negative-sense RNA levels rose from undetectable to about 0.1% of** total virus RNA and host mRNA.', 'lgd': {'word': 'Up', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**An earlier study of MHV-A59 transcription using [ 32 P] pulse labelling in the presence of actinomycin D provided a similar but** slightly different order (mRNA7 > mRNA6 > mRNA5 > mRNA1/mRNA3/ mRNA4 > mRNA2) [97] although it should be noted that, while mRNAs 7 and 6 are nearly always the most abundant subgenomic transcripts, the relative abundances of the other transcripts can vary greatly between different isolates, strains and mutants of MHV [42, 97] .', 'lgd': {'word': 'An', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'but'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The translation of virus proteins was detectable at a very early** stage of infection.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'early'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Indeed, using a high MOI** infection, we were able to visualize input gRNA translation at 1 h p.i., a stage when the majority of ribosomes had not yet reached the pp1b ORF.', 'lgd': {'word': 'Indeed', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'MOI'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Using RiboSeq (HAR) data at this time point, we were able to estimate a translation rate** of 4.2 amino acids s 1 , consistent with previous estimates for mammalian systems [8] .', 'lgd': {'word': 'Using', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'rate'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**At 2.5 h p.i., when gRNA is unlikely to be a substrate for** packaging, its translational efficiency was still low, but at this point in the replication cycle, the formation of replicationtranscription complexes would preclude the massive amplification of viral RNA that takes place between 2 and 6 h p.i.', 'lgd': {'word': 'At', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'for'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**It may also be the case that the pp1a and pp1b ORFs on the gRNA are inherently poorly translatable, e.**g.', 'lgd': {'word': 'It', 'seed': 'e'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**We also observed significant amounts of RNASeq reads mapping to the N ORF region at 1 h p.**i., a time point at which negative-sense RNASeq reads were essentially absent.', 'lgd': {'word': 'We', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'p'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Further, the absence of similar amounts of RNASeq density in the leader region, together with a very low** translation efficiency, suggest that the N ORF RNA does not correspond to bona fide mRNA7.', 'lgd': {'word': 'Further', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'low'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**There has been considerable debate regarding the presence of subgenomic RNAs in coronavirus particles [99, 100] but recent analyses [101] suggest that there is a very selective** incorporation of MHV gRNA into virus particles and, although immunopurified virus particles may contain detectable amounts of mRNA7, it is minimal.', 'lgd': {'word': 'There', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'selective'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The N ORF RNA observed in our study may represent a part of** a defective viral genome with some structural similarity to DI-like RNAs.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Our data indicate that in MHV-infected cells, in addition to the \"standard\" coding sequences, ribosomes access and translate a number of** short ORFs.', 'lgd': {'word': 'Our', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The AUG-initiated uORF of the gRNA has been characterised previously and may play a role in** attenuation of translation of ORFs 1a and 1b, with a beneficial but non-essential role in coronavirus replication in cell culture [80, 103] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'in'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**We found that translation of this uORF occurred at a level similar** to that of ORF1a, reflecting its upstream position but poorer initiation context.', 'lgd': {'word': 'We', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'similar'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Interestingly, ribosomes on this uORF paused predominantly at the second codon (proline), probably as a consequence of** the restrained geometry of this amino acid in the decoding centre [83] .', 'lgd': {'word': 'Interestingly', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Other translated uORFs included a UUG-initiated** 1-codon ORF in the leader sequence, an AUU-initiated 15-codon ORF upstream of ORF4a, and a CUG-initiated 13-codon ORF upstream of ORF5.', 'lgd': {'word': 'Other', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'initiated'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The function, if any, of the first two is unknown, but we speculate that the latter uORF may play a role in** expression of the E protein, which is encoded downstream of ORF5 on mRNA5.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'in'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**E is a small, hydrophobic** viroporin that plays multiple roles during infection, including a role in virion morphogenesis [104] .', 'lgd': {'word': 'E', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'hydrophobic'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**We now hypothesize, however, that E could also be expressed via a form of** leaky scanning, where, after translating the short uORF on mRNA5, the small subunit of the ribosome remains associated with the mRNA, resumes scanning, and re-initiates at the AUG of the E ORF.', 'lgd': {'word': 'We', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**We were also able to confirm expression of the previously characterized internal (I) ORF embedded within the N gene [93] through western blotting, while analysis of profiling data (taking advantage of the phasing quality to gauge translation levels in different frames) was consistent with translation of I at a level not** more than 12% of N protein expression.', 'lgd': {'word': 'We', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'not'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The mechanism of I expression is uncertain, but leaky scanning of ribosomes that fail to initiate at the N AUG is a possibility and** the low level of I expression is consistent with such a mechanism.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'and'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Note that failure to detect I ORF initiation (and weak detection of E ORF initiation) may indicate a shortcoming of** the ribosomal profiling technique in the detection of initiation codons accessed by non-standard mechanisms.', 'lgd': {'word': 'Note', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Coronaviru**s 1** PRF sign**als have been useful models for studies of ribosomal frameshifting in vitro, both from the perspective of structure-function relationships of RNA pseudoknots, and also because they stimulate efficient frameshifting [58] .', 'lgd': {'word': 'Coronavirus', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'signals'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**We find little evidence tha**t 1** PRF** is modulated by MHV infection, with similar efficiencies observed both in infected cells and in transfected cells expressing a frameshift-reporter mRNA.', 'lgd': {'word': 'We', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'is'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In this study, we did identify a number of** strong ribosomal pauses, however, and confirmed the occurrence of pausing within nsp3 in an in vitro translation assay.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The nsp3 pausing site is located in the linker region between two modular domains of the protein, i.e.**', 'lgd': {'word': 'The', 'seed': 'e'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Ribosomal pausing as a way to** optimize protein folding has been reported increasingly in recent years [109] [110] [111] .', 'lgd': {'word': 'Ribosomal', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'to'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Together with RNASeq, the datasets provide a high-resolution** examination of MHV replication and gene expression and provides a basis for the subsequent analyses of virus-host responses (manuscript in preparation).', 'lgd': {'word': 'Together', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'resolution'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': \"**Murine 17 clone 1 (17Cl-**1) [112] and BHK-21 [C-13] (ATCC CCL-10) cells were maintained in Dulbecco's modification of Eagle's medium supplemented with 10% (vol/vol) fetal calf serum (FCS).\", 'lgd': {'word': 'Murine', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '17Cl'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**17Cl-1 cells (10 7 ) were plated in 10 cm dishes and, upon reaching 70-80% confluence, were infected with MHV-A59 at a** multiplicity of infection (MOI) of 10 PFU/cell (or 200 PFU/ cell in the \"High MOI\" experiment) in Hank\\'s balanced salt solution (HBSS) containing 50 g/ ml DEAE-dextran and 0.2% bovine serum albumin (BSA).', 'lgd': {'word': '17Cl', 'seed': 'a'}, 'rgd': {'seed': '1', 'word': '10'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Cells were rinsed with 5 ml of ice-cold PBS, the dishes were submerged in a reservoir of** liquid nitrogen for 10 s and then transferred to dry ice and 400 l of lysis buffer [20 mM Tris-HCl pH 7.5, 150 mM NaCl, 5 mM MgCl 2 , 1 mM DTT, 1% Triton X-100, 100 g/ml cycloheximide and 25 U/ml TURBO DNase (Life Technologies)] dripped onto the cells.', 'lgd': {'word': 'Cells', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The cells were scraped extensively to ensure lysis, collected and triturated with a 26-G** needle ten times.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'G'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The methodologies employed were based on the original protocols of Ingolia and colleagues [7, 114] , except ribosomal RNA contamination was removed by treatment with duplex-specific nuclease (DSN) and library amplicons were constructed using a small RNA** cloning strategy [115] adapted to Illumina smallRNA v2 to allow multiplexing.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'RNA'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The methods used were as described [16] , with minor modifications for the analysis of ribosomal pausing at the MH**V 1** PRF sig**nal, namely a broader range of RPFs, migrating between 28 and 80 nt, were harvested prior to amplicon construction, and longer PCR amplicons of~150-206 bp were gel purified.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'signal'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Amplicon libraries were deep sequenced using an Illumina HiSeq 2000 platform (repeat 1 samples at** the Wellcome Trust Centre for Human Genetics-Oxford Genomics Centre; repeat 2, MOI 200, and long read samples at the Beijing Genomics Institute).', 'lgd': {'word': 'Amplicon', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'at'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Trimmed reads were mapped first to Mus musculus rRNA (GenBank accession numbers NR_003278, NR_003279, NR_003280, NR_030686, NR_046233 and GU372691), followed by the MHV genome (GenBank accession number AY700211.1) and** subsequently Mus musculus mRNA, ncRNA and genomic DNA databases.', 'lgd': {'word': 'Trimmed', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'and'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Reads were mapped using bowtie version 1 [116]** with parameters -v 2 -best (i.e.', 'lgd': {'word': 'Reads', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '116'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The order of mapping was tested to check that virus-derived reads were not lost accidentally due to mis-mapping to host RNA, or vice versa; a slight reduction** (~0.05%) in virus-derived reads was observed only on mapping to the entire host genome (gDNA) and thus mapping to virus RNA and host mRNA was considered to be specific.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'reduction'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**To compare read densities between CDSs and 3 0 UTRs (S9 Fig and S10 Fig) , we used reads whose 5 0 end offset by +12 nt (i.e.**', 'lgd': {'word': 'To', 'seed': 'e'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The presence of transcript isoforms with 3 0 UTRs shorter than the annotated (!300 nt) 3 0 UTRs leads to a modest underestimation** of the actual 3 0 UTR density.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'underestimation'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Transcripts with !20 RPFs with 5 0 ends mapping between 30 and +15 relative to the annotated initiation codon were used, and histograms of 5 0 end positions for individual transcripts were down-weighted by the number of RPFs mapping to this region before summing over the different transcripts (i.e.**', 'lgd': {'word': 'Transcripts', 'seed': 'e'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**a weighted sum** of \"highly expressed\" mRNAs, [7] ).', 'lgd': None, 'rgd': {'seed': 'a', 'word': 'sum'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Plots showing reads mapped to the MHV genome (Figs 1, 2,** 7, 9 , 11, 12A, 13 and 14 and S11 Fig) show histograms of the positions to which the 5 0 ends of reads map, with a +12 nt offset to indicate (for RPFs) the approximate P-site.', 'lgd': {'word': 'Plots', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '2'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**(More precisely, the +12 nt offset means that RPFs whose 5 0 end aligns to the first position **of a** codon** are mapped to the first nucleotide of the P-site codon, and RPFs whose 5 0 end aligns to the third position of a codon are mapped to the last nucleotide of the codon preceding the P-site codon.)', 'lgd': {'word': 'More', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'are'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**We used a similar procedure** to calculate RNASeq densities for each inter-TRS region (Fig 4, RNASeq) , with the inter-TRS regions (prior to the 30 nt 3 0 buffer) being 72 to 21748 (mRNA1), 21754 to 23923 (mRNA2), 23929 to 27936 (mRNA3), 27942 to 28319 (mRNA4), 28325 to 28959 (mRNA5), 28965 to 29656 (mRNA6), and 29662 to 31335 (mRNA7).', 'lgd': {'word': 'We', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'procedure'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Frameshifting efficiencies (Fig 8) were calculated using reads whose 5 0 end offset by +12 nt (i.e.**', 'lgd': {'word': 'Frameshifting', 'seed': 'e'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**These coordinates leave a 150 nt** buffer after the ORF1a initiation codon (nt 211), before the frameshift site (nt 13602), after the ORF1a termination codon (nt 13623) and before the ORF1b termination codon (nt 21746), respectively.', 'lgd': {'word': 'These', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'nt'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Phasing distributions in the N and I ORFs (Fig 14E) were calculated with respect to the N reading frame, using reads whose 5 0 end offset by +12 nt (i.e.**', 'lgd': {'word': 'Phasing', 'seed': 'e'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**For the analysis of RiboSeq and RNASeq count variability within ORF1a (Fig 9A) , counts were first smoothed with a 3-nt** running mean filter and then the fold-change relative to mean was calculated using reads whose 5 0 end mapped between nt 211 (the start of ORF1a) and nt 13572 (30 nt 5 0 of the frameshift site).', 'lgd': {'word': 'For', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'nt'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**For the above analyses, virus reads with discontiguous mappings to the MHV genome (i.e.**', 'lgd': {'word': 'For', 'seed': 'e'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**We then pooled the remaining unmapped reads with the reads that mapped to the virus genome and, for each library, searched this set of reads for the query sequence UUUAAAUCUAA (AY700211.1 nt 55** to 65; 5 0 -adjacent to the leader TRS).', 'lgd': {'word': 'We', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '55'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**These criteria were motivated by previous data indicating that, in leader/ body chimeras, nucleotides up to and including UUUAAAUCUAA are templated by the leader, nucleotides at +1 and +2** may be templated by leader or genome, and nucleotides at +3 and above are templated by the genome sequence [41] .', 'lgd': {'word': 'These', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '2'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Read counts were normalized for library size as above, and for CDS length according to the sum of all coding exon fragment lengths for a given gene** ID in the genes.gtf file.', 'lgd': {'word': 'Read', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'gene'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**While this is likely to have only a modest effect** on RiboSeq/RNASeq translation efficiencies (Fig 6, y-axis) it will tend to result in underestimates for RNASeq RPKM values (Fig 6, x-axis) .', 'lgd': {'word': 'While', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'effect'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The MHV frameshift signal, and the N, nsp3 and nsp6 protein coding sequences were amplified using specific oligonucleotides (S4 Table) and cDNA derived from 17 Cl-1 cells infected** with MHV-A59 at an MOI 10 and harvested at 8 h p.i.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'infected'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**BamH1-XhoI-digested PCR fragments were cloned into pcDNA 3.1 (+) (Life** Technologies) previously digested with BamH1-XhoI.', 'lgd': {'word': 'BamH1', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'Life'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In pPS0 plasmids, PCR reactions were carried out using the pcDNA.3 nsp3 plasmid as a template and** cloned into a digested XhoI/PvuII-pPS0 plasmid.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'and'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**For all pcDNA.3 and pPS0 constructs, a \"pause** control\" was also generated in which a UAA stop codon was introduced to generate a protein whose size corresponded to that produced by the predicted ribosomal pause.', 'lgd': {'word': 'For', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'pause'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Frameshifting assays in tissue culture 17 Cl-1 and BHK-**21 cells were seeded in dishes of a 24-well plate and grown for 16 h until 80% confluence was reached.', 'lgd': {'word': 'Frameshifting', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'BHK'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Plasmids were transfected using a commercial liposome** method (TransIT-LT1, Mirus).', 'lgd': {'word': 'Plasmids', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'liposome'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**and reporter gene expression was determined using a dual-luciferase** assay system kit (Promega).', 'lgd': {'word': 'and', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'luciferase'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**These were blocked for 30-60 min with 5% powdered milk (Marvel) in PBST [137 mM NaCl, 2.7 mM KCl, 10 mM Na 2 HPO 4 , 1.5 mM** KH 2 PO 4 (pH 6.7), and 0.1% Tween 20] and probed with mouse monoclonal antibodies raised against nsp9 (AM08450PU-N, Acris Antibodies, Inc, 1:500 in Marvel-PBST), N (1:1,000), S (1:500), GAPDH (G8795, Sigma-Aldrich, 1:20,000) or a polyclonal rabbit anti-I (1:1,000, a kind gift of Prof. P. S. Masters, Wadsworth Center, New York State Department of Health).', 'lgd': {'word': 'These', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'mM'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**RNAs were recovered by a single extraction** with phenol-chloroform (1:1 vol/vol) followed by ethanol precipitation.', 'lgd': {'word': 'RNAs', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'extraction'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Remaining unincorporated nucleotides were removed by gel filtration through a NucAway spin** column (Ambion).', 'lgd': {'word': 'Remaining', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'spin'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A typical reaction mixture had a volume of** 10 l and was composed of 90% (vol/ vol) RRL, 20 M amino acids (lacking methionine), and 0.2 MBq [ 35 S]-methionine.', 'lgd': {'word': 'A', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In ribosomal pausing assays, conditions were the same except that the reaction mixture had a volume of** 40 l and the translational inhibitor edeine was added 3 min after the start of the reaction in order to obtain synchronous initiation (final concentration, 5 M).', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Aliquots of 1.5 l** were withdrawn from the translation reaction mixture at specified intervals and mixed with an equal volume of EDTA/RNase A mixture, as above.', 'lgd': {'word': 'Aliquots', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'l'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Dried gels were exposed to a Carestream Kodak** Biomax MR film (Sigma-Aldrich) and scanned.', 'lgd': {'word': 'Dried', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'Kodak'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Chimeric reads utilizing the leader TRS were identified by searching for all reads containing the sequence UUUAAAUCUAA (AY700211.1 nt 55** to 65), and classified according to the identity of the following nucleotides at positions +3 to +17.', 'lgd': {'word': 'Chimeric', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '55'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Nucleotides at positions +1 to +2** in the RNASeq read are listed in column 3.', 'lgd': {'word': 'Nucleotides', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '2'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Also, the 5 nucleotides preceding these in the genome are listed in column 1.**', 'lgd': {'word': 'Also', 'seed': '1'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The numbers of junction/body chimeric reads containing each sequence are given in column 6 (repeat 1) and** column 7 (repeat 2).', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'and'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Only sequences with three or more occurrences in repeat 1 and ten** or more occurrences in repeat 2 are shown.', 'lgd': {'word': 'Only', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'ten'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Alpha-ketoamides as broad-spectrum inhibitors of coronavirus and enterovirus replication Structure-based design, synthesis, and activity assessment"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The main protease of coronaviruses and the 3C protease of enteroviruses share a similar active-**site architecture and a unique requirement for glutamine in the P1 position of the substrate.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'active'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In order to obtain nearequipotent, broad-spectrum antivirals against alphacoronaviruses, betacoronaviruses, and enteroviruses, we pursued structure-based design of peptidomimetic a-ketoamides as** inhibitors of main and 3C proteases.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'as'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**We chose the chemical class of peptidomimetic a-ketoamides to** assess the feasibility of achieving antiviral drugs targeting coronaviruses and enteroviruses with near-equipotency.', 'lgd': {'word': 'We', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'to'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Here we describe the structure-based design, synthesis, and evaluation of inhibitory activity of a series of** compounds with broad-spectrum activities afforded by studying the structure-activity relationships mainly with respect to the P2 position of the peptidomimetics.', 'lgd': {'word': 'Here', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Our efforts to design novel a-ketoamides as** broad-spectrum inhibitors of coronavirus M pro s and enterovirus 3C pro s started with a detailed analysis of the following crystal structures of unliganded target enzymes: SARS-CoV M pro (ref.', 'lgd': {'word': 'Our', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'as'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**25-27; PDB entries 1UJ1, 2BX3, 2BX4); bat coronavirus HKU4 M pro as a surrogate for** the closely related MERS-CoV protease (our unpublished work (Ma, Xiao et al.', 'lgd': {'word': '25', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'for'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**During the course of the present study, we determined crystal structures of a number of** lead a-ketoamide compounds in complex with SARS-CoV M pro , HCoV-NL63 M pro , and CVB3 3C pro , in support of the design of improvements in the next round of lead optimization.', 'lgd': {'word': 'During', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The structural foundation of these was elucidated in detail in a subproject involving** the M pro of HCoV NL63; because of its volume, this work will be published separately (Zhang et al., in preparation) and only some selected findings are referred to here.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'involving'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**1 One of** the reasons is that despite the devastating consequences of SARS for the affected patients, the development of an antiviral drug against this virus would not be commercially viable in view of the fact that the virus has been rapidly contained and did not reappear since 2004.', 'lgd': None, 'rgd': {'seed': '1', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**As a result, we were empty-handed when the Middle-East respiratory syndrome coronavirus (MERS-CoV), a** close relative of SARS-CoV, emerged in 2012.', 'lgd': {'word': 'As', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'we'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The potential for human-to-human transmission of MERS-CoV has been impressively demonstrated by the 2015 outbreak in South Korea, where 186 cases could be traced back to a single infected** traveller returning from the Middle East.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'infected'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**4 SARS-like coronaviruses are still circulating in bats in China, [5] [6] [7] [8] from where they may spill over into the human population; this is probably what caused the current outbreak of atypical pneumonia in Wuhan, which is linked to a seafood and** animal market.', 'lgd': {'word': '4', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'and'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In spite of the considerable threat posed by SARS-CoV and related viruses, as well as by MERS-CoV, it is obvious that the number of cases so far does not warrant the commercial development of an antiviral drug targeting MERS-and SARS-CoV even if a projected steady** growth of the number of MERS cases is taken into account.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'steady'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A possible solution to the problem could be the development of broadspectrum antiviral drugs that are directed against the major viral protease, a target that** is shared by all coronavirus genera as well as, in a related form, by members of the large genus Enterovirus in the picornavirus family.', 'lgd': {'word': 'A', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'that'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**While both of them have a single-stranded** RNA genome of positive polarity, that of enteroviruses is very small (just 7 -9 kb) whereas coronaviruses feature the largest RNA genome known to date (27 -34 kb) .', 'lgd': {'word': 'While', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'stranded'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Nevertheless, a related feature** shared by these two groups of viruses is their type of major protease, 21 which in the enteroviruses is encoded by the 3C region of the genome (hence the protease is designated 3C pro ).', 'lgd': {'word': 'Nevertheless', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'feature'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Similar to the enteroviral 3C pro , it is a cysteine protease** in the vast majority of cases and has therefore also been called \"3C-like protease\" (3CL pro ).', 'lgd': {'word': 'Similar', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'protease'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The first crystal structure of a CoV M** pro or 3CL pro (ref.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'M'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**22) revealed that two of the three domains of the enzyme together resemble the chymotrypsin-like fold of the enteroviral 3C pro , but there is an additional a-helical domain** that is involved in the dimerization of the protease (Fig.', 'lgd': {'word': '22', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'domain'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**1B) functions as a** monomer.', 'lgd': {'word': '1B', 'seed': 'a'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Further, the enteroviral 3C pro features a classical Cys.**..His...Glu/Asp catalytic triad, whereas the CoV M pro only has a Cys...His dyad.', 'lgd': {'word': 'Further', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'Cys'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': \"**22 Yet, there are a number of** common features shared between the two types of proteases, in particular their almost absolute requirement for Gln in the P1 position of the substrate and space for only small amino-acid residues such as Gly, Ala, or Ser in the P1' position, encouraging us to explore the coronaviral M pro and the enteroviral 3C pro as a common target for the design of broad-spectrum antiviral compounds.\", 'lgd': {'word': '22', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The fact that there is no known human protease with a specificity for** Gln at the cleavage site of the substrate increases the attractivity of this viral target, as there is hope that the inhibitors to be developed will not show toxicity versus the host cell.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'for'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Indeed, neither the enterovirus 3C pro inhibitor rupintrivir, which was developed as a treatment of** the common cold caused by HRV, nor the peptide aldehyde inhibitor of the coronavirus M pro that was recently demonstrated to lead to complete recovery of cats from the normally fatal infection with Feline Infectious Peritonitis Virus (FIPV), showed any toxic effects on humans or cats, respectively.', 'lgd': {'word': 'Indeed', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**23, 24 Figure 1: Crystal** structures of SARS-CoV main protease (M pro , ref.', 'lgd': {'word': '23', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'Crystal'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The coronavirus M pro is a homodimer, with** each monomer comprising three domains.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'with'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**As the proteases targeted in our study all specifically cleave the peptide bond following a P1-glutamine** residue (HCoV-NL63 M pro uniquely also accepts P1 = His at the Nsp13/Nsp14 cleavage site 31 ), we decided to use a 5-membered ring (g-lactam) derivative of glutamine (henceforth called GlnLactam) as the P1 residue in all our aketoamides (see Scheme 1).', 'lgd': {'word': 'As', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'glutamine'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This moiety has been found to be a good mimic** of glutamine and enhance the power of the inhibitors by up to 10-fold, most probably because compared to the flexible glutamine side-chain, the more rigid lactam leads to a reduction of the loss of entropy upon binding to the target protease.', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'mimic'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': \"**29, 32 Our synthetic efforts therefore aimed at optimizing the substituents at the P1', P2, and P3 positions of the a-**ketoamides.\", 'lgd': {'word': '29', 'seed': 'a'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Synthesis (Scheme 1) started** with the dianionic alkylation of N-Boc glutamic acid dimethyl ester with bromoacetonitrile.', 'lgd': {'word': 'Synthesis', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'started'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**As expected, this alkylation occurred in a highly stereoselective** manner, giving 1 as the exclusive product.', 'lgd': {'word': 'As', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'stereoselective'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In the following step, the cyano group of 1 was subjected** to hydrogenation.', 'lgd': {'word': 'In', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'subjected'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**On the other hand, the amidation of acyl chloride and a-amino acid** methyl ester afforded the intermediates 4, which gave rise to the acids 5 via alkaline hydrolysis.', 'lgd': {'word': 'On', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'acid'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Then, the a-hydroxyamides 10** were prepared by removing the acetyl group of compounds 9.', 'lgd': {'word': 'Then', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': '10'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In the final step, the oxidation of the exposed alcohol group in compounds 10 generated our target a-ketoamides 11.**', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': '11'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The inhibitory potencies of candidate a-ketoamides were** evaluated against purified recombinant SARS-CoV M pro , HCoV-NL63 M pro , CVB3 3C pro , and EV-A71 3C pro .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'were'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The most potent compounds were further tested against viral replicons and against SARS-CoV, MERS-CoV, or a whole range** of enteroviruses in cell culture-based assays (Tables 1 -3 and Supplementary Table 1', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'range'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**To enable the rapid and biosafe screening of antivirals against corona-and enteroviruses, a non-infectious,** but replication-competent SARS-CoV replicon was used 33 along with subgenomic replicons of CVB3 34 and EV-A71 (kind gift of B. Zhang, Wuhan, China).', 'lgd': {'word': 'To', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'infectious'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': \"**The initial compound to be designed and synthesized was 11a, which carries a cinnamoyl N-**cap in the P3 position, a benzyl group in P2, the glutamine lactam (GlnLactam) in P1, and benzyl in P1' (Table  1) .\", 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'N'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This compound showed good to mediocre activities against recombinant SARS-CoV M pro (IC50 = 1.95 M;** for all compounds, see Tables 1 -3 for standard deviations), CVB3 3C pro (IC50 = 6.6 M), and EV-A71 3C pro (IC50 = 1.2 M), but was surprisingly completely inactive (IC50 > 50 M) against HCoV-NL63 M pro .', 'lgd': {'word': 'This', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'M'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In virus-infected cell cultures, the results obtained were also good to mediocre (Table 3) : SARS-CoV (EC50 = 5.8 M in Vero E6 cells), MERS-CoV (EC50 = 0.0047 M in Huh7 cells), HCoV 229E (EC50 = 11.8 M in Huh7 cells), or a host of** enteroviruses (EC50 = 9.8 M against EV-A71 in RD cells; EC50 = 0.48 M against EV-D68 in HeLa Rh cells; EC50 = 5.6 M against HRV2 in HeLa Rh cells).', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The reason for this difference is that the oxygen atom of the thiohemiketal accepts a hydrogen bond** from the catalytic His40 in the CVB3 protease, rather than from the main-chain amides of the oxyanion hole as in the SARS-CoV and HCoV-NL63 enzymes (Fig.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'bond'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**It is remarkable that we succeeded in obtaining a crystal structure** of compound 11a in complex with the HCoV-NL63 M pro , even though it has no inhibitory effect on the activity of the enzyme (IC50 > 50 M), (Fig.', 'lgd': {'word': 'It', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'structure'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A similar observation has been made in one of our previous studies, where we were able to determine the crystal structure of a complex between** the inactive Michaelacceptor compound SG74 and the EV-D68 3C pro (ref.', 'lgd': {'word': 'A', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'between'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Note that the S2 pocket is also covered by a \"lid\"** centred around Thr47.', 'lgd': {'word': 'Note', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'lid'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The S2 site is large and not covered by a \"lid\"**.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'lid'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The crystal structures of SARS-CoV M pro , HCoV-NL63 M pro , and CVB3 3C pro in complex with 11a revealed a fundamental difference** between the S2 pockets of the coronavirus proteases and the enterovirus proteases: The cavities are covered by a \"lid\" in the former but are open to one side in the latter (Fig.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'difference'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The SARS-CoV enzyme features the largest S2 pocket, with a volume of** 252  3 embraced by the residues (Gln189, His41) defining the side-walls of the pocket in the ligand-free enzyme, as calculated by using Chimera, 36 followed by the CVB3 3C pro S2 pocket with about 180  3 (space between Thr130 and His40).', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The HCoV-NL63 M pro has by far the smallest S2 pocket of the three enzymes, with a free space** of only 45  3 between Pro189 and His41, according to Chimera.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'space'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In agreement with these observations, a good fit** is observed between the P2 benzyl group of 11a and the S2 subsite of the SARS-CoV M pro as well as that of the CVB3 3C pro (Fig.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'fit'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Thus, the properties of our target proteases with respect to the S2 pocket were defined at this point as \"small\" and \"covered by a lid\" for** HCoV-NL63 M pro , \"large\" and \"covered\" for SARS-CoV M pro , and \"large\" and \"open\" for CVB3 3C pro .', 'lgd': {'word': 'Thus', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'for'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**To explore the sensitivity of the S2 pocket towards a polar substituent** in the para position of the benzyl group, we synthesized compound 11m carrying a 4-fluorobenzyl group in P2.', 'lgd': {'word': 'To', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'substituent'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**From this, we concluded that the introduction of the polar fluorine atom is not compatible with the geometry of the S2 pocket of SARS-CoV M pro , whereas the fluorine can accept a hydrogen bond** from Arg39 in EV-A71 3C pro (ref.', 'lgd': {'word': 'From', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'bond'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In SARS-CoV M pro , however, the carbonyl groups of residues 186 and 188 might lead to a repulsion of** the fluorinated benzyl group.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**https://doi.org/10.1101/2020.02.10.936898 doi: bioRxiv preprint Table 2 : a-ketoamide-induced** inhibition of subgenomic RNA synthesis using replicons in a cell-based assay (EC50, M) 1  2  3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60 author/funder.', 'lgd': {'word': 'https', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'induced'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The stereochemistry of the thiohemiketal is S, as the group accepts a hydrogen bond** from His41, whereas the amide keto group accepts three H-bonds from the oxyanion hole (residues 145 -147).', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'bond'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**D. The crystal structure of 11f in complex with HCoV-NL63 M pro shows that this short (inactive) compound lacking a P3 residue** has its P2-Boc group inserted into the S2 pocket of the protease.', 'lgd': {'word': 'D', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'residue'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This resulted in improved activities against SARS-CoV M pro (IC50 = 0.33 M) and in a very good** activity against HCoV-NL63 M pro (IC50 = 1.08 M, compare with the inactive 11a).', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'good'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Our interpretation of this result is that the smaller P2-isobutyl substituent of 11n can still interact with the \"lid\" (in particular, Met49) of the SARS-CoV M pro S2 site, but is unable to reach the \"back-wall\" of the EV-A71 3C pro pocket and thus, in the absence of a \"lid\"**, cannot generate sufficient enthalpy of binding.', 'lgd': {'word': 'Our', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'lid'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**We will see from examples to follow that this trend persists among all inhibitors with a smaller P2** substituent: Even though the SARS-CoV M pro S2 pocket has a larger volume than that of the enterovirus 3C pro , the enzyme can be efficiently inhibited by compounds carrying a small P2 residue that makes hydrophobic interactions with the lid (Met49) and floor (Met165) residues.', 'lgd': {'word': 'We', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'P2'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This is not only a consequence of** the smaller size of the isobutyl group compared to the benzyl group, but also of its larger conformational flexibility, which allows a better fit to the binding site.', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The decreased activity in case of SARS-CoV M pro and the total inactivity against HCoV-NL63 M pro indicate that the n-butyl chain is too long for the S2 pocket of these proteases, whereas the slight improvement against EV-A71 3C pro and CVB3 3C pro is probably a consequence of** the extra space that is available to long and flexible substituents because of the lack of a lid covering the enterovirus 3C pro pocket.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**As the n-butyl substituent in P2 of 11o was obviously too long, we next synthesized a derivative with** the shorter propargyl (ethinylmethyl) as the P2 residue (compound 11p).', 'lgd': {'word': 'As', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'with'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Obviously, the P2 side-chain requires a methylene group** in the b-position in order to provide the necessary flexibility for the substituent to be embedded in the S2 pocket.', 'lgd': {'word': 'Obviously', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'group'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The cyclohexylmethyl derivative 11r exhibited IC50 = 0.7 M against SARS-CoV M pro , 12.3 M against HCoV-NL63 M pro , 1.7 M** against EV-A71 3C pro , and 0.9 M against CVB3 3C pro .', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'M'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Thus, the replace- 1  2**  3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60 author/funder.', 'lgd': {'word': 'Thus', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '2'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**https://doi.org/10.1101/2020.02.10.936898 doi: bioRxiv preprint ment of the phenyl group by the cyclohexyl group led to a significant improvement** of the inhibitory activity against the recombinant SARS-CoV M pro and to a dramatic improvement in case of CVB3 3C pro .', 'lgd': {'word': 'https', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'improvement'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Even for the HCoV-NL63 M pro , against which 11a was completely inactive, greatly improved albeit still weak activity was observed (Table 1** ).', 'lgd': {'word': 'Even', 'seed': '1'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In the viral replicons, 11r performed very well, with EC50 = 0.8 -0.9 M for the EV-A71 replicon, 0.45 M for CVB3, and 1.4 M** for SARS-CoV (Table 2 ).', 'lgd': {'word': 'In', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'M'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Against HCoV 229E in Huh7 cells, the EC50 was surprisingly low (1.8 M)**.', 'lgd': {'word': 'Against', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'M'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Even in Vero cells, EC50 against MERS-CoV was 5 M, and the EC50 against SARS-CoV in Vero E6 cells was 1.8 -2.**1 M, i.e.', 'lgd': {'word': 'Even', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '2'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The next substituent to be tried was cyclopropylmethyl (compound 11s, which showed good activities against SARS-CoV M pro (IC50 = 0.24 M) and HCoV-NL63 M pro (1.4 M)**, but poor values against EV-A71 3C pro (IC50 = 18.5 M) and CVB3 3C pro (IC50 = 4.3 M) (Table 1) .', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'M'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**By contrast, the compound inhibited HCoV 229E and MERS-CoV in Huh7 cells with EC50 of 1.3 and** 0.08 M, respectively.', 'lgd': {'word': 'By', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'and'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This plasticity is expressed in a conformational change** of residue Gln189, both in the main chain and in the side-chain.', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'change'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The main-chain conformational change is connected with a flip of** the peptide between Gln189 and Thr190.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**37 As a consequence of** these changes, the sidechain oxygen of Gln189 can accept a 2.54- hydrogen bond from the main-chain NH of the P2 residue in the 11s complex (see Fig.', 'lgd': {'word': '37', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In Huh7 cells infected with MERS-CoV, this compound exhibited EC50 = 0.1 M (but** 9.8 M in Vero cells), whereas EC50 was 7.0 M against SARS-CoV in Vero E6 cells.', 'lgd': {'word': 'In', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'but'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The CC50 of 11t in HeLa cells was 65 M, i.e.**', 'lgd': {'word': 'The', 'seed': 'e'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Obviously, this substituent was still a bit too** small for the enterovirus proteases, so as the next step, we tested P2 = cyclopentylmethyl (compound 11u).', 'lgd': {'word': 'Obviously', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'too'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This turned out to be the one compound with acceptable IC50 values against all tested enzymes: 1.3 M** against SARS-CoV M pro , 5.4 M against HCoV-NL63 M pro , 4.7 M against EV-A71 3C pro , and 1.9 M against CVB3 3C pro ( Table 1 ).', 'lgd': {'word': 'This', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'M'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In Huh7 cells infected with HCoV 229E or MERS-CoV, 11u showed EC50 = 2.5 or 0.03 M (11.1 M for** MERS-CoV in Vero cells), while EC50 was 4.9 M against SARS-CoV in Vero E6 cells (Table 3) .', 'lgd': {'word': 'In', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'for'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In other words, the nearly equipotent 11u is indeed a** compromise.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This compound exhibited submicromolar IC50 values against CVB3 3C pro and SARS-CoV M pro , and IC50 = 1.7 M** against EV-A71 3C pro (Table 1) , as well as similarly low EC50 values in the replicons of these viruses (Table 2 ).', 'lgd': {'word': 'This', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'M'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In Huh7 cells infected with MERS-CoV, the performance of this compound was excellent, with EC50 = 0.0004 M, and even against HCoV 229E in Huh7 cells and SARS-CoV in Vero E6 cells, EC50 values of 1.8 and** 2.1 M, respectively, were observed (Table 3) .', 'lgd': {'word': 'In', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'and'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The only concern is the activity of the compound against EV-A71 in RD cells, for which the EC50 value was 3.7 M, resulting in too low a therapeutic index.**', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'index'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Preliminary pharmacokinetics tests with the compound in mice did not indicate a toxicity problem** (to be published elsewhere).', 'lgd': {'word': 'Preliminary', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'problem'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**We describe here the structure-based design, the synthesis, and the assessment of capped dipeptide a-ketoamides that** target the main protease of alpha-or betacoronaviruses as well as the 3C protease of enteroviruses.', 'lgd': {'word': 'We', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'that'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Through crystallographic analyses of a total of** six inhibitor complexes of three different proteases in this study, we found the a-ketoamide warhead (-CO-CO-NH-) to be sterically more versatile than other warheads such as Michael acceptors (-CH=CH-CO-) and aldehydes (-CH=O), because it features two acceptors for hydrogen bonds from the protein, namely the a-keto oxygen and the amide oxygen, whereas the other warheads have only one such acceptor.', 'lgd': {'word': 'Through', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In the various complexes, the hydroxy group (or oxyanion) of the thiohemiketal that is formed by the nucleophilic attack of the activesite cysteine residue onto the a-keto carbon,** can accept one or two hydrogen bonds from the main-chain amides of the oxyanion hole.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'carbon'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In addition, the amide oxygen of the inhibitor accepts a hydrogen bond** from the catalytic His side-chain.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'bond'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**We have previously observed a similar difference** in case of aldehyde inhibitors, where the single interaction point -the oxyanion of the thiohemiacetal -can accept a hydrogen bond either from the oxyanion hole or from the catalytic His side-chain, 37 resulting in different stereochemistry of the thiohemiacetal carbon.', 'lgd': {'word': 'We', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'difference'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': \"**In addition to better matching the H-bonding donor/acceptor properties of the catalytic center through offering two hydrogen-bond acceptors instead of one, a-ketoamides have** another big advantage over aldehydes and a,b-unsaturated esters (Michael acceptors) in that they allow easy extension of the inhibitors to probe the primed specificity subsites beyond S1', although this has so far rarely been explored (e.g., ref.\", 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'have'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The most prominent a-ketoamide drugs** are probably telaprivir and boceprivir, peptidomimetic inhibitors of the HCV NS3/4A protease, 39, 40 which have helped revolutionize the treatment of chronic HCV infections.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'drugs'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**For viral cysteine proteases, a-ketoamides have** only occasionally been described as inhibitors and few systematic studies have been carried out.', 'lgd': {'word': 'For', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'have'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A number of capped dipeptidyl a-ketoamides have** been described as inhibitors of the norovirus 3C-like protease.', 'lgd': {'word': 'A', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'have'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Although we did not include the norovirus 3CL pro in our study, expanding the target range of our inhibitors to norovirus is probably a realistic undertaking.**', 'lgd': {'word': 'Although', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'undertaking'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**42 published a series of** a-ketoamides as inhibitors of the EV-A71 3C pro .', 'lgd': {'word': '42', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Interestingly, they also reported that a six-membered** d-lactam in the P1 position led to 2 -3 times higher activities, compared to the fivemembered g-lactam.', 'lgd': {'word': 'Interestingly', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'membered'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': \"**43 described a series of** five a-ketoamides with P1' = cyclopropyl that showed submicromolar activity against EV-D68 and two HRV strains.\", 'lgd': {'word': '43', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Occasionally, individual a-ketoamides have** been reported in the literature as inhibitors of both the enterovirus 3C protease and the coronavirus main protease.', 'lgd': {'word': 'Occasionally', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'have'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A single capped dipeptidyl a-ketoamide, Cbz-**Leu-GlnLactam-CO-CO-NH-iPr, was described that inhibited the recombinant transmissible gastroenteritis virus (TGEV) and SARS-CoV M pro s as well as human rhinovirus and poliovirus 3C pro s in the one-digit micromolar range.', 'lgd': {'word': 'A', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'Cbz'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': 'Among **a series of** aldehydes, Prior et al.', 'lgd': None, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**45 described the capped tripeptidyl a-ketoamide Cbz-**1-naphthylalanine-Leu-GlnLactam-CO-CO-NH-iPr, which showed IC50 values in the 3-digit nanomolar range against HRV 3C pro and SARS-CoV M pro , as well as EC50 values of 0.03 M against HRV18 and 0.5 M against HCoV 229E in cell culture.', 'lgd': {'word': '45', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'Cbz'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**For compounds with warheads other than a-ketoamides, in-**vitro activity against both corona-and enteroviruses has also occasionally been reported.', 'lgd': {'word': 'For', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'in'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**44 reported a dipeptidyl aldehyde** and its bisulfite adduct, both of which exhibited good inhibitory activities against the isolated 3C proteases of human rhinovirus and poliovirus as well as against the 3C-like proteases of a number of coronaviruses, but antiviral activities in cell culture against EV-A71 were poor (EC50 >10 M), again most probably due to P2 being isobutyl (leucine).', 'lgd': {'word': '44', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'aldehyde'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In the SARS-CoV M pro , the S2 subsite is a deep hydrophobic** pocket that is truly three-dimensional in shape: the \"walls\" of the groove are formed by the polypeptide main chain around residues 186 -188 as well as by the side-chains of His41 (of the catalytic dyad) and Gln189, whereas the \"floor\" is formed by Met165 and the \"lid\" by residues 45 -51, in particular Met49.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'hydrophobic'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**37 Because the pocket offers so many opportunities for interaction and features a pronounced plasticity,** P2 substituents such as isobutyl (from Leu), which are too small to fill the pocket entirely, can still generate sufficient binding enthalpy.', 'lgd': {'word': '37', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'plasticity'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In the S2 pocket of the HCoV-NL63 M pro , Gln189 is replaced by proline and this change is accompanied by a significant loss** of flexibility; whereas the side-chain of Gln189 of SARS-CoV M pro is found to accommodate its conformation according to the steric requirements of the P2 substituent, the proline is less flexible, leading to a much smaller space at the entrance to the pocket.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'loss'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**As a consequence, a** P2benzyl substituent is hindered from penetrating deeply into the pocket, whereas the smaller and more flexible isobutyl group of P2-Leu is not.', 'lgd': {'word': 'As', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'a'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Finally, in the 3C pro s of EV-A71 and CVB3, the S2 pocket lacks a lid, i.**e.', 'lgd': {'word': 'Finally', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'i'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': 'As **a consequence, it** offers less interaction points for P2 substituents of inhibitors, so that such substituents must reach the \"back-wall\" of the pocket (formed by Arg39, Asn69, Glu71) in order to create sufficient binding energy.', 'lgd': None, 'rgd': {'seed': 'a', 'word': 'it'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**When we introduced a fluoro substituent** in the para position of the P2-benzyl group of our lead compound, 11a, we observed good activity against the enterovirus 3C pro s but complete inactivity against the coronavirus M pro s (see Table 1 , compound 11m).', 'lgd': {'word': 'When', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'substituent'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This is easily explained on the basis of the crystal structures: In the enterovirus 3C pro s, the fluorine can accept a hydrogen bond** from Arg39 (ref.', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'bond'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In agreement with this, rupintrivir (which has P2 = p-fluorobenzyl) is a good inhibitor** of the enteroviral 3C pro s, 46 but not of the coronaviral main proteases, as we predicted earlier.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'inhibitor'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Whereas the inhibitor exhibits excellent activity against MERS-CoV when Huh7 cells are the host cells (400 pM), the inhibitory activity is weaker by a factor of** up to 12,500 when Vero cells are used (EC50 = 5 M).', 'lgd': {'word': 'Whereas', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**On the other hand, 11r exhibits excellent anti-MERS-CoV activity in human Calu3 lung cells, i.e.**', 'lgd': {'word': 'On', 'seed': 'e'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**in the primary target cells where the compound will have to act in a therapeutic setting** (A. Kupke, personal communication).', 'lgd': {'word': 'in', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'setting'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**As we tested antiviral activity against SARS-CoV exclusively in Vero cells, the EC50 values determined for our compounds against this virus are in the one-digit micromolar range or higher; the best is again compound 11r with EC50 = 2.1** M.', 'lgd': {'word': 'As', 'seed': '1'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**We observed a good correlation** between the inhibitory activity of the designed compounds against the isolated proteases, in viral replicons, and in virus-infected Huh7 cells.', 'lgd': {'word': 'We', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'correlation'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**48, 49 Using an Amicon YM10 membrane (EMD Millipore), the purified SARS-CoV M pro was concentrated to 21 mgmL -1 in buffer** A (20 mM Tris-HCl, 150 mM NaCl, 1 mM DTT, 1 mM EDTA, pH 7.5).', 'lgd': {'word': '48', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'buffer'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Crystallization was performed by equilibrating 1 L protein** (mixed with 1 L precipitant solution) against 500 l reservoir containing 6 -8% polyethylene glycol (PEG) 6,000, 0.1 M MES (pH 6.0), at 20 using the vapor diffusion sitting-drop method.', 'lgd': {'word': 'Crystallization', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'protein'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A crystal of the free enzyme was soaked in cryo-protectant buffer containing 20% MPD, 6% PEG 6,000, 0.1 M MES,** 7.5 mM 11a, pH 6.0, for 2 h at 20.', 'lgd': {'word': 'A', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'MES'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Another set of free enzyme crystals was soaked in another cryoprotectant buffer with 6% PEG 6,000, 5% MPD, 0.1 M MES,** 15% glycerol, 10 mM 11s, pH 6.0, for 2 h. Subsequently, crystals were fished and flash-cooled in liquid nitrogen prior to data collection.', 'lgd': {'word': 'Another', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'MES'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The concentrated HCoV-NL63 M pro (45 mgmL -1 ) was** incubated with 5 mM 11a for 4 h at 20, followed by setting up crystallization using the vapor diffusion sitting-drop method at 20 with equilibration of 1 L protein (mixed with 1 L mother liquor) against 500 L reservoir composed of 0.1 M lithium sulfate monohydrate, 0.1 M sodium citrate tribasic dihydrate, 25% PEG 1,000, pH 6.0.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'was'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The crystals were protected by a cryo-buffer** containing 0.1 M lithium sulfate monohydrate, 0.1 M sodium citrate tribasic dihydrate, 25% PEG 1,000, 15% glycerol, 2 mM 11a, pH 6.0 and flash-cooled in liquid nitrogen.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'buffer'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Several free-enzyme crystals were soaked in cryo-protectant buffer containing 0.1 M lithium** sulfate monohydrate, 0.1 M sodium citrate tribasic dihydrate, 25% PEG 1,000, 15% glycerol, 5 mM 11n (or 11f), pH 6.0.', 'lgd': {'word': 'Several', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'lithium'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Freshly prepared CVB3 3C pro at a concentration of** 21.8 mgmL -1 was incubated with 5 mM 11a pre-dissolved in 100% DMSO at room tempature for 1 h. Some white precipitate appeared in the mixture.', 'lgd': {'word': 'Freshly', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Single rod-like crystals were detected both from the Index TM screen, under the condition of 0.1 M MgCl2** hexahydrate, 0.1 M Bis-Tris, 25% PEG 3,350, pH 5.5, and from the Sigma TM screen at 0.2 M Li2SO4, 0.1 M Tris-HCl, 30% PEG 4,000, pH 8.5.', 'lgd': {'word': 'Single', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'MgCl2'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Crystal optimization was performed by using the vapor-diffusion sitting-drop method, with 1 L CVB3** 3C pro -inhibitor complex mixed with 1 L precipitant solution, and equilibration against 500 L reservoir containing 0.1 M Tris-HCl, 0.2 M MgCl2, pH 8.5, and PEG 3,350 varied from 22% to 27%.', 'lgd': {'word': 'Crystal', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'CVB3'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Another optimization screen was also performed against a different reservoir,** 0.1 M Tris-HCl, 0.2 M MgCl2, pH range from 7.5 to 8.5, and PEG 4,000 varied from 24% to 34%.', 'lgd': {'word': 'Another', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'reservoir'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Diffraction data from the crystal of the SARS-CoV M pro in complex with 11a were collected at 100 K at synchrotron beamline PXI-X06SA (PSI, Villigen, Switzerland) using a Pilatus 6M** detector (DECTRIS).', 'lgd': {'word': 'Diffraction', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': '6M'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**52, 53 The complex structures of HCoV-NL63 M pro with 11a, 11f, and 11n were also determined with MOLREP, using as a search model** the structure of the free enzyme determined by us (LZ et al., unpublished) .', 'lgd': {'word': '52', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'model'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A buffer containing 20 mM Tris-HCl, 100 mM NaCl, 1 mM EDTA,** 1 mM DTT, pH 7.3, was used for all the enzymatic assays.', 'lgd': {'word': 'A', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'EDTA'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Two substrates with the cleavage sites of M pro and 3C pro , respectively (indicated by the arrow, ), Dabcyl-KTSAVLQSGFRKM-E(Edans)-NH2 and Dabcyl-KEALFQGPPQF-E(Edans)-NH2 (95% purity; Biosyntan), were employed in the fluorescence resonance energy transfer (FRET)based cleavage assay, using a 96-well** microtiter plate.', 'lgd': {'word': 'Two', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'well'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The dequenching of the Edans fluorescence due to the cleavage of the substrate as catalyzed by the proteases was monitored at 460 nm with excitation at 360 nm, using a Flx800 fluorescence** spectrophotometer (BioTek).', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'fluorescence'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Curves of relative fluorescence units (RFU) against substrate concentration were linear for all substrates up to beyond 50 M, indicating a minimal influence** of the inner-filter effect.', 'lgd': {'word': 'Curves', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'influence'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**For the determination of the IC50, different proteases at a specified final** concentration (0.5 M SARS-CoV or HCoV-NL63 M pro , 2 M CVB3 3C pro , 3 M EV-A71 3C pro ) were separately incubated with the inhibitor at various concentrations (0 to 100 M) in reaction buffer at 37 for 10 min.', 'lgd': {'word': 'For', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'final'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Assessment of inhibitory activity of a-ketoamides using** viral replicons and virus-infected cells Cells and viruses.', 'lgd': {'word': 'Assessment', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'using'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': \"**59) were grown in Dulbecco's modified minimal essential medium (DMEM) supplemented with 2 mM glutamine, 100 UmL -1 penicillin, 100** gmL -1 streptomycin sulfate, and fetal calf serum (10% in growth medium and 2% in maintenance medium).\", 'lgd': {'word': '59', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '100'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Huh-T7 cells were additionally supplemented with geneticin (G-418 sulfate, 400 gmL -1** ).', 'lgd': {'word': 'Huh', 'seed': '1'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**For enterovirus (except CVB3) infection experiments, human rhabdomyosarcoma cells (RD; for EV-A71; BRCR strain) and HeLa Rh cells (for EV-D68 and human rhinoviruses) were grown in MEM Rega 3 medium supplemented with 1% sodium** bicarbonate, 1% Lglutamine, and fetal calf serum (10% in growth medium and 2% in maintenance medium).', 'lgd': {'word': 'For', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'sodium'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**For HCoV-229E (a kind gift** from Volker Thiel (Bern, Switzerland)), culture and infection experiments were carried out as described.', 'lgd': {'word': 'For', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'gift'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**61, 62 Infection of Vero and Huh7 cells with MERS-CoV (strain EMC/2012) and SARS-CoV infection of Vero E6 cells (strain Frankfurt-1) at** low multiplicity of infection (MOI) were done as described before.', 'lgd': {'word': '61', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'at'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**AY278741), in the context of a bacterial artificial** chromosome (BAC) under the control of the CMV promoter, has been described previously (pBAC-REP-RLuc).', 'lgd': {'word': 'AY278741', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'artificial'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': \"**33 Apart from the replicase polyprotein, the replicon encodes the following features: the 5'-and 3'-non-translated regions (NTR), a ribozyme (Rz)**, the bovine growth hormone sequence, and structural protein N.\", 'lgd': {'word': '33', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'Rz'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Huh-T7 cells grown in 12-well plates to a confluency of** 80% -90% (2 -3 x 10 5 cells/well) were washed with 1 mL OptiMEM (Invitrogen) and transfected with 0.25 g of the replication-competent replicon and Lipofectamin2000 or X-tremeGENE9 in 300 l OptiMEM (final volume) as recommended by the manufacturer (Invitrogen or Roche, respectively).', 'lgd': {'word': 'Huh', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Initially, we performed a quick assessment** of the inhibitory activity of the candidate compounds towards the enteroviral and coronaviral replicons at a concentration of 40 M in Huh-T7 cells.', 'lgd': {'word': 'Initially', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'assessment'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Compounds that were relatively powerful and non-toxic at this concentration, were assayed in a dose-dependent** manner to estimate their half-maximal effective concentration (EC50) as well as their cytotoxicity (CC50), as described.', 'lgd': {'word': 'Compounds', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'dependent'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**29 In brief, different concentrations of a-ketoamides (40** M in screening experiments or increasing concentration (0, 1.25, 2.5, 5, 10, 20, 40 M) when determining the EC50) were added to growth medium of replicon-transfected Huh-T7 cells.', 'lgd': {'word': '29', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': '40'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Twenty-four hours later, the cells were washed with 1 mL phosphate-**buffered saline (PBS or OPTIMEM, Invitrogen) and lysed in 0.15 mL Passive lysis buffer (Promega) at room temperature (RT) for 10 min.', 'lgd': {'word': 'Twenty', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'phosphate'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**After freezing (-80 o C) and thawing (RT), the cell debris was removed by centrifugation (16,000 x g, 1 min) and** the supernatant (10 or 20 l) was assayed for Firefly or Renilla luciferase activity (Promega or Biotrend Chemikalien) using an Anthos Lucy-3 luminescence plate reader (Anthos Microsystem).', 'lgd': {'word': 'After', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'and'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The antiviral activity of the compounds was evaluated in a cytopathic effect** (CPE) readout assay using MTS [3-(4,5-dimethylthiazol-2-yl)-5-(3carboxymethoxyphenol)-2-(4-sulfophenyl)-2H-tetrazolium, innersalt]-based assay.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'effect'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Briefly, 24 h prior to infection, cells were seeded in 96-well plates at a density of** 2.5 x 10 4 (RD cells) or of 1.7 x 10 4 (HeLa Rh) per well in medium supplemented with 2% FCS.', 'lgd': {'word': 'Briefly', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Plates were incubated for 1 h at** 37C, then the optical density at 498 nm (OD498) of each well was measured by microtiter plate reader (Saffire 2 , Tecan).', 'lgd': {'word': 'Plates', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'at'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The OD values were converted to percentage of controls and the EC50 was calculated by logarithmic interpolation as the concentration of compound that results in a 50% protective** effect against virus-induced CPE.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'protective'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**61, 63 In brief, Huh7, Vero, or Vero E6 cells were seeded in 96-well plates at a density of** 1  10 4 (Huh7 and Vero E6) or 2  10 4 cells (Vero) per well.', 'lgd': {'word': '61', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Absorbance at 490 nm (A490) was measured using a Berthold Mithras** LB 940 96-well plate reader (Berthold).', 'lgd': {'word': 'Absorbance', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'Mithras'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': \"**HCoV-229E-Rev: 5-ggTCGTTTAGTTGAGAAAAGT -3, and 229E-ZNA probe: 5'-6-Fam-AGA (pdC)TT(pdU)G(pdU)GT(pdC)TA(pdC)T-ZNA-3-BHQ-1 -3'** (Metabion).\", 'lgd': {'word': 'HCoV', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '3'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Data were analysed using GraphPad Prism 5.0; EC50 values were calculated based on a 4parameter logistic** statistics equation.', 'lgd': {'word': 'Data', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'logistic'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**CC50 values were calculated using inhibitor versus normalized response statistics equation by including proper controls (no inhibitor and 1% Triton-**X-100-treated cells).', 'lgd': {'word': 'CC50', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'Triton'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**29, 65 Chemical synthesis of a-ketoamides General** procedure.', 'lgd': {'word': '29', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'General'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**1 H NMR** spectra were recorded on 300-MHz, 400-MHz, or 500-MHz instruments.', 'lgd': None, 'rgd': {'seed': '1', 'word': 'NMR'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Mass spectra were recorded using a Bruker ESI** ion-trap HCT Ultra.', 'lgd': {'word': 'Mass', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'ESI'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**HPLC spectra were recorded by LC20A or LC10A (Shimadzu Corporation) with Shim-pack GIST C18 (5 m, 4.6x150mm) with three solvent systems (methanol/water, methanol/ 0.1% HCOOH** in water or methanol/0.1% ammonia in water).', 'lgd': {'word': 'HPLC', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'HCOOH'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Synthesis of (2S,4R)-dimethyl 2-(tert-butoxycarbonylamino)-4-(cyanomethyl) pentanedioate (1)**.', 'lgd': {'word': 'Synthesis', 'seed': '1'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**To a solution of N-Boc-L-glutamic acid dimethyl ester (6.0 g, 21.8 mmol) in THF (60 mL) was added dropwise a** solution of lithium bis(trimethylsilyl)amide (LHMDS) in THF (47 mL, 1 M) at 78C under nitrogen.', 'lgd': {'word': 'To', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Meanwhile, bromoacetonitrile (1.62 mL,** 23.3 mmol) was added dropwise to the dianion solution over a period of 1 h while keeping the temperature below 70C.', 'lgd': {'word': 'Meanwhile', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'mL'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The organic layer was concentrated and purified by flash column chromatography (petroleum ether/ethyl acetate = 4/1) to** give product 1 (4.92 g, 72%) as colorless oil.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'to'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**1 H NMR (CDCl3, 400 MHz)  5.23 (1H, d, J = 9.0 Hz), 4.43-4.36 (1H, m), 3.77(1H, s), 3.76 (1H, s), 2.89-2.69 (3H, m), 2.20-2.14 (2H, m), 1.**45 (9H, s).', 'lgd': {'word': '1', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'NMR'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In a hydrogenation flask was placed compound 1** (4.0 g, 12.7 mmol), 5 mL of chloroform and 60 mL of methanol before the addition of PtO2.', 'lgd': {'word': 'In', 'seed': '1'}, 'rgd': {'seed': 'a', 'word': 'flask'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The light-brown filtrate was concentrated and purified by silica gel column chromatography (petroleum ether/ethyl acetate = 4/1) to** give the product 2 (2.20 g, 61%) as white solid.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'to'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**1 H NMR** (CDCl3)  6.02 (1H, br), 5.', 'lgd': None, 'rgd': {'seed': '1', 'word': 'NMR'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Compound 2 (1.0 g,** 3.5 mmol) was dissolved in 10 mL dichloromethane (DCM), then 10 mL trifluoroacetic acid (TFA) was added.', 'lgd': {'word': 'Compound', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'g'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The reaction mixture was stirred at 20 o C for 0.5 h, and concentrated in vacuo to get a colorless oil,** which could be used for the following step without purification.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'oil'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The methyl amino-acid ester hydrochloride (6.0 mmol) was dissolved in 20 mL CH2Cl2, and then acyl chloride (6.0 mmol), triethylamine (1.69 mL,** 12.0 mmol) were added, before the reaction was stirred for 2 h at 20 o C. The reaction mixture was diluted with 20 mL CH2Cl2, washed with 50 mL of saturated brine (2  25 mL), and dried over Na2SO4.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'mL'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The methyl L-phenylalaninate hydrochloride (1.30 g,** 6.0 mmol) was dissolved in 20 mL CH2Cl2, and then cinnamoyl chloride (1.00 g, 6.0 mmol), triethylamine (1.69 mL, 12.0 mmol) were added, before the reaction was stirred for 2 h at room temperature.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'g'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The solvent was evaporated and the product 4a was obtained as white solid (1.75 g,** 95%), which could be used for the next step without further purification.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'g'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**1 M NaOH (5 mL) was added to a** solution of compound 4 (3.0 mmol) in methanol (5 mL).', 'lgd': {'word': '1', 'seed': 'a'}, 'rgd': {'seed': '1', 'word': 'NaOH'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The reaction was stirred for 20 min at 20 o C. Then 1 M HCl** was added to the reaction solution until pH = 1.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'HCl'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The solvent was evaporated and the crude material purified on silica, eluted with mixtures of CH2Cl2/MeOH (20/1) to** afford the product 5 (90-96% yield) as a white solid.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'to'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**To this solution, 1.5 equiv** (1.54 g) of 1-[bis(dimethylamino)methylene]-1H-1,2,3triazolo[4,5-b]pyridinium 3-oxide hexafluorophosphate (HATU) was added, and the reaction was stirred for 0.5 h at 20 o C. Then compound 3 (500 mg, 2.7 mmol) and TEA (0.70 mL, 5.42 mmol) was added to the reaction.', 'lgd': {'word': 'To', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'equiv'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The solvent was evaporated and the crude material purified on silica, eluted with a mixture of** CH2Cl2/MeOH (40/1) to give the product 6 (62-84% yield).', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Compound 6 (1.1 mmol)** was dissolved in methanol (40 mL), then NaBH4 (0.34 g, 8.8 mmol) was added under ambient conditions.', 'lgd': {'word': 'Compound', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'mmol'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The resulting mixture was stirred at 20 o C for 1 h. The** mixture was concentrated and purified by column chromatography on silica gel (CH2Cl2/MeOH = 20/1) to give the product 8 as white solid (88-95% yield).', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'The'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The reaction was stirred at 20 o C for 24 h. Then the solvent was evaporated and the crude material purified on silica, eluted with a mixture of** CH2Cl2/MeOH (20/1) to give the product 9 (46-84%).', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**1 M NaOH (0.5 mL) was added to a** solution of compound 9 (0.164 mmol) in methanol (5 mL).', 'lgd': {'word': '1', 'seed': 'a'}, 'rgd': {'seed': '1', 'word': 'NaOH'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Then, 1 M HCl** was added to the reaction solution until pH = 7.', 'lgd': {'word': 'Then', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'HCl'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The resulting mixture was stirred at 20 o C for 1 h. The** mixture was concentrated and purified by column chromatography on silica gel (CH2Cl2/MeOH = 20/1) to give the a-ketoamides 11 as light yellow solid (52-79% in two steps).', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'The'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': \"**Detailed results of the variation of the P1' and P3 substituents Synthesis of a-ketoamides 11b** -11e and 11g -11l Supplemental Table 1 : inhibitory activities (IC50 (M)) of -ketoamides with P1' and P3 modifications against viral proteases Supplemental Table 2 : crystallographic data for complexes between viral proteases and -ketoamides Molecular formula strings and biological data (CSV)\", 'lgd': {'word': 'Detailed', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': '11b'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**1  2**  3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  2**1  2**2  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60 author/funder.', 'lgd': None, 'rgd': {'seed': '1', 'word': '2'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**https://doi.org/10.1101/2020.02.10.936898 doi: bioRxiv preprint  1  2**  3  4  5  6  7  8  9  10  11  12  13  14  15  16  17  18  19  20  21  22  23  24  25  26  27  28  29  30  31  32  33  34  35  36  37  38  39  40  41  42  43  44  45  46  47  48  49  50  51  52  53  54  55  56  57  58  59  60 author/funder.', 'lgd': {'word': 'https', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '2'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Potential Broad Spectrum Inhibitors of the Coronavirus 3CL pro : A Virtual Screening and Structure-Based Drug Design Study"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Human coronaviruses represent a significant disease** burden; however, there is currently no antiviral strategy to combat infection.', 'lgd': {'word': 'Human', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'disease'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The coronavirus 3CL pro provides a highly validated** drug target and as there is a high degree of sequence homology and conservation in main chain architecture the design of broad spectrum inhibitors is viable.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'validated'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The ZINC drugs-now library was screened in a consensus high-**throughput pharmacophore modeling and molecular docking approach by Vina, Glide, GOLD and MM-GBSA.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'high'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**As these compounds were extensively validated by a consensus approach** and by molecular dynamics, the likelihood that at least one of these compounds is bioactive is excellent.', 'lgd': {'word': 'As', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'approach'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Additionally, the compounds segregate into 15 significantly dissimilar (p < 0.05) clusters based on shape and features, which represent valuable scaffolds that can be used as a basis for** future anti-coronaviral inhibitor discovery experiments.', 'lgd': {'word': 'Additionally', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'for'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Coronaviruses (CoV) affect a diverse group** of animal hosts, and cause a plethora of diseases in animals including progressive peritonitis, acute and chronic hepatitis, gastroenteritis, nephritis, and encephalitis [1] .', 'lgd': {'word': 'Coronaviruses', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'group'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Another two human coronaviruses, namely severe acute respiratory syndrome (SARS)-CoV and Middle East respiratory syndrome (MERS)-CoV, cause severe respiratory syndromes and present a significant threat** with their high fatality rates.', 'lgd': {'word': 'Another', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'threat'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Given the high prevalence of respiratory infections, human coronaviruses represent a substantial disease** burden, which is exacerbated by the high implications of healthcare workers in coronavirus outbreaks [12] .', 'lgd': {'word': 'Given', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'disease'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Misdiagnosis of these outbreaks pose a further substantial** threat to healthcare workers with nosocomial spread to other patients putting further pressure on an already strained healthcare system.', 'lgd': {'word': 'Misdiagnosis', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'substantial'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The 3CL pro has proven to be a valuable target** in drug discovery efforts and has been validated as an effective drug target in several studies.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'target'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The first generation of 3CL pro inhibitors were irreversible peptidomimetic structures, often five residues in length with at reactive warhead at the terminus that formed a covalent bond** between the thiolate anion of the catalytic Cys145 residue and the reactive atom of the warhead [19] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'bond'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**All coronavirus 3CL pro share a high sequence** homology, as well as main chain architecture and substrate conservation [30, 31] , which makes the identification of broad spectrum lead compounds more viable.', 'lgd': {'word': 'All', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'sequence'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The substrate binding site of the 3CL pro has two deeply buried S 1 and S** 2 subsites, as well as shallow S 1 1 , S 3 and S 4 subsites with varying degrees of solvent exposure.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'S'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Substrate specificity of coronavirus 3CL pro is mainly determined by the P 1 , P** 2 and P 1 1 positions [31] .', 'lgd': {'word': 'Substrate', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'P'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The P 1 position has** an absolute specificity for glutamine which stabilizes the S 1 subsite via a hydrogen bond with the imidazole N2 of His162/3 and van der Waals interactions with surrounding residues of the S 1 pocket.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'has'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The P 2 site has a preference for** leucine or methionine to fill the hydrophobic S 2 pocket.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'for'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The sidechains of the S 3 site are solvent-exposed and therefore this site is expected to tolerate a wide range** of functionality, but shows a preference for basic residues [32] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'range'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Sidechains and backbones of residues surrounding the S 4 site create a highly congested** pocket which favors a small, hydrophobic residue in the P 4 position, either Ser, Thr, Val or Pro [32] [33] [34] .', 'lgd': {'word': 'Sidechains', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'congested'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The S 1 1 and** S 2 1 subsites also accommodate small residues in the P 1 1 and P 2 1 positions, which may include Ser, Ala or Gly [33, 35] .', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'and'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Vina performed the initial high throughput screen of the Drugs-Now dataset from ZINC, comprising a total of** approximately 6.5 million compounds.', 'lgd': {'word': 'Vina', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The grid box for searching was centered on the active His41 residue with grid box spacing set to 1  and** extended 26 grid points in each direction to encompass the entire binding pocket.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'and'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The lowest binding score within the hit-list corresponded to a binding affinity** of9.5 kcal/mol and therefore all compounds with an equivalent binding affinity were progressed.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'affinity'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Receptors were minimized by a restrained minimization** using the OPLS2005 force field [40, 41] , during which, heavy atoms were restrained to remain within 0.3  of the original structure.', 'lgd': {'word': 'Receptors', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'minimization'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The grid box was generated 10  in each direction and a constraint was** set for the ligand to form a hydrogen bond with the pivotal His162 residue of the S 1 active pocket.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'was'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This cut-off was predetermined based on the binding affinity of a known inhibitor.**', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'inhibitor'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The GOLDScore fitness scoring function was used to rank ligands based on a GOLDScore cut-**off of 60 [42] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'cut'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**As a final filter a** binding free energy cut-off of 80 kcal/mol was defined by determining the free energy of binding of a known active inhibitor.', 'lgd': {'word': 'As', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'filter'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Features included an essential hydrogen bond with the His163 residue and a hydrophobic moiety** in the P 2 position.', 'lgd': {'word': 'Features', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'moiety'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Excluded volumes of 1  were** set to better consider the steric effects.', 'lgd': {'word': 'Excluded', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'were'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Non-essential features included a hydrogen bond** donor and acceptor in the Glu166 position, a hydrogen bond acceptor in the P 4 pocket, a single hydrogen bond donor in the P 2 and P 3 pocket and an acceptor in the P 1 1 pocket.', 'lgd': {'word': 'Non', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'bond'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A total of four pharmacophoric features were required within the excluded volumes of 1** .', 'lgd': {'word': 'A', 'seed': '1'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A hydrogen bond constraint with His163 was used as a primary filter.**', 'lgd': {'word': 'A', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'filter'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The above mentioned techniques conducted a high throughput** virtual screen of the ZINC Drugs-Now dataset by docking and pharmacophore modeling.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'throughput'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Side chains of residues in the active pocket assuming orientations that resulted in a steric clash** with the ligand were refined and the ligand was minimized with Prime to better fit the pocket.', 'lgd': {'word': 'Side', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'clash'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**As before, a binding energy** of80 kcal/mol was used as a cut-off and the hydrogen bond constraint in the S 1 pocket was again applied to discriminate between potential binders and non-binders.', 'lgd': {'word': 'As', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'energy'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The protein-ligand complex was then placed in the center of a solvated, cubic** box, 1 nm from the box edge.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'cubic'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The system was energy minimized by a brief dynamics** simulation using Steepest Descent Minimization until steepest descent converged to Fmax <1000.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'dynamics'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Long range electrostatic interactions were calculated with the Particle Mesh Ewald (PME) method [45] with a cutoff of** 1 nm and periodic boundary conditions were applied.', 'lgd': {'word': 'Long', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Both NVT and NPT equilibrations were run for 50,000 steps or 100 ps using a 2 fs** time step and a leap-frog integrator.', 'lgd': {'word': 'Both', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'fs'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**PME was again used for long range electrostatics with a 0.16** Fourier spacing.', 'lgd': {'word': 'PME', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': '16'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Short range electrostatic and van der Waals cutoffs were 1.0 nm** and 1.4 nm respectively.', 'lgd': {'word': 'Short', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'nm'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**During NVT equilibration temperature coupling was achieved by V-rescale algorithm with a target temperature** of 300 K. Pressure coupling during NPT equilibration was achieved via the Parrinello-Rahman algorithm and is incorporated in the NPT equilibration once temperature is stable to ensure the proper density is reached (approximately 1000 kg/m 3 ).', 'lgd': {'word': 'During', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'temperature'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The 3CL pro of 229E was previously solved in complex with an inhibitor by X-ray crystallography at a high resolution** of 1.8  [46] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'resolution'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The corresponding crystallographic structure, with Protein Data Bank (PDB) code 2ZU2, shows a dimeric structure** with the inhibitor, zinc N-ethyl-N-phenyldithiocarbamate (EPDTC), coordinated to the catalytic dyad residues of His41 and Cys144 with a zinc-centered tetrahedral geometry.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'structure'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The apo form of the NL63 3CL pro was solved in its dimeric form by X-ray crystallography to a high resolution** of 1.6  with the respective PDB code, 3TLO.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'resolution'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**It was previously solved by X-ray crystallography to a resolution of** 2.5  in complex with a peptidomimetic, covalent inhibitor [30] .', 'lgd': {'word': 'It', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The PDB entry contains a total of** four chains, two chains form the active dimeric structure with the remaining two found in monomeric forms.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**As this inhibitor possesses several pharmacophoric features common to the natural substrate and favorable to drug development it was utilized to develop a pharmacophore model** for virtual screening.', 'lgd': {'word': 'As', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'model'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**However, the majority of these include structures in complex with a peptidomimetic, covalent** inhibitor [46] [47] [48] or mutational studies [49] [50] [51] .', 'lgd': {'word': 'However', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'covalent'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**At the time this study commenced, only one structure could be identified in complex with a small molecule** inhibitor.', 'lgd': {'word': 'At', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'molecule'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This structure, with PDB code 3V3M [19] , also represented the only coronavirus 3CL pro with a noncovalent inhibitor.**', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'inhibitor'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The crystallographic structure contains a noncovalent, furyl** amide ligand, ML188, which was identified by a high throughput screen, with subsequent lead optimization based on structure-activity relationships within the S 1 , S 1 1 and S 2 binding pockets.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'furyl'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Molecular docking algorithms are often calibrated on a training set** of experimental ligand-protein complexes and accuracy of these docking programs is often highly dependent on the training set used [52] .', 'lgd': {'word': 'Molecular', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'set'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This highlights the importance of confirming that the docking software used for virtual screening is capable of replicating the binding mode of a known, experimental** inhibitor for the class of enzymes studied [53, 54] .', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'experimental'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In order to confirm that the molecular docking algorithms used in this study are well suited to the 3CL pro of human coronaviruses, and their related class, a previously solved** experimental structure with a bound, noncovalent inhibitor is required.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'solved'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Initial studies on structure-based design of inhibitors, targeting the 3CL pro of coronaviruses, were focused on peptidomimetic compounds with a reactive warhead** group that formed an irreversible, covalent interaction with the active Cys residue [19] .', 'lgd': {'word': 'Initial', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'warhead'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**An RMSD below 2  for heavy atoms (excluding hydrogens) between the experimental structure and predicted pose of docking is a well-defined** benchmark to assess the accuracy of molecular docking sampling algorithms [56] .', 'lgd': {'word': 'An', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'defined'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': 'Figure **1 represents the** experimental pose of the ML188 ligand with overlaid predicted pose of Vina and Glide.', 'lgd': None, 'rgd': {'seed': '1', 'word': 'the'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**An RMSD below 2  for heavy atoms (excluding hydrogens) between the experimental structure and predicted pose of docking is a well-defined** benchmark to assess the accuracy of molecular docking sampling algorithms [56] .', 'lgd': {'word': 'An', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'defined'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The orientation of the P1 furan is somewhat distorted with the conformation of the ring structure being assumed to orientate at a right angle** to the experimental pose.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'angle'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The overall predicted pose of Vina corresponds to the experimental pose with a high level** of accuracy.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'level'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**These results strongly support the use of Vina as high throughput tool for virtual screening and indicates that with the use of Glide as a subsequent refinement** tool, the accuracy of predicting the binding pose of novel ligands will be high.', 'lgd': {'word': 'These', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'refinement'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The ML188 lead compound also provided a means to** set benchmarks for scoring functions and free energy of binding estimation.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'to'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The inhibitor achieved a binding affinity** estimation of 8.1 kcal/mol when bound to 3V3M.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'affinity'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The orientation of the P 1 1 furan** is somewhat distorted with the conformation of the ring structure being assumed to orientate at a right angle to the experimental pose.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'furan'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The overall predicted pose of Vina corresponds to the experimental pose with a high level** of accuracy.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'level'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**These results strongly support the use of Vina as high throughput tool for virtual screening and indicates that with the use of Glide as a subsequent refinement** tool, the accuracy of predicting the binding pose of novel ligands will be high.', 'lgd': {'word': 'These', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'refinement'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The ML188 lead compound also provided a means to** set benchmarks for scoring functions and free energy of binding estimation.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'to'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The inhibitor achieved a binding affinity** estimation of8.1 kcal/mol when bound to 3V3M.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'affinity'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This was done to ensure that ligands with potential broad spectrum activity were not missed based on their predicted inability to bind to a single receptor.**', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'receptor'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The primary hit-list consisted of the top 1500 ranked compounds from the high throughput virtual screen (HTVS) of the ZINC Drugs-Now dataset conducted by Vina, a quantity where** further refinement and analysis would be achievable.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'where'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Ranking the initial virtual screen based on docking score alone serves as a starting point** in the selection criteria and assumes that the predicted binding affinity somewhat reflects true binding affinity [58] [59] [60] .', 'lgd': {'word': 'Ranking', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'point'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**For this reason, all ligands which achieved this score were also included in the study, yielding a total of** 1527 hit compounds.', 'lgd': {'word': 'For', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**To improve the likelihood of identifying actual active compounds, three sampling algorithms and four scoring functions were applied to the hit-list identified by Vina in a consensus based** approach.', 'lgd': {'word': 'To', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'based'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This approach has frequently improved enrichment rates of active compounds in a virtual screening** context, in comparison to the use of a single algorithm for docking and scoring [62] [63] [64] .', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'screening'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In order to define a threshold binding** score cut-off to prefilter hits, the crystallographic structure of 3V3M, with bound inhibitor ML188, was used.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'binding'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**As most scoring functions cannot be compared between structures it is important to define a binding score** cut-off based on a related experimental structure [57] .', 'lgd': {'word': 'As', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'score'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The inhibitor ML188 achieved a binding affinity** estimation of8.1 kcal/mol when bound to 3V3M.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'affinity'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': 'As **a subsequent stage** of hit-list refinement the identified hit compounds and 2ZU2 receptor were directly imported into GOLD for redocking and rescoring.', 'lgd': None, 'rgd': {'seed': 'a', 'word': 'stage'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**At this stage the hit-list was refined to a point where** the binding pose of each hit could be analyzed by manual inspection.', 'lgd': {'word': 'At', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'where'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The natural substrate of the 3CL pro forms a hydrogen bond** with His162/3 in the S 1 pocket where the S 2 pocket is filled by a hydrophobic moiety.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'bond'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**These comprehensive filters of three sampling and scoring algorithms and pharmacophoric feature constraints provided us with a highly defined** hit-list of 116 compounds.', 'lgd': {'word': 'These', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'defined'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Rescoring the final hit-list with MM-GBSA identified a total of** 22 ligands that passed all filters and pharmacophoric constraints for the 2ZU2 receptor.', 'lgd': {'word': 'Rescoring', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This included a total of** three sampling algorithms and four scoring functions from Vina, Glide and GOLD and the highly advanced MM-GBSA, respectively, and two pharmacophoric constraints with a hydrogen bond acceptor in the S 1 pocket and a hydrophobic moiety filling the S 2 pocket.', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This consensus docking and scoring approach is in comparison to several studies, which identified novel lead compounds from a virtual screen,** where only a single molecular docking algorithm was incorporated [65] [66] [67] [68] [69] .', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'screen'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The comprehensive list of techniques used in this study heightens the probability that the ligands identified in the final hit-list do represent novel scaffolds of active molecules and with the high homology between the 3CL pro of various human coronaviruses these compounds should exhibit a high potential** for broad spectrum effects.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'potential'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The use of multiple techniques therefore improves the probability of identifying a diverse set** of lead compounds.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'set'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The pharmacophore model was based largely on a single covalent** inhibitor.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'covalent'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This inhibitor was chosen as it possesses several favorable noncovalent features for the generation of a pharmacophore model.**', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'model'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**As general practice, several inhibitors are often used to generate a pharmacophore model** where features that are frequently repeated are included in the pharmacophore [73] .', 'lgd': {'word': 'As', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'model'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The essential features of the pharmacophore model included a hydrogen bond** acceptor in the P 1 position and a hydrophobic moiety in the P 2 position.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'bond'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Non-essential features included a hydrogen bond** donor and acceptor in the position to interact with Glu166, a hydrogen bond acceptor in the P 4 pocket, a single hydrogen bond donor in the P 2 and P 3 pockets and an acceptor in the P 1 1 pocket.', 'lgd': {'word': 'Non', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'bond'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A total of four features, including the two essential features and any two of the non-essential features, within an excluded volume of 1  were** required for a potential hit to satisfy the pharmacophore model.', 'lgd': {'word': 'A', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'were'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**To ensure a primary hit-**list of manageable size, only ligands satisfying the pharmacophore within an RMSD of 0.5  where filtered to the initial hit-list, which included approximately 25,000 compounds.', 'lgd': {'word': 'To', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'hit'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Since MOE utilizes a united atom** force field, it was necessary to add hydrogens with Open Babel.', 'lgd': {'word': 'Since', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'atom'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**As the pharmacophore model only filters compounds based on the presence of a hydrogen bond** acceptor in the P 1 position, and does not take into account distance and geometry essential for the formation of a hydrogen bond, the Pose Filter script of Schrdinger was again used to further refine the hit-list, which now contained 3569 hits.', 'lgd': {'word': 'As', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'bond'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': 'In **a dynamic, physiological** system, these sidechains would accommodate ligand binding by the induced-fit phenomena, however in molecular docking this process is circumvented by the static nature of the receptor [76] .', 'lgd': None, 'rgd': {'seed': 'a', 'word': 'physiological'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Residue sidechains resulting in a steric clash** with the ligand were refined using the Prime module of Schrdinger, which replicates their dynamic, flexible movements allowing the sidechains to accommodate the ligand in the pocket.', 'lgd': {'word': 'Residue', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'clash'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The binding mode predicted using this approach was then further assessed using the pharmacophoric constraints of a hydrogen bond** in the S 1 and a hydrophobic moiety in the S 2 pockets and scoring with MM-GBSA where the cut-off previously defined was again implemented.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'bond'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Of the 22 compounds identified in the hit-list of potential 2ZU2 binders, 15 compounds could replicate the interactions in the S 1 and S** 2 pockets and fulfil the MM-GBSA cut-off (Table 1) , where four of the compounds identified by pharmacophore modeling could fulfil these requirements ( Table 2 ).', 'lgd': {'word': 'Of', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'S'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Free energy of binding between a known active** inhibitor, ML188, and respective coronavirus 3CL pro is also shown in Tables 1 and 2 for comparative purposes.', 'lgd': {'word': 'Free', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'active'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The dendogram was generated using similarity score of 1.04, which** is recommended to achieve statistical significance at the 95% confidence interval.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'which'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Molecular dynamics utilizes Newtonian physics to simulate atomic movements in a solvated system** and remains the most accurate computational method for simulating protein-drug interactions.', 'lgd': {'word': 'Molecular', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'system'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**To confirm the findings seen in docking and pharmacophore modeling and validate the binding mode of representative hit-compounds, a 10 ns** molecular dynamic simulation allowed for an assessment of non-bonded energies and hydrogen bonding networks between ligand and receptor and ligand stability via RMSD, radius of gyration and changes in solvent accessible surface area.', 'lgd': {'word': 'To', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'ns'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A pyridine filled the S1 pocket and formed a hydrogen bond** between the pyridine nitrogen and imidazole N2 of His163.', 'lgd': {'word': 'A', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'bond'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A 1,2,4-triazole formed a** linker in the center of the structure and mediated the formation of a hydrogen bond between N2 and the backbone of Glu165/6, which formed a third hydrogen bond via the backbone carbonyl and ligand NH.', 'lgd': {'word': 'A', 'seed': 'a'}, 'rgd': {'seed': '1', 'word': '4'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**OC43 3CL pro exhibited a fourth hydrogen** bond between N4 of the 1,2,4-triazole and Gln189.', 'lgd': {'word': 'OC43', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'hydrogen'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In 2ZU2 and 3TLO the position of Gln is replaced by Pro188/9, which therefore did not contribute to the formation of a** bond.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The dendogram was generated using similarity score of 1.04, which** is recommended to achieve statistical significance at the 95% confidence interval.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'which'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Molecular dynamics utilizes Newtonian physics to simulate atomic movements in a solvated system** and remains the most accurate computational method for simulating protein-drug interactions.', 'lgd': {'word': 'Molecular', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'system'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**To confirm the findings seen in docking and pharmacophore modeling and validate the binding mode of representative hit-compounds, a 10 ns** molecular dynamic simulation allowed for an assessment of non-bonded energies and hydrogen bonding networks between ligand and receptor and ligand stability via RMSD, radius of gyration and changes in solvent accessible surface area.', 'lgd': {'word': 'To', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'ns'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A pyridine filled the S 1 pocket and** formed a hydrogen bond between the pyridine nitrogen and imidazole N2 of His163.', 'lgd': {'word': 'A', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'and'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A 1,2,4-triazole formed a** linker in the center of the structure and mediated the formation of a hydrogen bond between N2 and the backbone of Glu165/6, which formed a third hydrogen bond via the backbone carbonyl and ligand NH.', 'lgd': {'word': 'A', 'seed': 'a'}, 'rgd': {'seed': '1', 'word': '4'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**OC43 3CL pro exhibited a fourth hydrogen** bond between N4 of the 1,2,4-triazole and Gln189.', 'lgd': {'word': 'OC43', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'hydrogen'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In 2ZU2 and 3TLO the position of Gln is replaced by Pro188/9, which therefore did not contribute to the formation of a** bond.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The remaining four structures exhibited similar profiles however with varying degrees of occupancy, with an average bond formation of 2.5 for 2ZU2, 1.7 for** 3D23, 2.6 for 3TLO and 2.1 for OC43 3CL pro .', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'for'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**All structures displayed high occupancy at both Glu166 positions, however a decreased occupancy** at the His162/3 position.', 'lgd': {'word': 'All', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'occupancy'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The loss of hydrogen bond formation at this position was often caused by the pyridine ring orientating at a right angle** to the His162/3 N2.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'angle'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**These bonds were predominantly formed via the backbone, however 2ZU2 was capable of forming with both the backbone and sidechain, where the sidechain exhibits a higher frequency.**', 'lgd': {'word': 'These', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'frequency'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The bond formed between Glu189 and the N4 of the 1,2,4-**triazole, as seen in docking results for OC43 3CL pro , exhibits a relatively low occupancy of 17.95%.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '4'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The compound does, however, still provide an excellent scaffold for lead optimization with the substructures filling the S2, S3 and S4 subsites making extensive and stable interactions with surrounding residues including two hydrogen bonds between the central 1,2,4-**triazole and NH and the backbone of Glu165/6.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '4'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Further lead optimization may explore the replacement of the pyridine ring with the related lactam ring, which is known to mediate two hydrogen bonds, one between the ligand carbonyl and the His162/3 N2 and a second hydrogen** bond between the lactam NH and Glu166 [46] .', 'lgd': {'word': 'Further', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'hydrogen'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Solvent accessible surface area of the ligand also remained consistent throughout the duration of the simulation and fluctuated within a range of** 4.8 nm 2 -5.4 nm 2 (Figure 4) .', 'lgd': {'word': 'Solvent', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The remaining four structures exhibited similar profiles however with varying degrees of occupancy, with an average bond formation of 2.5 for 2ZU2, 1.7 for** 3D23, 2.6 for 3TLO and 2.1 for OC43 3CL pro .', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'for'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**All structures displayed high occupancy at both Glu166 positions, however a decreased occupancy** at the His162/3 position.', 'lgd': {'word': 'All', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'occupancy'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The loss of hydrogen bond formation at this position was often caused by the pyridine ring orientating at a right angle** to the His162/3 N2.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'angle'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**These bonds were predominantly formed via the backbone, however 2ZU2 was capable of forming with both the backbone and sidechain, where the sidechain exhibits a higher frequency.**', 'lgd': {'word': 'These', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'frequency'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The bond formed between Glu189 and the N4 of the 1,2,4-**triazole, as seen in docking results for OC43 3CL pro , exhibits a relatively low occupancy of 17.95%.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '4'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The compound does, however, still provide an excellent scaffold for lead optimization with the substructures filling the S 2 , S 3 and S 4 subsites making extensive and stable interactions with surrounding residues including two hydrogen bonds between the central 1,2,4-**triazole and NH and the backbone of Glu165/6.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '4'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Further lead optimization may explore the replacement of the pyridine ring with the related lactam ring, which is known to mediate two hydrogen bonds, one between the ligand carbonyl and the His162/3 N2 and a second hydrogen** bond between the lactam NH and Glu166 [46] .', 'lgd': {'word': 'Further', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'hydrogen'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The presence of this second hydrogen bond and van der Waals interactions, mediated via ring substituents, may further stabilize the compound within the S 1** pocket.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Solvent accessible surface area of the ligand also remained consistent throughout the duration of the simulation and fluctuated within a range of** 4.8 nm 2 -5.4 nm 2 (Figure 4) .', 'lgd': {'word': 'Solvent', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**As with the previous compound a pyridine ring** fills the S1 pocket and formed a hydrogen bond with the imidazole N2 of His162/3.', 'lgd': {'word': 'As', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'ring'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A backbone carbonyl of His164 for 3V3M formed a hydrogen bond** with a ligand NH.', 'lgd': {'word': 'A', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'bond'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This residue was replaced by a Gln in** all remaining structures, however a backbone carbonyl of these two residues were perfectly aligned thereby allowing the formation of the hydrogen bond in all structures.', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'in'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The backbone NH of Glu165/6 again formed a hydrogen bond** with a ligand carbonyl.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'bond'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A fourth hydrogen bond is observed in 3V3M, 3D23 and OC43 3CL pro between the sidechain of Gln189 and a ligand carbonyl.**', 'lgd': {'word': 'A', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'carbonyl'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The position of this Gln in 2ZU2 and 3TLO was not aligned structurally to 3V3M, 3D23 or OC43 3CL pro , with structural superimposition showing a Pro in** this position accounting for the loss of the hydrogen bond in 2ZU2 and 3TLO.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'in'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A potential problematic feature of this ligand was a solvent exposed** 1,3-dioxolane, which made few interactions with surrounding residues.', 'lgd': {'word': 'A', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'exposed'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Molecular dynamics trajectories again showed a strong correlation** with that seen in our pharmacophore models, with the exception of the 2ZU2-ZINC09411012 complex.', 'lgd': {'word': 'Molecular', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'correlation'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In this structure the pyridine ring gradually dissociated out of the S1 pocket at around 4 ns followed by almost complete dissociation of the ligand from the active site, where the 1,3-dioxolane** became completely solvent exposed.', 'lgd': {'word': 'In', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'dioxolane'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Representation of elements mediating hydrogen bonds with respective residues and the P 1 pyridine, hydrophobic** P 2 and solvent exposed dioxolane.', 'lgd': {'word': 'Representation', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'hydrophobic'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**As with the previous compound a pyridine ring** fills the S 1 pocket and formed a hydrogen bond with the imidazole N2 of His162/3.', 'lgd': {'word': 'As', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'ring'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A backbone carbonyl of His164 for 3V3M formed a hydrogen bond** with a ligand NH.', 'lgd': {'word': 'A', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'bond'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This residue was replaced by a Gln in** all remaining structures, however a backbone carbonyl of these two residues were perfectly aligned thereby allowing the formation of the hydrogen bond in all structures.', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'in'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The backbone NH of Glu165/6 again formed a hydrogen bond** with a ligand carbonyl.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'bond'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A fourth hydrogen bond is observed in 3V3M, 3D23 and OC43 3CL pro between the sidechain of Gln189 and a ligand carbonyl.**', 'lgd': {'word': 'A', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'carbonyl'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The position of this Gln in 2ZU2 and 3TLO was not aligned structurally to 3V3M, 3D23 or OC43 3CL pro , with structural superimposition showing a Pro in** this position accounting for the loss of the hydrogen bond in 2ZU2 and 3TLO.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'in'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A potential problematic feature of this ligand was a solvent exposed** 1,3-dioxolane, which made few interactions with surrounding residues.', 'lgd': {'word': 'A', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'exposed'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**As with the previous compound a pyridine ring** fills the S1 pocket and formed a hydrogen bond with the imidazole N2 of His162/3.', 'lgd': {'word': 'As', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'ring'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A backbone carbonyl of His164 for 3V3M formed a hydrogen bond** with a ligand NH.', 'lgd': {'word': 'A', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'bond'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This residue was replaced by a Gln in** all remaining structures, however a backbone carbonyl of these two residues were perfectly aligned thereby allowing the formation of the hydrogen bond in all structures.', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'in'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The backbone NH of Glu165/6 again formed a hydrogen bond** with a ligand carbonyl.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'bond'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A fourth hydrogen bond is observed in 3V3M, 3D23 and OC43 3CL pro between the sidechain of Gln189 and a ligand carbonyl.**', 'lgd': {'word': 'A', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'carbonyl'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The position of this Gln in 2ZU2 and 3TLO was not aligned structurally to 3V3M, 3D23 or OC43 3CL pro , with structural superimposition showing a Pro in** this position accounting for the loss of the hydrogen bond in 2ZU2 and 3TLO.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'in'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A potential problematic feature of this ligand was a solvent exposed** 1,3-dioxolane, which made few interactions with surrounding residues.', 'lgd': {'word': 'A', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'exposed'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Molecular dynamics trajectories again showed a strong correlation** with that seen in our pharmacophore models, with the exception of the 2ZU2-ZINC09411012 complex.', 'lgd': {'word': 'Molecular', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'correlation'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In this structure the pyridine ring gradually dissociated out of the S1 pocket at around 4 ns followed by almost complete dissociation of the ligand from the active site, where the 1,3-dioxolane** became completely solvent exposed.', 'lgd': {'word': 'In', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'dioxolane'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The two methyl groups that filled the hydrophobic S2 pocket remained the only Molecular dynamics trajectories again showed a strong correlation** with that seen in our pharmacophore models, with the exception of the 2ZU2-ZINC09411012 complex.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'correlation'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In this structure the pyridine ring gradually dissociated out of the S 1 pocket at** around 4 ns followed by almost complete dissociation of the ligand from the active site, where the 1,3-dioxolane became completely solvent exposed.', 'lgd': {'word': 'In', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'at'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The orientation of all S 1 subsite residues** including His162, Phe139, Glu165, His171 and Tyr160 remained unaltered and therefore could not attribute to this loss of stability.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'residues'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The movement may most likely be attributed to by an initial minor loss of stability caused by the movement of Ala143 closer to the pyridine ring, with subsequent bond formation between the Ala backbone and pyridine N. This movement destabilized the pyridine ring, which subsequently formed a new hydrogen** bond with Gly142.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'hydrogen'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**At this point the pyridine had almost completely lost its orientation in the S 1 pocket, which** resulted in further loss of the van der Waals interactions leading to complete dissociation from the pocket ( Figure 5 ).', 'lgd': {'word': 'At', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'which'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This movement also corresponded to a gradually increasing** RMSD, assuming few stable conformations, indicative of an unstable ligand.', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'increasing'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The orientation of the pyridine ring in 3TLO is somewhat stabilized by a bifurcated interaction** with the sidechain of Glu166 which promoted the movement of the pyridine N away from Ala143 without disrupting the interaction with His162.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'interaction'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Lead optimization steps may therefore preferentially incorporate a core structure** that prevents rotation of the pyridine by resulting steric clashes.', 'lgd': {'word': 'Lead', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'structure'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**As mentioned previously, the exploration of a phenylalanine substituent,** imidazole or lactam ring in this position may also contribute positively to the stability of the ligand within the S 1 pocket [46, 47, 77] .', 'lgd': {'word': 'As', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'substituent'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Interactions with the Glu165/6 were also highly conserved with a frequency of** 86% for 3TLO, 91% for OC43 3CL pro and 3V3M and 94% for 3D23.', 'lgd': {'word': 'Interactions', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The position of 164, occupied by a His in** 3V3M and a Gln in the remaining structures, showed varying degrees of bond formation, with the presence of the His in 3V3M resulting in a significant reduction in bond formation.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'in'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The sidechain of Gln189 was shown to form a hydrogen bond** with the ligand in 3V3M, 3D23 and OC43 3CL pro , however this residue was highly solvent exposed and therefore resulted in a high degree of flexibility of the sidechain, which should hinder the formation of this bond.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'bond'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In 3V3M the ligand was however able to stabilize the sidechain to maintain the formation of the bond with a high degree** of frequency at 73%.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'degree'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Ligand RMSD again showed similar profiles to the previously described complexes, where initially a large movement** was observed followed by subsequent stabilization of the ligand within the active site.', 'lgd': {'word': 'Ligand', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'movement'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Radius of gyration again confirmed RMSD results where all structures fluctuate between 4.7  and 5.0 , with the exception of 2ZU2 which deterred greatly at approximately 6 ns that signified the point at which the pyridine ring completely dissociated from the S 1** pocket.', 'lgd': {'word': 'Radius', 'seed': '1'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**These compounds are generally large and have a high likelihood** of off target effects through covalent modification and, as yet, there remains no commercially available molecular entity for the treatment of coronaviruses [14] .', 'lgd': {'word': 'These', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'likelihood'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Different techniques used in virtual screening have led to the identification of a diverse set** of lead compounds, where one technique is able to identify compounds the other may have missed [70, 71] .', 'lgd': {'word': 'Different', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'set'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**For this reason, we used a joint docking** and pharmacophore modeling based virtual screening strategy to identify a highly defined list of potential lead compounds.', 'lgd': {'word': 'For', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'docking'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Molecular dynamic simulations also provide a mechanistic explanation** for any ligand inactivity, which would be impossible to identify with in vitro techniques.', 'lgd': {'word': 'Molecular', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'explanation'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**As ligand inactivity may be a result of** a single misplaced substituent this mechanistic view point provides extensive information for future lead optimization.', 'lgd': {'word': 'As', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The essential pharmacophoric constraints, including an acceptor in the P 1 and hydrophobic** moiety in P 2 , used in both docking and pharmacophore modeling drug discovery methods would be expected to generate a hit-list with similar scaffolds.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'hydrophobic'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The final hit-list, containing a total of** 19 ligands (15 from molecular docking and four identified by pharmacophore modeling), however were highly divergent and represented 15 distinct clusters.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**As all cluster members were already preselected for features to fill the P 1 and P** 2 pockets, have similar estimated binding affinities and the structure clusters are significantly dissimilar, they represent 15 valuable scaffolds that can be further experimentally validated and to be used as a basis for future lead optimization and anti-coronaviral inhibitor discovery experiments.', 'lgd': {'word': 'As', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'P'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The pyridine ring that fills the S 1 pocket in** 3V3M is stable in both modelled ligand-protein complexes and with the known inhibitor, ML188, it is evident that this feature contributes positively to antiviral effects of the inhibitor [19] .', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'in'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**However, with the remaining 3CL pro , several complexes showed instability of the pyridine ring within the S 1** pocket.', 'lgd': {'word': 'However', 'seed': '1'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**We therefore propose that future lead optimization explores the effects of incorporating a lactam ring** in this position as it is known to better replicate the interactions of the natural substrate and is also effective against the SARS-CoV 3CL pro [46] .', 'lgd': {'word': 'We', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'ring'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A phenylalanine substituent and an imidazole ring have also been shown to be efficacious in this P 1 position [47,** 77] .', 'lgd': {'word': 'A', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '47'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The diverse and comprehensive approach of hit-list refinement also ensures a high likelihood** of the presence of bioactives within the list.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'likelihood'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This study provides a promising start** in the pursuit of active lead compounds for the broad spectrum inhibition of the 3CL pro associated with human coronaviruses.', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'start'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In conclusion, in the drug-design sphere, a variety of** computational approaches have been used at the various stages of the drug-design process in support of the experimental techniques.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Importantly, however, predictions from virtual screening methods are not meant to replace experimental affinity determination, but can complement the experimental methods by producing an enriched subset of a large chemical** database, with an increased possibility of the compounds actually binding to the drug target of interest [80, 81] .', 'lgd': {'word': 'Importantly', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'chemical'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### MERS coronavirus: diagnostics, epidemiology and transmission"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The first known cases of Middle East respiratory syndrome (MERS), associated with infection by a novel coronavirus** (CoV), occurred in 2012 in Jordan but were reported retrospectively.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'coronavirus'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Since then, MERS-CoV sequences have been found in a bat and** in many dromedary camels (DC).', 'lgd': {'word': 'Since', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'and'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Precisely how virus transmits to humans remains unknown but close and lengthy exposure appears to be a** requirement.', 'lgd': {'word': 'Precisely', 'seed': 'a'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In humans, MERS is mostly known as a lower respiratory** tract (LRT) disease involving fever, cough, breathing difficulties and pneumonia that may progress to acute respiratory distress syndrome, multiorgan failure and death in 20 % to 40 % of those infected.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'respiratory'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**An email from Dr Ali Mohamed Zaki, an Egyptian virologist working at the Dr Soliman Fakeeh Hospital in Jeddah in the Kingdom of Saudi Arabia (KSA) announced the first culture of a new coronavirus** to the world.', 'lgd': {'word': 'An', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'coronavirus'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The email was published on the website of the professional emerging diseases (ProMED) network on 20 th September 201**2 [1]** (**Fig.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'Fig'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**1) and described the first reported case, a** 60 year old man from Bisha in the KSA.', 'lgd': {'word': '1', 'seed': 'a'}, 'rgd': {'seed': '1', 'word': 'and'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This information led to the rapid discovery of a second case** of the virus, this time in an ill patient in the United Kingdom, who had been transferred from Qatar for care [2] .', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'case'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**As of 2 nd of September 2015, there have been 1,493 detections** of viral RNA or virus-specific antibodies across 26 countries (Additional file 1: Figure S1 ) confirmed by the World Health Organization (WHO), with over a third of the positive people dying (at least 527, 35 %) [3] .', 'lgd': {'word': 'As', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'detections'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Since that first report, a slow discovery** process over the following two to three years revealed a virus that had infected over 90 % of adult dromedary camels (DC; Camelus dromedarius) in the KSA [4] , also DCs across the Arabian Peninsula and parts of Africa that are a source of DC imports for the KSA [5] .', 'lgd': {'word': 'Since', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'discovery'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Subsequent epidemiology and research has identified the cell receptor as exopeptidase dipeptidyl peptidase 4 (DPP4; also called CD26); that MERS-CoV has a broad tropism,** replicating better in some cells lines and eliciting a more proinflammatory response than SARS-CoV; is widespread in DCs; has the potential to infect other animals and that MERS kills its human host more often than SARS did (20-40 % versus 9 % for SARS [14] ) [15] [16] [17] [18] [19] .', 'lgd': {'word': 'Subsequent', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'tropism'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': \"**Although DCs appear to suffer the equivalent of a 'common** cold' from MERS-CoV infection, in humans, the virus can be a more serious and opportunistic pathogen associated with the death of up to 40 % of reported cases.\", 'lgd': {'word': 'Although', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'common'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**It has yet to be established whether infections thought to have been acquired from an animal source produce a more severe** outcome than those spread between humans [20] .', 'lgd': {'word': 'It', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'severe'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Fever and gastrointestinal symptoms may form a prodrome, after** which symptoms decline, only to be followed by a more severe systemic and respiratory syndrome [25, 26] .', 'lgd': {'word': 'Fever', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'after'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**It also included roles for contact with a probable or** confirmed case or for travel or residence within the Arabian Peninsula.', 'lgd': {'word': 'It', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'or'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**From July 2013, the revised WHO case definition included the importance of seeking out and understanding the role of asymptomatic cases and from June 2014, the WHO definition more clearly stated that a confirmed case** included any person whose sample was RT-PCR positive for MERS-CoV, or who produced a seroconversion, irrespective of clinical signs and symptoms.', 'lgd': {'word': 'From', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'case'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The KSA definition of a case became** more strict on 13 th May 2014, relying on the presence of both clinical features and laboratory confirmation [33] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'became'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Testing of asymptomatic people was recommended against from December 2014 [34] , reinforced by a case definition** released by the KSA Ministry of Health in June 2015 [35] .', 'lgd': {'word': 'Testing', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'definition'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Interestingly in June 2015, an outbreak in South Korea followed a similar distribution** [39, 40] .', 'lgd': {'word': 'Interestingly', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'distribution'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Among laboratory confirmed cases, fever, cough and upper respiratory tract (URT) signs and symptoms usually occur first, followed within a week by** progressive LRT distress and lymphopaenia [37] .', 'lgd': {'word': 'Among', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'by'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Patients often present to a hospital with** pneumonia, or worse, and secondary bacterial infections have been reported [37, 41] .', 'lgd': {'word': 'Patients', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'with'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Children under the age of 14 years are rarely reported to be positive for MERS-CoV, comprising only 1.1 % (n** = 16) of total reported cases.', 'lgd': {'word': 'Children', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'n'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Between 1 st September 2012 and 2 nd December 2013, a** study described the then tally of paediatric cases in the KSA, which stood at 11 (two to 16 years of age; median 13 years); nine were asymptomatic (72 %) and one infant died [44] .', 'lgd': {'word': 'Between', 'seed': 'a'}, 'rgd': {'seed': '1', 'word': 'September'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In Amman, Jordan, 1,005 samples** from hospitalized children under the age of two years with fever and/or respiratory signs and symptoms were tested but none were positive for MERS-CoV RNA, despite being collected at a similar time to the first known outbreak of MERS-CoV in the neighbouring town of Al-Zarqa [45] .', 'lgd': {'word': 'In', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'samples'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A second trimester stillbirth occurred in a pregnant woman** during an acute respiratory illness and while not RT-rtPCR positive, the mother did subsequently develop antibodies to MERS-CoV, suggestive of recent infection [46] .', 'lgd': {'word': 'A', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'woman'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Her exposure history to a MERS-CoV** RT-rtPCR positive relative and an antibody-reactive husband, her incubation period and her symptom history met the WHO criteria for being a probable MERS-CoV case [46] .', 'lgd': {'word': 'Her', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'CoV'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Created using Geneious v8.1 [211]** and annotated using Adobe Illustrator.', 'lgd': {'word': 'Created', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '211'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Beneath this is a schematic depicting** the location of RT-PCR primers (blue arrows indicate direction) and oligoprobes (green rectangles) used in the earliest RT-rtPCR screening assays and conventional, semi-nested (three primers) RT-PCR confirmatory sequencing assays [47, 48] .', 'lgd': {'word': 'Beneath', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'depicting'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**However, as is well described, cell culture is a slow, specialised** and insensitive method [50] while PCR-based techniques are the preferred method for MERS-CoV detection.', 'lgd': {'word': 'However', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'specialised'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**2 ) have become a key diagnostic** and taxonomic target for CoV species identification.', 'lgd': {'word': '2', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'diagnostic'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**With less than 80 % identity between the amino acid sequence of MERS ORF 1ab and betacoronavirus relatives, Tylonycteris bat HKU4 and Pipistrellus bat HKU5, it can be concluded that it is a novel and** distinct virus.', 'lgd': {'word': 'With', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'and'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A different approach used a monoclonal antibody-**based capture ELISA targeting the MERS-CoV nucleocapsid protein with a sensitivity of 10 3 TCID 50 and 100 % specificity [62] .', 'lgd': {'word': 'A', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'antibody'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Demonstration of a seroconversion to** a MERS-CoV infection meets the current WHO definition of a case so optimized and thoroughly validated sero-assays employed alongside good clinical histories are useful to both identify prior MERS-CoV infection and help support transmission studies.', 'lgd': {'word': 'Demonstration', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'to'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Because serology testing is, by its nature, retrospective, it is usual to detect a viral footprint,** in the form of antibodies, in the absence of any signs or symptoms of disease and often in the absence of any viral RNA [63] .', 'lgd': {'word': 'Because', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'footprint'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Much of the Arabian Peninsula and all of the Horn of Africa lack baseline data describing the proportion of the community who may have been infected by a MERS-CoV.**', 'lgd': {'word': 'Much', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'CoV'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**However, sero-surveys have had widespread use in elucidating the role of DCs as a transmission source** for MERS-CoV.', 'lgd': {'word': 'However', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'source'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Also, no diagnostically relevant variation in neutralization activity have been found from among a range of** circulating tested MERS-CoV isolates and sera, so whole virus or specific protein-based sero-assays should perform equivalently in detecting serological responses to the single MERS-CoV serotype [49] .', 'lgd': {'word': 'Also', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Nor did IFA methods detect any sign of prior MERS-CoV infection among a small sample** of 130 healthy blood donors from another Hospital in Jeddah (collected between Jan and Dec 2012) [70] .', 'lgd': {'word': 'Nor', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'sample'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The study indicated that HCoV-HKU1 was a likely source** of crossreactive antigen in the whole virus IFA [70] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'source'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**IFA using recombinant proteins instead of whole-virus IFA, has been shown to be a more specific** tool [31] .', 'lgd': {'word': 'IFA', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'specific'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Since asymptomatic zoonoses have been posited [72] , an absence of antibodies to MERS-CoV among some humans who have regular and close contact with camels may reflect the rarity of actively infected animals at butcheries, a limited transmission** risk associated with slaughtering DCs [70] , a pre-existing cross-protective immune status or some other factor(s) resulting in a low risk of disease and concurrent seroconversion developing after exposure in this group.', 'lgd': {'word': 'Since', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'transmission'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Some sero-assays have bypassed the risks of working with infectious virus by creating transfected cells expressing recombinant portions of the MERS-CoV nucleocapsid and spike proteins [48, 73] , or using a recombinant lentivirus** expressing MERS-CoV spike protein and luciferase [74, 75] .', 'lgd': {'word': 'Some', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'lentivirus'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Similarly, a study of** four herdsmen in contact with an infected DC herd in Al-Ahsa, eight people who had intermittent contact with the herd, 30 veterinary surgeons and support staff who were not exposed to the herd, three unprotected abattoir workers in Al-Ahsa and 146 controls who were not exposed to DCs in any professional role, found none with serological evidence of past MERS-CoV infection using the ppNT assay [10] .', 'lgd': {'word': 'Similarly', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The implications for our ability to detect any response in mild or asymptomatic cases was not explored but may be a signifcant factor** in understanding exposure in the wider community.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'factor'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A Jordanian outbreak of acute LRT disease in a hospital in** 2012 was retrospectively found to be associated with MERS-CoV infection, initially using RT-rtPCR, but subsequently, and on a larger scale, through positivity by ELISA and IFA or MNT test.', 'lgd': {'word': 'A', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'in'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The ELISA used a recombinant nucleocapsid** protein from the group 2 betacoronavirus bat-CoV HKU5 to identify antibodies against the equivalent crossreactive MERS-CoV protein [71] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'nucleocapsid'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Detection of MERS-CoV infection using ELISA or S1 subunit protein microarray [84] is usually followed by confirmatory IFA and/ or a plaque-reduction** neutralization (PRNT) [69, 70, 85] or MNT test.', 'lgd': {'word': 'Detection', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'reduction'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': \"**In the largest study of human sera, a tiered diagnostic** process assigned both recombinant IFA and recombinant ELISA positive sera to 'stage 1' seropositivity.\", 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'diagnostic'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A stage 2 seropositive result additionally required a suitably titred** PRNT result [87] .', 'lgd': {'word': 'A', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'titred'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**MERS-CoV does not appear to be easily transmitted from DCs to humans, or perhaps it is [72] , but generally does not trigger a detectable immune** response if only mild disease or asymptomatic infection results.', 'lgd': {'word': 'MERS', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'immune'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Serology assays are in need of further validation in this area so care is required when moving newly developed diagnostic serology algorithms from a research setting** to one that informs public health decisions.', 'lgd': {'word': 'Serology', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'setting'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This was reinforced when a false positive** US case, purported to have been infected after a handshake and two face-to-face meetings, did not withstand further confirmatory analysis using a more specific, NT assay and was subsequently retracted [88, 89] .', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'positive'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The WHO recommends sampling from the LRT for MERS-CoV RT-rtPCR testing, especially when sample collection is delayed by a week or** more after onset of symptoms.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'or'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The URT is a less invasive** and more convenient sampling site however, and an oropharyngeal and throat swab or a nasopharyngeal aspirate/wash are recommended when URT sampling is to be conducted [90] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'invasive'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Paired sera, collected two to three weeks apart are preferable for serological testing while a single sample** is suggested to be sufficient if collected two weeks after onset of disease or a single serum collected during the first 10-12 days if conducting RT-rtPCR [53, 90] .', 'lgd': {'word': 'Paired', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'sample'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**RNAaemia may also correlate with disease severity; signs of virus were cleared from the serum of a recovered patient,** yet lingered until the death of another [92] .', 'lgd': {'word': 'RNAaemia', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'patient'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This may be due to poor sampling when a cough is** absent or non-productive or because the viral load is low [95] .', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'is'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**At writing, no human data exist to define whether the virus replicates solely or preferentially in the LRT or URT, or replicates in other human tissues in vivo although MERS-CoV RNA has been detected from both the URT and LRT in a macaque monkey** model [100] .The distribution of DPP4 in the human upper airways is also not well described.', 'lgd': {'word': 'At', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'monkey'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**[25, 69, 99, 101] In one instanc**e, a** HCW s**hed viral RNA for 42 days in the absence of disease [99] .', 'lgd': {'word': '25', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'shed'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**When viruses are sought, they have included human herpesvirus (HHV), rhinoviruses (HRV), enteroviruses (EV), respiratory syncytial virus (RSV), parainfluenzavirus types 1, 2** and 3 (PIVs),influenzaviruses (IFVs), endemic HCoVs, adenoviruses (AdVs) metapneumovirus (MPV) and influenza A\\\\H1N1 virus; co-detections with MERS-CoV have been found on occasion [2, 22, 37, 69, 97] .', 'lgd': {'word': 'When', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '2'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**[107] [108] [109] Over time, increased influenza vaccination has been credited f**or a** fall** in the prevalence of influenza like illnesses among Hajj pilgrims.', 'lgd': {'word': '107', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'in'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**[110] A LRT sample is often not collected for these studies [98, 107, 109] , so false negative findings a**re a** possibility altho**ugh little is known about the initial site of MERS-CoV infection and replication; it may have been assumed it was the LRT because disease was first noticed there but the URT may be the site of the earliest replication.', 'lgd': {'word': '110', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'although'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**3 ), there was a rapid increase** in MERS cases, accompanied by intense screening; approximately 5,000 samples from in and around the region were tested in a month yielding around 140 MERS-CoV detections (~3 % prevalence) [111] .', 'lgd': {'word': '3', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'increase'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Among 5,065 individuals sampled and tested across the KSA between October 2012 and September 2013,108 (2.1 %) detections** were made in a hospital-centric population which included hospitalized cases (n = 2,908; 57.4 %), their families (n = 462; 9.1 %) and associated HCWs (n = 1,695; 33.5 %) [32] .', 'lgd': {'word': 'Among', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'detections'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**[112] However, the proportion of deaths among those infected with MERS-CoV is much higher than that known for the HCoVs NL63, HKU1, 229E or OC43 in other countries, and even above that for SARS-CoV; it is n**ot a** virus t**hat could reasonably be described as a \"storm in a teacup\".', 'lgd': {'word': '112', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'that'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This definition made no specific allowance for acquisition from DCs through a droplet-based** route, which is very likely route for acquisition of a virus that initially and predominantly causes respiratory disease [23] .', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'based'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Providing support for a droplet transmission** route and perhaps indicating the presence of RNA in smaller, drier droplet nuclei, MERS-CoV RNA was identified in a high volume air sample collected from a barn housing an infected DC [117] .', 'lgd': {'word': 'Providing', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'transmission'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**These factors may prove important for human cases who do not describe any DC contact [119] nor any contact with a confirmed case.**', 'lgd': {'word': 'These', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'case'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Wording focuses on consumption of DC products but does not specifically ascribe risk to a droplet route** for acquisition of MERS-CoV from DC [120] .', 'lgd': {'word': 'Wording', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'route'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**What constitutes a definition of** \"contact\" during these interviews has been defined for one study [72] .', 'lgd': {'word': 'What', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Conclusive evidence supporting bats as a source for** human infections by MERS-CoV has yet to be found, but bats do appear to host ancestral representatives [53, 125] .', 'lgd': {'word': 'Conclusive', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'for'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**However, these are not variants of the same virus nor always within the same phylogenetic lineage as MERS-CoV; they are each a genetically distinct** virus.', 'lgd': {'word': 'However', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'distinct'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Bat-to-human infection by MERS-CoV is a purely speculative** event.', 'lgd': {'word': 'Bat', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'speculative'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The only piece of MERS-CoV-specific evidence pointing to bats originates from amplification of a 190 nt** fragment of the RNAdependent RNA polymerase gene of the MERS-CoV genome, identified in a faecal pellet from an insectivorous Emballonuridae bat, Taphozous perforatus found in Bisha, the KSA [121] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'nt'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**While very short, the sequence of the fragment defined it as a diagnostic discovery.**', 'lgd': {'word': 'While', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'discovery'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Subsequently a link to DCs was reported [85] and that link has matured into a** verified association [38, 126] (Fig.', 'lgd': {'word': 'Subsequently', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'to'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Earlier and subsequent versions of this chart are maintained on a personal blog** [210] .', 'lgd': {'word': 'Earlier', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'blog'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Virus Res 2015 Vol 202:60-88 with permission from Elsevier [5] DCs, which make up 95 % of all camels, have a central presence** in the Arabian Peninsula where human-DC contact ranges from little to close [119] .', 'lgd': {'word': 'Virus', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'presence'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Despite camel butchery being a local occupation,** neither butchers nor other at-risk groups are identifiable among MERS cases; this may simply be a reporting issue rather than an unexplainable absence of MERS.', 'lgd': {'word': 'Despite', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'occupation'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**DCs were sampled from a mostly Canary** Island-born herd and from Omani DCs (originally imported from the Horn of Africa) [85] .', 'lgd': {'word': 'DCs', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'Canary'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**c Hypothetical ways in which subclinical (when infection may not meet a previously defined** clinical threshold of signs and/or symptoms) or asymptomatic (no obvious signs or measured, noticed or recalled symptoms of illness) MERS-CoV infection may be implicated in transmission DC sera could neutralise MERS-CoV while all Omani DC sera had high levels of specific MERS-CoV neutralizing antibody [85] .', 'lgd': {'word': 'c', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'defined'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This indicated that DCs had in the past been infected by MERS-CoV, or a very similar** virus.', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'similar'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Since this study, a host of** peer-reviewed reports have looked at both DCs and other animals, and the possibility that they may host MERS-CoV infection.', 'lgd': {'word': 'Since', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**No virology or serology studies of human samples from areas in Africa where there are camels with a history of** MERS-CoV have been reported to date.', 'lgd': {'word': 'No', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**However,an absence of unexplained pneumonia that may be attributable to MERS-CoV infection may not signal the absence of virus among humans in each country but simply reflect a lack of** expensive epidemiology studies conducted by resource-poor countries.', 'lgd': {'word': 'However', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Researchers sought to determine a direction for** infection; were DCs transmitting virus to humans or were humans infecting DCs?', 'lgd': {'word': 'Researchers', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'for'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**At a Qatari site, a** farm owner and his employee became ill in mid-October 2013 and tested positive for MERS-CoV RNA in a sputum and throat swab sample, respectively.', 'lgd': {'word': 'At', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'site'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The results indicated a recent outbreak** had occurred in this herd; the first indication of MERS-CoV RNA found within DCs with a temporal association to human infections.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'outbreak'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Three positive DC samples were confirmed by sequencing a 358 nt** portion of the spike gene; these sequences were identical to each other, again with close homology to other human and DC MERS-CoV sequences [138] .', 'lgd': {'word': 'Three', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'nt'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The DCs and human contacts yielded ORF1a and ORF4b sequences differing by only a single nucleotide** each, clustering closely with the Hafr-Al-Batin_1_2013 variant [138] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'nucleotide'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Subsequent case studies found evidence of a concurrent human** and DC infection and the direction of that infection was inferred to be from the ill DCs and to their human owners [117, 142, 146] .', 'lgd': {'word': 'Subsequent', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'human'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Partial genome sequences indicated that a human and** a MERS-CoV RT-rtPCR positive DC had been infected by a variant of the same virus, harbouring the same distinct pattern of nucleotide polymorphisms.', 'lgd': {'word': 'Partial', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'and'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': \"**[142] All nine DC in the owner's herd, serially sampled, reacted **in a** recombinant** S1 antigen ELISA, with the two animals that had been RT-rtPCR positive showing a small, verifiable rise in antibody titre [142] .\", 'lgd': {'word': '142', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'S1'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The authors suggested that the rise in titre in DC sera which occurred alongside a declining RNA** load, while the patient was actively ill and hospitalized, indicated that the DCs were infected first followed by the owner [117, 142] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'RNA'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**3 ) and this may be a time when** there is increased risk to humans of spill-over due to new infections among nave DC populations [128] .', 'lgd': {'word': '3', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'when'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Juvenile DCs appear to host active infection more often than adult DCs and thus the sacrificial slaughter of DCs, which must be five years of age or older (termed a thane), may** not be accompanied by significant risk of exposure to infection.', 'lgd': {'word': 'Juvenile', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'may'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': 'In **a subsequent study,** MERS-CoV RNA was identified in the milk, nasal secretion and faeces of DCs from Qatar [151] .', 'lgd': None, 'rgd': {'seed': 'a', 'word': 'study'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**By comparison, a well known** and efficently transmitted respiratory virus, influenza A virus, could not be recovered in culture beyond four hours under any conditions [150] .', 'lgd': {'word': 'By', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'known'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': \"**For context, pathogenic bacteria can remain viable and airborne for 45 min in a coughed aerosol** and can spread 4 m. MERS-CoV's ability to remain viable over long time periods gives it the capacity to thoroughly contaminate a room's surfaces when occupied by an infected and symptomatic patient [153] .\", 'lgd': {'word': 'For', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'aerosol'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The provision of evidence supporting the best formulation of personal protective equipment to be worn by HCWs who receive, manage or conduct procedures on infectious cases remains a** priority.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**However, transmission of MERS-CoV is defined as sporadic (not sustained), intra-familial, often healthcare associated, inefficient and requiring close and prolonged contact [22, 31, 63, 93, 97, 102, 156] In a household study,** 14 of 280 (5 %) contacts of 26 MERS-CoV positive index patients were RNA or antibody positive; the rate of general transmission, even in outbreaks is around 3 % [31] .', 'lgd': {'word': 'However', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'study'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**That is to say, the basic reproduction number (R 0 ) -the average number of infections caused by one infected individual in a fully susceptible** populationhas been close to one throughout various clusters and outbreaks.', 'lgd': {'word': 'That', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'susceptible'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**If R 0 was greater than 1, a** sustained increase in case numbers would be expected.', 'lgd': {'word': 'If', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'a'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Some R o calculations may be affected by incomplete case contact tracing, limited community testing and how a case is** defined.', 'lgd': {'word': 'Some', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'is'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**That MERS has had a constant presence** in the Arabian Peninsula since 2012 is due to ongoing, sporadic spill-over events from DCs amplified by poorly controlled hospital outbreaks.', 'lgd': {'word': 'That', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'presence'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The first known MERS human-to-human transmission event was one characterized by acute LRT disease in a healthcare setting** in Jordan.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'setting'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In stark contrast, a sero-survey** of HCW who were sometimes in close and prolonged contact with the first, fatal MERS-CoV case in 2012 [162] , found none of the HCW had seroconverted four months later, despite an absence of eye protection and variable compliance with required PPE standards [162] .', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'survey'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Case-control studies were not a** focus.', 'lgd': {'word': 'Case', 'seed': 'a'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A rise in the cases termed asymptomatic (which enlarge the denominator for calculations of the proportion of fatal cases, defined in [164] ) resulted in a drop in** the proportion of fatal cases during the Jeddah-2014 outbreak.', 'lgd': {'word': 'A', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'in'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Historically, such rises are consistent with changing definitions and laboratory responses and clinical management of a newly discovered** virus infection that was first noted only among the severely ill.', 'lgd': {'word': 'Historically', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'discovered'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**However the proportion of male deaths from total males with MERS is a similar figure** to that for females.', 'lgd': {'word': 'However', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'figure'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In the KSA, there is a greater proportion** of younger males among cases and deaths than were observed from the 2015 South Korean or the Jeddah-2014 outbreaks (Additional file 2: Figure S2 ).', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'proportion'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Why these aspects have differed may be due to differences in the time to presentation and diagnosis, the nature and quality of supportive care, the way a person became** infected (habits, exposure to a human or zoonotic source, viral load, route of infection) or the extent to which different populations are burdened by underlying diseases [40] .', 'lgd': {'word': 'Why', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'became'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': 'As **a group, HCWs** comprised 16 % of MERS cases in the KSA and South Korea.', 'lgd': None, 'rgd': {'seed': 'a', 'word': 'HCWs'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In May 2013, the WHO published guidelines for IPC during care of probable or confirmed cases of MERS-CoV infection in a healthcare setting** [166] .', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'setting'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Healthcare facilities have been a regular target** for suggested improvements aimed at improving infection prevention and control (IPC) procedures [115, 118] .', 'lgd': {'word': 'Healthcare', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'target'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Earlier and subsequent versions of this chart are maintained on a personal blog** [210] length coverage in a single experiment with highly repetitious measurement of each nucleotide position [52, 140] .', 'lgd': {'word': 'Earlier', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'blog'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Two studies during 2015, one looking at Jeddah-2014 MERS-CoV variants and another looking at a variant exported** from South Korea to China, have now identified signs of genetic recombination among MERS-CoV variants.', 'lgd': {'word': 'Two', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'exported'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Over a month, a** DC virus sequenced on different occasions did not change at all indicating a degree of genomic stability in its host, supporting that DCs are the natural, rather than intermediate, host for the MERS-CoV we know today [77] .', 'lgd': {'word': 'Over', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'a'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This neighbour joining tree was created in MEGA v6 using an alignment of human and DCderived MERS-CoV sequences (Geneious v8.1 [211]** ).', 'lgd': {'word': 'This', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '211'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Genome sequence located to a distinct clade** were identified from an Egyptian DC that was probably imported from Sudan.', 'lgd': {'word': 'Genome', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'clade'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A virus sequenced from a Neoromicia capensis** bat was more closely related to MERS-CoV than other large bat-derived sequences had been to that point, but the genome of a variant of a MERS-CoV has yet to be discovered and deduced from any bat [125] .', 'lgd': {'word': 'A', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'capensis'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**At least nine MERS-CoV genomes contained amino acid substitutions in the receptor binding domain (RBD) of the spike protein and codons 158 (N-terminal region), 460 (RBD), 1020 (in heptad repeat 1), 1202** and 1208 bear investigation as markers of adaptive change [140, 169] .', 'lgd': {'word': 'At', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '1202'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The spike protein had not changed in the recombinant MERS-CoV genome identified in China in 2015 but was reported to have varied at a higher rate** than that for complete MERS-CoV genomes, among South Korean variants [172, 178] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'rate'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Despite this, one assay amplifying a 615 nucleotide** fragment of the spike S2 domain gene for Sanger sequencing agreed with the results generated by the sequencing of a some full genomes and was useful to define additional sequence groupings [177] .', 'lgd': {'word': 'Despite', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'nucleotide'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Genomic sequence can also be used to define the geographic boundaries of a cluster or** outbreak and monitor its progress, based on the similarity of the variants found among infected humans and animals when occurring together, or between different sites and times (Fig.', 'lgd': {'word': 'Genomic', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'or'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This approach was employed when defining the geographically constrained MERS hospital outbreak in Al-Ahsa, which occurred between 1 st April** and 23 rd May 2013, as well as clusters in Buraidah and a community outbreak in Hafr Al-Batin, the KSA.', 'lgd': {'word': 'This', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'April'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Genomic sequencing identified that approximately 12 MERS-CoV detections from a community outbreak** in Hafr Al-Batin between June and August 2013 may have been triggered by an index case becoming infected through DC contact [175] .', 'lgd': {'word': 'Genomic', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'outbreak'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Molecular epidemiology has revealed otherwise hidden links in transmission chains encompassing a period of** up to five months [179] .', 'lgd': {'word': 'Molecular', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In Riyadh-2014, genetic evidence supported the likelihood of multiple external introductions of virus, implicating a range of** healthcare facilities in an event that otherwise looked contiguous [23, 168, 179] .', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Riyadh is a nexus for** camel and human travel and has had more MERS cases than any other region of the KSA to date but also harbours a wide range of MERS-CoV variants [128, 167, 179] .', 'lgd': {'word': 'Riyadh', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'for'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**However the South Korean outbreak originated from a single infected** person, resulting in three to four generations of cases [180, 181] .', 'lgd': {'word': 'However', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'infected'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Contact tracing is essential to contain the emergence and transmission of a new virus** and today it is supported by molecular epidemiology.', 'lgd': {'word': 'Contact', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'virus'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**For example, there were 83 contacts, both symptomatic and asymptomatic, of a case treated** in Germany who travelled from the UAE but no sign of virus or antibody were found in any of them [73] .', 'lgd': {'word': 'For', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'treated'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In a study of 123 contacts of a** case treated in France, only seven matched the definition for a possible case and were tested; one who had shared a 20 m 2 hospital room while in a bed 1.5 m away from the index case for a prolonged period was positive [26] .', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Analyses of public data reveal many likely instances of nosocomial acquisition of infection in the Arabian Peninsula and these data may be accompanied by some details noting contact with a known case** or facility.', 'lgd': {'word': 'Analyses', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'case'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**One example identified the likely role of a patient with** a subclinical infection, present in a hospital during their admission for other reasons, as the likeliest index case triggering a family cluster [93] .', 'lgd': {'word': 'One', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'with'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Contact tracing was a significant factor** in the termination of a 2015 outbreak involving multiple South Korean hospitals [184] .', 'lgd': {'word': 'Contact', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'factor'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Such studies demonstrate the necessity of finding and understanding a role for** mild and asymptomatic cases, together with restricting close contact or prolonged exposure of infected people to others, especially older family members and friends with underlying disease (Fig.', 'lgd': {'word': 'Such', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'for'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The outbreak was mostly (>60 % of cases) associated with human-to-human spread within hospital environments and resulted from a lack of,** or breakdown in, infection prevention and control [37, 185, 186] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This was closely followed by a distinct outbreak** in Ar Riyad province in the KSA which appeared to come under control in early November.', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'outbreak'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**After staying in Bahrain for two weeks, a 68 year** old male (68 M) travelled home to South Korea via Qatar, arriving free of symptoms on the 4 th May 2015 [187] .', 'lgd': {'word': 'After', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'year'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**He developed fever, myalgia and a cough nearly** a week later (11 th ).', 'lgd': {'word': 'He', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'nearly'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**He visited a clinic as** an outpatient between the 12 th and 15 th of May and was admitted to Hospital A on the 15 th [188] .', 'lgd': {'word': 'He', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'as'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**During this second stay, a sputum sample** was taken and tested positive for MERS-CoV on the 20 th [187, 188] , triggering transfer to the designated isolation treatment facility.', 'lgd': {'word': 'During', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'sample'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Over a period of 10 days, the index case was seen at three different hospitals, demonstrating a** key feature of \"hospital shopping\" that shaped the South Korean outbreak.', 'lgd': {'word': 'Over', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In total 186 cases were generated in this outbreak, all linked through a single transmission** chain to 68 M; 37 cases died [189] .', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'transmission'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In South Korea, the national health insurance system provides for relatively low cost medical care, defraying some costs by making family members responsible for a portion of** the ministration of the sick, resulting in them sometimes staying for long periods in the rooms that often have more than four beds in them [24] .', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Other factors thought to have enabled this outbreak included unfamiliarity of local clinicians with MERS, ease with which the public can visit and be treated by tertiary hospitals, the custom of visiting sick friends and relatives in hospitals, the hierarchical nature of Korean society, crowded emergency rooms, poor IPC measures, a lack of** negative pressure isolation rooms and poor inter-hospital communication of patient disease histories [24, [190] [191] [192] .', 'lgd': {'word': 'Other', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The hospitals involved were initially not identified, governmental guidance and actions produced confusing messages and there was very limited communication at all early on which resulted in unnecessary concern, distrust and a distinct economic** impact [191, [196] [197] [198] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'economic'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Early in the outbreak, a infected traveller,** the son of an identified case in South Korea, passed through Hong Kong on his way to China where he was located, isolated and cared for in China [91, 199, 200] .', 'lgd': {'word': 'Early', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'traveller'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A review of public data showed that, as for MERS in the KSA, older age and the presence of underlying disease were significantly associated with a fatal outcome** in South Korea.', 'lgd': {'word': 'A', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'outcome'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**[40] Even though R 0 i**s <1,** su**per-spreading events facilitated by circumstances created in healthcare settings and characterized by cluster sizes over 150, such as this one, are not unexpected from MERS-CoV infection [204] .', 'lgd': {'word': '40', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'super'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In the region of Ar Riyad, including the capital city of Riyadh, a hospital based** cluster began, within a single hospital, from late June 2015 [205] .', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'based'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This opportunistic virus has had its greatest impact on those with underlying diseases and such vulnerable people, sometimes suffering multiple comorbidities, have been most often associated with hospitals, creating a perfect storm** of exposure, transmission and mortality.', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'storm'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**It remains unclear if this group are uniquely affected by MERS-CoV or if other respiratory virus infections, including those from HCoVs, produce a similarly serious** impact.', 'lgd': {'word': 'It', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'serious'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In South Korea, a single imported** case created an outbreak of 185 cases and 36 deaths that had a disproportionate impact on economic performance, community behaviour and trust in government and the health care system.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'imported'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Vigilance remains important for containment since MERS-CoV is a virus with** a genetic makeup that has been observed for only three years and is not stable.', 'lgd': {'word': 'Vigilance', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'with'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Global alignment of case definitions would further aid accurate calculation of a case fatality** ratio by including subclinical case numbers.', 'lgd': {'word': 'Global', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'fatality'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Whole genome sequencing has been used extensively to study MERS-CoV travel and variation and although it remains a tool for** experts, it appears to be the best tool for the job.', 'lgd': {'word': 'Whole', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'for'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**For the viruses, MERS-CoV has a broader tropism,** grows more rapidly in vitro, more rapidly induces cytopathogenic change, triggers distinct transcriptional responses, makes use of a different receptor, induces a more proinflammatory state and has a delayed innate antiviral response compared to SARS-CoV.', 'lgd': {'word': 'For', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'tropism'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**There appears to be a 2-3** % prevalence of MERS-CoV in the KSA with a 5 % chance of secondary transmission within the household.', 'lgd': {'word': 'There', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': '3'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**There is an increased risk of infection through certain occupations at certain times and a much greater** chance for spread to other humans during circumstances created by humans, which drives more effective transmission than any R 0 would predict on face value.', 'lgd': {'word': 'There', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'greater'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**There is no evidence that MERS-CoV is a virus of** pandemic concern.', 'lgd': {'word': 'There', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**As long as we facilitate the spread of MERS-CoV among our most vulnerable populations, the world must remain on alert for cases which may be exported more frequently when a host country** with infected camel reservoirs is experiencing human clusters or outbreaks.', 'lgd': {'word': 'As', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'country'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Serological tools continue to emerge but they are in need of further validation using samples from mild and asymptomatic infections and a densely sampled** cohort study to follow contacts of new cases may address this need.', 'lgd': {'word': 'Serological', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'sampled'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Additional file 1: Figure** S1 .', 'lgd': {'word': 'Additional', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'Figure'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### X-ray Structural and Biological Evaluation of a Series of Potent and Highly Selective Inhibitors of Human Coronavirus Papain-like Proteases"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Structure-guided design was used to generate a series of** noncovalent inhibitors with nanomolar potency against the papain-like protease (PLpro) from the SARS coronavirus (CoV).', 'lgd': {'word': 'Structure', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Selectivity and cytotoxicity studies established a more than** 100-fold preference for the coronaviral enzyme over homologous human deubiquitinating enzymes (DUBs), and no significant cytotoxicity in Vero E6 and HEK293 cell lines is observed.', 'lgd': {'word': 'Selectivity', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'than'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**X-ray structural analyses of inhibitor-bound crystal structures revealed subtle differences between binding modes of the initial benzodioxolane lead (15g) and the most potent analogues 3k and 3j, featuring a monofluoro substitution** at para and meta positions of the benzyl ring, respectively.', 'lgd': {'word': 'X', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'substitution'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**SARS-CoV was established as the causative agent of the fatal global outbreak of respiratory disease in humans during 20022003 that resulted in a case-fatality** rate (CFR) of 11%.', 'lgd': {'word': 'SARS', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'fatality'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**1 In October** 2012, the Centers for Disease Control and Prevention (CDC) added SARS-CoV to the select agents list of the Department of Health and Human Services (HHS).', 'lgd': None, 'rgd': {'seed': '1', 'word': 'October'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Among many aspects that make SARS-CoV a potential threat** to the human population, the lack of effective vaccines or anticoronaviral drugs had a significant impact in its classification as a select agent.', 'lgd': {'word': 'Among', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'threat'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**However, even with the most extensive preventive measures, the reemergence of SARS-CoV or other virulent human coronaviruses poses a continuing threat.**', 'lgd': {'word': 'However', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'threat'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A powerful reminder of this, as well as of the fatal repercussions of the interspecies transmission potential of CoVs, was brought to the forefront in September 2012 by the emergence of a new SARSlike** respiratory virus (previously termed HCoV-EMC, now designated Middle East respiratory syndrome coronavirus, MERS-CoV).', 'lgd': {'word': 'A', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'SARSlike'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**5 Reminiscent of the initial stages of SARS-CoV pandemic, global travel has contributed to the spread of MERS coronavirus, with a total of** 178 laboratory-confirmed cases and a CFR of 43%.', 'lgd': {'word': '5', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**6 The infected individuals display SARS-like symptoms, including a severe respiratory** infection (SRI), and sometimes exhibit an acute renal failure which is a unique signature of MERS infection.', 'lgd': {'word': '6', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'respiratory'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**2b,7 Today, a total of** 6 human coronaviruses are known, of which SARS-CoV and MERS-CoV are recognized as highly pathogenic with the potential for human-to-human transmission.', 'lgd': {'word': '2b', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': 'Although **a number of** coronaviral proteins have been identified as potential drug targets, 9 further development of drug candidates has been compromised by the general lack of antiviral data and biological evaluations, which can be done only in BSL-3 facilities with select agent certification for laboratories in the U.S. Two of the most promising drug targets are the SARS-CoV-encoded cysteine proteases, 3CLpro (chymotrypsin-like protease) and PLpro (papain-like protease).', 'lgd': None, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**PLpro, in addition to playing an essential role during virus replication, is proposed to be a key enzyme** in the pathogenesis of SARS-CoV.', 'lgd': {'word': 'PLpro', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'enzyme'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**13 The SARS-CoV PLpro inhibitors (compounds 24 14 and 15g,h 15 ), previously identified in our lab via high-throughput screening (HTS), have low micromolar inhibitory potency with minimal associated cytotoxicity in SARS-CoV-infected Vero E6 cells and are therefore viable leads for the development of drug candidates ( Figure 1** ).', 'lgd': {'word': '13', 'seed': '1'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Compounds 24, 15h, and 15g share a number of** chemical and structural features (Figure 1 ), including the presence of a naphthyl group adjacent to a stereogenic center containing a methyl group and a nitrogen-centered hydrogen bond (Hbond) donor (at a physiological pH).', 'lgd': {'word': 'Compounds', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Structureactivity relationships (SARs) have shown that in both inhibitor subtypes the naphthyl ring is optimally substituted at C-1 and the** presence of the methyl group at the aforementioned stereogenic center is important for potency.', 'lgd': {'word': 'Structure', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'the'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**14, 15 Interestingly, SARS-CoV PLpro exhibited significant stereopreference for the (R)-enantiomer of 24 (IC 50 = 0.6 M, EC 50 = 15 M), 14 while minimal stereochemical selectivity was observed between enantiomers (R)-15g and (S)-15h (IC 50 values of 0.67 and 0.56 M, respectively; EC 50 of 9.1 M for** both).', 'lgd': {'word': '14', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'for'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': \"**14, 15 Specifically, there is a highly mobile** -turn/loop (Gly 267 -Gly 272 ) adjacent to the active site that closes upon inhibitor binding, thereby changing the orientation of the enzyme's backbone to allow for H-bonding with the inhibitor's core.\", 'lgd': {'word': '14', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'mobile'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**However, relative to the conformation adopted with the smaller inhibitor 24, compound 15g requires a different and** slightly more opened conformation to accommodate the longer piperidine-4-carboxamide scaffold and the bulkier 1,3-benzodioxole ring.', 'lgd': {'word': 'However', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'and'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Toward this goal, we utilized our previous SAR analysis and the X-ray structure of compound 15g in complex with SARS-CoV PLpro to develop a second generation** of PLpro inhibitors.', 'lgd': {'word': 'Toward', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'generation'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Here, we disclose SAR of a new series** of potent SARS-CoV PLpro inhibitors, their selectivity, antiviral efficacy, biological evaluation, and a detailed report of the molecular interactions between SARS-CoV PLpro and inhibitors 3k and 3j aided by X-ray crystallographic studies.', 'lgd': {'word': 'Here', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'series'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In addition, we demonstrate the effective inhibition of PLP2 from HCoV-NL63, a potentially fatal** pathogen in children and elderly, by compound 3k and related analogues.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'fatal'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**(1) from a** primary HTS from which lead 24 was developed; (B) hit (2) from a primary HTS from which 15g and 15h were developed.', 'lgd': {'word': '1', 'seed': 'a'}, 'rgd': {'seed': '1', 'word': 'from'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The appropriate nitriles 8a,c were treated with 1-naphthyl Grignard** reagent, and the intermediary imines were hydrolyzed to the ketones 9a,c under acidic conditions (Scheme 1).', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'Grignard'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The displacement route used in Schemes 1 and 2** was not successful with the more hindered phenyl analogue 1d, so we had to employ the lengthier approach previously reported by Ghosh et al.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '2'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '15 **1-Naphthonitrile 18** was treated with benzylmagnesium chloride, and the resulting imine was hydrolyzed under acidic conditions to the ketone 19.', 'lgd': None, 'rgd': {'seed': '1', 'word': '18'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This amine was then condensed with the bis-aldehyde derived from dimethyl 2,2-bis((1,3-dioxolan-**2-yl)methyl)malonate to give the dihydropyridine dimethyl ester 21, which was readily reduced to the piperidine 22 by heterogeneous hydrogenation.', 'lgd': {'word': 'This', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'dioxolan'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The starting (R)-1-(1-**(naphthalen-1-yl)ethyl)-piperidine-4-carboxylic acid 25 was prepared according to the method of Ghosh et al.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '1'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Alternatively, carbinol 27d was prepared from 1-cyanoisoquinoline by** addition of methylmagnesium chloride followed by sodium borohydride reduction of the resulting ketone, as described in the Experimental Section.', 'lgd': {'word': 'Alternatively', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'by'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Conversion to piperidine ester 33 was effected with a three-step** sequence analogous to that Scheme 5.', 'lgd': {'word': 'Conversion', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'step'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Synthesis of Quinolinyl-and Isoquinolinylamides a employed in** Scheme 3.', 'lgd': {'word': 'Synthesis', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'in'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Saponifiction with LiOH in a mixture (3:**1:1) of THF/MeOH/H 2 O furnished the free acid.', 'lgd': {'word': 'Saponifiction', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': '3'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**It is worth noting that the shorter ethyl isonipecotate displacement route used in Schemes 1, 2,** and 5 was not successful with the mesylate derived from reduction/mesylation of 29, presumably because of excessive steric hindrance.', 'lgd': {'word': 'It', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '2'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Compound 15g is a competitive inhibitor** of SARS-CoV PLpro that displays potent enzymatic inhibition (IC 50 = 0.67 M) and low micromolar antiviral activity against SARS-CoV.', 'lgd': {'word': 'Compound', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'inhibitor'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**It evolved from a small SAR** study on the HTS hit 2 ( Figure 1 ) and is therefore an attractive lead for the development of anti-SARS drug candidates.', 'lgd': {'word': 'It', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'SAR'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**15 Compound 15g is composed of the central piperidine-4-carboxamide core decorated with 1-naphthalenylethyl and** 1,3-benzodioxolylmethyl substituents at the piperidinyl and carboxamide nitrogens, respectively.', 'lgd': {'word': '15', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'and'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Previous SAR has established the preference for the 1-naphthyl over** the 2naphthyl substitution pattern and the requirement of a single methyl substituent at the methylene linker for effective inhibition (when compared to both unsubstituted and gemdimethyl derivatives), with minimal stereochemical preference.', 'lgd': {'word': 'Previous', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'over'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**X-ray crystallographic studies of the SARS-CoV PLpro15g complex show that the methyl group of the (R)-enantiomer extends into a small pocket** that has both hydrophobic and polar features and is filled with water molecules.', 'lgd': {'word': 'X', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'pocket'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**To explore the dimensions and availability of this pocket to substituents as well as H-bond opportunities, the first set of analogues explores the incorporation of larger or polar substituents (1a, -Me; 1b, -OH; 1c, -OMe; 1d, -Ph) at the methyl group (position R 1 in Table**  1 ).', 'lgd': {'word': 'To', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'Table'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Addition of substituents to R 1 resulted in** higher IC 50 values for all substitutions, and loss of inhibitory potency was proportional to substituent size.', 'lgd': {'word': 'Addition', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'in'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**It also appears that the potential entropic gain by displacing the water molecules with larger R 1 groups is** not achieved likely because of a larger enthalpic penalty of breaking the H-bonds formed between these water molecules and Asp 165 , Arg 167 , Tyr 274 , Thr 302 , and Asp 303 .', 'lgd': {'word': 'It', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'is'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': 'As **a result, unlike** the previously reported predictions by fragment mapping program (FTMap) in which all water molecules were removed before computational analyses, 20 here we demonstrate experimentally that this pocket is unlikely to provide any extra ligand-binding sites or room for larger substituents.', 'lgd': None, 'rgd': {'seed': 'a', 'word': 'unlike'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**We next examined the effect of incorporating substituents at the benzylic position R 2 to form a second stereogenic** carbon, using unsubstituted compound 2a (IC 50 = 2.2  0.1 M) as the comparator.', 'lgd': {'word': 'We', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'stereogenic'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Diastereoisomeric epimers 2b and 2c, with a methyl group** at R 2 , decreased the inhibitory activity by a comparable extent (IC 50 of 13.5  1.2 and 12.7  0.3 M, respectively), indicative of a steric clash with SARS-CoV PLpro.', 'lgd': {'word': 'Diastereoisomeric', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'group'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Interestingly, a 10-fold** stereopreference was observed for (S)methoxymethyl 2e (IC 50 = 1.9  0.1 M), compared to epimeric 2d (IC 50 = 18.0  1.9 M), which could be indicative of a H-bond in that region or simply the ability of the more active enantiomer 2e to retain the methoxy in a solventexposed environment, thereby avoiding a desolvation penalty.', 'lgd': {'word': 'Interestingly', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'fold'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**15 In the X-ray structure of the SARS-CoV PLpro15g complex, one of the 1,3-dioxolane** oxygens is within 3  of the Gln 270 side chain amide nitrogen, suggesting the potential for forming an H-bond.', 'lgd': {'word': '15', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'dioxolane'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**As observed with compound 2a, removal of the dioxolane ring from 15g results in a 3-**fold decrease in potency, suggesting that the dioxolane group provides a significant contribution to inhibitory potency but not through interaction with Gln 270 .', 'lgd': {'word': 'As', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': '3'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Therefore, to elucidate the nature of interactions provided by the 1,3-benzodioxole** group, benzene ring derivatives with diverse substituents at position R 3 were explored.', 'lgd': {'word': 'Therefore', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'benzodioxole'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The incorporation of a p-ethyl** group at R 3 (3a) had a more than 4-fold improvement in potency over unsubstituted prototype 2a, surpassing that of the dioxolane lead (15g).', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'ethyl'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Further extension of the acetamido group from the meta position, favored in the previous pair, dramatically decreases activity of the corresponding derivative (3f, IC 50 = 20.4  1.2 M)**.', 'lgd': {'word': 'Further', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'M'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A reversal in the meta vs para trend is seen with the corresponding pair of chloro derivatives 3g (IC 50 = 27.2  4.1 M) and** 3h (IC 50 = 0.58  0.01 M), with the halogen tolerated only at the para position.', 'lgd': {'word': 'A', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'and'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Finally, derivatives 3j and 3k, featuring the monofluoro substitution at para and meta positions, respectively, displayed a significant improvement**', 'lgd': {'word': 'Finally', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'improvement'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Values are reported as mean  standard deviation based on a minimum of** triplicate measurements.', 'lgd': {'word': 'Values', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Surprisingly, 3,4-difluoro substitution in 3i (IC 50 = 29.2  2.1 M) had** a significantly detrimental effect, decreasing the inhibitory potency by 200-fold when compared to 3k.', 'lgd': {'word': 'Surprisingly', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'had'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In general, neither steric nor electronic factors can be invoked to rationalize systematically the effects of the substitution pattern at R 3 and R 4 on the inhibitory activity, with groups as diverse as 1,3-dioxolane,** 4-ethyl, 3-and 4carboxamido, 3-acetamido, 3-and 4-fluoro, and 4-chloro achieving a similar level of activity, despite presenting a wide range in polarity, size, and H-bonding capacity.', 'lgd': {'word': 'In', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'dioxolane'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Interestingly, the activity was rescued by the addition of a 3-methoxy** group to the 4-pyridinyl ring of the 5b analogue (5c, IC 50 = 0.35  0.02 M), confirming the benefit of having a H-bonding group at C-3 of the aromatic ring.', 'lgd': {'word': 'Interestingly', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'methoxy'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Extending the separation of the two aromatic centers in compounds 3h and 3k by one carbon atom resulted in weakening of inhibition activity in corresponding variants 6a (IC 50 = 1.6  0.**3 M) and 6b (IC 50 = 1.9  0.1 M).', 'lgd': {'word': 'Extending', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '0'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The effect is surprisingly minor, considering the nature of this perturbation, and is perhaps indicative of a significant amount** of flexibility in the active site of SARS-CoV PLpro in the region bound by the benzene moiety.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'amount'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Finally, tricyclic analogue 7 was prepared as a conformationally restricted** analogue of our most potent inhibitor 3k ( Figure  2 ).', 'lgd': {'word': 'Finally', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'restricted'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**It was designed to lock the conformation of the bond joining C-1 of the** naphthyl ring to the piperidinylethyl group to that observed in the X-ray structure of the SARS-CoV PLpro15g complex.', 'lgd': {'word': 'It', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'the'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This modification, however, led to a significant loss** of activity (IC 50 = 5.1  0.5 M).', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'loss'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**To gain structural insight into the enhanced potency of compound 3k, SARS-CoV PLpro was cocrystallized as a complex with** 3k (PDB code 4OW0) and 3j (PDB code 4OVZ) using an approach similar to that for 15g.', 'lgd': {'word': 'To', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'with'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Complete X-ray data sets were collected for the SARS-CoV PLpro3k and PLpro3j complexes to resolutions of 2.1 and 2.**5 , respectively (Table  3 ).', 'lgd': {'word': 'Complete', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '2'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**After identification of a molecular replacement** solution and performance of initial rounds of refinement of the SARS-CoV PLpro structural model in the absence of any ligand or water, strong (>3) residual electron density was observed in F o  F c maps for both 3k and 3j in the active sites of each monomer within the asymmetric unit.', 'lgd': {'word': 'After', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'replacement'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Upon inhibitor binding, the -turn/loop (Gly 267 -Gly 272 ) containing Tyr 269 adopts a closed conformation** via an induced-fit mechanism to interact with the inhibitors.', 'lgd': {'word': 'Upon', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'conformation'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This enables the formation of a 3 ** H-bond between the backbone carbonyl of Tyr 269 and the carboxyamide nitrogen of the inhibitors.', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': ''}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The puckering of the piperidine ring positions the cationic nitrogen within a distance of** 2.8  from an oxygen of the carboxylate of Asp 165 , thereby forming a charge-to-charge mediated H-bond.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A structural superimposition of the two SARS-CoV PLpro 3k and PLpro3j complexes ( Figure 3e ) shows that the 1naphthyl rings align identically and that they pack against the two tandem prolines, Pro 248 and Pro 249 , in a hydrophobic pocket** formed by the side chains of the prolines, Tyr 265 , Tyr 269 , and Thr 302 (Figure 3c and Figure 3d ).', 'lgd': {'word': 'A', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'pocket'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This pocket orients the (R)-methyl group into a small cavity** lined by hydrophobic and hydrophilic side chains wherein some H-bond opportunities exist with the side chains of Asp 303 , Thr 302 , and Tyr 274 .', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'cavity'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This smaller cavity explains our observed SAR with hydrophobic or polar extensions at the (R)-methyl (position R 1 in Table** 1 ) Figure 2 .', 'lgd': {'word': 'This', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'Table'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**As discussed above, previous structural and computational analyses of the 15g-bound structure showed that the 1,3-benzodioxole** moiety can move within 3  of the amide nitrogen of Gln 270 side chain.', 'lgd': {'word': 'As', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'benzodioxole'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Interestingly, while the superimposition between PLpro15g, PLpro3k, and PLpro3j complexes shows a near perfect** overlap of the 1-naphthyl rings, there is a 1  difference in the position of the carboxamide nitrogen of 15g when compared to 3k and 3j.', 'lgd': {'word': 'Interestingly', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'perfect'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This causes a slight tilt** on the orientations of the benzene rings to allow for the accommodation of the different benzene substituents.', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'tilt'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': \"As **a result, we** conclude that the slightly enhanced inhibitory activity of compound 3k compared to 3j and 15g is due to its ability to form collectively stronger van der Waals' interactions between the m-fluorobenzene and the side chain of Tyr 269 , and slightly stronger interactions with the backbone oxygen atoms of Tyr 269 and Gln 270 (Figure 3f) .\", 'lgd': None, 'rgd': {'seed': 'a', 'word': 'we'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**First, to assess compound selectivity, a set of** the most potent analogues were tested against a panel of human DUBs, including representative ubiquitin specific proteases (USPs) with structural similarity to PLpro, along with the human cysteine proteases caspase 3 and cathepsin K (Table 4 ).', 'lgd': {'word': 'First', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**All of the analogues in Tables 1 and 2** were counterscreened against the viral orthologue PLP2 enzyme from the human coronavirus NL63 (HCoV-NL63), a member of the coronaviruses.', 'lgd': {'word': 'All', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '2'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**For those analogues that showed >30% inhibition at a single dose** concentration of 100 M, full dose response curves of HCoV-NL63 PLP2 inhibition versus increasing compound concentration up to 100 M were determined.', 'lgd': {'word': 'For', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'dose'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The results show that inhibition of HCoV-NL63 PLP2 is achieved by this series of compounds, albeit significantly weaker inhibition than for SARS-CoV PLpro (micromolar versus nanomolar range), and suggest that SARS-CoV PLpro and HCoV-NL63 PLP2 may display sufficient structural similarities at the active site for the potential development of a dual-target** inhibitor.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'target'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**However, in the case of our best compound, 3k, which has a 4-fold** improved IC 50 value, an additional 2-fold improvement in antiviral activity was observed (EC 50 = 5.4  0.6 M) when compared to 15g.', 'lgd': {'word': 'However', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'fold'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This was not entirely unexpected because of the presence of the 3,4methylenedioxy moiety, which renders the benzyl aromatic ring electron-rich and is itself a known target** of cytochrome P450s.', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'target'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**None of these five compounds exhibit cytotoxicity or off-target inhibitory activity of a series of** human DUB and cysteine protease enzymes, nor do any of these five inhibitors bind to human serum albumin (HSA).', 'lgd': {'word': 'None', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**1 H NMR** and 13 C NMR spectra were recorded in parts per million (ppm) on the  scale from an internal standard of residual tetramethylsilane (0 ppm).', 'lgd': None, 'rgd': {'seed': '1', 'word': 'NMR'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Rotamers are described as a ratio of** rotamer A to rotamer B if possible.', 'lgd': {'word': 'Rotamers', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Mass spectrometry data were obtained on a Waters Corporation** LCT.', 'lgd': {'word': 'Mass', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'Corporation'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Purity of all tested compounds was assessed by HPLC using an Agilent 1100 series with an Agilent Zorbax Eclipse Plus C18 column (254 nm detection) with the following gradient: 10% ACN/water (1 min), 10**90% ACN/water (6 min), and 90% ACN/ water (2 min).', 'lgd': {'word': 'Purity', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '10'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '()**-N-(Benzo[d**] [1,**** 3] dioxol-5-ylmethyl)-1-(1-(naphthalen-1yl)propyl)piperidine-4-carboxamide (1a).', 'lgd': {'word': 'N', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '3'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Ethyl 1-(1-**(naphthalen-1-yl)propyl)piperidine-4-carboxylate 11a (480 mg, 1.48 mmol) was dissolved in a 1:1:2 mixture of THF/EtOH/1 M aqueous NaOH, total volume of 8 mL.', 'lgd': {'word': 'Ethyl', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '1'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The pH was then reduced by addition of concentrated HCl to pH 4, and the resulting white precipitate was collected over a filter to** give the desired carboxylic acid (364 mg, 83%) without further purification.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'to'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Then the following was added sequentially to anhydrous DMF (1 mL): 3**  molecular sieves, 1-(1-(naphthalen-1-yl)propyl)piperidine-4-carboxylic acid (50 mg, 0.17 mmol), DIEA (0.088 mL, 0.504 mmol), EDCI (36 mg, 0.19 mmol), HOBt (28 mg, 0.19 mmol), and finally piperonylamine (0.031 mL, 0.252 mmol).', 'lgd': {'word': 'Then', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '3'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This was stirred at room temperature for 16 h, at which time the material was partitioned between 10% Na 2 CO 3 solution and EtOAc, and the organic layer was washed with 10% Na 2 CO 3 solution (3  20 mL) and brine (1  20** mL).', 'lgd': {'word': 'This', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '20'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Purification was accomplished via flash chromatography (4 g silica RediSep gold column on a CombiFlash, eluted** with 70% EtOAc/ Hex) to give the desired product (34 mg, 47%).', 'lgd': {'word': 'Purification', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'eluted'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**1 ()-N-(Benzo[d] [1,** 3] dioxol-5-ylmethyl)-1-(2-hydroxy-1-(naphthalen-1-yl)ethyl)piperidine-4-carboxamide (1b).', 'lgd': {'word': '1', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'N'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Ethyl 1-(2-methoxy-1-**(naphthalen-1-yl)ethyl)piperidine-4-carboxylate 11c (80 mg, 0.23 mmol) was dissolved in EtOH (2 mL) and 6 M NaOH (5 mL).', 'lgd': {'word': 'Ethyl', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '2'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This was stirred at 50C for 3 h. The pH was dropped to 2 by dropwise addition of concentrated HCl, and material was extracted with 1:1 EtOAc/**Et 2 O (3  10 mL).', 'lgd': {'word': 'This', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'EtOAc'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The solvent was then dried with anhydrous MgSO 4 , filtered, and removed in vacuo to give 1-(2methoxy-**1-(naphthalen-1-yl)ethyl)piperidine-4-carboxylic acid (30 mg, 41%) without further purification.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '2methoxy'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Then the following was added sequentially to anhydrous DMF (1 mL): 3**  molecular sieves, 1-(2-methoxy-1-(naphthalen-1-yl)ethyl)piperidine-4-carboxylic acid (20 mg, 0.06 mmol), DIEA (0.033 mL, 0.191 mmol), EDCI (14 mg, 0.07 mmol), HOBt (11 mg, 0.07 mmol), and finally piperonylamine (0.012 mL, 0.096 mmol).', 'lgd': {'word': 'Then', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '3'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This was stirred at room temperature for 24 h, at which time the material was partitioned between 10% aqueous Na 2 CO 3 solution and EtOAc, and the organic layer was washed with 10% aqueous Na 2 CO 3 solution (3  10 mL) and brine (1  10** mL).', 'lgd': {'word': 'This', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '10'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**1 10 mmol) was dissolved in a** 1:1:2 mixture of THF/EtOH/1 M aqueous NaOH, total volume of 4 mL.', 'lgd': {'word': '1', 'seed': 'a'}, 'rgd': {'seed': '1', 'word': 'mmol'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This was stirred at 60C for 4 h. The mixture was allowed to cool, and the pH was reduced by addition of concentrated HCl to pH 4, and the resulting white precipitate was collected over a filter to** give 1-(1-(naphthalen-1-yl)-2phenylethyl)piperidine-4-carboxylic acid (30 mg, 82%) as a white powder.', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'to'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Then the following was added sequentially to anhydrous DMF (1 mL): 3**  molecular sieves, 1-(1-(naphthalen-1-yl)-2phenylethyl)piperidine-4-carboxylic acid (30 mg, 0.08 mmol), DIEA (44 L, 0.25 mmol), EDCI (18 mg, 0.09 mmol), HOBt (14 mg, 0.09 mmol), and piperonylamine (16 L, 0.13 mmol).', 'lgd': {'word': 'Then', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '3'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This was stirred at room temperature for 17 h, at which time the material was partitioned between 10% aqueous Na 2 CO 3 solution and EtOAc, and the organic layer was washed with 10% aqueous Na 2 CO 3 solution (3  20 mL) and brine (1  20** mL).', 'lgd': {'word': 'This', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '20'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The residue was then purified by flash chromatography (10 g silica, 50% EtOAc/Hex) to N-(3-Fluorobenzyl)-1-(1-**(quinolin-8-yl)ethyl)piperidine-4carboxamide (4a).', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '1'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Ethyl 1-(1-**(quinolin-8-yl)ethyl)piperidine-4carboxylate 28a (0.38 g, 1.30 mmol) was dissolved in methanol (20 mL), followed by the addition of 1 N sodium hydroxide (2.6 mL, 2.6 mmol) in one portion.', 'lgd': {'word': 'Ethyl', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '1'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The resulting mixture was stirred for 18 h at room temperature, then concentrated in vacuo and partitioned between ether and a minimum amount** of water.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'amount'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**To a solution of (R,S)-1-**(1-(quinolin-8-yl)ethyl)piperidine-4-carboxylic acid (0.1 g, 0.35 mmol) in dry DMF (5 mL) was added EDCI (0.07 g, 0.35 mmol), HOBt (0.07 g, 0.42 mmol), and DIEA (0.06 mL, 0.35 mmol) followed by (3-fluorophenyl)methanamine (0.04 g, 0.35 mmol).', 'lgd': {'word': 'To', 'seed': '1'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This reaction mixture was stirred overnight at room temperature, after which it was diluted with saturated NaHCO 3 and extracted with EtOAc (1**).', 'lgd': {'word': 'This', 'seed': '1'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The combined organic layers were washed with saturated NaCl (3) and dried (MgSO 4 ), then concentrated and purified by silica flash chromatography (15% MeOH/**DCM) to obtain 4a (0.05 g, 33% yield).', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'MeOH'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**1 N-(3-Fluorobenzyl)-1-**(1-(quinolin-5-yl)ethyl)piperidine-4carboxamide (4b).', 'lgd': {'word': '1', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '3'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Ethyl 1-(1-**(quinolin-5-yl)ethyl)piperidine-4carboxylate 28b (0.67 g, 2.15 mmol) was dissolved in methanol (20 mL), followed by 1 N sodium hydroxide (4.29 mL, 4.29 mmol) in one portion.', 'lgd': {'word': 'Ethyl', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '1'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The resulting mixture was stirred 18 h at room temperature, concentrated in vacuo, and partitioned between ether and a minimum amount** of water.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'amount'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**To a solution of (R,S)-1-**(1-(quinolin-5yl)ethyl)piperidine-4-carboxylic acid (0.100 g, 0.352 mmol) in dry DMF (5 mL) were added EDCI (0.067 g, 0.350 mmol), HOBt (0.060 g, 0.420 mmol), and DIEA (0.060 mL, 0.35 mmol) followed by (3fluorophenyl)methanamine (0.040 g, 0.350 mmol) .', 'lgd': {'word': 'To', 'seed': '1'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The combined organic layers were washed with saturated NaCl (3) and dried (MgSO 4 ), then concentrated and purified by silica gel flash chromatography (15% MeOH/**DCM) to provide the target compound (0.05 g, 33%) as a colorless oil.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'MeOH'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**1 washed with** 10% Na 2 CO 3 (3  25 mL).', 'lgd': None, 'rgd': {'seed': '1', 'word': 'with'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**1 ()-1-**(Naphthalen-1-yl)ethane-1,2-diol (13).', 'lgd': {'word': '1', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '1'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**2-Hydroxy-1-(naphthalen-**1-yl)ethanone 12 (200 mg, 1.07 mmol), prepared previously by the literature method, 18 was dissolved in anhydrous THF (2 mL) and was cooled in an ice bath.', 'lgd': {'word': '2', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'naphthalen'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**1 M LAH in THF (1.**07 mL, 1.07 mmol) was slowly added dropwise to the solution.', 'lgd': {'word': '1', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'LAH'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The mixture was then allowed to warm to room temperature and was stirred for 2 h. After this time, the reaction was worked up by the Fieser method, 27 the aluminum was filtered off, and the filtrate solvent was removed in vacuo to give a white solid.**', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'solid'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This was purified by flash chromatography (10 g silica, 50% EtOAc/Hex) to give the desired product (168 mg, 83%) as a white solid.**', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'solid'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**1 ()-Ethyl 1-**(2-((tert-Butyldimethylsilyl)oxy)-1-(naphthalen-1-yl)ethyl)piperidine-4-carboxylate (16) .', 'lgd': {'word': '1', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'Ethyl'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**2-((tert-Butyldimethylsilyl)oxy)-1-(naphthalen-**1-yl)ethanol 14 (320 mg, 1.06 mmol) was dissolved in DCM (6 mL) with 3  molecular sieves, followed by DIEA (0.28 mL, 1.59 mmol).', 'lgd': {'word': '2', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'naphthalen'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**MsCl (0.10 mL, 1.27 mmol)** was added dropwise, and the mixture was allowed to stir at room temperature for 3 h. Ethyl piperidine-4-carboxylate (0.82 mL, 5.29 mmol) was then added dropwise and the mixture stirred for 23 h at room temperature.', 'lgd': {'word': 'MsCl', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'mmol'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A 1:1 solution** of Et 2 O/EtOAc (30 mL) was added.', 'lgd': {'word': 'A', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'solution'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The solution was washed with aqueous 10% aqueous Na 2 CO 3 (3  20 mL) and brine (1  15** mL), dried with anhydrous MgSO 4 , filtered, and the filtrate was concentrated in vacuo.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '15'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**1 ()-N-(Benzo[d] [1,** 3] dioxol-5-ylmethyl)-1-(2-((tertbutyldimethylsilyl)oxy)-1-(naphthalen-1-yl)ethyl)piperidine-4carboxamide (17) .', 'lgd': {'word': '1', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'N'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Ethyl 1-(2-((tert-butyldimethylsilyl)oxy)-1-**(naphthalen-1-yl)ethyl)piperidine-4-carboxylate 16 (100 mg, 0.23 mmol) was dissolved in ethanol (6 mL) and 7 M aqueous NaOH (3 mL) and stirred at 40C for 3 h. The mixture was then cooled to 0C and acidified (pH 2) with concentrated HCl.', 'lgd': {'word': 'Ethyl', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '2'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The resulting white precipitate was collected over filter and dried under high vacuum overnight to give the desired carboxylic acid (70 mg, 75%) as a white powder.**', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'powder'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The crude solid was added to a solution of** DIEA (0.059 mL, 0.338 mmol), EDCI (42.2 mg, 0.220 mmol), and HOBt (33.7 mg, 0.220 mmol) in anhydrous DMF (1 mL) over 3  molecular sieves.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**After this time, a 1:1** solution of Et 2 O/EtOAc (25 mL) was added and washed with 10% aqueous Na 2 CO 3 (3  20 mL) and brine (1  10 mL), dried with anhydrous MgSO 4 , and concentrated in vacuo.', 'lgd': {'word': 'After', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': '1'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**1 1-(Naphthalen-1-**yl)-2-phenylethanone (19) .', 'lgd': {'word': '1', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'Naphthalen'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**1-Naphthonitrile 18 (1.**00 g, 6.53 mmol) was dissolved in anhydrous Et 2 O (20 mL) in dry glassware, followed by the addition of benzylmagnesium chloride (7.83 mL, 7.83 mmol) under N 2 .', 'lgd': {'word': '1', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '18'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This was stirred at room temperature for 18 h. At this time, 1 M HCl** was added (5 mL) and the mixture was stirred at 90C for 35 min (during which time it was necessary to add more Et 2 O), then allowed to cool to room temperature, at which point material was extracted by EtOAc/Et 2 O, dried with anhydrous MgSO 4 and the filtrate was concentrated in vacuo.', 'lgd': {'word': 'This', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'HCl'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The resulting residue was purified by flash chromatography (20 g silica, 10% EtOAc/Hex) to give the desired ketone (1.35 g,** 84%).', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'g'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**1 ()-Dimethyl 1-**(1-(Naphthalen-1-yl)-2-phenylethyl)pyridine-4,4(1H)-dicarboxylate (21).', 'lgd': {'word': '1', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'Dimethyl'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Dimethyl 2,2-bis((1,3-dioxolan-**2-yl)methyl)malonate (578 mg, 1.90 mmol) was dissolved in a 1:1 mixture of THF (10 mL) to 10% aqueous HCl (10 mL) and stirred at room temperature for 18 h. At this time, the reaction was neutralized with solid NaHCO 3 and a solution of 1-(naphthalen-1-yl)-2-phenylethanamine 20 (446 mg, 1.80 mmol), prepared previously by the literature method 19 from 19, in THF (6 mL) was added.', 'lgd': {'word': 'Dimethyl', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'dioxolan'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The residue was then purified via flash chromatography (15 g silica, 15% EtOAc/Hex) to give the desired condensed product (470 mg, 58%) as a yellow oil.**', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'oil'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**1 ()-Dimethyl 1-**(1-(Naphthalen-1-yl)-2-phenylethyl)piperidine-4,4-dicarboxylate (22).', 'lgd': {'word': '1', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'Dimethyl'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Dimethyl 1-(1-**(naphthalen-1yl)-2-phenylethyl)pyridine-4,4(1H)-dicarboxylate 21 (80 mg, 0.187 mmol) was dissolved in N 2 -sparged MeOH (5 mL), and 10% Pd/C was added.', 'lgd': {'word': 'Dimethyl', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '1'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**1 1-(Quinolin-**8-yl)ethanol (27a).', 'lgd': None, 'rgd': {'seed': '1', 'word': 'Quinolin'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': 'To **a 78**C solution of quinoline-8-carbaldehyde 26a (0.5 g, 3.18 mmol) in dry THF under N 2 was added methylmagnesium chloride (3.82 mL, 3.82 mmol) dropwise.', 'lgd': None, 'rgd': {'seed': 'a', 'word': '78'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**1 1-(Quinolin-**5-yl)ethanol (27b).', 'lgd': None, 'rgd': {'seed': '1', 'word': 'Quinolin'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': 'To **a 78**C solution of quinoline-5-carbaldehyde 26b (0.82 g, 5.2 mmol) in dry THF under N 2 was added methylmagnesium chloride (6.3 mL, 6.3 mmol) dropwise.', 'lgd': None, 'rgd': {'seed': 'a', 'word': '78'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The residue was purified by silica flash chromatography (1040% EtOAc/Hex) to afford the target compound (0.77 g, 86%) as a yellow oil.**', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'oil'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**1 1-(Isoquinolin-**5-yl)ethanol (27c).', 'lgd': None, 'rgd': {'seed': '1', 'word': 'Isoquinolin'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**To a 78C solution of isoquinoline-5-carbaldehyde 26c (1.**13 g, 7.19 mmol) in dry THF under N 2 was added methylmagnesium chloride (8.63 mL, 8.63 mmol) dropwise.', 'lgd': {'word': 'To', 'seed': '1'}, 'rgd': {'seed': 'a', 'word': '78'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Concentration provided a white solid** which was triturated in ether, filtered, and dried under high vacuum to give 1-(isoquinolin-5yl)ethanol 26c (1.1 g, 88% yield).', 'lgd': {'word': 'Concentration', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'solid'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**1 1-(Isoquinolin-1-**yl)ethanol (27d).', 'lgd': {'word': '1', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'Isoquinolin'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**To a solution of 1-**cyanoisoquinoline (301 mg, 1.96 mmol) in dry Et 2 O (10 mL) at 0C, 3.92 mL of 1 M MeMgBr (3.92 mmol) was added and allowed to stir for 3 h to produce a bright orange solution.', 'lgd': {'word': 'To', 'seed': '1'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The reaction was quenched with H 2 O (15 mL) and 6 M HCl (5 mL) and stirred for 1 h at** 80C.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'at'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The residue was purified by silca flash chromatography (30% EtOAc/Hex) to furnish 1-(isoquinolin-**1-yl)ethanone (304 mg, 91%) as a yellow oil.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'isoquinolin'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This compound (304 mg, 1.78 mmol)** was then dissolved in MeOH (6 mL), and NaBH 4 was added and the mixture was allowed to stir at 0C for 18 h. The mixture was concentrated in vacuo and treated with saturated aqueous NH 4 Cl.', 'lgd': {'word': 'This', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'mmol'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The residue was purified by silica flash chromatography (30% EtOAc/Hex) to furnish the desired compound (226.5 mg, 73.6%) as a clear oil.**', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'oil'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**1 Ethyl 1-**(1-(Quinolin-8-yl)ethyl)piperidine-4-carboxylate (28a).', 'lgd': {'word': '1', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '1'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**To a 0C solution of 1-**(quinolin-8-yl)ethanol 27a (0.47 g, 2.71 mmol) in dry DCM was added DIEA (1.05 g, 8.14 mmol) followed by Ms 2 O (0.71 g, 4.10 mmol).', 'lgd': {'word': 'To', 'seed': '1'}, 'rgd': {'seed': 'a', 'word': 'C'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The NL63-HCoV PLP2 counterscreening assay was performed in a 100 L** reaction volume including 400 nM enzyme, 100 M compound, and 50 M RLRGG-AMC substrate in assay buffer (50 mM HEPES, pH 7.5, 0.1 mg/mL BSA, 2 mM DTT).', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'L'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The assays were performed in triplicate using 31.6 M compounds and Ub-rhodamine 110 as a substrate for** USPs, Nedd8-EKL for DEN1, and commercially available fluorogenic peptide substrates for caspase 3 and cathepsin K.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'for'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**SARS-CoV antiviral activity in infected Vero E6 cells was performed as previously described in a BSL-3** laboratory.', 'lgd': {'word': 'SARS', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': '3'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**14b Briefly, Vero E6 cells were seeded into 96-well culture plates at 1  10** 3 cells per well 24 h prior to infection with SARS-CoV.', 'lgd': {'word': '14b', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '10'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Both the half maximal effective concentration (EC 50 ) and the 50% cytostatic concentration (CC 50 ) values were calculated using a four-parameter** logistic equation by means of Sigma Plot 10 or GraphPad Prism software.', 'lgd': {'word': 'Both', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'parameter'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Compounds (1 M) were** incubated with mouse liver microsomes (0.5 mg/mL) and NADPH in 0.1 M phosphate buffer at 37C as previously described.', 'lgd': {'word': 'Compounds', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'were'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Prior to crystallization, untagged SARS-CoV PLpro in 25 mM Tris, pH 7.5, 100 mM NaCl, 10 mM DTT at concentrations of 6 or 12 mg/mL was incubated with 2 mM inhibitor (dissolved in DMSO or ethanol) added to a 100 dilution** and incubated overnight at 4C.', 'lgd': {'word': 'Prior', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'dilution'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Crystals were transferred from liquid nitrogen into a stream of** dry nitrogen gas at 100 K for X-ray data collection.', 'lgd': {'word': 'Crystals', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Crystals belong to space group C2 with unit cell dimensions of a = 119** , b = 74 , c = 98 ,  = 104.', 'lgd': {'word': 'Crystals', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': '119'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The inhibitor-bound structures diffracted to resolutions of 2.1 and 2.**5  for 3k and 3j, respectively.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '2'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The initial phases were determined from molecular replacement solutions using the SARS-CoV PLpro15g inhibitor complex structure (pdb:3MJ5) as a search model** and the MolRep 31 program of the CCP4 suite.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'model'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**X-ray Structural and Biological Evaluation of a Series of** Potent and Highly Selective Inhibitors of Human Coronavirus Papain-like Proteases', 'lgd': {'word': 'X', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Coronavirus Cell Entry Occurs through the Endo-/ Lysosomal Pathway in a Proteolysis-Dependent Manner"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Enveloped viruses need to fuse with a host cell** membrane in order to deliver their genome into the host cell.', 'lgd': {'word': 'Enveloped', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'cell'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Using recombinant MHVs expressing reporter genes as well as a novel, replication-**independent fusion assay we confirmed the importance of clathrin-mediated endocytosis and demonstrated that trafficking of MHV to lysosomes is required for fusion and productive entry to occur.', 'lgd': {'word': 'Using', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'replication'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Fusion of MHV was severely inhibited by a pan-lysosomal** protease inhibitor, while trafficking of MHV to lysosomes and processing by lysosomal proteases was no longer required when a furin cleavage site was introduced in the S protein immediately upstream of the fusion peptide.', 'lgd': {'word': 'Fusion', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'lysosomal'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In contrast, MERS-CoV, which contains a minimal furin** cleavage site just upstream of the fusion peptide, was negatively affected by inhibition of furin, but not of lysosomal proteases.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'furin'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**We conclude that a proteolytic cleavage** site in the CoV S protein directly upstream of the fusion peptide is an essential determinant of the intracellular site of fusion.', 'lgd': {'word': 'We', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'cleavage'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**(2014) Coronavirus Cell Entry Occurs through the Endo-/Lysosomal Pathway **in a** Proteolysis-Depend**ent Manner.', 'lgd': {'word': '2014', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'Dependent'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**To achieve successful infection enveloped viruses need to fuse with a host cell** membrane to deliver the viral genome into the host cell.', 'lgd': {'word': 'To', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'cell'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Some viruses, such as herpes simplex virus, Sendai virus, and human immunodeficiency virus, appear to be capable of direct fusion at the plasma membrane after initial attachmen**t [1]**** [2] [3] [4] [5] .', 'lgd': {'word': 'Some', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '2'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**These triggers, which may include a decrease in** pH, changes in redox environment, and proteolytic activity [6] [7] [8] , induce conformational changes in the viral fusion proteins leading to the merger of viral and host membranes.', 'lgd': {'word': 'These', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'in'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**They are capable of infecting a wide variety** of mammalian and avian species.', 'lgd': {'word': 'They', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'variety'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Human coronaviruses (HCoVs) are known as major causes of the common cold (e.**g.', 'lgd': {'word': 'Human', 'seed': 'e'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The well-studied mouse hepatitis virus (MHV) is often used as a safe model** to study CoV infections.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'model'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**All CoV virions contain a canonical set** of four structural proteins.', 'lgd': {'word': 'All', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'set'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Trimers of the CoV S protein, a type I** membrane protein belonging to the class I fusion proteins, form the peplomers that protrude from the virion surface [20] .', 'lgd': {'word': 'Trimers', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'I'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Also feline infectious peritonitis virus (FIPV) was suggested to enter via a clathrin-and** caveolaeindependent endocytic route [23, 24] .', 'lgd': {'word': 'Also', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'and'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**For the HCoV-229E a caveolae-dependent** endocytic uptake has been suggested [25] .', 'lgd': {'word': 'For', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'dependent'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Although the ability of MHV S proteins to cause cell-cell fusion at a neutral pH** was initially interpreted as an indication for fusion of virions at the cell surface, more recent studies indicate the requirement for clathrin-mediated endocytosis for entry of MHV [26] [27] [28] [29] .', 'lgd': {'word': 'Although', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'pH'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**For SARS-CoV and infectious bronchitis virus (IBV), it appears that a proteolytic cleavage** of the S protein at a more downstream position than the S1/S2 boundary upon receptor binding is of importance for cell entry [40, 43, [46] [47] [48] [49] .', 'lgd': {'word': 'For', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'cleavage'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In the present study we performed a detailed investigation** of the entry of different CoVs.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'investigation'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**By using recombinant MHVs expressing reporter genes as well as by applying a novel, replication-**independent fusion assay we confirmed the importance of clathrin-mediated endocytosis and demonstrated that trafficking of MHV virions to lysosomal compartments and processing of the S protein by lysosomal proteases was required for productive entry to occur.', 'lgd': {'word': 'By', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'replication'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Our results indicate that a cleavage site** in the S protein of CoVs immediately upstream of the FP determines the site of fusion.', 'lgd': {'word': 'Our', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'site'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In contrast, MERS-CoV, which contains a minimal furin-**cleavage site consensus sequence in the S protein immediately upstream of the FP, was negatively affected by inhibition of furin, but not of lysosomal proteases.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'furin'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In an automated, high-throughput RNAi screen [50] targeting the druggable genome (approximately 7000 genes) a number of** proteins associated with endocytosis were found to be required for efficient infection of HeLa cells with GFP-expressing MHV.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**To validate these findings these proteins were subjected to a follow-up** analysis using siRNA-mediated gene silencing with oligonucleotides from a different supplier than the one used for the initial RNAi screen (Fig.', 'lgd': {'word': 'To', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'up'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The HOPS subunit VPS39 (reviewed in [54] ) was also found to be a strong hit** in the siRNA screen and therefore selected.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'hit'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Other proteins included SNX1, involved in retrograde transport of cargo between endosomes and the trans-Golgi network (reviewed in [55] ), VCL, inter alia involved in connecting the Arp2/3 complex with integrins during actin polymerization (reviewed in [56] ), and the Ser/Thr-protein kinase PAK1, which is activated by the Rho/Rac/Cdc42 family and is implicated in a variety of** downstream effects including modulation of the actin cytoskeleton (reviewed in [57] ).', 'lgd': {'word': 'Other', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Enveloped viruses need to fuse with a host cell** membrane in order to deliver their genome into the host cell.', 'lgd': {'word': 'Enveloped', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'cell'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**By combining cellbiological, infection, and fusion assays we demonstrated that murine hepatitis virus (MHV), a prototypic member** of the CoV family, enters cells via clathrin-mediated endocytosis.', 'lgd': {'word': 'By', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'member'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Moreover, although MHV does not depend on a low pH** for fusion, the virus was shown to rely on trafficking to lysosomes for proteolytic cleavage of its spike (S) protein and membrane fusion to occur.', 'lgd': {'word': 'Moreover', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'pH'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In conclusion, we elucidated the MHV entry pathway in detail and demonstrate that a proteolytic cleavage** site in the S protein of different CoVs is an essential determinant of the intracellular site of fusion.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'cleavage'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**siRNAs targeting RAB7A were shown to inhibit the expression of a RAB7a-fusion** protein ( Figure S1 in Text S1).', 'lgd': {'word': 'siRNAs', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'fusion'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**To confirm and extend our understanding of the role of endocytosis in MHV entry we subsequently selected a number of** proteins known to be involved in either caveolae-or clathrinmediated endocytosis, actin-or microtubule-mediated transport, as well as proteins associated with endosomal vesicles and endosomal maturation, to be screened using the siRNA silencing-approach described above.', 'lgd': {'word': 'To', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Downregulation of most proteins associated with clathrin-mediated endocytosis inhibited MHV infection, Figure 1** .', 'lgd': {'word': 'Downregulation', 'seed': '1'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Actin cytoskeleton altering drugs Latrunculin A (LatA), Jasplakinolide (Jasp), Cytochalasin B (CytoB), and Cytochalasin D (CytoD), or the inducer of microtubule depolymerization Nocodazole (Noc) only decreased MHV infection when added early, indicating a role for** the actin and microtubule cytoskeleton in entry but not RNA replication ( Figure 2 , dark orange and orange).', 'lgd': {'word': 'Actin', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'for'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Likewise U18666A, a cholesterol transport-**affecting agent, which also prevents maturation of late endosomes [62] , had a strong inhibitory effect on MHV infection when added early (Figure 2 , turquoise).', 'lgd': {'word': 'Likewise', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'transport'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**To unequivocally demonstrate the importance of clathrin-mediated endocytosis and endosome maturation for MHV entry, we therefore made use of a fusion assay** we recently developed [63] .', 'lgd': {'word': 'To', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'assay'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The assay is based on minimal complementation of defective bgalactosidase (b-galactosidase DM15) with the short a-peptide [64]** .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': '64'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**MHV-aN, a recombinant MHV** containing an N protein tagged with the a-peptide (aN), is used to infect DM152fragment expressing target cells.', 'lgd': {'word': 'MHV', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'MHV'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Upon fusion of the virion with a host cell** membrane aN is released into the cytoplasm resulting in complementation of the defective b-galactosidase thereby reconstituting a functional enzyme.', 'lgd': {'word': 'Upon', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'cell'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**After 100 min incubation of cells with virus at 37uC, cells were collected and the amount of FIC generated as a results of** enzyme complementation analyzed by FACS.', 'lgd': {'word': 'After', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**MHV fusion was barely affected by replication inhibitor BrefA, whereas MG132 had a clear negative** effect, in agreement with the conclusion drawn previously that MG132 inhibits entry of MHV as well as RNA synthesis [61] .', 'lgd': {'word': 'MHV', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'negative'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Disturbance of the actin cytoskeleton by EIPA or by LatA, CytoB, or CytoD reduced fusion by 75-80% ( Figure 3B , dark orange), while interference with microtubule polymerization by Noc had a smaller effect** ( Figure 3B , orange).', 'lgd': {'word': 'Disturbance', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'effect'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Cells were imaged using a spinning-disc** confocal microscope acquiring zstacks in 30 s intervals over 10 min time frames from 10-70 min post warming.', 'lgd': {'word': 'Cells', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'disc'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A virion was categorized as associating with a certain endosomal** marker only if this co-localization was observed over at least four sequential 30 s interval images.', 'lgd': {'word': 'A', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'endosomal'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Complete disappearance of a virus particle** (including in other z-stacks) while immediately previously co-localizing with an endosomal marker was categorized as a fusion event (Figures S3 and S4 in Text S1).', 'lgd': {'word': 'Complete', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'particle'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': 'When **a viral particle** co-localized with endosomal compartments but did neither dissociate nor fade during the 10 min acquisition period it was classified as non-fusing.', 'lgd': None, 'rgd': {'seed': 'a', 'word': 'particle'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': 'Only **a very small** percentage of virions were categorized as fusing while in early endosomes.', 'lgd': None, 'rgd': {'seed': 'a', 'word': 'small'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Luciferase expression levels indicated that infection of cells with VSV and IAV is much more affected by BafA1, with an IC50 values of 0.80 and 0.63 nM, respectively, compared to MHV, which displays a three to** four fold higher IC50 of 2.34 nM ( Figure 5A ).', 'lgd': {'word': 'Luciferase', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'to'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**To confirm and extend these observations, we made use of haploid HAP1 cells lacking a functional HOPS** complex resulting from lentiviralmediated knockout of the VPS33A subunit (H1-DV33 cells; [74] ).Both HAP1 cells and H1-DV33 cells were modified to stably express the MHV receptor.', 'lgd': {'word': 'To', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'HOPS'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': 'As **a control, the** H1-DV33 cells were in addition stably transfected with FLAG-tagged VPS33A (H1-DV33-fV33).', 'lgd': None, 'rgd': {'seed': 'a', 'word': 'the'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The lack of a functional HOPS** complex had no effect on VSV and IAV infection; however, MHV infection was strongly reduced in the knockout, but not in the re-transfected cells ( Figure 5B ).', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'HOPS'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**These observations confirm the conclusion that late endosome-tolysosome maturation is required for efficient entry of MHV, a characteristic that** is not shared with the pH-sensitive VSV and IAV.', 'lgd': {'word': 'These', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'that'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Considering that MHV was much less affected by perturbation of the endosomal pH than IAV and VSV while it requires trafficking to lysosomes for efficient entry, we hypothesized that entry might depend on cleavage of a viral protein** by lysosomal proteases.', 'lgd': {'word': 'Considering', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'protein'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Exceptions were AEBSF, which has been shown to cause aggregation of early endosomal vesicles [75] , and a pan-lysosomal** protease inhibitor (CPI; cystatinpepstatin inhibitor) capable of inhibiting the three major protease family members found in lysosomes.', 'lgd': {'word': 'Exceptions', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'lysosomal'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**From these results we conclude that inhibition of a broad range** of endosomal proteases efficiently blocks fusion of MHV, indicating that efficient entry requires the activity of lysosomal proteases.', 'lgd': {'word': 'From', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'range'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Introduction of a furin cleavage** site immediately upstream of the fusion peptide renders MHV independent of lysosomal proteases', 'lgd': {'word': 'Introduction', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'cleavage'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**To test this hypothesis, we introduced an optimal furin cleavage site (FCS) by substituting three amino acids by Arg (AIRGRRRRRRR) immediately upstream of a highly conserved** Arg (indicated in bold) that occurs just N-terminal of the FP.', 'lgd': {'word': 'To', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'conserved'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Western blot analysis of the S protein of a purified stock** of this virus using an antibody recognizing the S2 subunit showed no evidence of cleavage at the newly introduced FCS (S29 site).', 'lgd': {'word': 'Western', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'stock'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**5) , infection with MHV carrying a wild type** S was severely reduced in cells lacking a functional HOPS complex and addition of the FI did not alter this effect (Figure 8, red bars) .', 'lgd': {'word': '5', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'type'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In contrast, infection with MHV-S29FCS was not decreased by the lack of a functional HOPS** complex.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'HOPS'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**However, FI treatment had a clearly negative** effect on this virus, which was much more dramatic in the absence of a functional HOPS complex in H1-DV33 cells (Figure 8 , blue).', 'lgd': {'word': 'However', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'negative'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In conclusion, MHV-S29FCS lost the requirement for a functional HOPS** complex in parallel with this virus becoming insensitive to the pan-lysosomal protease inhibitor CPI.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'HOPS'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**1) , infection** with wild type S protein carrying MHV was reduced after gene silencing of RAB5, RAB7, VPS11, and VPS41 (Figure 9, red bars) .', 'lgd': None, 'rgd': {'seed': '1', 'word': 'infection'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**From these results we conclude that introduction of a FCS immediately** upstream of the FP abolishes the requirement for trafficking of virions to lysosomes and for processing by lysosomal proteases.', 'lgd': {'word': 'From', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'immediately'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Interestingly, MERS-CoV and IBV-Beaudette contain a minimal furin** cleavage site Arg-X-X-Arg just upstream of the fusion peptide ( Figure 10A ).', 'lgd': {'word': 'Interestingly', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'furin'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Since the requirement for a functional HOPS** complex is indicative of fusion in lysosomes, as we observed for MHV, we analyzed whether FIPV requires processing by lysosomal proteases for efficient entry as well.', 'lgd': {'word': 'Since', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'HOPS'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**As MERS-CoV carries a FCS in** its S protein immediately upstream of the FP, we hypothesized this virus not to require trafficking to lysosomes and processing by lysosomal proteases for efficient entry.', 'lgd': {'word': 'As', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'in'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Cells were subsequently inoculated with MERS-CoV at a MOI of** 0.1 in the presence of these inhibitors.', 'lgd': {'word': 'Cells', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The results indicate that, in contrast to wild type MHV and FIPV, but similarly to recombinant MHV carrying a FCS immediately** upstream of the FP, infection with MERS-CoV is strongly inhibited by the FI but not by CPI (Figure 11 ), indicating that MERS-CoV does not require trafficking to lysosomes for efficient entry.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'immediately'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Based on these results we conclude that the cleavage site in the CoV S protein immediately upstream of the FP is a key determinant** of the intracellular site of fusion.', 'lgd': {'word': 'Based', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'determinant'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**By using a replication-independent** fusion assay, we confirmed that MHV entry requires clathrin-mediated endocytosis despite the wellknown ability of the MHV S protein to cause cell-cell fusion at neutral pH.', 'lgd': {'word': 'By', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'independent'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Efficient inhibition of MHV entry was only observed using a pan-lysosomal** protease inhibitor, and could not be achieved using more specific protease inhibitors.', 'lgd': {'word': 'Efficient', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'lysosomal'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Consistent with a common mechanism** for the entry of CoVs, FIPV, but not MERS-CoV, the latter of which contains a furin cleavage site immediately upstream of the FP, was shown to require trafficking to lysosomes and processing by lysosomal proteases for efficient entry.', 'lgd': {'word': 'Consistent', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'mechanism'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Based on these results we propose a model in** which the cleavage site immediately upstream of the FP is an essential determinant of the intracellular site of CoV fusion ( Figure 12) .', 'lgd': {'word': 'Based', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'in'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Furthermore, we made use of three independent siRNAs per target gene, and a target was** only classified as a hit when at least two out three siRNAs showed the same phenotype.', 'lgd': {'word': 'Furthermore', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'was'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Finally, the results obtained were corroborated by using a large panel** of inhibitors and by making use of haploid knockout cells, in which late endosome-to-lysosome trafficking was inhibited.', 'lgd': {'word': 'Finally', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'panel'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Endocytic uptake is also required for a mutant virus** carrying a S protein with a FCS immediately upstream of its FP, despite the relative insensitivity to high concentrations of BafA1.', 'lgd': {'word': 'Endocytic', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'virus'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Therefore, the ability of a virus to** infect cells in the presence of BafA1 does not necessarily imply virus entry to occur at the cell surface.', 'lgd': {'word': 'Therefore', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'to'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': 'Also **a recombinant MHV** carrying the spike protein of MHV-4 (MHV-JHM) was found to enter via clathrin-mediated endocytosis (MHV-S4; Figure S10 in Text S1) despite its ability to cause extensive cell-cell fusion [80] [81] [82] .', 'lgd': None, 'rgd': {'seed': 'a', 'word': 'MHV'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Although our replicationdependent assays indicate a requirement for** caveolin 2 (CAV2) for infection, this protein was shown not to be involved in virus entry using our fusion assay.', 'lgd': {'word': 'Although', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'for'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Also depletion of other proteins involved in caveolae-mediated endocytosis, including caveolin 1 (CAV1)** and flotillins 1 and 2 (FLOT1 and FLOT2) did not affect MHV infection or fusion.', 'lgd': {'word': 'Also', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'CAV1'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Interestingly, fusion of MHV was severely inhibited by EIPA, an inhibitor of the Na + /H + exchanger NHE1, which is regarded as a hallmark inhibitor** of macropinocytosis.', 'lgd': {'word': 'Interestingly', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'inhibitor'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Apparently, inhibition of virus entry by EIPA does not prove by itself that a virus enters** via this particular pathway.', 'lgd': {'word': 'Apparently', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'enters'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The importance of lysosomes for entry was confirmed by using knockout cells lacking a functional HOPS** complex (For a review on the HOPS complex see [54] ).', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'HOPS'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Interestingly, in these cells lysosomes are clustered in a perinuclear region** of the cell rather than dispersed throughout the cytoplasm.', 'lgd': {'word': 'Interestingly', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'region'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**However, we cannot exclude that actin also plays a role in** the clathrin-mediated uptake of MHV particles, as has been observed for VSV and other pathogens that depend on clathrin-mediated endocytosis (reviewed in [91] ).', 'lgd': {'word': 'However', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'in'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The absence of a functional HOPS** complex, which is required for late endosome-to-lysosome maturation, did not affect infection of cells with VSV or IAV, while entry of MHV was severely reduced.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'HOPS'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**We propose that the sequence immediately upstream of the FP is a key determinant** of the intracellular site of fusion.', 'lgd': {'word': 'We', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'determinant'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**MERS-CoV, which contains a minimal FCS,** is inhibited by furin inhibitor (FI) but not by the pan-lysosomal protease inhibitor (CPI).', 'lgd': {'word': 'MERS', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'FCS'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**10) , to fuse in early endosomes in a furin-dependent** manner.', 'lgd': {'word': '10', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'dependent'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Other CoVs that do not contain a FCS at** this position are predicted to fuse in lysosomes.', 'lgd': {'word': 'Other', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'at'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Strikingly, other inhibitors that affect members of a single protease** family had none or only little impact on MHV fusion.', 'lgd': {'word': 'Strikingly', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'protease'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**These results are in consistence with a functional redundancy** of protease family members [47, 76] and may explain why previous studies using specific lysosome protease inhibitors [27, 92] failed to detect entry inhibition.', 'lgd': {'word': 'These', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'redundancy'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Although we did not demonstrate cleavage of MHV S at the FP proximal position directly, a recent study** found a cleaved form of the MHV S2 subunit to correspond with the fusion-active form [49] .', 'lgd': {'word': 'Although', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'study'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Furthermore, introduction of an optimal FCS at the FP proximal position abolished the entry inhibition by the pan-lysosomal protease inhibitor whilst introducing a dependency on** furin-related enzymes.', 'lgd': {'word': 'Furthermore', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'on'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**However, in the presence of furin inhibitor, entry of this mutant MHV was much more efficient in wild type cells than in cells lacking a functional HOPS** complex (Fig.', 'lgd': {'word': 'However', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'HOPS'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**8) , indicating that under certain circumstances lysosomal proteases may play a role in** entry of this virus as well.', 'lgd': {'word': '8', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'in'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Trafficking of virions to lysosomes was shown to be also important for entry of FIPV, but not of MERS-CoV, in agreement with the latter virus containing a putative FCS** immediately upstream of the FP.', 'lgd': {'word': 'Trafficking', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'FCS'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Based on the present study and on the work of others, we conclude that cleavage at the FP proximal position is likely to be a general requirement** for CoV entry.', 'lgd': {'word': 'Based', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'requirement'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Our results furthermore show that cleavage at a FP-proximal** position is an important determinant of the intracellular site of fusion.', 'lgd': {'word': 'Our', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'proximal'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Stable cell lines were generated using a Moloney murine** leukemia (MLV) retroviral vector.', 'lgd': {'word': 'Stable', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'murine'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**MLV was produced in HEK293T cells by triple plasmid transfection of a transfer vector** containing the DM15 or mCC1a gene as well as a puromycin or neomycin resistance marker gene, respectively, in combination with expression vectors encoding the MLV Gag-Pol, and the VSV spike protein G. Upon MLV transduction, stably transduced cells were selected at 2 mg/ml puromycin and/or 0.5 mg/ml G418 (both Sigma), maintenance at 1 mg/ml puromycin and/or 0.5 mg/ml G418 in DMEM, supplemented with 10% FBS.', 'lgd': {'word': 'MLV', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'vector'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**H1-DV33 cells were stably transfected with FLAG-tagged VPS33A (H1-DV33-fV33) using MLV transduction as described above using a blasticidin resistance** marker gene in the transfer vector.', 'lgd': {'word': 'H1', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'resistance'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The DM15 gene was isolated from a DH5 E.** coli strain by DNA extraction and PCR.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'E'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The gene was cloned into a pCAGGS vector** for (transient) expression and into a MLV-based pQCXIP transfer vector (Clontech), resulting pQCXIP-DM15, for the generation of stable cell lines.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'vector'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**pMH54 containing a GFP expression** cassette between the E and M gene was generated as described previously for firefly luciferase [59] .', 'lgd': {'word': 'pMH54', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'expression'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The transcription vector used to generate MHV-S29FCS (pXHERLM-S29FCS+) was generated by site-directed mutagenesis, thereby changing the sequence encoding AIRGR immediately upstream of the FP into a RRRRR-encoding** sequence in vector pXHERLM [59] (GCA9ATC9CGA9GGG9CGT to AGA9CGC9CGA9AGG9CGT).', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'encoding'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The transcription vector used to generate MHV-S4 expressing firefly luciferase, was generated by introducing the firefly luciferase expression cassette between the E and M genes similarly as described previously [59] in a pMH54derived transcription** vector that contains the gene encoding the S protein of MHV-4 (MHV-JHM) [82] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'transcription'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Recombinant MHV-EGFPM virus, containing a GFP expression** cassette between the E and the M gene, MHV-S29FCS, containing a Renilla luciferase expression cassette between the E and the M gene and a FCS at the FP-proximal position, and MHV-S4 containing the spike gene of MHV-4 (JHM) and a luciferase expression cassette were generated by targeted RNA recombination as described before [98] .', 'lgd': {'word': 'Recombinant', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'expression'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Briefly, donor RNA was generated from linearized pMH54derived transfer vectors described above, and electroporated into FCWF cells infected with interspecies chimeric fMHV coronavirus (an MHV-A59 derivative, in which the ectodomain of the spike protein has been replaced by that of a feline coronavirus,** thereby changing host cell tropism).', 'lgd': {'word': 'Briefly', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'coronavirus'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The electroporated FCWF cells were seeded onto a monolayer of** LR7 cells.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Generation of MHV-EFLM and MHV-ERLM, containing a firefly or** Renilla luciferase expression cassette between the E and the M gene, and MHV-aN, containing a N protein tagged with the a-peptide, has been described before [63, 103] .', 'lgd': {'word': 'Generation', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'or'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**30,000 HeLa-mCC1a-(DM15) cells were seeded one day prior to transfection in a 24-well** dish.', 'lgd': {'word': '30', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'well'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**mRNA levels of genes were analyzed by qRT-PCR using a custom designed** pair of specific primers to the gene resulting in about 150 bp products.', 'lgd': {'word': 'mRNA', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'designed'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': \"**RNA levels were measured using the GoTaq 1-Step RT-**qPCR system (Promega) according to the manufacturers' instructions on a LightCycler 480 (Roche).\", 'lgd': {'word': 'RNA', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'RT'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Expression levels were corrected for cell number and viability as determined by the Wst-1 assay (Roche)**.', 'lgd': {'word': 'Expression', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'Roche'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Cells were washed and taken up in FACS buffer (2% FBS, 0.05M EDTA, 0.2% NaN 3 in PBS) and GFP expression was quantified by FACS analysis on a FACS Calibur** (Benson Dickson) using FlowJo software.', 'lgd': {'word': 'Cells', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'Calibur'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Firefly luciferase expression was assessed using the firefly luciferase assay system from Promega or using a homemade system** (50 mM tricine, 100 mM EDTA, 2.5 mM MgSO 4 , 10 mM DTT, 1.25 mM ATP, 12.5 mM D-Luciferin).', 'lgd': {'word': 'Firefly', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'system'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Light emission was measured on a Centro LB** 960 luminometer.', 'lgd': {'word': 'Light', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'LB'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Luciferase expression levels (in relative light units, RLU) were corrected for cell number and viability as determined by the Wst-1 assay (Roche)**.', 'lgd': {'word': 'Luciferase', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'Roche'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Huh-7 cells were inoculated with MERS-CoV at a MOI of** 0.1 in FBScontaining DMEM.', 'lgd': {'word': 'Huh', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Cells were stained using rabbit anti-SARS-CoV nsp4 antibodies that are cross-reactive for MERS-CoV, according to a standard protocol** using a FITC-conjugated swineanti-rabbit antibody.', 'lgd': {'word': 'Cells', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'protocol'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Number of infected cells was determined by cell counts on a wide-field** fluorescent microscope.', 'lgd': {'word': 'Number', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'field'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': 'After **a recovery period** of 30 min at 37uC, cells were (mock) treated with the different inhibitors for 30 min.', 'lgd': None, 'rgd': {'seed': 'a', 'word': 'period'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**MHV-aN virus was bound to cells in DMEM with 2%FCS (in the absence or presence of inhibitors) at a MOI = 20** for 90 min at 4uC to synchronize infection, after which cells were shifted to 37uC for 2 h. Cells were trypsinized and transferred to Eppendorf tubes, washed and immediately analyzed by FACS.', 'lgd': {'word': 'MHV', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': '20'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**MHV wt virus was grown on LR7 cells and purified over a 20% sucrose** cushion in TN buffer by centrifugation at 110,000 rcf for 2.5 h. Supernatant was removed and pellet resuspended in 200 ml TN buffer overnight on ice.', 'lgd': {'word': 'MHV', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'sucrose'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Concentrated virus solution was subjected to further purification on a Pfefferkorn gradient** (10-20%, 25-50%, 50% cushion).', 'lgd': {'word': 'Concentrated', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'gradient'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**After spinning for 1 h at** 150,000 rcf a clear virus band was visible.', 'lgd': {'word': 'After', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'at'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The virus was pelleted by centrifugation at 110,000 rcf for 1 h and** resuspended in 200 ml 0.1M sodium phosphate, 0.15M NaCl buffer pH 7.2 overnight on ice.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'and'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**24 h after transfection MHV-DyLight488 was bound to cells on ice at MOI = 20 for 1.5 h** in DMEM, 2% FBS.', 'lgd': {'word': '24', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'h'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**For recording a Zeiss Axio** Observer Z1 inverse spinning-disk confocal microscope, equipped with full box stage incubation, including CO 2 (Pecon), argon-krypton and helium-neon laser, two Photometrics Evolve 512 back-illuminated electron-multiplying charge-coupled-device (EM-CCD) cameras, and 10061.46NA Oil alpha Plan Apochromat objective was used.', 'lgd': {'word': 'For', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'Axio'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**For each virus spot the area underlying a circle of** 0.213 mm 2 was measured for its gray mean value.', 'lgd': {'word': 'For', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': \"If **a virus particle** faded and disappeared (and could not be found in other z-stacks) whilst co-localizing in previous intervals with an endosomal vesicle it was categorized as 'fusing' (Figure S2 and S3 in Text S1).\", 'lgd': None, 'rgd': {'seed': 'a', 'word': 'particle'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': \"When **a viral particle** co-localized with endosomal compartments but did neither dissociate nor fade during the 10 min acquisition period it was classified as 'non-fusing'.\", 'lgd': None, 'rgd': {'seed': 'a', 'word': 'particle'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Alignments were performed over the entire length of the spike proteins using MegAlign (Lasergene DNASTAR) using a ClustalW alignment,** gap penalty 10, gap length penalty 0.2, delay divergent sequences 30%, DNA translation weight 0.5, protein weight matrix: PAM series, DNA weight matrix: ClustalW.', 'lgd': {'word': 'Alignments', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'alignment'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': \"**Briefly, 7'500 HeLa cells were seeded one day prior to transfection in a 96-well** plate.\", 'lgd': {'word': 'Briefly', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'well'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Transfection mix for one well contained 2.5 ml of 1 mM siRNA,** 10 ng plasmid, and 0.5 ml Oligofectamine in 12.5 ml OptiMEM (Gibco).', 'lgd': {'word': 'Transfection', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'siRNA'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Confluent HAP1, H1-DV33, and H1-DV33-fV33 cells and their stably mCeacam1a expressing counterparts were detached using a cell scraper,** homogenized, and fixed.', 'lgd': {'word': 'Confluent', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'scraper'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**After 30 min incubation in blocking buffer (3% BSA (Sigma), in PBS) for 1 h cells** were incubated in 1:100 N-CEACAM-Fc [80] antibody, washed, and stained with 2ry AF488 goat-anti-rabbit antibody (Life Technologies).', 'lgd': {'word': 'After', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'cells'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Viruses were purified and concentrated over a 20% sucrose** cushion (in TN buffer) at 110,000 rcf.', 'lgd': {'word': 'Viruses', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'sucrose'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Upon transfer to a nitrocellulose membrane** (Millipore), the presence of cellular and viral proteins was probed with antibodies against GM130 (rabbit pAb, Abcam), FLAG (HRPlabeled mouse anti-FLAG mAb, Sigma) or the S2 subunit of MHV A59 [105] (mouse anti-S2 mAb) diluted 1:1000.', 'lgd': {'word': 'Upon', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'membrane'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**When necessary, the blots were subsequently incubated with HRP-labeled rabbit anti-mouse or swine anti-rabbit antibodies (both diluted 1:5000; DAKO)**.', 'lgd': {'word': 'When', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'DAKO'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Cells were fixed in 4% formaldehyde in PBS for 15 min at RT, washed with PBS, and subsequently permeabilized in PBS containing 0.1% Triton-**X-100 for 10 min.', 'lgd': {'word': 'Cells', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'Triton'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Cells were incubated with antibody against LAMP1 (rabbit anti-LAMP1 pAb, 1:100 dilution;** Abcam) in 3% BSA in PBS followed by incubation with secondary antibodies coupled to AF488, AF-568 phalloidin, and DAPI (all Life Technologies).', 'lgd': {'word': 'Cells', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'dilution'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The samples were analyzed using a confocal laser-**scanning microscope (Leica SPE-II).', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'laser'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**LR7 cells were infected at MOI = 0.1 or MOI** = 4.0 with MHV-ERLM or MHV-S29FCS in DMEM containing 2% FBS and 25 mM HEPES (infection medium).', 'lgd': {'word': 'LR7', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'MOI'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**After 3 h of infection supernatant was replaced by fresh infection medium and infection was allowed to progress over a period of** 24 h. Every 3 h a small sample of the culture supernatant was collected and immediately frozen.', 'lgd': {'word': 'After', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The relative amount of viral RNA present was determined with a LightCycler 480** using LightCycler 480 RNA Master Hydrolysis kit (Roche Applied Biosciences) and specific primers and probe targeted against the MHV 1b gene by comparison with a standard curve.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': '480'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Coronavirus Cell Entry Occurs through the Endo-/ Lysosomal Pathway in a Proteolysis-Dependent** Manner', 'lgd': {'word': 'Coronavirus', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'Dependent'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Protein-Protein Interactions of Viroporins in Coronaviruses and Paramyxoviruses: New Targets for Antivirals?"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Viroporins are members of a rapidly growing** family of channel-forming small polypeptides found in viruses.', 'lgd': {'word': 'Viroporins', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'growing'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A deeper analysis of these interactions should not only provide a more complete** overview of the multiple functions of these viroporins, but also suggest novel strategies that target protein-protein interactions as much needed antivirals.', 'lgd': {'word': 'A', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'complete'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**CoVs belong to the subfamily Coronavirinae in the family Coronaviridae, and are organized into four gener**a [1]**** [2] [3] : , ,  and .', 'lgd': {'word': 'CoVs', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '2'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Of particular medical interest are the virus responsible for the severe acture respiratory syndrome (SARS), which produced a near pandemic** in 2003 [6] , and the recently emerged Middle East respiratory syndrome coronavirus (MERS-CoV), which after 3 years has caused hundreds of deaths [7] .', 'lgd': {'word': 'Of', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'pandemic'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Currently, no effective licensed treatments exist against coronavirus infection [8] [9] [10] , although live vaccines consisting of attenuated viruses are a promising strategy** [11, 12] , along with fusion inhibitors (reviewed in [13] ).', 'lgd': {'word': 'Currently', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'strategy'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The envelope (E) proteins are short polypeptides (76-109 amino acids) and are encoded by a CoV subgenomic** RNA either as a monocistronic or a polycistronic mRNA [14, 16] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'subgenomic'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**However, when cells were infected with a recombinant SARS-**CoV lacking the E gene-a more biologically relevant system-it was found that this increased apoptosis in cell culture [28] , whereas administration of E protein in trans reduced the stress response in cells infected with rSARS-CoV-E.', 'lgd': {'word': 'However', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'SARS'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Although the precise causes of attenuation are not known, they appear to be contributed by different parts of the E protein, which encompass the cytoplasmic C-terminal tail [33] as well as its transmembrane (TM) domain [34] (see Figure 1** ).', 'lgd': {'word': 'Although', 'seed': '1'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Proline in the C-terminal domain (highlighted grey) is flanked by predicted -sheet structure at both sides, forming a hypothetical -**coil- motif (BCBM) (red square).', 'lgd': {'word': 'Proline', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': ''}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In IBV, E protein has been shown to play a role in** the secretory pathway, altering lumenal environments and rearranging secretory organelles, and leading to efficient trafficking of virions [35] [36] [37] [38] [39] .', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'in'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**These rearrangement of host cell membranes, e.g., the** Golgi complex [37] , are observed during CoV infections, and virions appear in large vacuoles derived from Golgi/ERGIC membranes [38] .', 'lgd': {'word': 'These', 'seed': 'e'}, 'rgd': {'seed': 'e', 'word': 'the'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The IBV E mutation T16A, aligned with channel-inactive mutation N15A in SARS-CoV E (Figure 1 , arrow)**, resulted in decreased Golgi disruption [35, 36] , and this effect could not be restored by conservative polar substitutions Ser, Asn, or Gln [35] .', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'arrow'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In addition, a mutant IBV** E where the TM domain was replaced with a non-oligomerizing TM domain of vesicular stomatitis virus glycoprotein (VSV G) was defective in release of infectious virus particles [36] .', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'IBV'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**These effects could be due either to a loss of** channel activity or the loss of TM domain integrity, with subsequent disruption of protein-protein interactions.', 'lgd': {'word': 'These', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Although channel activity of the T16A mutant that disrupted Golgi rearrangement was not measured [35] , we have later found this mutant to be channel-inactive [40] , suggesting a possible link** between channel activity and Golgi rearrangement.', 'lgd': {'word': 'Although', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'link'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Although structural data are not available for most CoV E proteins, SARS-CoV E [41] [42] [43] [44] [45] [46] , MERS E [47] and IBV E [40] have all been shown to have a single -**helical TM domain ( Figure 1 ).', 'lgd': {'word': 'Although', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': ''}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The topology of E protein channels has been a subject of** controversy [35, 50, 51] , but a recent study [26] of untagged SARS-CoV E protein in infected cells produced a model with Figure 1 .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The fully conserved Proline in the C-terminal domain (highlighted grey) is flanked by predicted -sheet structure at both sides, forming a hypothetical -**coil- motif (BCBM) (red square).', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': ''}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In IBV, E protein has been shown to play a role in** the secretory pathway, altering lumenal environments and rearranging secretory organelles, and leading to efficient trafficking of virions [35] [36] [37] [38] [39] .', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'in'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**These rearrangement of host cell membranes, e.g., the** Golgi complex [37] , are observed during CoV infections, and virions appear in large vacuoles derived from Golgi/ERGIC membranes [38] .', 'lgd': {'word': 'These', 'seed': 'e'}, 'rgd': {'seed': 'e', 'word': 'the'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The IBV E mutation T16A, aligned with channel-inactive mutation N15A in SARS-CoV E (Figure 1 , arrow)**, resulted in decreased Golgi disruption [35, 36] , and this effect could not be restored by conservative polar substitutions Ser, Asn, or Gln [35] .', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'arrow'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In addition, a mutant IBV** E where the TM domain was replaced with a non-oligomerizing TM domain of vesicular stomatitis virus glycoprotein (VSV G) was defective in release of infectious virus particles [36] .', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'IBV'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**These effects could be due either to a loss of** channel activity or the loss of TM domain integrity, with subsequent disruption of protein-protein interactions.', 'lgd': {'word': 'These', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Although channel activity of the T16A mutant that disrupted Golgi rearrangement was not measured [35] , we have later found this mutant to be channel-inactive [40] , suggesting a possible link** between channel activity and Golgi rearrangement.', 'lgd': {'word': 'Although', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'link'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Although structural data are not available for most CoV E proteins, SARS-CoV E [41] [42] [43] [44] [45] [46] , MERS E [47] and IBV E [40] have all been shown to have a single -**helical TM domain ( Figure 1 ).', 'lgd': {'word': 'Although', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': ''}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The topology of E protein channels has been a subject of** controversy [35, 50, 51] , but a recent study [26] of untagged SARS-CoV E protein in infected cells produced a model with cytoplasmic C-terminal domain and lumenal N-terminus, and this topology is also likely in other E proteins [20, 24, 26, 35] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The only structural data available for a CoV E** protein is for SARS-CoV E, where the TM domain has been characterized in some detail in lipid membranes [42] and in DPC micelles [46] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'E'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Mutations at these residues abolished channel activity in vitro [52] , and introduction of these mutations in a recombinant SARS-**CoV resulted in in vivo attenuation in a mouse model [34] .', 'lgd': {'word': 'Mutations', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'SARS'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Synthetic peptides corresponding to the E protein TM domain bearing these \"revertant\" mutations, e.g., N15D,** V25L, or V25F-L19A, regained channel activity as measured in black lipid membranes [34] .', 'lgd': {'word': 'Synthetic', 'seed': 'e'}, 'rgd': {'seed': 'e', 'word': 'N15D'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**These revertant mutants also recovered pentameric integrity, as shown by an analysis of oligomeric size for mutants of a truncated form** of SARS-CoV E protein, E TR i.e., E (8-65) (Figure 2c ).', 'lgd': {'word': 'These', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'form'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In this assay, we used perfluorooctanoic acid polyacrylamide gel electrophoresis (PFO-NuPAGE), a system that** preserves the native oligomeric size in membrane proteins [53, 54] .', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'that'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Destabilization was obvious for V25F, where a monomeric species** can be observed at the bottom of the gel (black arrow).', 'lgd': {'word': 'Destabilization', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'species'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The only structural data available for a CoV E** protein is for SARS-CoV E, where the TM domain has been characterized in some detail in lipid membranes [42] and in DPC micelles [46] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'E'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Mutations at these residues abolished channel activity in vitro [52] , and introduction of these mutations in a recombinant SARS-**CoV resulted in in vivo attenuation in a mouse model [34] .', 'lgd': {'word': 'Mutations', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'SARS'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Synthetic peptides corresponding to the E protein TM domain bearing these \"revertant\" mutations, e.g., N15D,** V25L, or V25F-L19A, regained channel activity as measured in black lipid membranes [34] .', 'lgd': {'word': 'Synthetic', 'seed': 'e'}, 'rgd': {'seed': 'e', 'word': 'N15D'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**These revertant mutants also recovered pentameric integrity, as shown by an analysis of oligomeric size for mutants of a truncated form** of SARS-CoV E protein, ETR i.e., E (8-65) (Figure 2c ).', 'lgd': {'word': 'These', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'form'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In this assay, we used perfluorooctanoic acid polyacrylamide gel electrophoresis (PFO-NuPAGE), a system that** preserves the native oligomeric size in membrane proteins [53, 54] .', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'that'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Destabilization was obvious for V25F, where a monomeric species** can be observed at the bottom of the gel (black arrow).', 'lgd': {'word': 'Destabilization', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'species'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**However, both V25F revertant mutants, V25F-L27S and V25L, exhibited a more compact** pentamer than V25F, and no low molecular weight band corresponding to monomers (Figure 2c ).', 'lgd': {'word': 'However', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'compact'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In fact, the sedimentation equilibrium profile of these two revertant mutants, V25F-L27S and V25L, could be fitted to a monomer-pentamer** model with log K a = 16.7 and 17.5, respectively [40] , comparable to the wild-type E TR (log K a = 16).', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'pentamer'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The C-terminal domain of E proteins contains a totally conserved** proline residue, which in and -coronaviruses is at the center of a predicted -coil- motif, or BCBM ( Figure 1 ).', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'conserved'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Nevertheless, fragment 46-60 adopts 100% -structure when studied as a synthetic peptide** [44, 55] , and synthetic peptides inspired in that domain form amyloid fibers [55, 56] .', 'lgd': {'word': 'Nevertheless', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'peptide'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Viruses 2015, 7 5 However, both V25F revertant mutants, V25F-L27S and V25L, exhibited a more compact** pentamer than V25F, and no low molecular weight band corresponding to monomers (Figure 2c ).', 'lgd': {'word': 'Viruses', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'compact'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In fact, the sedimentation equilibrium profile of these two revertant mutants, V25F-L27S and V25L, could be fitted to a monomer-pentamer** model with log Ka = 16.7 and 17.5, respectively [40] , comparable to the wild-type ETR (log Ka = 16).', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'pentamer'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The C-terminal domain of E proteins contains a totally conserved** proline residue, which in and -coronaviruses is at the center of a predicted -coil- motif, or BCBM ( Figure 1 ).', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'conserved'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Nevertheless, fragment 46-60 adopts 100% -structure when studied as a synthetic peptide** [44, 55] , and synthetic peptides inspired in that domain form amyloid fibers [55, 56] . )', 'lgd': {'word': 'Nevertheless', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'peptide'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This discrepancy may point to the existence of a frustrated structure,** poised for conformational change to -structure.', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'structure'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In SARS-CoV E, this region was found to redirect a plasma membrane** protein to the Golgi region, and mutations designed to increase its -helical propensity, V56A-Y57A-V58A-Y59A (\"AAAA\" mutant), disrupted membrane localization [57] .', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'membrane'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**However, although it is possible that a small fraction** of the population has a BCBM folded as -sheet structure, the reported in detergent micelles that shows that the predicted -structure after the conserved proline residue (Pro54 to Lys63) is in fact -helical; adapted from [45] ; (right) pentameric model reconstructed from the E TR monomer [45] .', 'lgd': {'word': 'However', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'fraction'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This discrepancy may point to the existence of a frustrated structure,** poised for conformational change to -structure.', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'structure'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In SARS-CoV E, this region was found to redirect a plasma membrane** protein to the Golgi region, and mutations designed to increase its -helical propensity, V56A-Y57A-V58A-Y59A (\"AAAA\" mutant), disrupted membrane localization [57] .', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'membrane'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**However, although it is possible that a small fraction** of the population has a BCBM folded as -sheet structure, the reported cellular effects caused by the said \"AAAA\" mutant could have been equally caused by disruption of protein-protein interactions mediated by that exposed cytoplasmic domain.', 'lgd': {'word': 'However', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'fraction'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The proposition of a \"frustrated\"** BCBM is reminiscent of viral internal fusion peptides [58] [59] [60] [61] [62] [63] [64] and regions of high structural plasticity, e.g., the hinge region of fusion protein HIV-1 gp41 [65] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'frustrated'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In fusion proteins such as influenza hemagglutinin [66, 67] , vesicular stomatitis virus (VSV) G-protein [68] and presynaptic SNAREs [69] , the TM domain may translate a conformational change** in the extramembrane domain into local bilayer stress [70, 71] , and fusogenicity of TM domains in SNARE or VSV G-protein is inversely correlated to its helicity in solution [72, 73] .', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'change'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This is consistent with the presence in SNARE and other viral fusion protein TM domains [72, 74] of -sheet-promoting residues [75] , i.e., -**branched (I, V, T), glycine, or bulky (W, Y, F).', 'lgd': {'word': 'This', 'seed': 'e'}, 'rgd': {'seed': 'e', 'word': ''}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In both TM domain and C-terminal tail of CoV E proteins there is a high abundance** of -branched and bulky residues that can potentially destabilize -helical conformation (Figure 1 , red underlined).', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'abundance'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Although fusion/fission activity of CoV E proteins has never been demonstrated, we hypothesize that this could be due to a lack of** suitable in vitro assays, e.g., involving both E and M proteins.', 'lgd': {'word': 'Although', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In addition to its three predicted TM domains, it has a large C-**terminal extramembrane domain ( 120 residues) exposed to the cytoplasm or to the interior of the virion [85] , which forms contacts with the C-terminal tail of the E protein [23, 33, 82, 83] .', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'C'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**E mutants in MHV produced revertants wi**th a** partial duplicat**ion of the M gene.', 'lgd': {'word': 'E', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'duplication'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**These results suggested a common role** of the \"new M\" and E proteins in \"dispersing or de-aggregating\" M during packaging [87] .', 'lgd': {'word': 'These', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'role'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Recent yeast-two hybrid searches of interacting partners of SARS-CoV E using the C-terminal extramembrane domain as a bait have** produced abundant data, although the significance of only a few of these binders has been explored and reported [88, 89] .', 'lgd': {'word': 'Recent', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'have'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**One of these binders is the protein associated with Lin Seven 1 (PALS1)** [88] , which is part of a complex that controls polarity and tight junction formation in epithelia.', 'lgd': {'word': 'One', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'PALS1'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This interaction was found to involve PALS1 PDZ domain and the last four C-terminal residues of SARS-CoV E protein, through a X--**X- motif, where  is a hydrophobic amino acid.', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': ''}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**PDZ [post synaptic density protein (PSD95), Drosophila disc large tumor suppressor (Dlg1), and zonula occludens-1 protein (zo-**1)] domains are common structural domains of 80-90 amino acids found in any organism.', 'lgd': {'word': 'PDZ', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'zo'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The latter suggests a wider implication** of this newly found interaction, as viruses could harness the alteration of these pathways to their own advantage.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'implication'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This indicated a predominant random** coil structure in that part of the protein.', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'random'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In addition, a synthetic peptide** encompassing the last 18 residues (59 to 76) also adopted a random coil conformation, whether in solution or in presence of lipid zwitterionic membranes [55] .', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'peptide'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This is somewhat surprising, since PDZ-binding motifs adopt typically a -strand** conformation [90] .', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'strand'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In addition to binding to C-terminal peptides, some PDZ domains recognize internal peptide fragments [92] [93] [94] , e.g., Par** (partitioning defective)-6 PDZ domain can bind to an internal peptide fragment from PALS1 that adopts an extended conformation [92] .', 'lgd': {'word': 'In', 'seed': 'e'}, 'rgd': {'seed': 'e', 'word': 'Par'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This C-terminal PDZ-binding motif found in SARS-CoV E protein was later shown to be a determinant of** virulence, as infection of mice with viruses lacking this domain lowered their immune response [89] .', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The latter is a scaffolding protein** that can initiate a signaling cascade resulting in the phosphorylation and activation of p38 mitogen-activated protein kinase (p38-MAPK) [95] , leading to expression of proinflammatory cytokines.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'protein'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**However, a 12-residue** deletion at the C-terminus resulted in a virus with high pathogenicity [33] .', 'lgd': {'word': 'However', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'residue'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Noticeably, several other viruses, e.g., human** papillomavirus and influenza A, have been found to enhance pathogenesis through proteins containing PDZ binding motifs (reviewed in [97] ), which suggests that this is a particular case of a widely used viral strategy.', 'lgd': {'word': 'Noticeably', 'seed': 'e'}, 'rgd': {'seed': 'e', 'word': 'human'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**hRSV was first isolated in 1956 from a chimpanzee with** a respiratory illness, and later found to be a human virus [102] .', 'lgd': {'word': 'hRSV', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'with'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In developing countries, RSV is a significant cause** of death, with global estimates of more than 70,000 deaths in young children.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'cause'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The epithelial cells of the respiratory tract are the major sites of virus replication, but hRSV can infect a wide variety** of human and animal cells.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'variety'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The fusion (F) protein facilitates viral entry through the cell membrane [106] through formation of a 6-helix** bundle.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'helix'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This critical step has been targeted in vitro, e.g., peptides** that mimic conserved domains of RSV-F protein [107, 108] , peptides based on F-interacting RhoA GTPase [109] , dendrimer-like molecule RFI-641 [110] , or other organic compounds [111, 112] .', 'lgd': {'word': 'This', 'seed': 'e'}, 'rgd': {'seed': 'e', 'word': 'peptides'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Vaccines have been recently designed based on a stabilized RSV-**F form which preserves a highly antigenic site in its prefusion state, yielding RSV-specific neutralizing antibodies in mice and macaques [115, 116] .', 'lgd': {'word': 'Vaccines', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'RSV'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Recently, a vaccine candidate** based on the extracellular domain (C-terminal) of RSV viroporin, the small hydrophobic (SH) protein, has been reported [117] , and prevention of nasopulmonary infection in mice caused by RSV has been reported using stapled peptides targeting the fusogenic F-protein 6-helix bundle [118] .', 'lgd': {'word': 'Recently', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'candidate'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Palivizumab is a humanized monoclonal** antibody (IgG) directed against F protein but is only moderately effective [119] , which combined with its high cost [120] limits its use to a small fraction of patients worldwide.', 'lgd': {'word': 'Palivizumab', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'monoclonal'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The only licensed drug for use in infected individuals is ribavirin, a nucleoside analog,** but its efficacy is very limited [121] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'analog'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The hRSV genome transcribes 11 proteins [122] , including the three membrane proteins: fusion (F), small hydrophobic (SH), and attachment (G), which plays a role in** the initial interaction of the virus with the cell [123, 124] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'in'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The small hydrophobic (SH) protein is 64 (RSV subgroup A) or 65 (RSV subgroup B) amino acids long, with a single -**helical TM domain [125, 126] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': ''}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In fact, a recombinant RSV** with deletion of the SH gene has been proposed as a live vaccine in calves [136] and in humans [133] .', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'RSV'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In common to SARS-CoV E protein, SH protein blocks or delays apoptosis in infected cells [137] , and a similar anti-**apoptotic effect of SH protein has been observed for other members of the Paramyxoviridae family that encode SH proteins, e.g., J Paramyxovirus (JPV) [138, 139] , mumps virus (MuV), and the parainfluenza virus 5 (PIV5), formerly known as simian virus 5 (SV5)-see below.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'anti'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**For example, apoptosis induced by PIV5 SH was blocked by neutralizing antibodies against TNF- and TNF- receptor 1 (TNF-**R1), but not by an antibody against TNF-R2 [140] .', 'lgd': {'word': 'For', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'TNF'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Viruses are able to activate the inflammasome, which activates caspase-1 to produce** pro-inflammatory cytokines, e.g., IL-1.', 'lgd': {'word': 'Viruses', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'produce'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**For example, influenza A virus (IAV) activates NLRP3 as a result of** H + or ion flux from Golgi mediated by the M2 ion channel [142] .', 'lgd': {'word': 'For', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The channel activity of SARS-CoV E protein is required for the processing of IL-1 [34] , which requires caspase-1** activation.', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Further, an hRSV SH mutant led to reduced IL-1 secretion and caspase-1 expression, whereas** lipid raft disruptors, which may affect SH protein targeting to Golgi lipid rafts, blocked inflammasome activation [145] .', 'lgd': {'word': 'Further', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'whereas'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Therefore, although a suggestion has** been made to link inflammasome activation to SH protein channel activity and ion leakage from the Golgi during infection [145] , this has not been confirmed.', 'lgd': {'word': 'Therefore', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'has'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The TM domain of SH protein has a funnel-like** architecture [126] (Figure 4 ), as observed in other viroporins, e.g., influenza M2 [149] , SARS E protein [46] and HCV p7 [150] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'like'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A narrower region [126] in the TM domain is lined with hydrophobic side chains (Ile32, Ile36, Ile40 and Leu44) whereas the more open N-terminal region is lined by polar residues, i.e., His22,** Thr25 and Ser29 (Figure 4b ).', 'lgd': {'word': 'A', 'seed': 'e'}, 'rgd': {'seed': 'e', 'word': 'His22'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Therefore, although a suggestion has** been made to link inflammasome activation to SH protein channel activity and ion leakage from the Golgi during infection [145] , this has not been confirmed.', 'lgd': {'word': 'Therefore', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'has'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The TM domain of SH protein has a funnel-like** architecture [126] (Figure 4) , as observed in other viroporins, e.g., influenza M2 [149] , SARS E protein [46] and HCV p7 [150] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'like'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A narrower region [126] in the TM domain is lined with hydrophobic side chains (Ile32, Ile36, Ile40 and Leu44) whereas the more open N-terminal region is lined by polar residues, i.e., His22,** Thr25 and Ser29 (Figure 4b) .', 'lgd': {'word': 'A', 'seed': 'e'}, 'rgd': {'seed': 'e', 'word': 'His22'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In RSV SH protein, the TM -helix extends up to His-51 in the C-terminal region, followed by a loop, whereas** the N-terminal cytoplasmic extramembrane domain forms a short -helix (residues 5-14) (Figure 4b) , which is present both in micelles and in bicelles [148] .', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'whereas'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**For example, addition of a hemagglutinin [148]** .', 'lgd': {'word': 'For', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': '148'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In RSV SH protein, the TM -helix extends up to His-51 in the C-terminal region, followed by a loop, whereas** the N-terminal cytoplasmic extramembrane domain forms a short -helix (residues 5-14) (Figure 4b) , which is present both in micelles and in bicelles [148] .', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'whereas'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**For example, addition of a hemagglutinin antigen** epitope tag at the cytoplasmic tail of PIV5 SH protein abolished its ability to inhibit TNF- signaling [141] .', 'lgd': {'word': 'For', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'antigen'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Second, while the SH proteins of RSV and MuV (both human pathogens) have opposite topologies, they share a high degree** of similarity in a 10-residue stretch at the cytoplasmic domain (N-terminus for RSV SH and C-terminus for MuV SH) ( Figure 5) .', 'lgd': {'word': 'Second', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'degree'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Second, while the SH proteins of RSV and MuV (both human pathogens) have opposite topologies, they share a high degree** of similarity in a 10-residue stretch at the cytoplasmic domain (N-terminus for RSV SH and C-terminus for MuV SH) ( Figure 5) .', 'lgd': {'word': 'Second', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'degree'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Note that MuV SH is a type I** integral membrane protein, i.e., its C terminus faces the cytoplasm [154, 155] , therefore its orientation has been reversed for the alignment.', 'lgd': {'word': 'Note', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'I'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The TM domain is underlined and bold, and a \"10-**residue\" conserved fragment of high similarity has been highlighted in grey.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': '10'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**However, a tri-component** complex between SH, G and F proteins was not observed [156] .', 'lgd': {'word': 'However', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'component'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Recently, a membrane-based** yeast two-hybrid system (MbY2H) was used to identify a cellular binding partner of hRSV SH protein, the B-Cell receptor-associated protein 31 (BAP31) [160] , in a human lung cDNA library.', 'lgd': {'word': 'Recently', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'based'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**BAP31 is a membrane protein** located at the endoplasmic reticulum (ER) and has an essential role in sorting newly synthesized membrane proteins [161] .', 'lgd': {'word': 'BAP31', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'protein'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Additionally, BAP31 has a cytoplasmic C-**terminus that form two coiled coils [162, 163] , one of them containing a variant of the death effector domain (vDED) [164] flanked by two caspase-8 cleavage sites.', 'lgd': {'word': 'Additionally', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'C'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This domain is excised upon activation of caspase-8 [165, 166] to produce a fragment p20,** known to function as a proapoptotic factor [166, 167] .', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'p20'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**BAP31 also forms a complex with** the mitochondrial fission 1 (Fis1) [168] membrane protein, which spans the ER and mitochondria, and serves as a platform for activation of caspase-8.', 'lgd': {'word': 'BAP31', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'with'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**These contacts may interfere between BAP31 and caspase-8 interaction, e.g., by** SH protein binding to BAP31 vDED domain.', 'lgd': {'word': 'These', 'seed': 'e'}, 'rgd': {'seed': 'e', 'word': 'by'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Note that MuV SH is a type I** integral membrane protein, i.e., its C terminus faces the cytoplasm [154, 155] , therefore its orientation has been reversed for the alignment.', 'lgd': {'word': 'Note', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'I'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The TM domain is underlined and bold, and a \"10-**residue\" conserved fragment of high similarity has been highlighted in grey.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': '10'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**However, a tri-component** complex between SH, G and F proteins was not observed [156] .', 'lgd': {'word': 'However', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'component'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Recently, a membrane-based** yeast two-hybrid system (MbY2H) was used to identify a cellular binding partner of hRSV SH protein, the B-Cell receptor-associated protein 31 (BAP31) [160] , in a human lung cDNA library.', 'lgd': {'word': 'Recently', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'based'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**BAP31 is a membrane protein** located at the endoplasmic reticulum (ER) and has an essential role in sorting newly synthesized membrane proteins [161] .', 'lgd': {'word': 'BAP31', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'protein'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Additionally, BAP31 has a cytoplasmic C-**terminus that form two coiled coils [162, 163] , one of them containing a variant of the death effector domain (vDED) [164] flanked by two caspase-8 cleavage sites.', 'lgd': {'word': 'Additionally', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'C'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This domain is excised upon activation of caspase-8 [165, 166] to produce a fragment p20,** known to function as a proapoptotic factor [166, 167] .', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'p20'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**BAP31 also forms a complex with** the mitochondrial fission 1 (Fis1) [168] membrane protein, which spans the ER and mitochondria, and serves as a platform for activation of caspase-8.', 'lgd': {'word': 'BAP31', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'with'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**These contacts may interfere between BAP31 and caspase-8 interaction, e.g., by** SH protein binding to BAP31 vDED domain.', 'lgd': {'word': 'These', 'seed': 'e'}, 'rgd': {'seed': 'e', 'word': 'by'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Schematic depicting a hypothetical model** for apoptosis delay through interaction of SH and BAP31 proteins.', 'lgd': {'word': 'Schematic', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'model'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**(a) Activation of caspase 8 via TNF- or other effectors cleaves the cytoplasmic vDED domain of BAP31, producing pro-apoptotic p20; (b) SH protein may bind to various proteins in this pathway, e.**g., BAP31, blocking its cleavage by caspase-8; (c) ion channel activity through SH protein may contribute to production of IL-1 through activation of the inflammasome.', 'lgd': {'word': 'a', 'seed': 'e'}, 'rgd': {'seed': 'a', 'word': 'Activation'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The screening for SH protein binders referred to above [160] used a truncated form** of SH protein, encompassing only the TM domain and the cytoplasmic domain (M1-L44), therefore only these domains are involved in interaction with BAP31.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'form'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In the latter report, a stretch of** residues in the N-terminal cytoplasmic helix of SH protein was perturbed by addition of the BAP31 vDED domain to full length SH protein in micelles, with major shifts observed at residues Ile6, Ile8, Ser12 and Trp15 (Figure 7a ).', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Although there is conservation in that region between hRSV SH and mumps virus SH protein ( Figure 5 ), the residues indicated above are not conserved, whereas PIV5 SH protein does not show any homology in this region, suggesting that the latter do not bind BAP31, or they do but through a different binding** site.', 'lgd': {'word': 'Although', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'binding'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Indeed, apoptosis induced by Fas ligand (FasL) or by tumor necrosis factor-related apoptosis-inducing ligand (TRAIL) was strongly suppressed in keratinocytes expressing E5 [169] , a viroporin also** found to bind BAP31 [170] .', 'lgd': {'word': 'Indeed', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'also'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**E5 impaired formation of the death-inducing signaling complex triggered by TRAIL, showing that inhibition of ligand-mediated apoptosis in human keratinocytes is a primary function** of the HPV-16 E5 protein.', 'lgd': {'word': 'E5', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'function'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The mechanism of inhibition of TNF- signaling by SH proteins is not clear, but a model has** been proposed [141] where SH protein interacts directly or indirectly with TNF-R1, blocks TNF- signaling, and prevents more TNF- from being produced.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'has'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Schematic depicting a hypothetical model** for apoptosis delay through interaction of SH and BAP31 proteins.', 'lgd': {'word': 'Schematic', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'model'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**(a) Activation of caspase 8 via TNF- or other effectors cleaves the cytoplasmic vDED domain of BAP31, producing pro-apoptotic p20; (b) SH protein may bind to various proteins in this pathway, e.**g., BAP31, blocking its cleavage by caspase-8; (c) ion channel activity through SH protein may contribute to production of IL-1 through activation of the inflammasome.', 'lgd': {'word': 'a', 'seed': 'e'}, 'rgd': {'seed': 'a', 'word': 'Activation'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The screening for SH protein binders referred to above [160] used a truncated form** of SH protein, encompassing only the TM domain and the cytoplasmic domain (M1-L44), therefore only these domains are involved in interaction with BAP31.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'form'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In the latter report, a stretch of** residues in the N-terminal cytoplasmic helix of SH protein was perturbed by addition of the BAP31 vDED domain to full length SH protein in micelles, with major shifts observed at residues Ile6, Ile8, Ser12 and Trp15 (Figure 7a ).', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Although there is conservation in that region between hRSV SH and mumps virus SH protein ( Figure 5 ), the residues indicated above are not conserved, whereas PIV5 SH protein does not show any homology in this region, suggesting that the latter do not bind BAP31, or they do but through a different binding** site.', 'lgd': {'word': 'Although', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'binding'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Indeed, apoptosis induced by Fas ligand (FasL) or by tumor necrosis factor-related apoptosis-inducing ligand (TRAIL) was strongly suppressed in keratinocytes expressing E5 [169] , a viroporin also** found to bind BAP31 [170] .', 'lgd': {'word': 'Indeed', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'also'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**E5 impaired formation of the death-inducing signaling complex triggered by TRAIL, showing that inhibition of ligand-mediated apoptosis in human keratinocytes is a primary function** of the HPV-16 E5 protein.', 'lgd': {'word': 'E5', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'function'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The mechanism of inhibition of TNF- signaling by SH proteins is not clear, but a model has** been proposed [141] where SH protein interacts directly or indirectly with TNF-R1, blocks TNF- signaling, and prevents more TNF- from being produced.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'has'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Obvious candidates are polar residues that face the lumen of the channel (Figure 4b ), e.g., mutations** H22A, T25A and S29A, and mutants that disrupt channel structure, i.e., obtained by introducing bulky side chains in residues involved in TM helix-helix interactions.', 'lgd': {'word': 'Obvious', 'seed': 'e'}, 'rgd': {'seed': 'e', 'word': 'mutations'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Prior to the final FMOC-deprotection step, the peptide-resin is subjected to ring-closing olefin metathesis (i.e., stapling)** which increases peptide stability and helicity.', 'lgd': {'word': 'Prior', 'seed': 'e'}, 'rgd': {'seed': 'e', 'word': 'stapling'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In principle, both membrane and extramembrane parts of proteins can be targeted, as shown recently for Halobacterium salinarium where efflux by a small multidrug** resistance protein was inhibited with peptides targeting its TM domain [172] .', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'multidrug'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Viruses 2015, 7 13 e.g., mutations** H22A, T25A and S29A, and mutants that disrupt channel structure, i.e., obtained by introducing bulky side chains in residues involved in TM helix-helix interactions.', 'lgd': {'word': 'Viruses', 'seed': 'e'}, 'rgd': {'seed': 'e', 'word': 'mutations'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Prior to the final FMOC-deprotection step, the peptide-resin is subjected to ring-closing olefin metathesis (i.e., stapling)** which increases peptide stability and helicity.', 'lgd': {'word': 'Prior', 'seed': 'e'}, 'rgd': {'seed': 'e', 'word': 'stapling'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In principle, both membrane and extramembrane parts of proteins can be targeted, as shown recently for Halobacterium salinarium where efflux by a small multidrug** resistance protein was inhibited with peptides targeting its TM domain [172] .', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'multidrug'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Once defined the interactions between the SARS-CoV E protein C-terminal tail and other cellular or viral binders, e.g., between** E and M, a similar strategy could be followed by designing -helical peptides that mimic the cytoplasmically exposed -helical domain in the C-terminal tail of CoV E proteins (Figure 7b ).', 'lgd': {'word': 'Once', 'seed': 'e'}, 'rgd': {'seed': 'e', 'word': 'between'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Once defined the interactions between the SARS-CoV E protein C-terminal tail and other cellular or viral binders, e.g., between** E and M, a similar strategy could be followed by designing -helical peptides that mimic the cytoplasmically exposed -helical domain in the C-terminal tail of CoV E proteins (Figure 7b ).', 'lgd': {'word': 'Once', 'seed': 'e'}, 'rgd': {'seed': 'e', 'word': 'between'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This effect may be related to a common cytoplasmically** oriented membrane-bound -helix, connected to the TM domain by a flexible loop.', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'cytoplasmically'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Coronavirus Cell Entry Occurs through the Endo-/ Lysosomal Pathway in a Proteolysis-Dependent Manner"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Enveloped viruses need to fuse with a host cell** membrane in order to deliver their genome into the host cell.', 'lgd': {'word': 'Enveloped', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'cell'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Using recombinant MHVs expressing reporter genes as well as a novel, replication-**independent fusion assay we confirmed the importance of clathrin-mediated endocytosis and demonstrated that trafficking of MHV to lysosomes is required for fusion and productive entry to occur.', 'lgd': {'word': 'Using', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'replication'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Fusion of MHV was severely inhibited by a pan-lysosomal** protease inhibitor, while trafficking of MHV to lysosomes and processing by lysosomal proteases was no longer required when a furin cleavage site was introduced in the S protein immediately upstream of the fusion peptide.', 'lgd': {'word': 'Fusion', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'lysosomal'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In contrast, MERS-CoV, which contains a minimal furin** cleavage site just upstream of the fusion peptide, was negatively affected by inhibition of furin, but not of lysosomal proteases.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'furin'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**We conclude that a proteolytic cleavage** site in the CoV S protein directly upstream of the fusion peptide is an essential determinant of the intracellular site of fusion.', 'lgd': {'word': 'We', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'cleavage'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**(2014) Coronavirus Cell Entry Occurs through the Endo-/Lysosomal Pathway **in a** Proteolysis-Depend**ent Manner.', 'lgd': {'word': '2014', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'Dependent'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**To achieve successful infection enveloped viruses need to fuse with a host cell** membrane to deliver the viral genome into the host cell.', 'lgd': {'word': 'To', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'cell'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Some viruses, such as herpes simplex virus, Sendai virus, and human immunodeficiency virus, appear to be capable of direct fusion at the plasma membrane after initial attachmen**t [1]**** [2] [3] [4] [5] .', 'lgd': {'word': 'Some', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '2'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**These triggers, which may include a decrease in** pH, changes in redox environment, and proteolytic activity [6] [7] [8] , induce conformational changes in the viral fusion proteins leading to the merger of viral and host membranes.', 'lgd': {'word': 'These', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'in'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**They are capable of infecting a wide variety** of mammalian and avian species.', 'lgd': {'word': 'They', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'variety'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Human coronaviruses (HCoVs) are known as major causes of the common cold (e.**g.', 'lgd': {'word': 'Human', 'seed': 'e'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The well-studied mouse hepatitis virus (MHV) is often used as a safe model** to study CoV infections.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'model'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**All CoV virions contain a canonical set** of four structural proteins.', 'lgd': {'word': 'All', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'set'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Trimers of the CoV S protein, a type I** membrane protein belonging to the class I fusion proteins, form the peplomers that protrude from the virion surface [20] .', 'lgd': {'word': 'Trimers', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'I'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Also feline infectious peritonitis virus (FIPV) was suggested to enter via a clathrin-and** caveolaeindependent endocytic route [23, 24] .', 'lgd': {'word': 'Also', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'and'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**For the HCoV-229E a caveolae-dependent** endocytic uptake has been suggested [25] .', 'lgd': {'word': 'For', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'dependent'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Although the ability of MHV S proteins to cause cell-cell fusion at a neutral pH** was initially interpreted as an indication for fusion of virions at the cell surface, more recent studies indicate the requirement for clathrin-mediated endocytosis for entry of MHV [26] [27] [28] [29] .', 'lgd': {'word': 'Although', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'pH'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**For SARS-CoV and infectious bronchitis virus (IBV), it appears that a proteolytic cleavage** of the S protein at a more downstream position than the S1/S2 boundary upon receptor binding is of importance for cell entry [40, 43, [46] [47] [48] [49] .', 'lgd': {'word': 'For', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'cleavage'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In the present study we performed a detailed investigation** of the entry of different CoVs.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'investigation'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**By using recombinant MHVs expressing reporter genes as well as by applying a novel, replication-**independent fusion assay we confirmed the importance of clathrin-mediated endocytosis and demonstrated that trafficking of MHV virions to lysosomal compartments and processing of the S protein by lysosomal proteases was required for productive entry to occur.', 'lgd': {'word': 'By', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'replication'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Our results indicate that a cleavage site** in the S protein of CoVs immediately upstream of the FP determines the site of fusion.', 'lgd': {'word': 'Our', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'site'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In contrast, MERS-CoV, which contains a minimal furin-**cleavage site consensus sequence in the S protein immediately upstream of the FP, was negatively affected by inhibition of furin, but not of lysosomal proteases.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'furin'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In an automated, high-throughput RNAi screen [50] targeting the druggable genome (approximately 7000 genes) a number of** proteins associated with endocytosis were found to be required for efficient infection of HeLa cells with GFP-expressing MHV.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**To validate these findings these proteins were subjected to a follow-up** analysis using siRNA-mediated gene silencing with oligonucleotides from a different supplier than the one used for the initial RNAi screen (Fig.', 'lgd': {'word': 'To', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'up'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The HOPS subunit VPS39 (reviewed in [54] ) was also found to be a strong hit** in the siRNA screen and therefore selected.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'hit'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Other proteins included SNX1, involved in retrograde transport of cargo between endosomes and the trans-Golgi network (reviewed in [55] ), VCL, inter alia involved in connecting the Arp2/3 complex with integrins during actin polymerization (reviewed in [56] ), and the Ser/Thr-protein kinase PAK1, which is activated by the Rho/Rac/Cdc42 family and is implicated in a variety of** downstream effects including modulation of the actin cytoskeleton (reviewed in [57] ).', 'lgd': {'word': 'Other', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Enveloped viruses need to fuse with a host cell** membrane in order to deliver their genome into the host cell.', 'lgd': {'word': 'Enveloped', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'cell'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**By combining cellbiological, infection, and fusion assays we demonstrated that murine hepatitis virus (MHV), a prototypic member** of the CoV family, enters cells via clathrin-mediated endocytosis.', 'lgd': {'word': 'By', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'member'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Moreover, although MHV does not depend on a low pH** for fusion, the virus was shown to rely on trafficking to lysosomes for proteolytic cleavage of its spike (S) protein and membrane fusion to occur.', 'lgd': {'word': 'Moreover', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'pH'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In conclusion, we elucidated the MHV entry pathway in detail and demonstrate that a proteolytic cleavage** site in the S protein of different CoVs is an essential determinant of the intracellular site of fusion.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'cleavage'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**siRNAs targeting RAB7A were shown to inhibit the expression of a RAB7a-fusion** protein ( Figure S1 in Text S1).', 'lgd': {'word': 'siRNAs', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'fusion'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**To confirm and extend our understanding of the role of endocytosis in MHV entry we subsequently selected a number of** proteins known to be involved in either caveolae-or clathrinmediated endocytosis, actin-or microtubule-mediated transport, as well as proteins associated with endosomal vesicles and endosomal maturation, to be screened using the siRNA silencing-approach described above.', 'lgd': {'word': 'To', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Downregulation of most proteins associated with clathrin-mediated endocytosis inhibited MHV infection, Figure 1** .', 'lgd': {'word': 'Downregulation', 'seed': '1'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Actin cytoskeleton altering drugs Latrunculin A (LatA), Jasplakinolide (Jasp), Cytochalasin B (CytoB), and Cytochalasin D (CytoD), or the inducer of microtubule depolymerization Nocodazole (Noc) only decreased MHV infection when added early, indicating a role for** the actin and microtubule cytoskeleton in entry but not RNA replication ( Figure 2 , dark orange and orange).', 'lgd': {'word': 'Actin', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'for'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Likewise U18666A, a cholesterol transport-**affecting agent, which also prevents maturation of late endosomes [62] , had a strong inhibitory effect on MHV infection when added early (Figure 2 , turquoise).', 'lgd': {'word': 'Likewise', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'transport'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**To unequivocally demonstrate the importance of clathrin-mediated endocytosis and endosome maturation for MHV entry, we therefore made use of a fusion assay** we recently developed [63] .', 'lgd': {'word': 'To', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'assay'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The assay is based on minimal complementation of defective bgalactosidase (b-galactosidase DM15) with the short a-peptide [64]** .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': '64'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**MHV-aN, a recombinant MHV** containing an N protein tagged with the a-peptide (aN), is used to infect DM152fragment expressing target cells.', 'lgd': {'word': 'MHV', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'MHV'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Upon fusion of the virion with a host cell** membrane aN is released into the cytoplasm resulting in complementation of the defective b-galactosidase thereby reconstituting a functional enzyme.', 'lgd': {'word': 'Upon', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'cell'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**After 100 min incubation of cells with virus at 37uC, cells were collected and the amount of FIC generated as a results of** enzyme complementation analyzed by FACS.', 'lgd': {'word': 'After', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**MHV fusion was barely affected by replication inhibitor BrefA, whereas MG132 had a clear negative** effect, in agreement with the conclusion drawn previously that MG132 inhibits entry of MHV as well as RNA synthesis [61] .', 'lgd': {'word': 'MHV', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'negative'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Disturbance of the actin cytoskeleton by EIPA or by LatA, CytoB, or CytoD reduced fusion by 75-80% ( Figure 3B , dark orange), while interference with microtubule polymerization by Noc had a smaller effect** ( Figure 3B , orange).', 'lgd': {'word': 'Disturbance', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'effect'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Cells were imaged using a spinning-disc** confocal microscope acquiring zstacks in 30 s intervals over 10 min time frames from 10-70 min post warming.', 'lgd': {'word': 'Cells', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'disc'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A virion was categorized as associating with a certain endosomal** marker only if this co-localization was observed over at least four sequential 30 s interval images.', 'lgd': {'word': 'A', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'endosomal'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Complete disappearance of a virus particle** (including in other z-stacks) while immediately previously co-localizing with an endosomal marker was categorized as a fusion event (Figures S3 and S4 in Text S1).', 'lgd': {'word': 'Complete', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'particle'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': 'When **a viral particle** co-localized with endosomal compartments but did neither dissociate nor fade during the 10 min acquisition period it was classified as non-fusing.', 'lgd': None, 'rgd': {'seed': 'a', 'word': 'particle'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': 'Only **a very small** percentage of virions were categorized as fusing while in early endosomes.', 'lgd': None, 'rgd': {'seed': 'a', 'word': 'small'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Luciferase expression levels indicated that infection of cells with VSV and IAV is much more affected by BafA1, with an IC50 values of 0.80 and 0.63 nM, respectively, compared to MHV, which displays a three to** four fold higher IC50 of 2.34 nM ( Figure 5A ).', 'lgd': {'word': 'Luciferase', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'to'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**To confirm and extend these observations, we made use of haploid HAP1 cells lacking a functional HOPS** complex resulting from lentiviralmediated knockout of the VPS33A subunit (H1-DV33 cells; [74] ).Both HAP1 cells and H1-DV33 cells were modified to stably express the MHV receptor.', 'lgd': {'word': 'To', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'HOPS'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': 'As **a control, the** H1-DV33 cells were in addition stably transfected with FLAG-tagged VPS33A (H1-DV33-fV33).', 'lgd': None, 'rgd': {'seed': 'a', 'word': 'the'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The lack of a functional HOPS** complex had no effect on VSV and IAV infection; however, MHV infection was strongly reduced in the knockout, but not in the re-transfected cells ( Figure 5B ).', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'HOPS'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**These observations confirm the conclusion that late endosome-tolysosome maturation is required for efficient entry of MHV, a characteristic that** is not shared with the pH-sensitive VSV and IAV.', 'lgd': {'word': 'These', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'that'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Considering that MHV was much less affected by perturbation of the endosomal pH than IAV and VSV while it requires trafficking to lysosomes for efficient entry, we hypothesized that entry might depend on cleavage of a viral protein** by lysosomal proteases.', 'lgd': {'word': 'Considering', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'protein'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Exceptions were AEBSF, which has been shown to cause aggregation of early endosomal vesicles [75] , and a pan-lysosomal** protease inhibitor (CPI; cystatinpepstatin inhibitor) capable of inhibiting the three major protease family members found in lysosomes.', 'lgd': {'word': 'Exceptions', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'lysosomal'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**From these results we conclude that inhibition of a broad range** of endosomal proteases efficiently blocks fusion of MHV, indicating that efficient entry requires the activity of lysosomal proteases.', 'lgd': {'word': 'From', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'range'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Introduction of a furin cleavage** site immediately upstream of the fusion peptide renders MHV independent of lysosomal proteases', 'lgd': {'word': 'Introduction', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'cleavage'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**To test this hypothesis, we introduced an optimal furin cleavage site (FCS) by substituting three amino acids by Arg (AIRGRRRRRRR) immediately upstream of a highly conserved** Arg (indicated in bold) that occurs just N-terminal of the FP.', 'lgd': {'word': 'To', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'conserved'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Western blot analysis of the S protein of a purified stock** of this virus using an antibody recognizing the S2 subunit showed no evidence of cleavage at the newly introduced FCS (S29 site).', 'lgd': {'word': 'Western', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'stock'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**5) , infection with MHV carrying a wild type** S was severely reduced in cells lacking a functional HOPS complex and addition of the FI did not alter this effect (Figure 8 , red bars).', 'lgd': {'word': '5', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'type'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In contrast, infection with MHV-S29FCS was not decreased by the lack of a functional HOPS** complex.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'HOPS'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**However, FI treatment had a clearly negative** effect on this virus, which was much more dramatic in the absence of a functional HOPS complex in H1-DV33 cells (Figure 8, blue) .', 'lgd': {'word': 'However', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'negative'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In conclusion, MHV-S29FCS lost the requirement for a functional HOPS** complex in parallel with this virus becoming insensitive to the pan-lysosomal protease inhibitor CPI.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'HOPS'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**1) , infection** with wild type S protein carrying MHV was reduced after gene silencing of RAB5, RAB7, VPS11, and VPS41 (Figure 9 , red bars).', 'lgd': None, 'rgd': {'seed': '1', 'word': 'infection'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**From these results we conclude that introduction of a FCS immediately** upstream of the FP abolishes the requirement for trafficking of virions to lysosomes and for processing by lysosomal proteases.', 'lgd': {'word': 'From', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'immediately'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Interestingly, MERS-CoV and IBV-Beaudette contain a minimal furin** cleavage site Arg-X-X-Arg just upstream of the fusion peptide ( Figure 10A ).', 'lgd': {'word': 'Interestingly', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'furin'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Since the requirement for a functional HOPS** complex is indicative of fusion in lysosomes, as we observed for MHV, we analyzed whether FIPV requires processing by lysosomal proteases for efficient entry as well.', 'lgd': {'word': 'Since', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'HOPS'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**As MERS-CoV carries a FCS in** its S protein immediately upstream of the FP, we hypothesized this virus not to require trafficking to lysosomes and processing by lysosomal proteases for efficient entry.', 'lgd': {'word': 'As', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'in'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Cells were subsequently inoculated with MERS-CoV at a MOI of** 0.1 in the presence of these inhibitors.', 'lgd': {'word': 'Cells', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The results indicate that, in contrast to wild type MHV and FIPV, but similarly to recombinant MHV carrying a FCS immediately** upstream of the FP, infection with MERS-CoV is strongly inhibited by the FI but not by CPI (Figure 11 ), indicating that MERS-CoV does not require trafficking to lysosomes for efficient entry.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'immediately'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Based on these results we conclude that the cleavage site in the CoV S protein immediately upstream of the FP is a key determinant** of the intracellular site of fusion.', 'lgd': {'word': 'Based', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'determinant'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**By using a replication-independent** fusion assay, we confirmed that MHV entry requires clathrin-mediated endocytosis despite the well- known ability of the MHV S protein to cause cell-cell fusion at neutral pH.', 'lgd': {'word': 'By', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'independent'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Efficient inhibition of MHV entry was only observed using a pan-lysosomal** protease inhibitor, and could not be achieved using more specific protease inhibitors.', 'lgd': {'word': 'Efficient', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'lysosomal'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Consistent with a common mechanism** for the entry of CoVs, FIPV, but not MERS-CoV, the latter of which contains a furin cleavage site immediately upstream of the FP, was shown to require trafficking to lysosomes and processing by lysosomal proteases for efficient entry.', 'lgd': {'word': 'Consistent', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'mechanism'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Based on these results we propose a model in** which the cleavage site immediately upstream of the FP is an essential determinant of the intracellular site of CoV fusion ( Figure 12 ).', 'lgd': {'word': 'Based', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'in'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Furthermore, we made use of three independent siRNAs per target gene, and a target was** only classified as a hit when at least two out three siRNAs showed the same phenotype.', 'lgd': {'word': 'Furthermore', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'was'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Finally, the results obtained were corroborated by using a large panel** of inhibitors and by making use of haploid knockout cells, in which late endosome-to-lysosome trafficking was inhibited.', 'lgd': {'word': 'Finally', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'panel'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Endocytic uptake is also required for a mutant virus** carrying a S protein with a FCS immediately upstream of its FP, despite the relative insensitivity to high concentrations of BafA1.', 'lgd': {'word': 'Endocytic', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'virus'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Therefore, the ability of a virus to** infect cells in the presence of BafA1 does not necessarily imply virus entry to occur at the cell surface.', 'lgd': {'word': 'Therefore', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'to'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': 'Also **a recombinant MHV** carrying the spike protein of MHV-4 (MHV-JHM) was found to enter via clathrin-mediated endocytosis (MHV-S4; Figure S10 in Text S1) despite its ability to cause extensive cell-cell fusion [80] [81] [82] .', 'lgd': None, 'rgd': {'seed': 'a', 'word': 'MHV'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Although our replicationdependent assays indicate a requirement for** caveolin 2 (CAV2) for infection, this protein was shown not to be involved in virus entry using our fusion assay.', 'lgd': {'word': 'Although', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'for'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Also depletion of other proteins involved in caveolae-mediated endocytosis, including caveolin 1 (CAV1)** and flotillins 1 and 2 (FLOT1 and FLOT2) did not affect MHV infection or fusion.', 'lgd': {'word': 'Also', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'CAV1'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Interestingly, fusion of MHV was severely inhibited by EIPA, an inhibitor of the Na + /H + exchanger NHE1, which is regarded as a hallmark inhibitor** of macropinocytosis.', 'lgd': {'word': 'Interestingly', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'inhibitor'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Apparently, inhibition of virus entry by EIPA does not prove by itself that a virus enters** via this particular pathway.', 'lgd': {'word': 'Apparently', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'enters'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The importance of lysosomes for entry was confirmed by using knockout cells lacking a functional HOPS** complex (For a review on the HOPS complex see [54] ).', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'HOPS'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Interestingly, in these cells lysosomes are clustered in a perinuclear region** of the cell rather than dispersed throughout the cytoplasm.', 'lgd': {'word': 'Interestingly', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'region'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**However, we cannot exclude that actin also plays a role in** the clathrin-mediated uptake of MHV particles, as has been observed for VSV and other pathogens that depend on clathrin-mediated endocytosis (reviewed in [91] ).', 'lgd': {'word': 'However', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'in'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The absence of a functional HOPS** complex, which is required for late endosome-to-lysosome maturation, did not affect Trafficking of MHV from late endosomes to lysosomes (RAB7/LAMP1-positive compartments) is required for processing of MHV by lysosomal proteases and viral fusion to occur.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'HOPS'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**We propose that the sequence immediately upstream of the FP is a key determinant** of the intracellular site of fusion.', 'lgd': {'word': 'We', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'determinant'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**MERS-CoV, which contains a minimal FCS,** is inhibited by furin inhibitor (FI) but not by the pan-lysosomal protease inhibitor (CPI).', 'lgd': {'word': 'MERS', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'FCS'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**10) , to fuse in early endosomes in a furin-dependent** manner.', 'lgd': {'word': '10', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'dependent'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Other CoVs that do not contain a FCS at** this position are predicted to fuse in lysosomes.', 'lgd': {'word': 'Other', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'at'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Strikingly, other inhibitors that affect members of a single protease** family had none or only little impact on MHV fusion.', 'lgd': {'word': 'Strikingly', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'protease'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**These results are in consistence with a functional redundancy** of protease family members [47, 76] and may explain why previous studies using specific lysosome protease inhibitors [27, 92] failed to detect entry inhibition.', 'lgd': {'word': 'These', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'redundancy'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Although we did not demonstrate cleavage of MHV S at the FP proximal position directly, a recent study** found a cleaved form of the MHV S2 subunit to correspond with the fusion-active form [49] .', 'lgd': {'word': 'Although', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'study'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Furthermore, introduction of an optimal FCS at the FP proximal position abolished the entry inhibition by the pan-lysosomal protease inhibitor whilst introducing a dependency on** furin-related enzymes.', 'lgd': {'word': 'Furthermore', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'on'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**However, in the presence of furin inhibitor, entry of this mutant MHV was much more efficient in wild type cells than in cells lacking a functional HOPS** complex (Fig.', 'lgd': {'word': 'However', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'HOPS'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**8) , indicating that under certain circumstances lysosomal proteases may play a role in** entry of this virus as well.', 'lgd': {'word': '8', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'in'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Trafficking of virions to lysosomes was shown to be also important for entry of FIPV, but not of MERS-CoV, in agreement with the latter virus containing a putative FCS** immediately upstream of the FP.', 'lgd': {'word': 'Trafficking', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'FCS'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Based on the present study and on the work of others, we conclude that cleavage at the FP proximal position is likely to be a general requirement** for CoV entry.', 'lgd': {'word': 'Based', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'requirement'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Our results furthermore show that cleavage at a FP-proximal** position is an important determinant of the intracellular site of fusion.', 'lgd': {'word': 'Our', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'proximal'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Stable cell lines were generated using a Moloney murine** leukemia (MLV) retroviral vector.', 'lgd': {'word': 'Stable', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'murine'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**MLV was produced in HEK293T cells by triple plasmid transfection of a transfer vector** containing the DM15 or mCC1a gene as well as a puromycin or neomycin resistance marker gene, respectively, in combination with expression vectors encoding the MLV Gag-Pol, and the VSV spike protein G. Upon MLV transduction, stably transduced cells were selected at 2 mg/ml puromycin and/or 0.5 mg/ml G418 (both Sigma), maintenance at 1 mg/ml puromycin and/or 0.5 mg/ml G418 in DMEM, supplemented with 10% FBS.', 'lgd': {'word': 'MLV', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'vector'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**H1-DV33 cells were stably transfected with FLAG-tagged VPS33A (H1-DV33-fV33) using MLV transduction as described above using a blasticidin resistance** marker gene in the transfer vector.', 'lgd': {'word': 'H1', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'resistance'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The DM15 gene was isolated from a DH5 E.** coli strain by DNA extraction and PCR.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'E'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The gene was cloned into a pCAGGS vector** for (transient) expression and into a MLV-based pQCXIP transfer vector (Clontech), resulting pQCXIP-DM15, for the generation of stable cell lines.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'vector'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**pMH54 containing a GFP expression** cassette between the E and M gene was generated as described previously for firefly luciferase [59] .', 'lgd': {'word': 'pMH54', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'expression'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The transcription vector used to generate MHV-S29FCS (pXHERLM-S29FCS+) was generated by site-directed mutagenesis, thereby changing the sequence encoding AIRGR immediately upstream of the FP into a RRRRR-encoding** sequence in vector pXHERLM [59] (GCA9ATC9CGA9GGG9CGT to AGA9CGC9CGA9AGG9CGT).', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'encoding'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The transcription vector used to generate MHV-S4 expressing firefly luciferase, was generated by introducing the firefly luciferase expression cassette between the E and M genes similarly as described previously [59] in a pMH54derived transcription** vector that contains the gene encoding the S protein of MHV-4 (MHV-JHM) [82] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'transcription'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Recombinant MHV-EGFPM virus, containing a GFP expression** cassette between the E and the M gene, MHV-S29FCS, containing a Renilla luciferase expression cassette between the E and the M gene and a FCS at the FP-proximal position, and MHV-S4 containing the spike gene of MHV-4 (JHM) and a luciferase expression cassette were generated by targeted RNA recombination as described before [98] .', 'lgd': {'word': 'Recombinant', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'expression'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Briefly, donor RNA was generated from linearized pMH54derived transfer vectors described above, and electroporated into FCWF cells infected with interspecies chimeric fMHV coronavirus (an MHV-A59 derivative, in which the ectodomain of the spike protein has been replaced by that of a feline coronavirus,** thereby changing host cell tropism).', 'lgd': {'word': 'Briefly', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'coronavirus'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The electroporated FCWF cells were seeded onto a monolayer of** LR7 cells.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Generation of MHV-EFLM and MHV-ERLM, containing a firefly or** Renilla luciferase expression cassette between the E and the M gene, and MHV-aN, containing a N protein tagged with the a-peptide, has been described before [63, 103] .', 'lgd': {'word': 'Generation', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'or'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**siRNA transfections 30 ,000 HeLa-mCC1a-(DM15) cells were seeded one day prior to transfection in a 24-well** dish.', 'lgd': {'word': 'siRNA', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'well'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Transfection mix for one well contained 2.5 ml of 1 mM siRNA** and 0.5 ml Oligofectamine in 50 ml OptiMEM (Gibco).', 'lgd': {'word': 'Transfection', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'siRNA'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**mRNA levels of genes were analyzed by qRT-PCR using a custom designed** pair of specific primers to the gene resulting in about 150 bp products.', 'lgd': {'word': 'mRNA', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'designed'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': \"**RNA levels were measured using the GoTaq 1-Step RT-**qPCR system (Promega) according to the manufacturers' instructions on a LightCycler 480 (Roche).\", 'lgd': {'word': 'RNA', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'RT'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Expression levels were corrected for cell number and viability as determined by the Wst-1 assay (Roche)**.', 'lgd': {'word': 'Expression', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'Roche'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Firefly luciferase expression was assessed using the firefly luciferase assay system from Promega or using a homemade system** (50 mM tricine, 100 mM EDTA, 2.5 mM MgSO 4 , 10 mM DTT, 1.25 mM ATP, 12.5 mM D-Luciferin).', 'lgd': {'word': 'Firefly', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'system'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Light emission was measured on a Centro LB** 960 luminometer.', 'lgd': {'word': 'Light', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'LB'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Luciferase expression levels (in relative light units, RLU) were corrected for cell number and viability as determined by the Wst-1 assay (Roche)**.', 'lgd': {'word': 'Luciferase', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'Roche'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Huh-7 cells were inoculated with MERS-CoV at a MOI of** 0.1 in FBScontaining DMEM.', 'lgd': {'word': 'Huh', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Cells were stained using rabbit anti-SARS-CoV nsp4 antibodies that are cross-reactive for MERS-CoV, according to a standard protocol** using a FITC-conjugated swineanti-rabbit antibody.', 'lgd': {'word': 'Cells', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'protocol'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Number of infected cells was determined by cell counts on a wide-field** fluorescent microscope.', 'lgd': {'word': 'Number', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'field'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': 'After **a recovery period** of 30 min at 37uC, cells were (mock) treated with the different inhibitors for 30 min.', 'lgd': None, 'rgd': {'seed': 'a', 'word': 'period'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**MHV-aN virus was bound to cells in DMEM with 2%FCS (in the absence or presence of inhibitors) at a MOI = 20** for 90 min at 4uC to synchronize infection, after which cells were shifted to 37uC for 2 h. Cells were trypsinized and transferred to Eppendorf tubes, washed and immediately analyzed by FACS.', 'lgd': {'word': 'MHV', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': '20'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**24 h after transfection MHV-DyLight488 was bound to cells on ice at MOI = 20 for 1.5 h** in DMEM, 2% FBS.', 'lgd': {'word': '24', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'h'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**For recording a Zeiss Axio** Observer Z1 inverse spinning-disk confocal microscope, equipped with full box stage incubation, including CO 2 (Pecon), argon-krypton and helium-neon laser, two Photometrics Evolve 512 back-illuminated electron-multiplying charge-coupled-device (EM-CCD) cameras, and 10061.46NA Oil alpha Plan Apochromat objective was used.', 'lgd': {'word': 'For', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'Axio'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**For each virus spot the area underlying a circle of** 0.213 mm 2 was measured for its gray mean value.', 'lgd': {'word': 'For', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': \"If **a virus particle** faded and disappeared (and could not be found in other z-stacks) whilst co-localizing in previous intervals with an endosomal vesicle it was categorized as 'fusing' (Figure S2 and S3 in Text S1).\", 'lgd': None, 'rgd': {'seed': 'a', 'word': 'particle'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': \"When **a viral particle** co-localized with endosomal compartments but did neither dissociate nor fade during the 10 min acquisition period it was classified as 'non-fusing'.\", 'lgd': None, 'rgd': {'seed': 'a', 'word': 'particle'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Alignments were performed over the entire length of the spike proteins using MegAlign (Lasergene DNASTAR) using a ClustalW alignment,** gap penalty 10, gap length penalty 0.2, delay divergent sequences 30%, DNA translation weight 0.5, protein weight matrix: PAM series, DNA weight matrix: ClustalW.', 'lgd': {'word': 'Alignments', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'alignment'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': \"**Briefly, 7'500 HeLa cells were seeded one day prior to transfection in a 96-well** plate.\", 'lgd': {'word': 'Briefly', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'well'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Transfection mix for one well contained 2.5 ml of 1 mM siRNA,** 10 ng plasmid, and 0.5 ml Oligofectamine in 12.5 ml OptiMEM (Gibco).', 'lgd': {'word': 'Transfection', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'siRNA'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Confluent HAP1, H1-DV33, and H1-DV33-fV33 cells and their stably mCeacam1a expressing counterparts were detached using a cell scraper,** homogenized, and fixed.', 'lgd': {'word': 'Confluent', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'scraper'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**After 30 min incubation in blocking buffer (3% BSA (Sigma), in PBS) for 1 h cells** were incubated in 1:100 N-CEACAM-Fc [80] antibody, washed, and stained with 2ry AF488 goat-anti-rabbit antibody (Life Technologies).', 'lgd': {'word': 'After', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'cells'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Viruses were purified and concentrated over a 20% sucrose** cushion (in TN buffer) at 110,000 rcf.', 'lgd': {'word': 'Viruses', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'sucrose'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Upon transfer to a nitrocellulose membrane** (Millipore), the presence of cellular and viral proteins was probed with antibodies against GM130 (rabbit pAb, Abcam), FLAG (HRPlabeled mouse anti-FLAG mAb, Sigma) or the S2 subunit of MHV A59 [105] (mouse anti-S2 mAb) diluted 1:1000.', 'lgd': {'word': 'Upon', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'membrane'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**When necessary, the blots were subsequently incubated with HRP-labeled rabbit anti-mouse or swine anti-rabbit antibodies (both diluted 1:5000; DAKO)**.', 'lgd': {'word': 'When', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'DAKO'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Cells were fixed in 4% formaldehyde in PBS for 15 min at RT, washed with PBS, and subsequently permeabilized in PBS containing 0.1% Triton-**X-100 for 10 min.', 'lgd': {'word': 'Cells', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'Triton'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Cells were incubated with antibody against LAMP1 (rabbit anti-LAMP1 pAb, 1:100 dilution;** Abcam) in 3% BSA in PBS followed by incubation with secondary antibodies coupled to AF488, AF-568 phalloidin, and DAPI (all Life Technologies).', 'lgd': {'word': 'Cells', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'dilution'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The samples were analyzed using a confocal laser-**scanning microscope (Leica SPE-II).', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'laser'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**LR7 cells were infected at MOI = 0.1 or MOI** = 4.0 with MHV-ERLM or MHV-S29FCS in DMEM containing 2% FBS and 25 mM HEPES (infection medium).', 'lgd': {'word': 'LR7', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'MOI'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**After 3 h of infection supernatant was replaced by fresh infection medium and infection was allowed to progress over a period of** 24 h. Every 3 h a small sample of the culture supernatant was collected and immediately frozen.', 'lgd': {'word': 'After', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The relative amount of viral RNA present was determined with a LightCycler 480** using LightCycler 480 RNA Master Hydrolysis kit (Roche Applied Biosciences) and specific primers and probe targeted against the MHV 1b gene by comparison with a standard curve.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': '480'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Coronavirus Cell Entry Occurs through the Endo-/ Lysosomal Pathway in a Proteolysis-Dependent** Manner', 'lgd': {'word': 'Coronavirus', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'Dependent'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "---"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "### Coronavirus envelope protein: current knowledge"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': \"**The CoV envelope (E) protein is a small, integral** membrane protein involved in several aspects of the virus' life cycle, such as assembly, budding, envelope formation, and pathogenesis.\", 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'integral'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**It also compares E to other viral proteins of a similar nature** to speculate the relevance of these new findings.', 'lgd': {'word': 'It', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'nature'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Coronaviruses (CoVs) (order Nidovirales, family Coronaviridae, subfamily Coronavirinae) are enveloped viruses with a positive sense,** single-stranded RNA genome.', 'lgd': {'word': 'Coronaviruses', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'sense'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Based on genetic and antigenic criteria, CoVs have been organised into three groups: -CoVs, -CoVs, and -CoVs (Table 1) [1,** 2] .', 'lgd': {'word': 'Based', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '1'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Coronaviruses primarily infect birds and mammals, causing a variety of** lethal diseases that particularly impact the farming industry [3, 4] .', 'lgd': {'word': 'Coronaviruses', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The consequent outbreak of SARS in 2003 led to a near pandemic** with 8096 cases and 774 deaths reported worldwide, resulting in a fatality rate of 9.6% [24] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'pandemic'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Since the outbreak of MERS in April 2012 up until October 2018, 2229 laboratoryconfirmed cases have been reported globally, including 791 associated deaths with a case-fatality** rate of 35.5% [25] .', 'lgd': {'word': 'Since', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'fatality'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Clearly, the seriousness of these infections and the lack of effective, licensed treatments for CoV infections underpin the need for a more detailed** and comprehensive understanding of coronaviral molecular biology, with a specific focus on both their structural proteins as well as their accessory proteins [26] [27] [28] [29] [30] .', 'lgd': {'word': 'Clearly', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'detailed'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The coronaviral genome encodes four major structural proteins: the spike (S) protein, nucleocapsid (N) protein, membrane (M) protein, and the envelope (E) protein, all of which are required to produce a structurally complete** viral particle [29, 37, 38] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'complete'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**More recently, however, it has become clear that some CoVs do not require the full ensemble of structural proteins to form a complete, infectious** virion, suggesting that some structural proteins might be dispensable or that these CoVs might encode additional proteins with overlapping compensatory functions [35, 37, [39] [40] [41] [42] .', 'lgd': {'word': 'More', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'infectious'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Individually, each protein primarily plays a role in** the structure of the virus particle, but they are also involved in other aspects of the replication cycle.', 'lgd': {'word': 'Individually', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'in'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This formation of giant, multinucleated cells, or syncytia, has been proposed as a strategy to** allow direct spreading of the virus between cells, subverting virus-neutralising antibodies [45] [46] [47] .', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'to'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Interestingly, localisation of N to the endoplasmic reticulum (ER)-Golgi region has proposed a function for** it in assembly and budding [50, 51] .', 'lgd': {'word': 'Interestingly', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'for'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**During the replication cycle, E is abundantly expressed inside the infected cell, but only a small portion** is incorporated into the virion envelope [65] .', 'lgd': {'word': 'During', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'portion'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The CoV E protein is a short, integral** membrane protein of 76-109 amino acids, ranging from 8.4 to 12 kDa in size [69] [70] [71] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'integral'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The primary and secondary structure reveals that E has a short, hydrophilic** amino terminus consisting of 7-12 amino acids, followed by a large hydrophobic transmembrane domain (TMD) of 25 amino acids, and ends with a long, hydrophilic carboxyl terminus, which comprises the majority of the protein ( Fig.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'hydrophilic'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**1 ) [1,** 60, [72] [73] [74] [75] .', 'lgd': {'word': '1', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '1'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Comparative and phylogenetic analysis of SARS-CoV E revealed that a substantial portion** of the TMD consists of the two nonpolar, neutral amino acids, valine and leucine, lending a strong hydrophobicity to the E protein [79] .', 'lgd': {'word': 'Comparative', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'portion'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The C-terminus also exhibits some hydrophobicity but less than the TMD due to the presence of a cluster of** basic, positively charged amino acids [80] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Computational predictions regarding the secondary structure of E suggest that the Cterminus of and -CoVs also contains a conserved proline** residue centred in a -coil- motif [72] .', 'lgd': {'word': 'Computational', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'proline'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This motif likely functions as a Golgi-complex** targeting signal as mutation of this conserved proline was sufficient to disrupt the localization of a mutant chimeric protein to the Golgi complex and instead localized the protein to the plasma membrane [81] .', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'complex'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The SARS-CoV E protein has recently been found to contain a binding motif** known as the postsynaptic density protein 95 (PSD95)/Drosophila disc large tumour suppressor (Dlg1)/zonula occludens-1 protein (zo-1) (PDZ)-binding motif (PBM), located in the last four amino acids of the C terminus [82] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'motif'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The PDZ domain is a protein-protein** interaction module that can bind to the C-terminus of target proteins such as the cellular adapter proteins involved in host-cell processes important for viral infection [83] [84] [85] [86] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'protein'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The PBM domain was either mutated or deleted but reverted to a pathogenic state** after several passages in Vero E6 host cells.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'state'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Deletion of either the last nine resides of SARS-CoV E (PBM) or mutation of the four PBM residues to glycine (mutPBM) resulted in the acquisition of a PBM at** the C-terminus of E that was similar to the original PBM sequence.', 'lgd': {'word': 'Deletion', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'at'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This suggests that, at least for SARS-CoV E, some minor PBM mutations appear to be tolerated but that a reasonably intact** PBM domain is still necessary to avoid revertant mutants [34, 88] .', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'intact'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This would prove valuable for the design of a live, attenuated** vaccine with a PBM sufficiently mutated to remain intact, but also enough to be non-functional and abolish the pathogenicity of the virus.', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'attenuated'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**1 Amino Acid** Sequence and Domains of the SARS-CoV E Protein.', 'lgd': None, 'rgd': {'seed': '1', 'word': 'Acid'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The SARS-CoV E protein consists of three domains, i.e.**', 'lgd': {'word': 'The', 'seed': 'e'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Using SignalP, Wu, Zhang [79] reported a predicted signal** peptide cleavage site at the N-terminus of the SARS-CoV E protein.', 'lgd': {'word': 'Using', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'signal'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**They explored the possibility of a targeting signal** located in the luminal N-terminus but found the truncated terminus to be transported to the cell surface.', 'lgd': {'word': 'They', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'signal'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Conversely, truncation of the C-terminus and production of a chimeric E** protein both demonstrated retention at the Golgi complex, leading the authors to conclude that the Golgi-targeting information of the IBV E protein was located in its C-terminus.', 'lgd': {'word': 'Conversely', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'E'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Further truncation of the C-terminus narrowed down the bulk of the targeting information to a sequence motif** located between amino acid residues 44 and 72.', 'lgd': {'word': 'Further', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'motif'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The authors specifically demonstrated that neither the mutation of a highly conserved** proline residue nor the disruption of the predicted -strands, that stabilise the -hairpin on either side of the conserved proline residue, were sufficient to disrupt the targeting of the SARS-CoV E protein to the Golgi complex.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'conserved'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Conversely, the TGEV E protein exhibits a topology of** a luminal C-terminus and a cytoplasmic N-terminus [95] .', 'lgd': {'word': 'Conversely', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Prediction software demonstrated conflicting predictions between both the software and the experimental evidence; TMHMM and MEMSAT predicted a cytoplasmic N-**terminus and a luminal C-terminus, while HMMTop predicted a luminal N-terminus and a cytoplasmic C-terminus.', 'lgd': {'word': 'Prediction', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'N'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Moreover, transfected and infected cells expressing untagged SARS- CoV E exhibited a luminal N-**terminus and a cytoplasmic C-terminus topology [66] .', 'lgd': {'word': 'Moreover', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'N'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Nevertheless, the main reason likely stems from differences in the features unique to each algorithm, such as, whether the algorithm would include multiple features of the target protein(s) or only a clearly defined** set of criteria; how accurately the algorithm should discriminate between the different features; the point at which the specificity or sensitivity for a certain feature is defined as too broad or too narrow [102] .', 'lgd': {'word': 'Nevertheless', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'defined'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Nevertheless, some proteins prove challenging to isolate and not all biochemical techniques offer the needed high-resolution structural detail, in which case prediction programs are a good alternative** and offer valuable insight into the predicted outcomes [101] .', 'lgd': {'word': 'Nevertheless', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'alternative'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Many prediction programs also make use of a sliding window** method to predict certain structural features of a protein.', 'lgd': {'word': 'Many', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'window'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**It is based on a window size** that covers defined fragments of the amino acid sequence, rather than the whole sequence and takes into account that a given characteristic of an amino acid is not only determined by the residue itself, but also by the adjacent residues [103] .', 'lgd': {'word': 'It', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'size'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**It is possible that a standard window** size, corresponding to a stretch of residues in the sequence, was not used between the prediction programs, or even between different CoVs, which might have resulted in the different topological predictions for each of the CoVs in Table 2 .', 'lgd': {'word': 'It', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'window'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Based on a probabilistic approach,** the prediction of structural features, such as coils and strands, would benefit from smaller window sizes as residues up to three and six positions away from the central residue, respectively, can affect the formation of these structures.', 'lgd': {'word': 'Based', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'approach'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Conversely, helical structure formation can be affected by up to nine residues away from the central residue and would benefit from a larger window** size [103] .', 'lgd': {'word': 'Conversely', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'window'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Accordingly, the use of a standardised, optimal** window size could prove beneficial to obtain a more consistent and accurate topological prediction for CoV E.', 'lgd': {'word': 'Accordingly', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'optimal'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**By tagging the Nterminus of the IBV E protein with a FLAG tag,** Ruch and Machamer [109] succeeded in producing a membrane hairpin conformation, with the N-and C-termini oriented cytoplasmically.', 'lgd': {'word': 'By', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'tag'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**However, the untagged E protein exhibited the topological conformation of a single transmembrane-**spanning protein, demonstrating that the topology may be altered by the presence of the Nterminal tag [66] .', 'lgd': {'word': 'However', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'transmembrane'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The rationale for the multiple membrane topologies has been suggested in that, between the different CoV species, the E protein might not exhibit a uniform membrane** topology or that the orientation of E varies depending on the level of protein expression or oligomerization [69] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'membrane'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The hepatitis C virus (HCV) nucleocapsid core protein binds to ER membranes in a palmitoylation-dependent** manner for the formation of viral particles [116] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'dependent'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Rather, it is possible that a loss of** both the Golgitargeting information in the TMD and the palmitoylated cysteine residues leads to the loss of localization as well as membrane its association [65] .', 'lgd': {'word': 'Rather', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Linkage of myristic acid (C14:0) to the N-terminal of a glycine residue** found on some viral, cellular, or bacterial proteins, is known as N-terminal myristoylation [120] [121] [122] [123] .', 'lgd': {'word': 'Linkage', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'residue'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': \"**All of these proteins contain the conserved sequence motif 1 MGxxxS/T,** where 'x' can be any amino acid [80] .\", 'lgd': {'word': 'All', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'T'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Coronavirus E proteins, along with other members of the Nidovirales order, reportedly have no myristoylation motif and it is suggested to be a feature unique** only to the Arteriviridae family in the order of Nidovirales [80] .', 'lgd': {'word': 'Coronavirus', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'unique'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The SARS-CoV non-structural protein (nsp) 3 co-localises with E and its interaction was mediated through the N-terminal ubiquitin-like domain-1 of nsp3.**', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'nsp3'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Independently, a ubiquitination assay** further demonstrated that E can be ubiquitinated and that its ubiquitination status inversely correlates to its stability and half-life [128, 130] .', 'lgd': {'word': 'Independently', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'assay'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**However, this will need to be confirmed in the context of a natural infection.**', 'lgd': {'word': 'However', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'infection'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The IBV E protein has been suggested to contain a single glycosylation** site in its luminal N-terminus, while SARS-CoV E has been predicted to contain two potential glycosylation sites [132] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'glycosylation'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This suggests that glycosylation of N66 might function to prevent oligomerization of the E protein, possibly to promote a specific role** of the E protein.', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'role'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Westerbeck and Machamer [90] used both infected and transfected cells and reported the presence of two different forms of the IBV E protein, each associated with a specific function.**', 'lgd': {'word': 'Westerbeck', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'function'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The CoV E protein is unique in that it can form homotypic interactions, which allows it to oligomerise and generate an ion-channel protein known as a viroporin [135,** 136] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': '135'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Mutation of the TMD residues asparagine 15 (N15) to alanine (N15A) and valine 25 (V25) to phenylalanine (V25F) have been found to abolish the ion channelling capability of CoV E viroporin, a structure dependent** on its homopentameric conformation [75, 76, 138] .', 'lgd': {'word': 'Mutation', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'dependent'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The appearance of monomers in response to V25F clearly suggests that this residues plays a more crucial** role in oligomerisation, as opposed to N15A, which appears to reduce the amount of pentamers only slightly [139] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'crucial'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The ability of CoV E to assemble into homopentameric structures is clearly important in the formation of a functional CoV** E viroporin [75, 76, [135] [136] [137] [138] 140] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'CoV'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Only two studies have reported a possible interaction** between E and N, one for murine MHV and the other for SARS-CoV.', 'lgd': {'word': 'Only', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'interaction'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This suggests that there might, in fact, be a physical interaction** between E and N but the reason and exact requirements for this interaction remains to be determined.', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'interaction'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**More research is needed to understand this interaction and whether it offers a possible explanation** as to why or how VLP production is enhanced during the co-expression of M, E, and N [42, 52] .', 'lgd': {'word': 'More', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'explanation'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A sub-regional analysis of both E and S revealed a triple cysteine** motif located directly after the E protein TMD (NH 2 - L-Cys-A-Y-Cys-Cys-N  -COOH) and a similar motif located in the C-terminus of S (NH 2 - S-Cys-G-S-Cys-Cys-K  -COOH) [79] .', 'lgd': {'word': 'A', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'cysteine'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The authors proposed that the predicted orientation, position, and composition of these two motifs could serve as a structural basis** for the association between E and S, which would be mediated by the formation of disulphide bonds between the corresponding cysteine residues (Fig.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'basis'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Experimental data on a physical interaction** between CoV S and E is extremely limited with the exception of one study, which showed that SARS-CoV S is an interacting partner of E [128] .', 'lgd': {'word': 'Experimental', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'interaction'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Using a tagged E protein, the study aimed to identify SARS-CoV E protein interacting partners by a** tandem affinity purification (TAP) system coupled with mass spectrometry (MS; TAP-MS).', 'lgd': {'word': 'Using', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'E'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Protein 7a, a structural protein** unique to SARS-CoV, is incorporated into mature virions and plays an important part in the pathogenesis of SARS-CoV, where it functions to induce apoptosis, arrest the cell cycle, and promote the production of pro-inflammatory cytokines [144] [145] [146] [147] [148] .', 'lgd': {'word': 'Protein', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'protein'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': 'In **a mammalian two** hybrid system, SARS-CoV E was found to interact with 7a, but the importance of this interaction has not yet been determined [149] .', 'lgd': None, 'rgd': {'seed': 'a', 'word': 'two'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The domain mediating this PPI was only identified later when the SARS-CoV E protein was shown to interact with the protein associated with Caenorhabditis elegans lin-7 protein 1 (PALS1)** [82] .', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'PALS1'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The PDZ domain is a proteinprotein recognition** sequence found in cellular adaptor proteins that coordinate host cell signalling pathways by binding to other proteins that have a complementary PBM.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'recognition'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The PBM of SARS-CoV E is found in the last four amino acids (DLLV) of its C-terminu**s [1,**** 82] .', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': '82'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**To date, E has only been reported to interact with five host proteins, i.e.**', 'lgd': {'word': 'To', 'seed': 'e'}, 'rgd': None}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Bcl-xL, PALS1, syntenin, sodium/potassium (Na + /K + ) ATPase -1 subunit, and** stomatin [18, 66, 82, 87] .', 'lgd': {'word': 'Bcl', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'and'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Teoh, Siu [82] reported that the E-PALS1 interaction disrupts tight junctions in the lungs, suggesting a mechanism whereby** SARS-CoV virions can breach the alveolar wall and develop into a systemic infection.', 'lgd': {'word': 'Teoh', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'whereby'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Nieto-Torres, DeDiego [66] suggested that the interaction of E with Na + /K + ATPase -1 subunit and** stomatin, 2 proteins involved in maintaining ionic homeostasis, could be responsible for the reduced levels and activity of human epithelial sodium channels.', 'lgd': {'word': 'Nieto', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'and'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Jimenez-Guardeo, Nieto-Torres [18] is the only group to have shown that E is a determinant of** SARS-CoV pathogenesis in vivo.', 'lgd': {'word': 'Jimenez', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Aside from this, it is of therapeutic importance that more E interaction partners be identified as inhibitors of p38 mitogen-activated protein kinase (MAPK) were shown to increase the survival rate of mice, protecting them from a lethal infection** [18, 158] .', 'lgd': {'word': 'Aside', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'infection'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Identifying more interaction partners for CoV E could provide a more targeted** therapeutic approach where licensed coronaviral treatments are currently ineffective [26] [27] [28] .', 'lgd': {'word': 'Identifying', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'targeted'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The interaction between the cytoplasmic tails of the M and E proteins drives VLP production, suggesting that E participates in (1) viral** assembly [56, 61, 89] .', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'viral'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The coronaviral envelope consists predominantly of M while only a small portion** of E is incorporated into the viral envelope of virions [100, 167, 168] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'portion'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Extensive electron microscopy (EM) studies conducted on M from a variety of** CoVs provided no indication that M is capable of inducing membrane curvature on its own [51, 169, 170] .', 'lgd': {'word': 'Extensive', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In fact, various recombinant CoVs (rCoVs) lacking the E gene (E) exhibit a strikingly aberrant** morphology.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'aberrant'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Plaques of recombinant MHV-E exhibited a very similar** aberrant morphology, presenting as small, irregular-shaped plaques with jagged edges [39] .', 'lgd': {'word': 'Plaques', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'similar'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Cells infected with recombinant SARS-CoV-E (rSARS-CoV-E) contained a lower number** of mature virions but exhibited a higher proportion of vesicles containing a dense, granular material.', 'lgd': {'word': 'Cells', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'number'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Hagemeijer, Monastyrska [177] proposed a model in** which the luminal loops located between the transmembrane regions of nsp3 and 4 interact with one another to initiate the rearrangement of ER membranes and induce membrane curvature to form DMVs (Fig.', 'lgd': {'word': 'Hagemeijer', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'in'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This underpins the importance of establishing a unanimous topology** for the E protein as this model could be applied to the induction of membrane curvature by E, provided E can assume multiple topologies during an infection.', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'topology'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Should it be demonstrated that E can take on a topology with** a luminal loop, this would not be inconceivable as a possible mechanism for the induction of membrane curvature initiated by E or in which E participates.', 'lgd': {'word': 'Should', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'with'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Equally, as heterotypic interactions of nsp3 and 4 are required to induce ER membrane curvature, and the expression of both M and E is required for the formation of smooth, spherical CoV VLPs, it would be interesting to see if a heterotypic interaction** between M and E could drive membrane curvature by a similar mechanism [176] [177] [178] .', 'lgd': {'word': 'Equally', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'interaction'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Alternatively, no research exists on the exact purpose of the N-terminus of E. Perhaps homotypic interactions mediated by the N-termini of alternating E proteins could be responsible for inducing membrane curvature by a similar mechanism.**', 'lgd': {'word': 'Alternatively', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'mechanism'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**It is quite evident that although a lot of** progress has been made in determining the role of E in assembly, much still remains unknown.', 'lgd': {'word': 'It', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The viral envelope is formed primarily during assembly and culminates when the virion buds from the host membrane, a process known** as scission [179] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'known'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In the absence of scission machinery, the budding process begins but ultimately stops, and render budding virions attached to the membrane by a small membranous** neck.', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'membranous'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This causes virions to have an uncharacteristically elongated morphology sometimes referred to as \"beads-on-a-string\" and** is seen in viruses that lack the necessary machinery to release the budded virion [179] [180] [181] [182] [183] .', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'and'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This is clearly and elegantly demonstrated in the mutation of the matrix-2 (M2) protein, a viral protein** responsible for the budding and scission of the influenza virus.', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'protein'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Virions that have failed to undergo scission remain attached to the host cell membrane by a membranous neck.**', 'lgd': {'word': 'Virions', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'neck'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The budding process is reinitiated at the site where scission failed, and a new virion** is formed.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'virion'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**However, the new virion also remains attached to the membrane as well as the previous virion by a small membranous** neck.', 'lgd': {'word': 'However', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'membranous'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The continuation of this cycle and repeated initiation of budding results in the formation of consecutive scission-defective virions that resemble beads on a string [180,** 181] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': '180'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This demonstrates a necessity for** viral-host PPIs but given the shortage of information available on CoV E-host PPIs, it is nearly impossible to say whether E mediates scission in an ESCRT-dependent manner or not.', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'for'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**In the absence of the M2 protein, buds formed inside infected cells but failed to detach and such cells exhibited the beads-on-a-string morphology.**', 'lgd': {'word': 'In', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'morphology'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': \"**This suggests that M2 can serve as a substitute for** ESCRT complexes during influenza virus budding and, more importantly, raises the possibility of functionally equivalent M2's in other enveloped viruses.\", 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'for'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**While the accumulation of E at the ERGIC points largely to a role in** assembly and budding, only a small portion is incorporated into the viral envelope, suggesting that E has additional functions centred around the ER and Golgi region [66, 92, 109, 159] .', 'lgd': {'word': 'While', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'in'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**They tend to be small proteins (~60-120 amino acids) of a predominantly hydrophobic** nature that oligomerise in the membranes of infected cells, forming hydrophilic pores.', 'lgd': {'word': 'They', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'hydrophobic'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Most viroporins share certain structural features such as an amphipathic -helix in the hydrophobic domain (HD) along with a cluster of** positively charged, basic amino acids (such as lysine or arginine) which anchor the pore to the membrane through electrostatic interactions with the negatively charged phospholipids (Fig.', 'lgd': {'word': 'Most', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**It is, however, interesting to note that, at a neutral pH,** the ion selectivity of the respiratory syncytial virus (RSV) small hydrophobic (SH) protein can change from cationic to anionic [214] .', 'lgd': {'word': 'It', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'pH'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This suggests that viroporins are sensitive to changes in the cellular environment, a property that** could be of therapeutic value.', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'that'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Computational predictions and spectroscopic studies show that the SARS-CoV E TMD undergoes oligomerisation, characteristic of ion-channelling proteins, to form a stable pentamer** [75, [135] [136] [137] .', 'lgd': {'word': 'Computational', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'pentamer'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The CoV E viroporin is equally cation-selective when it comes to its ion-channelling properties, demonstrating a preference for** the monovalent cations Na + and K + [217, 218] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'for'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Synthetic peptides that correspond to E from HCoV-229E, MHV, and IBV exhibit a similar cation-**selectivity for MHV and IBV E as for SARS-CoV E. However, it is interesting that although the E viroporin synthetic peptides of HCoV-229E were still cation-selective, it exhibits a slightly higher selectivity for K + than for Na + [218] .', 'lgd': {'word': 'Synthetic', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'cation'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The SARS-CoV E synthetic peptide findings were corroborated using a full-length** SARS-CoV E protein [76] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'length'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**More recently, purified full-length MERS-CoV E has also demonstrated limited ion-channelling properties and would benefit from a more comprehensive** characterisation to establish whether it has ion-channelling properties similar to that of the other CoVs [140] .', 'lgd': {'word': 'More', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'comprehensive'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**5 Illustration of a typical viroporin** structure and motifs.', 'lgd': {'word': '5', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'viroporin'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The pore of the viroporin (brown) is created by the amphipathic -helix and the viroporin is anchored to a lipid bilayer** by terminal positively charged residues (lysine or arginine).', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'bilayer'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Mutant N15A reverted by incorporating a single mutation** that led to an amino acid change at the same position (A15D), creating a more stable mutant.', 'lgd': {'word': 'Mutant', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'mutation'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**It is interesting to note that in both cases SARS-CoV E and IBV E followed a similar trend** in their reversion: mutations at N15A and T16A both reverted by substitution of a single residue, whereas mutations at V25F and A26F produced revertants by acquisition of multiple residues.', 'lgd': {'word': 'It', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'trend'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**An intriguing study recently reported that both IBV infected and IBV E transfected cells exhibited a marked increase** in the pH of the Golgi lumen [227] .', 'lgd': {'word': 'An', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'increase'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**However, this increase in pH was found only in cells expressing a monomeric form** of IBV E and not the oligomeric form as required for viroporin formation.', 'lgd': {'word': 'However', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'form'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The authors proposed that the change in pH could be attributed to an interaction between the monomeric form of E and a host protein.**', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'protein'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Although possible, only a very small** number of host proteins have been shown to interact with CoV E. The monomeric and oligomeric forms were produced by transfection of mutated IBV E A26 to F26 (E A26F ) and T16 to A16 (E T16A ), respectively.', 'lgd': {'word': 'Although', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'small'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The ER can sustain a high load** of protein content without being overwhelmed [228] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'load'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**However, cells infected with rSARS-CoV and rSARS-CoVE, a more biologically** relevant system, demonstrated that SARS-CoV E may regulate the UPR as part of its pathogenesis [233] .', 'lgd': {'word': 'However', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'biologically'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Cells infected with SARS-CoVE exhibit a stronger stress** response compared to cells infected with the wild-type virus.', 'lgd': {'word': 'Cells', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'stress'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Moreover, a higher degree** of apoptosis was observed in SARS-CoVE-infected cells than in those infected with the wild-type virus.', 'lgd': {'word': 'Moreover', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'degree'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The study also shows that CoV E has an antiapoptotic function in infected cells by suppressing the UPR during infection, likely as a survival mechanism** and to continue viral propagation.', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'mechanism'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Viruses often encode proteins that interfere with the immune system to either inhibit a response or** enhance one as part of their pathogenicity.', 'lgd': {'word': 'Viruses', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'or'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**This protein also delays the transport of proteins along the secretory pathway by altering the Ca 2+ and H + concentrations of the Golgi and ER compartments and has been proposed to be a mechanism of** immune evasion as well [239] .', 'lgd': {'word': 'This', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'of'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The authors concluded that this ion-channelling function confers a selective advantage** to the virus [77] .', 'lgd': {'word': 'The', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'advantage'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Anti-viral candidates either exhibit only a narrow spectrum** of activity, are only effective at unusually high therapeutic dosages or cause serious side effects or immune suppression [248] .', 'lgd': {'word': 'Anti', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'spectrum'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**A few studies have investigated the potential of rCoVs with a mutated E** or lacking E, specifically focussing on SARS-and MERS-CoV, as live attenuated vaccine candidates with some promising results [34, 36, 165, 249, 250] .', 'lgd': {'word': 'A', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'E'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Stapled peptides fulfil these criteria to a large extent** and have been applied to various human diseases and fields such as cancer, infections, metabolism, neurology, and endocrinology [256] [257] [258] [259] [260] .', 'lgd': {'word': 'Stapled', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'extent'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': \"**To the best of the author's knowledge, only a few studies** so far have investigated the potential of stapled peptides as antiviral agents, with promising results for both intracellular and extracellular targets.\", 'lgd': {'word': 'To', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'studies'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**The focus so far has only been on HIV-1, RSV,** and HCV [260, [262] [263] [264] [265] .', 'lgd': {'word': 'The', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'RSV'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Stapled peptides (1) can** inhibit PPIs; (2) are more specific for their targets than small-molecule drugs, which also decreases the risk of unwanted side-effects; (3) can target diseases that are otherwise difficult to treat, referred to as \"undruggable\"; (4) can be modified easily to enhance membrane permeability, potency, and half-life; (5) have a short market time [253, 266, 267] .', 'lgd': {'word': 'Stapled', 'seed': '1'}, 'rgd': {'seed': '1', 'word': 'can'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Autophagy is a cellular process** that recycles excess or damaged cellular material to maintain the energy levels of the cell and ensure its survival.', 'lgd': {'word': 'Autophagy', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'process'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/markdown": [
       "- {'sent': '**Left: The binding of biotin to avidin occurs in a deep groove,** while the interaction between the human growth hormone (hGH) and the hGH receptor (hGHR) occurs over a larger, flatter area [254] a defence mechanism to promote viral propagation [274] [275] [276] .', 'lgd': {'word': 'Left', 'seed': 'a'}, 'rgd': {'seed': 'a', 'word': 'groove'}}"
      ],
      "text/plain": [
       "<IPython.core.display.Markdown object>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "ename": "KeyboardInterrupt",
     "evalue": "",
     "output_type": "error",
     "traceback": [
      "\u001b[0;31m---------------------------------------------------------------------------\u001b[0m",
      "\u001b[0;31mKeyboardInterrupt\u001b[0m                         Traceback (most recent call last)",
      "\u001b[0;32m<ipython-input-152-47e07ff4a5ca>\u001b[0m in \u001b[0;36m<module>\u001b[0;34m\u001b[0m\n\u001b[1;32m      3\u001b[0m     \u001b[0mdm\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34mf\"### {paper['metadata.title'][0]['text']}\"\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      4\u001b[0m     \u001b[0;32mfor\u001b[0m \u001b[0msnippet\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mpaper\u001b[0m\u001b[0;34m[\u001b[0m\u001b[0;34m'snippets'\u001b[0m\u001b[0;34m]\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 5\u001b[0;31m        \u001b[0mdm\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34mf'- {snippet}'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      6\u001b[0m     \u001b[0mdm\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m'---'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      7\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m<ipython-input-147-25a13fa77c1a>\u001b[0m in \u001b[0;36m<lambda>\u001b[0;34m(x)\u001b[0m\n\u001b[1;32m      2\u001b[0m \u001b[0;32mfrom\u001b[0m \u001b[0mnltk\u001b[0m \u001b[0;32mimport\u001b[0m \u001b[0msent_tokenize\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      3\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m----> 4\u001b[0;31m \u001b[0mdm\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;32mlambda\u001b[0m \u001b[0mx\u001b[0m\u001b[0;34m:\u001b[0m \u001b[0mdisplay\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mMarkdown\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mx\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m      5\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m      6\u001b[0m \u001b[0mseeds_pat\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m'|'\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mjoin\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mw\u001b[0m \u001b[0;32mfor\u001b[0m \u001b[0mw\u001b[0m \u001b[0;32min\u001b[0m \u001b[0mseeds\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/usr/local/anaconda3/lib/python3.7/site-packages/IPython/core/display.py\u001b[0m in \u001b[0;36mdisplay\u001b[0;34m(include, exclude, metadata, transient, display_id, *objs, **kwargs)\u001b[0m\n\u001b[1;32m    311\u001b[0m                 \u001b[0;31m# kwarg-specified metadata gets precedence\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    312\u001b[0m                 \u001b[0m_merge\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mmd_dict\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mmetadata\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 313\u001b[0;31m             \u001b[0mpublish_display_data\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mdata\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mformat_dict\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mmetadata\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mmd_dict\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    314\u001b[0m     \u001b[0;32mif\u001b[0m \u001b[0mdisplay_id\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    315\u001b[0m         \u001b[0;32mreturn\u001b[0m \u001b[0mDisplayHandle\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mdisplay_id\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/usr/local/anaconda3/lib/python3.7/site-packages/IPython/core/display.py\u001b[0m in \u001b[0;36mpublish_display_data\u001b[0;34m(data, metadata, source, transient, **kwargs)\u001b[0m\n\u001b[1;32m    120\u001b[0m         \u001b[0mdata\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mdata\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    121\u001b[0m         \u001b[0mmetadata\u001b[0m\u001b[0;34m=\u001b[0m\u001b[0mmetadata\u001b[0m\u001b[0;34m,\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 122\u001b[0;31m         \u001b[0;34m**\u001b[0m\u001b[0mkwargs\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    123\u001b[0m     )\n\u001b[1;32m    124\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/usr/local/anaconda3/lib/python3.7/site-packages/ipykernel/zmqshell.py\u001b[0m in \u001b[0;36mpublish\u001b[0;34m(self, data, metadata, source, transient, update)\u001b[0m\n\u001b[1;32m    110\u001b[0m             \u001b[0mIf\u001b[0m \u001b[0;32mTrue\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0msend\u001b[0m \u001b[0man\u001b[0m \u001b[0mupdate_display_data\u001b[0m \u001b[0mmessage\u001b[0m \u001b[0minstead\u001b[0m \u001b[0mof\u001b[0m \u001b[0mdisplay_data\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    111\u001b[0m         \"\"\"\n\u001b[0;32m--> 112\u001b[0;31m         \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_flush_streams\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    113\u001b[0m         \u001b[0;32mif\u001b[0m \u001b[0mmetadata\u001b[0m \u001b[0;32mis\u001b[0m \u001b[0;32mNone\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    114\u001b[0m             \u001b[0mmetadata\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0;34m{\u001b[0m\u001b[0;34m}\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/usr/local/anaconda3/lib/python3.7/site-packages/ipykernel/zmqshell.py\u001b[0m in \u001b[0;36m_flush_streams\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m     78\u001b[0m         \u001b[0;34m\"\"\"flush IO Streams prior to display\"\"\"\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     79\u001b[0m         \u001b[0msys\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mstdout\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mflush\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m---> 80\u001b[0;31m         \u001b[0msys\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mstderr\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mflush\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m     81\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m     82\u001b[0m     \u001b[0;34m@\u001b[0m\u001b[0mdefault\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;34m'_thread_local'\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/usr/local/anaconda3/lib/python3.7/site-packages/ipykernel/iostream.py\u001b[0m in \u001b[0;36mflush\u001b[0;34m(self)\u001b[0m\n\u001b[1;32m    349\u001b[0m                 \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mpub_thread\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mschedule\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mevt\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mset\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    350\u001b[0m                 \u001b[0;31m# and give a timeout to avoid\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 351\u001b[0;31m                 \u001b[0;32mif\u001b[0m \u001b[0;32mnot\u001b[0m \u001b[0mevt\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mwait\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mflush_timeout\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    352\u001b[0m                     \u001b[0;31m# write directly to __stderr__ instead of warning because\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    353\u001b[0m                     \u001b[0;31m# if this is happening sys.stderr may be the problem.\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/usr/local/anaconda3/lib/python3.7/threading.py\u001b[0m in \u001b[0;36mwait\u001b[0;34m(self, timeout)\u001b[0m\n\u001b[1;32m    550\u001b[0m             \u001b[0msignaled\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_flag\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    551\u001b[0m             \u001b[0;32mif\u001b[0m \u001b[0;32mnot\u001b[0m \u001b[0msignaled\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 552\u001b[0;31m                 \u001b[0msignaled\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mself\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0m_cond\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0mwait\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0mtimeout\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    553\u001b[0m             \u001b[0;32mreturn\u001b[0m \u001b[0msignaled\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    554\u001b[0m \u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;32m/usr/local/anaconda3/lib/python3.7/threading.py\u001b[0m in \u001b[0;36mwait\u001b[0;34m(self, timeout)\u001b[0m\n\u001b[1;32m    298\u001b[0m             \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    299\u001b[0m                 \u001b[0;32mif\u001b[0m \u001b[0mtimeout\u001b[0m \u001b[0;34m>\u001b[0m \u001b[0;36m0\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0;32m--> 300\u001b[0;31m                     \u001b[0mgotit\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mwaiter\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0macquire\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;32mTrue\u001b[0m\u001b[0;34m,\u001b[0m \u001b[0mtimeout\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[0m\u001b[1;32m    301\u001b[0m                 \u001b[0;32melse\u001b[0m\u001b[0;34m:\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n\u001b[1;32m    302\u001b[0m                     \u001b[0mgotit\u001b[0m \u001b[0;34m=\u001b[0m \u001b[0mwaiter\u001b[0m\u001b[0;34m.\u001b[0m\u001b[0macquire\u001b[0m\u001b[0;34m(\u001b[0m\u001b[0;32mFalse\u001b[0m\u001b[0;34m)\u001b[0m\u001b[0;34m\u001b[0m\u001b[0;34m\u001b[0m\u001b[0m\n",
      "\u001b[0;31mKeyboardInterrupt\u001b[0m: "
     ]
    }
   ],
   "source": [
    "for paper in sorted(covid_papers, key=lambda x: -len(x['snippets'])):\n",
    "    if not paper['snippets']: break\n",
    "    dm(f\"### {paper['metadata.title'][0]['text']}\")\n",
    "    for snippet in paper['snippets']:\n",
    "       dm(f'- {snippet}')\n",
    "    dm('---')\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 68,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "[nltk_data] Downloading package punkt to /root/nltk_data...\n",
      "[nltk_data]   Unzipping tokenizers/punkt.zip.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "True"
      ]
     },
     "execution_count": 68,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "nltk.download('punkt')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.0"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
